<html class="js" lang="en">
 <head>
  <link as="font" crossorigin="" href="/static/fonts/HardingText-Regular-Web-cecd90984f.woff2" rel="preload" type="font/woff2"/>
  <title>
   Highly accurate protein structure prediction with AlphaFold | Nature
  </title>
  <script id="save-data-connection-testing">
   function hasConnection() {
                return navigator.connection || navigator.mozConnection || navigator.webkitConnection || navigator.msConnection;
            }

            function createLink(src) {
                var preloadLink = document.createElement("link");
                preloadLink.rel = "preload";
                preloadLink.href = src;
                preloadLink.as = "font";
                preloadLink.type = "font/woff2";
                preloadLink.crossOrigin = "";
                document.head.insertBefore(preloadLink, document.head.firstChild);
            }

            var connectionDetail = {
                saveDataEnabled: false,
                slowConnection: false
            };

            var connection = hasConnection();
            if (connection) {
                connectionDetail.saveDataEnabled = connection.saveData;
                if (/\slow-2g|2g/.test(connection.effectiveType)) {
                    connectionDetail.slowConnection = true;
                }
            }

            if (!(connectionDetail.saveDataEnabled || connectionDetail.slowConnection)) {
                createLink("/static/fonts/HardingText-Regular-Web-cecd90984f.woff2");
            } else {
                document.documentElement.classList.add('save-data');
            }
  </script>
  <script data-consent="www-nature-com.proxy.lib.ohio-state.edu" src="/static/js/cookie-consent-es5-bundle-2b0f06c1e4.js">
  </script>
  <link crossorigin="" href="https://push-content.springernature.io" rel="preconnect"/>
  <meta content="IE=edge" http-equiv="X-UA-Compatible"/>
  <meta content="pc,mobile" name="applicable-device"/>
  <meta content="width=device-width,initial-scale=1.0,maximum-scale=5,user-scalable=yes" name="viewport"/>
  <meta content="5a2dc4ab3fcb9b0393241ffbbb490480" name="360-site-verification"/>
  <script data-test="dataLayer">
   window.dataLayer = [{"content":{"category":{"contentType":"article","legacy":{"webtrendsPrimaryArticleType":"research","webtrendsSubjectTerms":"computational-biophysics;machine-learning;protein-structure-predictions;structural-biology","webtrendsContentCategory":null,"webtrendsContentCollection":"Robotics and artificial intelligence;Method of the Year 2021: Protein structure prediction","webtrendsContentGroup":"Nature","webtrendsContentGroupType":null,"webtrendsContentSubGroup":"Article"}},"article":{"doi":"10.1038/s41586-021-03819-2"},"attributes":{"cms":null,"deliveryPlatform":"oscar","copyright":{"open":true,"legacy":{"webtrendsLicenceType":"http://creativecommons.org/licenses/by/4.0/"}}},"contentInfo":{"authors":["John Jumper","Richard Evans","Alexander Pritzel","Tim Green","Michael Figurnov","Olaf Ronneberger","Kathryn Tunyasuvunakool","Russ Bates","Augustin Žídek","Anna Potapenko","Alex Bridgland","Clemens Meyer","Simon A. A. Kohl","Andrew J. Ballard","Andrew Cowie","Bernardino Romera-Paredes","Stanislav Nikolov","Rishub Jain","Jonas Adler","Trevor Back","Stig Petersen","David Reiman","Ellen Clancy","Michal Zielinski","Martin Steinegger","Michalina Pacholska","Tamas Berghammer","Sebastian Bodenstein","David Silver","Oriol Vinyals","Andrew W. Senior","Koray Kavukcuoglu","Pushmeet Kohli","Demis Hassabis"],"publishedAt":1626307200,"publishedAtString":"2021-07-15","title":"Highly accurate protein structure prediction with AlphaFold","legacy":null,"publishedAtTime":null,"documentType":"aplusplus"},"journal":{"pcode":"nature","title":"nature","volume":"596","issue":"7873"},"authorization":{"status":true},"features":[{"name":"furtherReadingSection","present":true}],"collection":{"id":"dgebebiadj;dfejabhghd"}},"page":{"category":{"pageType":"article"},"attributes":{"template":"mosaic","featureFlags":[{"name":"nature-onwards-journey","active":false},{"name":"getftr-entitled","active":false}],"testGroup":null},"search":null},"privacy":{},"version":"1.0.0","product":null,"session":null,"user":null,"backHalfContent":true,"country":"US","hasBody":true,"uneditedManuscript":false,"twitterId":["o3xnx","o43y9","o3ef7"],"japan":false}];
    window.dataLayer.push({
        ga4MeasurementId: 'G-ERRNTNZ807',
        ga360TrackingId: 'UA-71668177-1',
        twitterId: ['3xnx', 'o43y9', 'o3ef7'],
        ga4ServerUrl: 'https://collect-nature-com.proxy.lib.ohio-state.edu',
        imprint: 'nature'
    });
  </script>
  <script>
   (function(w, d) {
        w.config = w.config || {};
        w.config.mustardcut = false;

        
        if (w.matchMedia && w.matchMedia('only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)').matches) {
            w.config.mustardcut = true;
            d.classList.add('js');
            d.classList.remove('grade-c');
            d.classList.remove('no-js');
        }
    })(window, document.documentElement);
  </script>
  <style>
   @media only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark) { .c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card--major .c-card__title,.c-card__title,.u-h2,.u-h3,h2,h3{-webkit-font-smoothing:antialiased;font-family:Harding,Palatino,serif;font-weight:700;letter-spacing:-.0117156rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card__title,.u-h3,h3{font-size:1.25rem;line-height:1.4rem}.c-reading-companion__figure-title,.u-h4{-webkit-font-smoothing:antialiased;font-weight:700;line-height:1.4rem}html{text-size-adjust:100%;box-sizing:border-box;font-size:100%;height:100%;line-height:1.15;overflow-y:scroll}body{background:#eee;color:#222;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;line-height:1.76;margin:0;min-height:100%}main{display:block}h1{font-size:2em;margin:.67em 0}a,sup{vertical-align:baseline}a{background-color:transparent;color:#069;overflow-wrap:break-word;text-decoration:underline;text-decoration-skip-ink:auto;word-break:break-word}b{font-weight:bolder}sup{font-size:75%;line-height:0;position:relative;top:-.5em}img{border:0;height:auto;max-width:100%;vertical-align:middle}button,input,select{font-family:inherit;font-size:100%;line-height:1.15;margin:0}button,input{overflow:visible}button,select{text-transform:none}[type=button],[type=submit],button{-webkit-appearance:button}[type=checkbox]{box-sizing:border-box;padding:0}[hidden]{display:none}button{border-radius:0;cursor:pointer;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}h1{-webkit-font-smoothing:antialiased;font-family:Harding,Palatino,serif;font-size:2rem;font-weight:700;letter-spacing:-.0390625rem;line-height:2.25rem}.c-card--major .c-card__title,.u-h2,.u-h3,h2{font-family:Harding,Palatino,serif;letter-spacing:-.0117156rem}.c-card--major .c-card__title,.u-h2,h2{-webkit-font-smoothing:antialiased;font-size:1.5rem;font-weight:700;line-height:1.6rem}.u-h3{font-size:1.25rem}.c-card__title,.c-reading-companion__figure-title,.u-h3,.u-h4,h4,h5,h6{-webkit-font-smoothing:antialiased;font-weight:700;line-height:1.4rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card__title,h3{font-family:Harding,Palatino,serif;font-size:1.25rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,h3{-webkit-font-smoothing:antialiased;font-weight:700;letter-spacing:-.0117156rem;line-height:1.4rem}.c-reading-companion__figure-title,.u-h4{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;letter-spacing:-.0117156rem}input+label{padding-left:.5em}nav ol,nav ul{list-style:none none}p:empty{display:none}.sans-serif{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.article-page{background:#fff}.c-article-header{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;margin-bottom:40px}.c-article-identifiers{color:#6f6f6f;display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3;list-style:none;margin:0 0 8px;padding:0}.c-article-identifiers__item{border-right:1px solid #6f6f6f;list-style:none;margin-right:8px;padding-right:8px}.c-article-identifiers__item:last-child{border-right:0;margin-right:0;padding-right:0}.c-article-title{font-size:1.5rem;line-height:1.25;margin:0 0 16px}@media only screen and (min-width:768px){.c-article-title{font-size:1.875rem;line-height:1.2}}.c-article-author-list{display:inline;font-size:1rem;list-style:none;margin:0 8px 0 0;padding:0;width:100%}.c-article-author-list__item{display:inline;padding-right:0}.c-article-author-list svg{margin-left:4px}.c-article-author-list__show-more{display:none;margin-right:4px}.c-article-author-list__button,.js .c-article-author-list__item--hide,.js .c-article-author-list__show-more{display:none}.js .c-article-author-list--long .c-article-author-list__show-more,.js .c-article-author-list--long+.c-article-author-list__button{display:inline}@media only screen and (max-width:539px){.js .c-article-author-list__item--hide-small-screen{display:none}.js .c-article-author-list--short .c-article-author-list__show-more,.js .c-article-author-list--short+.c-article-author-list__button{display:inline}}#uptodate-client,.js .c-article-author-list--expanded .c-article-author-list__show-more{display:none!important}.js .c-article-author-list--expanded .c-article-author-list__item--hide-small-screen{display:inline!important}.c-article-author-list__button,.c-button-author-list{background:#ebf1f5;border:4px solid #ebf1f5;border-radius:20px;color:#666;font-size:.875rem;line-height:1.4;padding:2px 11px 2px 8px;text-decoration:none}.c-article-author-list__button svg,.c-button-author-list svg{margin:1px 4px 0 0}.c-article-author-list__button:hover,.c-button-author-list:hover{background:#069;border-color:transparent;color:#fff}.c-article-info-details{font-size:1rem;margin-bottom:8px;margin-top:16px}.c-article-info-details__cite-as{border-left:1px solid #6f6f6f;margin-left:8px;padding-left:8px}.c-article-metrics-bar{display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3}.c-article-metrics-bar__wrapper{margin:16px 0}.c-article-metrics-bar__item{align-items:baseline;border-right:1px solid #6f6f6f;margin-right:8px}.c-article-metrics-bar__item:last-child{border-right:0}.c-article-metrics-bar__count{font-weight:700;margin:0}.c-article-metrics-bar__label{color:#626262;font-style:normal;font-weight:400;margin:0 10px 0 5px}.c-article-metrics-bar__details{margin:0}.c-article-main-column{font-family:Harding,Palatino,serif;margin-right:8.6%;width:60.2%}@media only screen and (max-width:1023px){.c-article-main-column{margin-right:0;width:100%}}.c-article-extras{float:left;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;width:31.2%}@media only screen and (max-width:1023px){.c-article-extras{display:none}}.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{border-bottom:2px solid #d5d5d5;font-size:1.25rem;margin:0;padding-bottom:8px}@media only screen and (min-width:768px){.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{font-size:1.5rem;line-height:1.24}}.c-article-associated-content__container .c-article-associated-content__title{margin-bottom:8px}.c-article-body p{margin-bottom:24px;margin-top:0}.c-article-section{clear:both}.c-article-section__content{margin-bottom:40px;padding-top:8px}@media only screen and (max-width:1023px){.c-article-section__content{padding-left:0}}.c-article-authors-search{margin-bottom:24px;margin-top:0}.c-article-authors-search__item,.c-article-authors-search__title{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.c-article-authors-search__title{color:#626262;font-size:1.05rem;font-weight:700;margin:0;padding:0}.c-article-authors-search__item{font-size:1rem}.c-article-authors-search__text{margin:0}.c-article-license__badge,c-card__section{margin-top:8px}.c-code-block{border:1px solid #eee;font-family:monospace;margin:0 0 24px;padding:20px}.c-code-block__heading{font-weight:400;margin-bottom:16px}.c-code-block__line{display:block;overflow-wrap:break-word;white-space:pre-wrap}.c-article-share-box__no-sharelink-info{font-size:.813rem;font-weight:700;margin-bottom:24px;padding-top:4px}.c-article-share-box__only-read-input{border:1px solid #d5d5d5;box-sizing:content-box;display:inline-block;font-size:.875rem;font-weight:700;height:24px;margin-bottom:8px;padding:8px 10px}.c-article-share-box__button--link-like{background-color:transparent;border:0;color:#069;cursor:pointer;font-size:.875rem;margin-bottom:8px;margin-left:10px}.c-article-editorial-summary__container{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem}.c-article-editorial-summary__container .c-article-editorial-summary__content p:last-child{margin-bottom:0}.c-article-editorial-summary__container .c-article-editorial-summary__content--less{max-height:9.5rem;overflow:hidden}.c-article-editorial-summary__container .c-article-editorial-summary__button{background-color:#fff;border:0;color:#069;font-size:.875rem;margin-bottom:16px}.c-article-editorial-summary__container .c-article-editorial-summary__button.active,.c-article-editorial-summary__container .c-article-editorial-summary__button.hover,.c-article-editorial-summary__container .c-article-editorial-summary__button:active,.c-article-editorial-summary__container .c-article-editorial-summary__button:hover{text-decoration:underline;text-decoration-skip-ink:auto}.c-article-editorial-summary__container .c-article-editorial-summary__button:focus{outline:3px solid #fece3e;will-change:transform}.c-article-associated-content__container .c-article-associated-content__collection-label{font-size:.875rem;line-height:1.4}.c-article-associated-content__container .c-article-associated-content__collection-title{line-height:1.3}.c-context-bar{box-shadow:0 0 10px 0 rgba(51,51,51,.2);position:relative;width:100%}.c-context-bar__title{display:none}.c-reading-companion{clear:both;min-height:389px}.c-reading-companion__sticky{max-width:389px}.c-reading-companion__scroll-pane{margin:0;min-height:200px;overflow:hidden auto}.c-reading-companion__tabs{display:flex;flex-flow:row nowrap;font-size:1rem;list-style:none;margin:0 0 8px;padding:0}.c-reading-companion__tabs>li{flex-grow:1}.c-reading-companion__tab{background-color:#eee;border:1px solid #d5d5d5;border-image:initial;border-left-width:0;color:#069;font-size:1rem;padding:8px 8px 8px 15px;text-align:left;width:100%}.c-reading-companion__tabs li:first-child .c-reading-companion__tab{border-left-width:1px}.c-reading-companion__tab--active{background-color:#fff;border-bottom:1px solid #fff;color:#222;font-weight:700}.c-reading-companion__sections-list{list-style:none;padding:0}.c-reading-companion__figures-list,.c-reading-companion__references-list{list-style:none;min-height:389px;padding:0}.c-reading-companion__references-list--numeric{list-style:decimal inside}.c-reading-companion__sections-list{margin:0 0 8px;min-height:50px}.c-reading-companion__section-item{font-size:1rem;padding:0}.c-reading-companion__section-item a{display:block;line-height:1.5;overflow:hidden;padding:8px 0 8px 16px;text-overflow:ellipsis;white-space:nowrap}.c-reading-companion__figure-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:16px 8px 16px 0}.c-reading-companion__figure-item:first-child{border-top:none;padding-top:8px}.c-reading-companion__reference-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:8px 8px 8px 16px}.c-reading-companion__reference-item:first-child{border-top:none}.c-reading-companion__reference-item a{word-break:break-word}.c-reading-companion__reference-citation{display:inline}.c-reading-companion__reference-links{font-size:.813rem;font-weight:700;list-style:none;margin:8px 0 0;padding:0;text-align:right}.c-reading-companion__reference-links>a{display:inline-block;padding-left:8px}.c-reading-companion__reference-links>a:first-child{display:inline-block;padding-left:0}.c-reading-companion__figure-title{display:block;margin:0 0 8px}.c-reading-companion__figure-links{display:flex;justify-content:space-between;margin:8px 0 0}.c-reading-companion__figure-links>a{align-items:center;display:flex}.c-reading-companion__figure-full-link svg{height:.8em;margin-left:2px}.c-reading-companion__panel{border-top:none;display:none;margin-top:0;padding-top:0}.c-cod,.c-reading-companion__panel--active{display:block}.c-cod{font-size:1rem;width:100%}.c-cod__form{background:#ebf0f3}.c-cod__prompt{font-size:1.125rem;line-height:1.3;margin:0 0 24px}.c-cod__label{display:block;margin:0 0 4px}.c-cod__row{display:flex;margin:0 0 16px}.c-cod__row:last-child{margin:0}.c-cod__input{border:1px solid #d5d5d5;border-radius:2px;flex-basis:75%;flex-shrink:0;margin:0;padding:13px}.c-cod__input--submit{background-color:#069;border:1px solid #069;color:#fff;flex-shrink:1;margin-left:8px;transition:background-color .2s ease-out 0s,color .2s ease-out 0s}.c-cod__input--submit-single{flex-basis:100%;flex-shrink:0;margin:0}.c-cod__input--submit:focus,.c-cod__input--submit:hover{background-color:#fff;color:#069}.c-pdf-download__link .u-icon{padding-top:2px}.c-pdf-download{display:flex;margin-bottom:16px;max-height:48px}@media only screen and (min-width:540px){.c-pdf-download{max-height:none}}@media only screen and (min-width:1024px){.c-pdf-download{max-height:48px}}.c-pdf-download__link{display:flex;flex:1 1 0%;padding:13px 24px}.c-pdf-download__link:hover{text-decoration:none}.c-pdf-download__text{padding-right:4px}@media only screen and (max-width:539px){.c-pdf-download__text{text-transform:capitalize}}@media only screen and (min-width:540px){.c-pdf-download__text{padding-right:8px}}.c-context-bar--sticky .c-pdf-download{display:block;margin-bottom:0;white-space:nowrap}@media only screen and (max-width:539px){.c-pdf-download .u-sticky-visually-hidden{clip:rect(0,0,0,0);border:0;height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}}.c-pdf-container{display:flex;justify-content:flex-end}@media only screen and (max-width:539px){.c-pdf-container .c-pdf-download{display:flex;flex-basis:100%}}.c-pdf-container .c-pdf-download+.c-pdf-download{margin-left:16px}.c-article-extras .c-pdf-container .c-pdf-download{width:100%}.c-article-extras .c-pdf-container .c-pdf-download+.c-pdf-download{margin-left:0}@media only screen and (min-width:540px){.c-context-bar--sticky .c-pdf-download__link{align-items:center;flex:1 1 183px}}@media only screen and (max-width:320px){.c-context-bar--sticky .c-pdf-download__link{padding:16px}}.article-page--commercial .c-article-main-column .c-pdf-button__container .c-pdf-download{display:none}@media only screen and (max-width:1023px){.article-page--commercial .c-article-main-column .c-pdf-button__container .c-pdf-download{display:block}}.c-status-message--success{border-bottom:2px solid #00b8b0;justify-content:center;margin-bottom:16px;padding-bottom:16px}.c-recommendations-title{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;font-weight:700;line-height:1.24;margin:0;padding-bottom:16px}.c-recommendations-close{background-color:transparent;border:0;cursor:pointer;height:2em;margin-right:-10px;margin-top:-5px;width:2em}.c-recommendations-authors{line-height:1.24;margin:0}.c-recommendations-list-container{position:relative}.c-recommendations-list{display:flex;flex-wrap:nowrap;justify-content:space-between;margin:0 auto;overflow-x:hidden;padding:0 0 16px;scroll-behavior:smooth;scroll-snap-type:x mandatory;width:calc(100% - 128px)}@media only screen and (max-width:539px){.c-recommendations-list{display:block;height:40vh;overflow-y:auto;width:100%}}.c-recommendations-list__item{display:flex;flex:0 0 calc(33.3333% - 24px);margin:0 24px 0 0;scroll-snap-align:center}@media only screen and (max-width:539px){.c-recommendations-list__item{margin:0;padding:0 0 16px}}.c-recommendations-list__item .c-card__image{align-items:baseline;flex:1 1 40%;margin:0 16px 0 0;max-width:150px}.c-recommendations-list__item .c-card__image img{border:1px solid #d5d5d5;height:auto;min-height:0;position:relative;transform:translateY(0)}@media only screen and (max-width:1023px){.c-recommendations-list__item .c-card__image{display:none}}.c-card__layout{display:flex;flex:1 1 auto;justify-content:space-between}.c-card__title-recommendation{-webkit-box-orient:vertical;-webkit-line-clamp:4;display:-webkit-box;font-size:1rem;font-weight:700;line-height:1.4;margin:0;max-height:5.6em;overflow:hidden!important;text-overflow:ellipsis}.c-card__title-recommendation .c-card__link{color:#069;text-decoration:none}.c-card__title-recommendation .c-card__link:hover{text-decoration:underline}@media only screen and (max-width:539px){.c-recommendations-column-switch{display:flex;flex-direction:column-reverse}}.js-greyout-page-background{background-color:rgba(34,34,34,.75);bottom:0;left:0;position:fixed;right:0;top:0}.c-article-metrics__heading a,.c-article-metrics__posts .c-card__title a{color:inherit}.c-article-metrics__posts .c-card__title{font-size:1.05rem}.c-article-metrics__posts .c-card__title+span{color:#6f6f6f;font-size:1rem}p{overflow-wrap:break-word;word-break:break-word}.c-ad{text-align:center}@media only screen and (min-width:320px){.c-ad{padding:8px}}.c-ad--728x90{background-color:#ccc;display:none}.c-ad--728x90 .c-ad__inner{min-height:calc(1.5em + 94px)}@media only screen and (min-width:768px){.js .c-ad--728x90{display:none}}.c-ad__label{color:#333;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;font-weight:400;line-height:1.5;margin-bottom:4px}.c-author-list{color:#6f6f6f;font-family:inherit;font-size:1rem;line-height:inherit;list-style:none;margin:0;padding:0}.c-author-list>li,.c-breadcrumbs>li,.c-footer__links>li,.js .c-author-list,.u-list-comma-separated>li,.u-list-inline>li{display:inline}.c-author-list>li:not(:first-child):not(:last-child):before{content:", "}.c-author-list>li:not(:only-child):last-child:before{content:" & "}.c-author-list--compact{font-size:.875rem;line-height:1.4}.c-author-list--truncated>li:not(:only-child):last-child:before{content:" ... "}.js .c-author-list__hide{display:none;visibility:hidden}.js .c-author-list__hide:first-child+*{margin-block-start:0}.c-meta{color:inherit;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.4;list-style:none;margin:0;padding:0}.c-meta--large{font-size:1rem}.c-meta--large .c-meta__item{margin-bottom:8px}.c-meta__item{display:inline-block;margin-bottom:4px}.c-meta__item:not(:last-child){border-right:1px solid #d5d5d5;margin-right:4px;padding-right:4px}@media only screen and (max-width:539px){.c-meta__item--block-sm-max{display:block}.c-meta__item--block-sm-max:not(:last-child){border-right:none;margin-right:0;padding-right:0}}@media only screen and (min-width:1024px){.c-meta__item--block-at-lg{display:block}.c-meta__item--block-at-lg:not(:last-child){border-right:none;margin-right:0;padding-right:0}}.c-meta__type{font-weight:700;text-transform:none}.c-skip-link{background:#069;bottom:auto;color:#fff;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;padding:8px;position:absolute;text-align:center;transform:translateY(-100%);z-index:9999}@media (prefers-reduced-motion:reduce){.c-skip-link{transition:top .3s ease-in-out 0s}}@media print{.c-skip-link{display:none}}.c-skip-link:link{color:#fff}.c-status-message{align-items:center;box-sizing:border-box;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;position:relative;width:100%}.c-card__summary>p:last-child,.c-status-message :last-child{margin-bottom:0}.c-status-message--boxed{background-color:#fff;border:1px solid #eee;border-radius:2px;line-height:1.4;padding:16px}.c-status-message__heading{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;font-weight:700}.c-status-message__icon{fill:currentcolor;display:inline-block;flex:0 0 auto;height:1.5em;margin-right:8px;transform:translate(0);vertical-align:text-top;width:1.5em}.c-status-message__icon--top{align-self:flex-start}.c-status-message--info .c-status-message__icon{color:#003f8d}.c-status-message--boxed.c-status-message--info{border-bottom:4px solid #003f8d}.c-status-message--error .c-status-message__icon{color:#c40606}.c-status-message--boxed.c-status-message--error{border-bottom:4px solid #c40606}.c-status-message--success .c-status-message__icon{color:#00b8b0}.c-status-message--boxed.c-status-message--success{border-bottom:4px solid #00b8b0}.c-status-message--warning .c-status-message__icon{color:#edbc53}.c-status-message--boxed.c-status-message--warning{border-bottom:4px solid #edbc53}.c-breadcrumbs{color:#000;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;list-style:none;margin:0;padding:0}.c-breadcrumbs__link{color:#666}svg.c-breadcrumbs__chevron{fill:#888;height:10px;margin:4px 4px 0;width:10px}.c-card{background-color:transparent;border:0;box-shadow:none;display:flex;flex-direction:column;font-size:14px;min-width:0;overflow:hidden;padding:0;position:relative}.c-card--no-shape{background:0 0;border:0;box-shadow:none}.c-card__image{display:flex;justify-content:center;overflow:hidden;padding-bottom:56.25%;position:relative}@supports (aspect-ratio:1/1){.c-card__image{padding-bottom:0}}.c-card__image img{left:0;min-height:100%;min-width:100%;position:absolute}@supports ((-o-object-fit:cover) or (object-fit:cover)){.c-card__image img{height:100%;object-fit:cover;width:100%}}.c-card__body{flex:1 1 auto;padding:16px}.c-card--no-shape .c-card__body{padding:0}.c-card--no-shape .c-card__body:not(:first-child){padding-top:16px}.c-card__title{letter-spacing:-.01875rem;margin-bottom:8px;margin-top:0}[lang=de] .c-card__title{hyphens:auto}.c-card__summary{line-height:1.4}.c-card__summary>p{margin-bottom:5px}.c-card__summary a{text-decoration:underline}.c-card__link:not(.c-card__link--no-block-link):before{bottom:0;content:"";left:0;position:absolute;right:0;top:0}.c-card--flush .c-card__body{padding:0}.c-card--major{font-size:1rem}.c-card--dark{background-color:#29303c;border-width:0;color:#e3e4e5}.c-card--dark .c-card__title{color:#fff}.c-card--dark .c-card__link,.c-card--dark .c-card__summary a{color:inherit}.c-header{background-color:#fff;border-bottom:5px solid #000;font-size:1rem;line-height:1.4;margin-bottom:16px}.c-header__row{padding:0;position:relative}.c-header__row:not(:last-child){border-bottom:1px solid #eee}.c-header__split{align-items:center;display:flex;justify-content:space-between}.c-header__logo-container{flex:1 1 0px;line-height:0;margin:8px 24px 8px 0}.c-header__logo{transform:translateZ(0)}.c-header__logo img{max-height:32px}.c-header__container{margin:0 auto;max-width:1280px;padding:0 16px}.c-header__menu{align-items:center;display:flex;flex:0 1 auto;flex-wrap:wrap;font-weight:700;gap:8px 8px;line-height:1.4;list-style:none;margin:0 -8px;padding:0}@media print{.c-header__menu{display:none}}@media only screen and (max-width:1023px){.c-header__menu--hide-lg-max{display:none;visibility:hidden}}.c-header__menu--global{font-weight:400;justify-content:flex-end}.c-header__menu--global svg{display:none;visibility:hidden}@media only screen and (min-width:540px){.c-header__menu--global svg{display:block;visibility:visible}}.c-header__menu--journal{font-size:.875rem;margin:8px 0 8px -8px}@media only screen and (min-width:540px){.c-header__menu--journal{flex-wrap:nowrap;font-size:1rem}}.c-header__item{padding-bottom:0;padding-top:0;position:static}.c-header__item--pipe{border-left:2px solid #eee;padding-left:8px}.c-header__item--padding{padding-bottom:8px;padding-top:8px}@media only screen and (min-width:540px){.c-header__item--dropdown-menu{position:relative}}@media only screen and (min-width:1024px){.c-header__item--hide-lg{display:none;visibility:hidden}}@media only screen and (max-width:767px){.c-header__item--hide-md-max{display:none;visibility:hidden}.c-header__item--hide-md-max:first-child+*{margin-block-start:0}}.c-header__link{align-items:center;color:inherit;display:inline-flex;gap:4px 4px;padding:8px;white-space:nowrap}.c-header__link svg{transition-duration:.2s}.c-header__show-text{display:none;visibility:hidden}.has-tethered .c-header__heading--js-hide:first-child+*{margin-block-start:0}@media only screen and (min-width:540px){.c-header__show-text{display:inline;visibility:visible}}.c-header__dropdown{background-color:#000;border-bottom:1px solid #2f2f2f;color:#eee;font-size:.875rem;line-height:1.2;padding:16px 0}@media print{.c-header__dropdown{display:none}}.c-header__heading{display:inline-block;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.25rem;font-weight:400;line-height:1.4;margin-bottom:8px}.c-header__heading--keyline{border-top:1px solid;border-color:#2f2f2f;margin-top:16px;padding-top:16px;width:100%}.c-header__list{display:flex;flex-wrap:wrap;gap:0 16px;list-style:none;margin:0 -8px;padding:0}.c-header__flush{margin:0 -8px}.c-header__visually-hidden{clip:rect(0,0,0,0);border:0;height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}.c-header__search-form{margin-bottom:8px}.c-header__search-layout{display:flex;flex-wrap:wrap;gap:16px 16px}.c-header__search-layout>:first-child{flex:999 1 auto}.c-header__search-layout>*{flex:1 1 auto}.c-header__search-layout--max-width{max-width:720px}.c-header__search-button{align-items:center;background-color:transparent;background-image:none;border:1px solid #fff;border-radius:2px;color:#fff;cursor:pointer;display:flex;font-family:sans-serif;font-size:1rem;justify-content:center;line-height:1.15;margin:0;padding:8px 16px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:100%}.u-button svg,.u-button--primary svg{fill:currentcolor}.c-header__input,.c-header__select{border:1px solid;border-radius:3px;box-sizing:border-box;font-size:1rem;padding:8px 16px;width:100%}.c-header__select{-webkit-appearance:none;background-image:url("data:image/svg+xml,%3Csvg height='16' viewBox='0 0 16 16' width='16' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z' fill='%23333' fill-rule='evenodd' transform='matrix(0 1 -1 0 11 3)'/%3E%3C/svg%3E");background-position:right .7em top 50%;background-repeat:no-repeat;background-size:1em;box-shadow:0 1px 0 1px rgba(0,0,0,.04);display:block;margin:0;max-width:100%;min-width:150px}@media only screen and (min-width:540px){.c-header__menu--journal .c-header__item--dropdown-menu:last-child .c-header__dropdown.has-tethered{left:auto;right:0}}@media only screen and (min-width:768px){.c-header__menu--journal .c-header__item--dropdown-menu:last-child .c-header__dropdown.has-tethered{left:0;right:auto}}.c-header__dropdown.has-tethered{border-bottom:0;border-radius:0 0 2px 2px;left:0;position:absolute;top:100%;transform:translateY(5px);width:100%;z-index:1}@media only screen and (min-width:540px){.c-header__dropdown.has-tethered{transform:translateY(8px);width:auto}}@media only screen and (min-width:768px){.c-header__dropdown.has-tethered{min-width:225px}}.c-header__dropdown--full-width.has-tethered{padding:32px 0 24px;transform:none;width:100%}.has-tethered .c-header__heading--js-hide{display:none;visibility:hidden}.has-tethered .c-header__list--js-stack{flex-direction:column}.has-tethered .c-header__item--keyline,.has-tethered .c-header__list~.c-header__list .c-header__item:first-child{border-top:1px solid #d5d5d5;margin-top:8px;padding-top:8px}.u-button{align-items:center;background-color:transparent;background-image:none;border:1px solid #069;border-radius:2px;color:#069;cursor:pointer;display:inline-flex;font-family:sans-serif;font-size:1rem;justify-content:center;line-height:1.3;margin:0;padding:8px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:auto}.u-button--primary{background-color:#069;background-image:none;border:1px solid #069;color:#fff}.u-button--full-width{display:flex;width:100%}.u-display-none{display:none}.js .u-js-hide,.u-hide{display:none;visibility:hidden}.u-hide:first-child+*{margin-block-start:0}.u-visually-hidden{clip:rect(0,0,0,0);border:0;height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}@media print{.u-hide-print{display:none}}@media only screen and (min-width:540px){.u-hide-at-sm{display:none;visibility:hidden}}@media only screen and (min-width:1024px){.u-hide-at-lg{display:none;visibility:hidden}.u-hide-at-lg:first-child+*{margin-block-start:0}}.u-clearfix:after,.u-clearfix:before{content:"";display:table}.u-clearfix:after{clear:both}.u-color-open-access{color:#b74616}.u-float-left{float:left}.u-icon{fill:currentcolor;display:inline-block;height:1em;transform:translate(0);vertical-align:text-top;width:1em}.u-full-height{height:100%}.u-list-reset{list-style:none;margin:0;padding:0}.u-sans-serif{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.u-container{margin:0 auto;max-width:1280px;padding:0 16px}.u-display-flex{display:flex;width:100%}.u-flex-direction-column{flex-direction:column}.u-justify-content-space-between{justify-content:space-between}.u-flex-static{flex:0 0 auto}.u-mt-32{margin-top:32px}.u-mb-8{margin-bottom:8px}.u-mb-16{margin-bottom:16px}.u-mb-24{margin-bottom:24px}.u-mb-32{margin-bottom:32px}.c-nature-box svg+.c-article__button-text,.u-ml-8{margin-left:8px}.u-pa-16{padding:16px}html *,html :after,html :before{box-sizing:inherit}.c-article-section__title,.c-article-title{font-weight:700}.c-card__title{line-height:1.4em}.c-article__button{background-color:#069;border:1px solid #069;border-radius:2px;color:#fff;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.4;margin-bottom:16px;padding:13px;transition:background-color .2s ease-out 0s,color .2s ease-out 0s}.c-article__button,.c-article__button:hover{text-decoration:none}.c-article__button--inverted,.c-article__button:hover{background-color:#fff;color:#069}.c-article__button--inverted:hover{background-color:#069;color:#fff}.c-header__link{text-decoration:inherit}.grade-c-hide{display:block}.u-lazy-ad-wrapper{background-color:#ccc;display:none;min-height:137px}@media only screen and (min-width:768px){.u-lazy-ad-wrapper{display:block}}.c-nature-box{background-color:#fff;border:1px solid #d5d5d5;border-radius:2px;box-shadow:0 0 5px 0 rgba(51,51,51,.1);line-height:1.3;margin-bottom:24px;padding:16px 16px 3px}.c-nature-box__text{font-size:1rem;margin-bottom:16px}.c-nature-box .c-pdf-download{margin-bottom:16px!important}.c-nature-box--version{background-color:#eee}.c-nature-box__wrapper{transform:translateZ(0)}.c-nature-box__wrapper--placeholder{min-height:165px} }
  </style>
  <link data-inline-css-source="critical-css" data-test="critical-css-handler" href="/static/css/enhanced-article-nature-branded-30b9d5ba44.css" media="only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)" onload="this.media='only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)';this.onload=null" rel="stylesheet"/>
  <noscript>
   <link href="/static/css/enhanced-article-nature-branded-30b9d5ba44.css" media="only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)" rel="stylesheet" type="text/css"/>
  </noscript>
  <link href="/static/css/article-print-122346e276.css" media="print" rel="stylesheet" type="text/css"/>
  <link href="/static/images/favicons/nature/apple-touch-icon-f39cb19454.png" rel="apple-touch-icon" sizes="180x180"/>
  <link href="/static/images/favicons/nature/favicon-32x32-3fe59ece92.png" rel="icon" sizes="32x32" type="image/png"/>
  <link href="/static/images/favicons/nature/favicon-16x16-951651ab72.png" rel="icon" sizes="16x16" type="image/png"/>
  <link crossorigin="use-credentials" href="/static/manifest.json" rel="manifest"/>
  <link color="#000000" href="/static/images/favicons/nature/safari-pinned-tab-69bff48fe6.svg" rel="mask-icon"/>
  <link href="/static/images/favicons/nature/favicon.ico" rel="shortcut icon"/>
  <meta content="#000000" name="msapplication-TileColor"/>
  <meta content="/static/browserconfig.xml" name="msapplication-config"/>
  <meta content="#000000" name="theme-color"/>
  <meta content="Nature" name="application-name"/>
  <script>
   (function () {
        if ( typeof window.CustomEvent === "function" ) return false;
        function CustomEvent ( event, params ) {
            params = params || { bubbles: false, cancelable: false, detail: null };
            var evt = document.createEvent( 'CustomEvent' );
            evt.initCustomEvent( event, params.bubbles, params.cancelable, params.detail );
            return evt;
        }

        CustomEvent.prototype = window.Event.prototype;

        window.CustomEvent = CustomEvent;
    })();
  </script>
  <!-- Google Tag Manager -->
  <script data-test="gtm-head">
   window.initGTM = function() {
        if (window.config.mustardcut) {
            (function (w, d, s, l, i) {
                w[l] = w[l] || [];
                w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                var f = d.getElementsByTagName(s)[0],
                        j = d.createElement(s),
                        dl = l != 'dataLayer' ? '&l=' + l : '';
                j.async = true;
                j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-MRVXSHQ');
        }
    }
  </script>
  <!-- End Google Tag Manager -->
  <script>
   (function(w,d,t) {
        function cc() {
            var h = w.location.hostname;
            if (h.indexOf('preview-www.nature.com') > -1) return;

            var e = d.createElement(t),
                    s = d.getElementsByTagName(t)[0];

            if (h.indexOf('nature.com') > -1) {
                if (h.indexOf('test-www.nature.com') > -1) {
                    e.src = 'https://cmp-static-nature-com.proxy.lib.ohio-state.edu/production_live/consent-bundle-8-23.js';
                    e.setAttribute('onload', "initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')");
                } else {
                    e.src = 'https://cmp-static-nature-com.proxy.lib.ohio-state.edu/production_live/consent-bundle-8-23.js';
                    e.setAttribute('onload', "initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')");
                }
            } else {
                e.src = '/static/js/cookie-consent-es5-bundle-2b0f06c1e4.js';
                e.setAttribute('data-consent', h);
            }
            s.insertAdjacentElement('afterend', e);
        }

        cc();
    })(window,document,'script');
  </script>
  <script id="js-position0">
   (function(w, d) {
        w.idpVerifyPrefix = 'https://verify-nature-com.proxy.lib.ohio-state.edu';
        w.ra21Host = 'https://wayf-springernature-com.proxy.lib.ohio-state.edu';
        var moduleSupport = (function() {
            return 'noModule' in d.createElement('script');
        })();

        var polyfillsUrl = function() {
            var features = {
                'IntersectionObserver': window.IntersectionObserver,
                'Promise': window.Promise,
                'URLSearchParams': window.URLSearchParams,
                'Symbol.iterator': window.Symbol && Symbol.iterator,
                'Array.from': Array.from,
                'Array.prototype.includes': Array.prototype.includes,
                'Array.prototype.find': Array.prototype.find,
                'Array.prototype.forEach': Array.prototype.forEach,
                'NodeList.prototype.forEach': NodeList.prototype.forEach,
                'Element.prototype.closest': Element.prototype.closest,
                'Element.prototype.prepend': Element.prototype.prepend,
                'Element.prototype.remove': Element.prototype.remove,
                'Object.assign': Object.assign
            };
            var req = [];
            for (var feature in features) {
                if (Object.prototype.hasOwnProperty.call(features, feature) && !features[feature]) {
                    req.push(feature);
                }
            }
            if (req.length) {
                return 'https://polyfill.io/v3/polyfill.min.js?features=' + req.join('%2C') + '&flags=always';
            }
            return null;
        };

        if (w.config.mustardcut === true) {
            w.loader = {
                index: 0,
                registered: [],
                scripts: [
                    {src: polyfillsUrl(), test: 'polyfills-js', noinit: true},
                    
                        {src: '/static/js/global-article-es6-bundle-62a1428781.js', test: 'global-article-js', module: true},
                        {src: '/static/js/global-article-es5-bundle-ca6a20c074.js', test: 'global-article-js', nomodule: true},
                        {src: '/static/js/shared-es6-bundle-7850fbb459.js', test: 'shared-js', module: true},
                        {src: '/static/js/shared-es5-bundle-bd334e676d.js', test: 'shared-js', nomodule: true},
                        {src: '/static/js/header-150-es6-bundle-5bb959eaa1.js', test: 'header-150-js', module: true},
                        {src: '/static/js/header-150-es5-bundle-c634a291c7.js', test: 'header-150-js', nomodule: true}
                    
                ].filter(function (s) {
                    if (s.src === null) return false;
                    if (moduleSupport && s.nomodule) return false;
                    return !(!moduleSupport && s.module);
                }),

                register: function (value) {
                    this.registered.push(value);
                },

                ready: function () {
                    if (this.registered.length === this.scripts.length) {
                        this.registered.forEach(function (fn) {
                            if (typeof fn === 'function') {
                                setTimeout(fn, 0); 
                            }
                        });
                        this.ready = function () {};
                    }
                },

                insert: function (s) {
                    var t = d.getElementById('js-position' + this.index);
                    if (t && t.insertAdjacentElement) {
                        t.insertAdjacentElement('afterend', s);
                    } else {
                        d.head.appendChild(s);
                    }
                    ++this.index;
                },

                createScript: function (script, beforeLoad) {
                    var s = d.createElement('script');
                    s.id = 'js-position' + (this.index + 1);
                    s.setAttribute('data-test', script.test);
                    if (beforeLoad) {
                        s.defer = 'defer';
                        s.onload = function () {
                            if (script.noinit) {
                                loader.register(true);
                            }
                            if (d.readyState === 'interactive' || d.readyState === 'complete') {
                                loader.ready();
                            }
                        };
                    } else {
                        s.async = 'async';
                    }
                    s.src = script.src;
                    return s;
                },

                init: function () {
                    this.scripts.forEach(function (s) {
                        loader.insert(loader.createScript(s, true));
                    });

                    d.addEventListener('DOMContentLoaded', function () {
                        loader.ready();
                        
                            var conditionalScripts = [
                                {match: 'div[data-pan-container]', src: '/static/js/pan-zoom-es6-bundle-464a2af269.js', test: 'pan-zoom-js',  module: true },
                                {match: 'div[data-pan-container]', src: '/static/js/pan-zoom-es5-bundle-2626f1bdf8.js', test: 'pan-zoom-js',  nomodule: true },
                                {match: 'math,span.mathjax-tex', src: '/static/js/math-es6-bundle-bd14bd0747.js', test: 'math-js', module: true},
                                {match: 'math,span.mathjax-tex', src: '/static/js/math-es5-bundle-2d5c465efd.js', test: 'math-js', nomodule: true}
                            ];
                        

                        if (conditionalScripts) {
                            conditionalScripts.filter(function (script) {
                                return !!document.querySelector(script.match) && !((moduleSupport && script.nomodule) || (!moduleSupport && script.module));
                            }).forEach(function (script) {
                                loader.insert(loader.createScript(script));
                            });
                        }
                    }, false);
                }
            };
            loader.init();
        }
    })(window, document);
  </script>
  <script data-test="global-article-js" defer="" id="js-position1" src="/static/js/global-article-es6-bundle-62a1428781.js">
  </script>
  <script data-test="shared-js" defer="" id="js-position2" src="/static/js/shared-es6-bundle-7850fbb459.js">
  </script>
  <script data-test="header-150-js" defer="" id="js-position3" src="/static/js/header-150-es6-bundle-5bb959eaa1.js">
  </script>
  <meta content="noarchive" name="robots"/>
  <meta content="Yes" name="access"/>
  <link href="https://www-nature-com.proxy.lib.ohio-state.edu/search" rel="search"/>
  <link href="https://www-nature-com.proxy.lib.ohio-state.edu/opensearch/opensearch.xml" rel="search" title="nature.com" type="application/opensearchdescription+xml"/>
  <link href="https://www-nature-com.proxy.lib.ohio-state.edu/opensearch/request" rel="search" title="nature.com" type="application/sru+xml"/>
  <script type="application/ld+json">
   {"mainEntity":{"headline":"Highly accurate protein structure prediction with AlphaFold","description":"Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. AlphaFold predicts protein structures with an accuracy competitive with experimental structures in the majority of cases using a novel deep learning architecture.","datePublished":"2021-07-15","dateModified":"2021-07-15","pageStart":"583","pageEnd":"589","license":"http://creativecommons.org/licenses/by/4.0/","sameAs":"https://doi-org.proxy.lib.ohio-state.edu/10.1038/s41586-021-03819-2","keywords":"Computational biophysics,Machine learning,Protein structure predictions,Structural biology,Science,Humanities and Social Sciences,multidisciplinary","image":"https://static-content.springer.com/image/art%3A10.1038%2Fs41586-021-03819-2/MediaObjects/41586_2021_3819_Fig1_HTML.png","isPartOf":{"name":"Nature","issn":["1476-4687","0028-0836"],"volumeNumber":"596","@type":["Periodical","PublicationVolume"]},"publisher":{"name":"Nature Publishing Group UK","logo":{"url":"https://www-springernature-com.proxy.lib.ohio-state.edu/app-sn/public/images/logo-springernature.png","@type":"ImageObject"},"@type":"Organization"},"author":[{"name":"John Jumper","url":"http://orcid.org/0000-0001-6169-6580","affiliation":[{"name":"DeepMind","address":{"name":"DeepMind, London, UK","@type":"PostalAddress"},"@type":"Organization"}],"email":"jumper@deepmind.com","@type":"Person"},{"name":"Richard Evans","affiliation":[{"name":"DeepMind","address":{"name":"DeepMind, London, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Alexander Pritzel","affiliation":[{"name":"DeepMind","address":{"name":"DeepMind, London, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Tim Green","url":"http://orcid.org/0000-0002-3227-1505","affiliation":[{"name":"DeepMind","address":{"name":"DeepMind, London, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Michael Figurnov","affiliation":[{"name":"DeepMind","address":{"name":"DeepMind, London, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Olaf Ronneberger","affiliation":[{"name":"DeepMind","address":{"name":"DeepMind, London, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Kathryn Tunyasuvunakool","affiliation":[{"name":"DeepMind","address":{"name":"DeepMind, London, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Russ Bates","affiliation":[{"name":"DeepMind","address":{"name":"DeepMind, London, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Augustin Žídek","affiliation":[{"name":"DeepMind","address":{"name":"DeepMind, London, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Anna Potapenko","affiliation":[{"name":"DeepMind","address":{"name":"DeepMind, London, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Alex Bridgland","affiliation":[{"name":"DeepMind","address":{"name":"DeepMind, London, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Clemens Meyer","affiliation":[{"name":"DeepMind","address":{"name":"DeepMind, London, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Simon A. A. Kohl","url":"http://orcid.org/0000-0003-4271-4418","affiliation":[{"name":"DeepMind","address":{"name":"DeepMind, London, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Andrew J. Ballard","affiliation":[{"name":"DeepMind","address":{"name":"DeepMind, London, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Andrew Cowie","affiliation":[{"name":"DeepMind","address":{"name":"DeepMind, London, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Bernardino Romera-Paredes","affiliation":[{"name":"DeepMind","address":{"name":"DeepMind, London, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Stanislav Nikolov","affiliation":[{"name":"DeepMind","address":{"name":"DeepMind, London, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Rishub Jain","affiliation":[{"name":"DeepMind","address":{"name":"DeepMind, London, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Jonas Adler","url":"http://orcid.org/0000-0001-9928-3407","affiliation":[{"name":"DeepMind","address":{"name":"DeepMind, London, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Trevor Back","affiliation":[{"name":"DeepMind","address":{"name":"DeepMind, London, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Stig Petersen","affiliation":[{"name":"DeepMind","address":{"name":"DeepMind, London, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"David Reiman","affiliation":[{"name":"DeepMind","address":{"name":"DeepMind, London, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Ellen Clancy","affiliation":[{"name":"DeepMind","address":{"name":"DeepMind, London, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Michal Zielinski","affiliation":[{"name":"DeepMind","address":{"name":"DeepMind, London, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Martin Steinegger","url":"http://orcid.org/0000-0001-8781-9753","affiliation":[{"name":"Seoul National University","address":{"name":"School of Biological Sciences, Seoul National University, Seoul, South Korea","@type":"PostalAddress"},"@type":"Organization"},{"name":"Seoul National University","address":{"name":"Artificial Intelligence Institute, Seoul National University, Seoul, South Korea","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Michalina Pacholska","url":"http://orcid.org/0000-0002-2160-6226","affiliation":[{"name":"DeepMind","address":{"name":"DeepMind, London, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Tamas Berghammer","affiliation":[{"name":"DeepMind","address":{"name":"DeepMind, London, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Sebastian Bodenstein","affiliation":[{"name":"DeepMind","address":{"name":"DeepMind, London, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"David Silver","url":"http://orcid.org/0000-0002-5197-2892","affiliation":[{"name":"DeepMind","address":{"name":"DeepMind, London, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Oriol Vinyals","affiliation":[{"name":"DeepMind","address":{"name":"DeepMind, London, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Andrew W. Senior","url":"http://orcid.org/0000-0002-2401-5691","affiliation":[{"name":"DeepMind","address":{"name":"DeepMind, London, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Koray Kavukcuoglu","affiliation":[{"name":"DeepMind","address":{"name":"DeepMind, London, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Pushmeet Kohli","affiliation":[{"name":"DeepMind","address":{"name":"DeepMind, London, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Demis Hassabis","url":"http://orcid.org/0000-0003-2812-9917","affiliation":[{"name":"DeepMind","address":{"name":"DeepMind, London, UK","@type":"PostalAddress"},"@type":"Organization"}],"email":"dhcontact@deepmind.com","@type":"Person"}],"isAccessibleForFree":true,"@type":"ScholarlyArticle"},"@context":"https://schema.org","@type":"WebPage"}
  </script>
  <link href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/s41586-021-03819-2" rel="canonical"/>
  <meta content="41586" name="journal_id"/>
  <meta content="Highly accurate protein structure prediction with AlphaFold" name="dc.title"/>
  <meta content="Nature 2021 596:7873" name="dc.source"/>
  <meta content="text/html" name="dc.format"/>
  <meta content="Nature Publishing Group" name="dc.publisher"/>
  <meta content="2021-07-15" name="dc.date"/>
  <meta content="OriginalPaper" name="dc.type"/>
  <meta content="En" name="dc.language"/>
  <meta content="2021 The Author(s)" name="dc.copyright"/>
  <meta content="2021 The Author(s)" name="dc.rights"/>
  <meta content="journalpermissions@springernature.com" name="dc.rightsAgent"/>
  <meta content="Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50&amp;nbsp;years9. Despite recent progress10–14, existing methods fall far&amp;nbsp;short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. AlphaFold predicts protein structures with an accuracy competitive with experimental structures in the majority of cases using a novel deep learning architecture." name="dc.description"/>
  <meta content="1476-4687" name="prism.issn"/>
  <meta content="Nature" name="prism.publicationName"/>
  <meta content="2021-07-15" name="prism.publicationDate"/>
  <meta content="596" name="prism.volume"/>
  <meta content="7873" name="prism.number"/>
  <meta content="OriginalPaper" name="prism.section"/>
  <meta content="583" name="prism.startingPage"/>
  <meta content="589" name="prism.endingPage"/>
  <meta content="2021 The Author(s)" name="prism.copyright"/>
  <meta content="journalpermissions@springernature.com" name="prism.rightsAgent"/>
  <meta content="https://www-nature-com.proxy.lib.ohio-state.edu/articles/s41586-021-03819-2" name="prism.url"/>
  <meta content="doi:10.1038/s41586-021-03819-2" name="prism.doi"/>
  <meta content="https://www-nature-com.proxy.lib.ohio-state.edu/articles/s41586-021-03819-2.pdf" name="citation_pdf_url"/>
  <meta content="https://www-nature-com.proxy.lib.ohio-state.edu/articles/s41586-021-03819-2" name="citation_fulltext_html_url"/>
  <meta content="Nature" name="citation_journal_title"/>
  <meta content="Nature" name="citation_journal_abbrev"/>
  <meta content="Nature Publishing Group" name="citation_publisher"/>
  <meta content="1476-4687" name="citation_issn"/>
  <meta content="Highly accurate protein structure prediction with AlphaFold" name="citation_title"/>
  <meta content="596" name="citation_volume"/>
  <meta content="7873" name="citation_issue"/>
  <meta content="2021/08" name="citation_publication_date"/>
  <meta content="2021/07/15" name="citation_online_date"/>
  <meta content="583" name="citation_firstpage"/>
  <meta content="589" name="citation_lastpage"/>
  <meta content="Article" name="citation_article_type"/>
  <meta content="" name="citation_fulltext_world_readable"/>
  <meta content="en" name="citation_language"/>
  <meta content="doi:10.1038/s41586-021-03819-2" name="dc.identifier"/>
  <meta content="10.1038/s41586-021-03819-2" name="DOI"/>
  <meta content="219300" name="size"/>
  <meta content="10.1038/s41586-021-03819-2" name="citation_doi"/>
  <meta content="http://api.springer-com.proxy.lib.ohio-state.edu/xmldata/jats?q=doi:10.1038/s41586-021-03819-2&amp;api_key=" name="citation_springer_api_url"/>
  <meta content="Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50&amp;nbsp;years9. Despite recent progress10–14, existing methods fall far&amp;nbsp;short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. AlphaFold predicts protein structures with an accuracy competitive with experimental structures in the majority of cases using a novel deep learning architecture." name="description"/>
  <meta content="Jumper, John" name="dc.creator"/>
  <meta content="Evans, Richard" name="dc.creator"/>
  <meta content="Pritzel, Alexander" name="dc.creator"/>
  <meta content="Green, Tim" name="dc.creator"/>
  <meta content="Figurnov, Michael" name="dc.creator"/>
  <meta content="Ronneberger, Olaf" name="dc.creator"/>
  <meta content="Tunyasuvunakool, Kathryn" name="dc.creator"/>
  <meta content="Bates, Russ" name="dc.creator"/>
  <meta content="Žídek, Augustin" name="dc.creator"/>
  <meta content="Potapenko, Anna" name="dc.creator"/>
  <meta content="Bridgland, Alex" name="dc.creator"/>
  <meta content="Meyer, Clemens" name="dc.creator"/>
  <meta content="Kohl, Simon A. A." name="dc.creator"/>
  <meta content="Ballard, Andrew J." name="dc.creator"/>
  <meta content="Cowie, Andrew" name="dc.creator"/>
  <meta content="Romera-Paredes, Bernardino" name="dc.creator"/>
  <meta content="Nikolov, Stanislav" name="dc.creator"/>
  <meta content="Jain, Rishub" name="dc.creator"/>
  <meta content="Adler, Jonas" name="dc.creator"/>
  <meta content="Back, Trevor" name="dc.creator"/>
  <meta content="Petersen, Stig" name="dc.creator"/>
  <meta content="Reiman, David" name="dc.creator"/>
  <meta content="Clancy, Ellen" name="dc.creator"/>
  <meta content="Zielinski, Michal" name="dc.creator"/>
  <meta content="Steinegger, Martin" name="dc.creator"/>
  <meta content="Pacholska, Michalina" name="dc.creator"/>
  <meta content="Berghammer, Tamas" name="dc.creator"/>
  <meta content="Bodenstein, Sebastian" name="dc.creator"/>
  <meta content="Silver, David" name="dc.creator"/>
  <meta content="Vinyals, Oriol" name="dc.creator"/>
  <meta content="Senior, Andrew W." name="dc.creator"/>
  <meta content="Kavukcuoglu, Koray" name="dc.creator"/>
  <meta content="Kohli, Pushmeet" name="dc.creator"/>
  <meta content="Hassabis, Demis" name="dc.creator"/>
  <meta content="Computational biophysics" name="dc.subject"/>
  <meta content="Machine learning" name="dc.subject"/>
  <meta content="Protein structure predictions" name="dc.subject"/>
  <meta content="Structural biology" name="dc.subject"/>
  <meta content="citation_journal_title=F1000Res.; citation_title=Advances in methods for atomic resolution macromolecular structure determination; citation_author=MC Thompson, TO Yeates, JA Rodriguez; citation_volume=9; citation_publication_date=2020; citation_pages=667; citation_doi=10.12688/f1000research.25097.1; citation_id=CR1" name="citation_reference"/>
  <meta content="citation_journal_title=Trends Biochem. Sci.; citation_title=How cryo-EM is revolutionizing structural biology; citation_author=X-C Bai, G McMullan, SHW Scheres; citation_volume=40; citation_publication_date=2015; citation_pages=49-57; citation_doi=10.1016/j.tibs.2014.10.005; citation_id=CR2" name="citation_reference"/>
  <meta content="citation_journal_title=FEBS J.; citation_title=A brief history of macromolecular crystallography, illustrated by a family tree and its Nobel fruits; citation_author=M Jaskolski, Z Dauter, A Wlodawer; citation_volume=281; citation_publication_date=2014; citation_pages=3985-4009; citation_doi=10.1111/febs.12796; citation_id=CR3" name="citation_reference"/>
  <meta content="citation_journal_title=Nat. Struct. Biol.; citation_title=The way to NMR structures of proteins; citation_author=K Wüthrich; citation_volume=8; citation_publication_date=2001; citation_pages=923-925; citation_doi=10.1038/nsb1101-923; citation_id=CR4" name="citation_reference"/>
  <meta content="citation_journal_title=Nucleic Acids Res.; citation_title=Protein Data Bank: the single global archive for 3D macromolecular structure data; citation_author=; citation_volume=47; citation_publication_date=2018; citation_pages=D520-D528; citation_doi=10.1093/nar/gky949; citation_id=CR5" name="citation_reference"/>
  <meta content="citation_journal_title=Nucleic Acids Res.; citation_title=MGnify: the microbiome analysis resource in 2020; citation_author=AL Mitchell; citation_volume=48; citation_publication_date=2020; citation_pages=D570-D578; citation_id=CR6" name="citation_reference"/>
  <meta content="citation_journal_title=Nat. Methods; citation_title=Protein-level assembly increases protein sequence recovery from metagenomic samples manyfold; citation_author=M Steinegger, M Mirdita, J Söding; citation_volume=16; citation_publication_date=2019; citation_pages=603-606; citation_doi=10.1038/s41592-019-0437-4; citation_id=CR7" name="citation_reference"/>
  <meta content="citation_journal_title=Annu. Rev. Biophys.; citation_title=The protein folding problem; citation_author=KA Dill, SB Ozkan, MS Shell, TR Weikl; citation_volume=37; citation_publication_date=2008; citation_pages=289-316; citation_doi=10.1146/annurev.biophys.37.092707.153558; citation_id=CR8" name="citation_reference"/>
  <meta content="citation_journal_title=Science; citation_title=Principles that govern the folding of protein chains; citation_author=CB Anfinsen; citation_volume=181; citation_publication_date=1973; citation_pages=223-230; citation_doi=10.1126/science.181.4096.223; citation_id=CR9" name="citation_reference"/>
  <meta content="citation_journal_title=Nature; citation_title=Improved protein structure prediction using potentials from deep learning; citation_author=AW Senior; citation_volume=577; citation_publication_date=2020; citation_pages=706-710; citation_doi=10.1038/s41586-019-1923-7; citation_id=CR10" name="citation_reference"/>
  <meta content="citation_journal_title=PLOS Comput. Biol.; citation_title=Accurate de novo prediction of protein contact map by ultra-deep learning model; citation_author=S Wang, S Sun, Z Li, R Zhang, J Xu; citation_volume=13; citation_publication_date=2017; citation_pages=e1005324; citation_doi=10.1371/journal.pcbi.1005324; citation_id=CR11" name="citation_reference"/>
  <meta content="citation_journal_title=Proteins; citation_title=Deep-learning contact-map guided protein structure prediction in CASP13; citation_author=W Zheng; citation_volume=87; citation_publication_date=2019; citation_pages=1149-1164; citation_doi=10.1002/prot.25792; citation_id=CR12" name="citation_reference"/>
  <meta content="citation_journal_title=Proteins; citation_title=A further leap of improvement in tertiary structure prediction in CASP13 prompts new routes for future assessments; citation_author=LA Abriata, GE Tamò, M Dal Peraro; citation_volume=87; citation_publication_date=2019; citation_pages=1100-1112; citation_doi=10.1002/prot.25787; citation_id=CR13" name="citation_reference"/>
  <meta content="citation_journal_title=Curr. Opin. Struct. Biol.; citation_title=Deep learning techniques have significantly impacted protein structure prediction and protein design; citation_author=R Pearce, Y Zhang; citation_volume=68; citation_publication_date=2021; citation_pages=194-207; citation_doi=10.1016/j.sbi.2021.01.007; citation_id=CR14" name="citation_reference"/>
  <meta content="Moult, J., Fidelis, K., Kryshtafovych, A., Schwede, T. &amp; Topf, M. Critical assessment of techniques for protein structure prediction, fourteenth round. CASP 14 Abstract Book 
                  https://www.predictioncenter.org/casp14/doc/CASP14_Abstracts.pdf
                  
                 (2020)." name="citation_reference"/>
  <meta content="citation_journal_title=Science; citation_title=Protein storytelling through physics; citation_author=E Brini, C Simmerling, K Dill; citation_volume=370; citation_publication_date=2020; citation_pages=eaaz3041; citation_doi=10.1126/science.aaz3041; citation_id=CR16" name="citation_reference"/>
  <meta content="citation_journal_title=J. Mol. Biol.; citation_title=Calculation of conformational ensembles from potentials of mean force. An approach to the knowledge-based prediction of local structures in globular proteins; citation_author=MJ Sippl; citation_volume=213; citation_publication_date=1990; citation_pages=859-883; citation_doi=10.1016/S0022-2836(05)80269-4; citation_id=CR17" name="citation_reference"/>
  <meta content="citation_journal_title=J. Mol. Biol.; citation_title=Comparative protein modelling by satisfaction of spatial restraints; citation_author=A Šali, TL Blundell; citation_volume=234; citation_publication_date=1993; citation_pages=779-815; citation_doi=10.1006/jmbi.1993.1626; citation_id=CR18" name="citation_reference"/>
  <meta content="citation_journal_title=Nat. Protocols; citation_title=I-TASSER: a unified platform for automated protein structure and function prediction; citation_author=A Roy, A Kucukural, Y Zhang; citation_volume=5; citation_publication_date=2010; citation_pages=725-738; citation_doi=10.1038/nprot.2010.5; citation_id=CR19" name="citation_reference"/>
  <meta content="citation_journal_title=J. Mol. Biol.; citation_title=Correlation of co-ordinated amino acid substitutions with function in viruses related to tobacco mosaic virus; citation_author=D Altschuh, AM Lesk, AC Bloomer, A Klug; citation_volume=193; citation_publication_date=1987; citation_pages=693-707; citation_doi=10.1016/0022-2836(87)90352-4; citation_id=CR20" name="citation_reference"/>
  <meta content="citation_journal_title=Protein Eng.; citation_title=Can three-dimensional contacts in protein structures be predicted by analysis of correlated mutations?; citation_author=IN Shindyalov, NA Kolchanov, C Sander; citation_volume=7; citation_publication_date=1994; citation_pages=349-358; citation_doi=10.1093/protein/7.3.349; citation_id=CR21" name="citation_reference"/>
  <meta content="citation_journal_title=Proc. Natl Acad. Sci. USA; citation_title=Identification of direct residue contacts in protein–protein interaction by message passing; citation_author=M Weigt, RA White, H Szurmant, JA Hoch, T Hwa; citation_volume=106; citation_publication_date=2009; citation_pages=67-72; citation_doi=10.1073/pnas.0805923106; citation_id=CR22" name="citation_reference"/>
  <meta content="citation_journal_title=PLoS ONE; citation_title=Protein 3D structure computed from evolutionary sequence variation; citation_author=DS Marks; citation_volume=6; citation_publication_date=2011; citation_pages=e28766; citation_doi=10.1371/journal.pone.0028766; citation_id=CR23" name="citation_reference"/>
  <meta content="citation_journal_title=Bioinformatics; citation_title=PSICOV: precise structural contact prediction using sparse inverse covariance estimation on large multiple sequence alignments; citation_author=DT Jones, DWA Buchan, D Cozzetto, M Pontil; citation_volume=28; citation_publication_date=2012; citation_pages=184-190; citation_doi=10.1093/bioinformatics/btr638; citation_id=CR24" name="citation_reference"/>
  <meta content="citation_journal_title=Proteins; citation_title=A large-scale experiment to assess protein structure prediction methods; citation_author=J Moult, JT Pedersen, R Judson, K Fidelis; citation_volume=23; citation_publication_date=1995; citation_pages=ii-iv; citation_doi=10.1002/prot.340230303; citation_id=CR25" name="citation_reference"/>
  <meta content="citation_journal_title=Proteins; citation_title=Critical assessment of methods of protein structure prediction (CASP)-round XIII; citation_author=A Kryshtafovych, T Schwede, M Topf, K Fidelis, J Moult; citation_volume=87; citation_publication_date=2019; citation_pages=1011-1020; citation_doi=10.1002/prot.25823; citation_id=CR26" name="citation_reference"/>
  <meta content="citation_journal_title=Proteins; citation_title=Scoring function for automated assessment of protein structure template quality; citation_author=Y Zhang, J Skolnick; citation_volume=57; citation_publication_date=2004; citation_pages=702-710; citation_doi=10.1002/prot.20264; citation_id=CR27" name="citation_reference"/>
  <meta content="citation_journal_title=IEEE Trans. Pattern Anal. Mach. Intell.; citation_title=Auto-context and its application to high-level vision tasks and 3D brain image segmentation; citation_author=Z Tu, X Bai; citation_volume=32; citation_publication_date=2010; citation_pages=1744-1757; citation_doi=10.1109/TPAMI.2009.186; citation_id=CR28" name="citation_reference"/>
  <meta content="Carreira, J., Agrawal, P., Fragkiadaki, K. &amp; Malik, J. Human pose estimation with iterative error feedback. In Proc. IEEE Conference on Computer Vision and Pattern Recognition 4733–4742 (2016)." name="citation_reference"/>
  <meta content="citation_journal_title=PLoS ONE; citation_title=rawMSA: end-to-end deep learning using raw multiple sequence alignments; citation_author=C Mirabello, B Wallner; citation_volume=14; citation_publication_date=2019; citation_pages=e0220182; citation_doi=10.1371/journal.pone.0220182; citation_id=CR30" name="citation_reference"/>
  <meta content="Huang, Z. et al. CCNet: criss-cross attention for semantic segmentation. In Proc. IEEE/CVF International Conference on Computer Vision 603–612 (2019)." name="citation_reference"/>
  <meta content="citation_journal_title=Proteins; citation_title=Comparison of multiple Amber force fields and development of improved protein backbone parameters; citation_author=V Hornak; citation_volume=65; citation_publication_date=2006; citation_pages=712-725; citation_doi=10.1002/prot.21123; citation_id=CR32" name="citation_reference"/>
  <meta content="citation_journal_title=Nucleic Acids Res.; citation_title=LGA: a method for finding 3D similarities in protein structures; citation_author=A Zemla; citation_volume=31; citation_publication_date=2003; citation_pages=3370-3374; citation_doi=10.1093/nar/gkg571; citation_id=CR33" name="citation_reference"/>
  <meta content="citation_journal_title=Bioinformatics; citation_title=lDDT: a local superposition-free score for comparing protein structures and models using distance difference tests; citation_author=V Mariani, M Biasini, A Barbato, T Schwede; citation_volume=29; citation_publication_date=2013; citation_pages=2722-2728; citation_doi=10.1093/bioinformatics/btt473; citation_id=CR34" name="citation_reference"/>
  <meta content="Xie, Q., Luong, M.-T., Hovy, E. &amp; Le, Q. V. Self-training with noisy student improves imagenet classification. In Proc. IEEE/CVF Conference on Computer Vision and Pattern Recognition 10687–10698 (2020)." name="citation_reference"/>
  <meta content="citation_journal_title=Nucleic Acids Res.; citation_title=Uniclust databases of clustered and deeply annotated protein sequences and alignments; citation_author=M Mirdita; citation_volume=45; citation_publication_date=2017; citation_pages=D170-D176; citation_doi=10.1093/nar/gkw1081; citation_id=CR36" name="citation_reference"/>
  <meta content="Devlin, J., Chang, M.-W., Lee, K. &amp; Toutanova, K. BERT: pre-training of deep bidirectional transformers for language understanding. In Proc. 2019 Conference of the North American Chapter of the Association for Computational Linguistics: Human Language Technologies 1, 4171–4186 (2019)." name="citation_reference"/>
  <meta content="Rao, R. et al. MSA transformer. In Proc. 38th International Conference on Machine Learning PMLR 139, 8844–8856 (2021)." name="citation_reference"/>
  <meta content="Tunyasuvunakool, K. et al. Highly accurate protein structure prediction for the human proteome. Nature 
                  https://doi-org.proxy.lib.ohio-state.edu/10.1038/s41586-021-03828-1
                  
                 (2021)." name="citation_reference"/>
  <meta content="citation_journal_title=Nat. Rev. Mol. Cell Biol.; citation_title=Advances in protein structure prediction and design; citation_author=B Kuhlman, P Bradley; citation_volume=20; citation_publication_date=2019; citation_pages=681-697; citation_doi=10.1038/s41580-019-0163-x; citation_id=CR40" name="citation_reference"/>
  <meta content="citation_journal_title=Nat. Biotechnol.; citation_title=Protein structure prediction from sequence variation; citation_author=DS Marks, TA Hopf, C Sander; citation_volume=30; citation_publication_date=2012; citation_pages=1072-1080; citation_doi=10.1038/nbt.2419; citation_id=CR41" name="citation_reference"/>
  <meta content="citation_journal_title=J. Mol. Biol.; citation_title=Predicting the secondary structure of globular proteins using neural network models; citation_author=N Qian, TJ Sejnowski; citation_volume=202; citation_publication_date=1988; citation_pages=865-884; citation_doi=10.1016/0022-2836(88)90564-5; citation_id=CR42" name="citation_reference"/>
  <meta content="citation_journal_title=Protein Eng.; citation_title=Prediction of contact maps with neural networks and correlated mutations; citation_author=P Fariselli, O Olmea, A Valencia, R Casadio; citation_volume=14; citation_publication_date=2001; citation_pages=835-843; citation_doi=10.1093/protein/14.11.835; citation_id=CR43" name="citation_reference"/>
  <meta content="citation_journal_title=Proc. Natl Acad. Sci. USA; citation_title=Improved protein structure prediction using predicted interresidue orientations; citation_author=J Yang; citation_volume=117; citation_publication_date=2020; citation_pages=1496-1503; citation_doi=10.1073/pnas.1914677117; citation_id=CR44" name="citation_reference"/>
  <meta content="citation_journal_title=PLOS Comput. Biol.; citation_title=Deducing high-accuracy protein contact-maps from a triplet of coevolutionary matrices through deep residual convolutional networks; citation_author=Y Li; citation_volume=17; citation_publication_date=2021; citation_pages=e1008865; citation_doi=10.1371/journal.pcbi.1008865; citation_id=CR45" name="citation_reference"/>
  <meta content="He, K., Zhang, X., Ren, S. &amp; Sun, J. Deep residual learning for image recognition. In Proc. IEEE Conference on Computer Vision and Pattern Recognition 770–778 (2016)." name="citation_reference"/>
  <meta content="citation_journal_title=Cell Syst.; citation_title=End-to-end differentiable learning of protein structure; citation_author=M AlQuraishi; citation_volume=8; citation_publication_date=2019; citation_pages=292-301; citation_doi=10.1016/j.cels.2019.03.006; citation_id=CR47" name="citation_reference"/>
  <meta content="citation_journal_title=Proteins; citation_title=Protein structure prediction using multiple deep neural networks in the 13th Critical Assessment of Protein Structure Prediction (CASP13); citation_author=AW Senior; citation_volume=87; citation_publication_date=2019; citation_pages=1141-1148; citation_doi=10.1002/prot.25834; citation_id=CR48" name="citation_reference"/>
  <meta content="Ingraham, J., Riesselman, A. J., Sander, C. &amp; Marks, D. S. Learning protein structure with a differentiable simulator. in Proc. International Conference on Learning Representations (2019)." name="citation_reference"/>
  <meta content="Li, J. Universal transforming geometric network. Preprint at 
                  https://arxiv-org.proxy.lib.ohio-state.edu/abs/1908.00723
                  
                 (2019)." name="citation_reference"/>
  <meta content="citation_journal_title=Nat. Mach. Intell.; citation_title=Improved protein structure prediction by deep learning irrespective of co-evolution information; citation_author=J Xu, M McPartlon, J Li; citation_volume=3; citation_publication_date=2021; citation_pages=601-609; citation_doi=10.1038/s42256-021-00348-5; citation_id=CR51" name="citation_reference"/>
  <meta content="Vaswani, A. et al. Attention is all you need. In Advances in Neural Information Processing Systems 5998–6008 (2017)." name="citation_reference"/>
  <meta content="Wang, H. et al. Axial-deeplab: stand-alone axial-attention for panoptic segmentation. in European Conference on Computer Vision 108–126 (Springer, 2020)." name="citation_reference"/>
  <meta content="citation_journal_title=Nat. Methods; citation_title=Unified rational protein engineering with sequence-based deep representation learning; citation_author=EC Alley, G Khimulya, S Biswas, M AlQuraishi, GM Church; citation_volume=16; citation_publication_date=2019; citation_pages=1315-1322; citation_doi=10.1038/s41592-019-0598-1; citation_id=CR54" name="citation_reference"/>
  <meta content="citation_journal_title=BMC Bioinformatics; citation_title=Modeling aspects of the language of life through transfer-learning protein sequences; citation_author=M Heinzinger; citation_volume=20; citation_publication_date=2019; citation_doi=10.1186/s12859-019-3220-8; citation_id=CR55" name="citation_reference"/>
  <meta content="citation_journal_title=Proc. Natl Acad. Sci. USA; citation_title=Biological structure and function emerge from scaling unsupervised learning to 250 million protein sequences; citation_author=A Rives; citation_volume=118; citation_publication_date=2021; citation_pages=e2016239118; citation_doi=10.1073/pnas.2016239118; citation_id=CR56" name="citation_reference"/>
  <meta content="citation_journal_title=Proteins; citation_title=High-accuracy protein structure prediction in CASP14; citation_author=J Pereira; citation_publication_date=2021; citation_doi=10.1002/prot.26171; citation_id=CR57" name="citation_reference"/>
  <meta content="Gupta, M. et al. CryoEM and AI reveal a structure of SARS-CoV-2 Nsp2, a multifunctional protein involved in key host processes. Preprint at 
                  https://doi-org.proxy.lib.ohio-state.edu/10.1101/2021.05.10.443524
                  
                 (2021)." name="citation_reference"/>
  <meta content="Ingraham, J., Garg, V. K., Barzilay, R. &amp; Jaakkola, T. Generative models for graph-based protein design. in Proc. 33rd Conference on Neural Information Processing Systems (2019)." name="citation_reference"/>
  <meta content="citation_journal_title=BMC Bioinformatics; citation_title=Hidden Markov model speed heuristic and iterative HMM search procedure; citation_author=LS Johnson, SR Eddy, E Portugaly; citation_volume=11; citation_publication_date=2010; citation_doi=10.1186/1471-2105-11-431; citation_id=CR60" name="citation_reference"/>
  <meta content="citation_journal_title=Nat. Methods; citation_title=HHblits: lightning-fast iterative protein sequence searching by HMM-HMM alignment; citation_author=M Remmert, A Biegert, A Hauser, J Söding; citation_volume=9; citation_publication_date=2012; citation_pages=173-175; citation_doi=10.1038/nmeth.1818; citation_id=CR61" name="citation_reference"/>
  <meta content="citation_journal_title=Nucleic Acids Res.; citation_title=UniProt: the universal protein knowledgebase in 2021; citation_author=; citation_volume=49; citation_publication_date=2020; citation_pages=D480-D489; citation_doi=10.1093/nar/gkaa1100; citation_id=CR62" name="citation_reference"/>
  <meta content="citation_journal_title=Nat. Commun.; citation_title=Clustering huge protein sequence sets in linear time; citation_author=M Steinegger, J Söding; citation_volume=9; citation_publication_date=2018; citation_doi=10.1038/s41467-018-04964-5; citation_id=CR63" name="citation_reference"/>
  <meta content="citation_journal_title=Nat. Biotechnol.; citation_title=MMseqs2 enables sensitive protein sequence searching for the analysis of massive data sets; citation_author=M Steinegger, J Söding; citation_volume=35; citation_publication_date=2017; citation_pages=1026-1028; citation_doi=10.1038/nbt.3988; citation_id=CR64" name="citation_reference"/>
  <meta content="citation_journal_title=Sci. Rep.; citation_title=FAMSA: fast and accurate multiple sequence alignment of huge protein families; citation_author=S Deorowicz, A Debudaj-Grabysz, A Gudyś; citation_volume=6; citation_publication_date=2016; citation_doi=10.1038/srep33964; citation_id=CR65" name="citation_reference"/>
  <meta content="citation_journal_title=BMC Bioinformatics; citation_title=HH-suite3 for fast remote homology detection and deep protein annotation; citation_author=M Steinegger; citation_volume=20; citation_publication_date=2019; citation_doi=10.1186/s12859-019-3019-7; citation_id=CR66" name="citation_reference"/>
  <meta content="citation_journal_title=Bioinformatics; citation_title=UniRef clusters: a comprehensive and scalable alternative for improving sequence similarity searches; citation_author=BE Suzek, Y Wang, H Huang, PB McGarvey, CH Wu; citation_volume=31; citation_publication_date=2015; citation_pages=926-932; citation_doi=10.1093/bioinformatics/btu739; citation_id=CR67" name="citation_reference"/>
  <meta content="citation_journal_title=PLOS Comput. Biol.; citation_title=Accelerated profile HMM searches; citation_author=SR Eddy; citation_volume=7; citation_publication_date=2011; citation_pages=e1002195; citation_doi=10.1371/journal.pcbi.1002195; citation_id=CR68" name="citation_reference"/>
  <meta content="citation_journal_title=PLOS Comput. Biol.; citation_title=OpenMM 7: rapid development of high performance algorithms for molecular dynamics; citation_author=P Eastman; citation_volume=13; citation_publication_date=2017; citation_pages=e1005659; citation_doi=10.1371/journal.pcbi.1005659; citation_id=CR69" name="citation_reference"/>
  <meta content="Ashish, A. M. A. et al. TensorFlow: large-scale machine learning on heterogeneous systems. Preprint at 
                  https://arxiv-org.proxy.lib.ohio-state.edu/abs/1603.04467
                  
                 (2015)." name="citation_reference"/>
  <meta content="Reynolds, M. et al. Open sourcing Sonnet – a new library for constructing neural networks. DeepMind 
                  https://deepmind.com/blog/open-sourcing-sonnet/
                  
                 (7 April 2017)." name="citation_reference"/>
  <meta content="citation_journal_title=Nature; citation_title=Array programming with NumPy; citation_author=CR Harris; citation_volume=585; citation_publication_date=2020; citation_pages=357-362; citation_doi=10.1038/s41586-020-2649-2; citation_id=CR72" name="citation_reference"/>
  <meta content="Van Rossum, G. &amp; Drake, F. L. Python 3 Reference Manual (CreateSpace, 2009)." name="citation_reference"/>
  <meta content="Bisong, E. in Building Machine Learning and Deep Learning Models on Google Cloud Platform: A Comprehensive Guide for Beginners 59–64 (Apress, 2019)." name="citation_reference"/>
  <meta content="TensorFlow. XLA: Optimizing Compiler for TensorFlow. 
                  https://www.tensorflow.org/xla
                  
                 (2018)." name="citation_reference"/>
  <meta content="citation_journal_title=Bioinformatics; citation_title=Analysis of several key factors influencing deep learning-based inter-residue contact prediction; citation_author=T Wu, J Hou, B Adhikari, J Cheng; citation_volume=36; citation_publication_date=2020; citation_pages=1091-1098; citation_doi=10.1093/bioinformatics/btz679; citation_id=CR76" name="citation_reference"/>
  <meta content="citation_journal_title=PLoS Pathog.; citation_title=MrpH, a new class of metal-binding adhesin, requires zinc to mediate biofilm formation; citation_author=W Jiang; citation_volume=16; citation_publication_date=2020; citation_pages=e1008707; citation_doi=10.1371/journal.ppat.1008707; citation_id=CR77" name="citation_reference"/>
  <meta content="Dunne, M., Ernst, P., Sobieraj, A., Pluckthun, A. &amp; Loessner, M. J. The M23 peptidase domain of the Staphylococcal phage 2638A endolysin. PDB 
                  https://doi-org.proxy.lib.ohio-state.edu/10.2210/pdb6YJ1/pdb
                  
                 (2020)." name="citation_reference"/>
  <meta content="citation_journal_title=Nature; citation_title=Structure and function of virion RNA polymerase of a crAss-like phage; citation_author=AV Drobysheva; citation_volume=589; citation_publication_date=2021; citation_pages=306-309; citation_doi=10.1038/s41586-020-2921-5; citation_id=CR79" name="citation_reference"/>
  <meta content="citation_journal_title=EMBO J.; citation_title=Structural basis for loading and inhibition of a bacterial T6SS phospholipase effector by the VgrG spike; citation_author=N Flaugnatti; citation_volume=39; citation_publication_date=2020; citation_pages=e104129; citation_doi=10.15252/embj.2019104129; citation_id=CR80" name="citation_reference"/>
  <meta content="citation_journal_title=ACS Synth. Biol.; citation_title=An interface-driven design strategy yields a novel, corrugated protein architecture; citation_author=M ElGamacy; citation_volume=7; citation_publication_date=2018; citation_pages=2226-2235; citation_doi=10.1021/acssynbio.8b00224; citation_id=CR81" name="citation_reference"/>
  <meta content="citation_journal_title=Science; citation_title=The structure of human CST reveals a decameric assembly bound to telomeric DNA; citation_author=CJ Lim; citation_volume=368; citation_publication_date=2020; citation_pages=1081-1085; citation_doi=10.1126/science.aaz9649; citation_id=CR82" name="citation_reference"/>
  <meta content="citation_journal_title=Nat. Struct. Mol. Biol.; citation_title=An embedded lipid in the multidrug transporter LmrP suggests a mechanism for polyspecificity; citation_author=V Debruycker; citation_volume=27; citation_publication_date=2020; citation_pages=829-835; citation_doi=10.1038/s41594-020-0464-y; citation_id=CR83" name="citation_reference"/>
  <meta content="citation_journal_title=Proc. Natl Acad. Sci. USA; citation_title=Structure of SARS-CoV-2 ORF8, a rapidly evolving immune evasion protein; citation_author=TG Flower; citation_volume=118; citation_publication_date=2021; citation_pages=e2021785118; citation_doi=10.1073/pnas.2021785118; citation_id=CR84" name="citation_reference"/>
  <meta content="Jumper, John" name="citation_author"/>
  <meta content="DeepMind, London, UK" name="citation_author_institution"/>
  <meta content="Evans, Richard" name="citation_author"/>
  <meta content="DeepMind, London, UK" name="citation_author_institution"/>
  <meta content="Pritzel, Alexander" name="citation_author"/>
  <meta content="DeepMind, London, UK" name="citation_author_institution"/>
  <meta content="Green, Tim" name="citation_author"/>
  <meta content="DeepMind, London, UK" name="citation_author_institution"/>
  <meta content="Figurnov, Michael" name="citation_author"/>
  <meta content="DeepMind, London, UK" name="citation_author_institution"/>
  <meta content="Ronneberger, Olaf" name="citation_author"/>
  <meta content="DeepMind, London, UK" name="citation_author_institution"/>
  <meta content="Tunyasuvunakool, Kathryn" name="citation_author"/>
  <meta content="DeepMind, London, UK" name="citation_author_institution"/>
  <meta content="Bates, Russ" name="citation_author"/>
  <meta content="DeepMind, London, UK" name="citation_author_institution"/>
  <meta content="Žídek, Augustin" name="citation_author"/>
  <meta content="DeepMind, London, UK" name="citation_author_institution"/>
  <meta content="Potapenko, Anna" name="citation_author"/>
  <meta content="DeepMind, London, UK" name="citation_author_institution"/>
  <meta content="Bridgland, Alex" name="citation_author"/>
  <meta content="DeepMind, London, UK" name="citation_author_institution"/>
  <meta content="Meyer, Clemens" name="citation_author"/>
  <meta content="DeepMind, London, UK" name="citation_author_institution"/>
  <meta content="Kohl, Simon A. A." name="citation_author"/>
  <meta content="DeepMind, London, UK" name="citation_author_institution"/>
  <meta content="Ballard, Andrew J." name="citation_author"/>
  <meta content="DeepMind, London, UK" name="citation_author_institution"/>
  <meta content="Cowie, Andrew" name="citation_author"/>
  <meta content="DeepMind, London, UK" name="citation_author_institution"/>
  <meta content="Romera-Paredes, Bernardino" name="citation_author"/>
  <meta content="DeepMind, London, UK" name="citation_author_institution"/>
  <meta content="Nikolov, Stanislav" name="citation_author"/>
  <meta content="DeepMind, London, UK" name="citation_author_institution"/>
  <meta content="Jain, Rishub" name="citation_author"/>
  <meta content="DeepMind, London, UK" name="citation_author_institution"/>
  <meta content="Adler, Jonas" name="citation_author"/>
  <meta content="DeepMind, London, UK" name="citation_author_institution"/>
  <meta content="Back, Trevor" name="citation_author"/>
  <meta content="DeepMind, London, UK" name="citation_author_institution"/>
  <meta content="Petersen, Stig" name="citation_author"/>
  <meta content="DeepMind, London, UK" name="citation_author_institution"/>
  <meta content="Reiman, David" name="citation_author"/>
  <meta content="DeepMind, London, UK" name="citation_author_institution"/>
  <meta content="Clancy, Ellen" name="citation_author"/>
  <meta content="DeepMind, London, UK" name="citation_author_institution"/>
  <meta content="Zielinski, Michal" name="citation_author"/>
  <meta content="DeepMind, London, UK" name="citation_author_institution"/>
  <meta content="Steinegger, Martin" name="citation_author"/>
  <meta content="School of Biological Sciences, Seoul National University, Seoul, South Korea" name="citation_author_institution"/>
  <meta content="Artificial Intelligence Institute, Seoul National University, Seoul, South Korea" name="citation_author_institution"/>
  <meta content="Pacholska, Michalina" name="citation_author"/>
  <meta content="DeepMind, London, UK" name="citation_author_institution"/>
  <meta content="Berghammer, Tamas" name="citation_author"/>
  <meta content="DeepMind, London, UK" name="citation_author_institution"/>
  <meta content="Bodenstein, Sebastian" name="citation_author"/>
  <meta content="DeepMind, London, UK" name="citation_author_institution"/>
  <meta content="Silver, David" name="citation_author"/>
  <meta content="DeepMind, London, UK" name="citation_author_institution"/>
  <meta content="Vinyals, Oriol" name="citation_author"/>
  <meta content="DeepMind, London, UK" name="citation_author_institution"/>
  <meta content="Senior, Andrew W." name="citation_author"/>
  <meta content="DeepMind, London, UK" name="citation_author_institution"/>
  <meta content="Kavukcuoglu, Koray" name="citation_author"/>
  <meta content="DeepMind, London, UK" name="citation_author_institution"/>
  <meta content="Kohli, Pushmeet" name="citation_author"/>
  <meta content="DeepMind, London, UK" name="citation_author_institution"/>
  <meta content="Hassabis, Demis" name="citation_author"/>
  <meta content="DeepMind, London, UK" name="citation_author_institution"/>
  <meta content="https://www-nature-com.proxy.lib.ohio-state.edu/platform/readcube-access" name="access_endpoint"/>
  <meta content="@nature" name="twitter:site"/>
  <meta content="summary_large_image" name="twitter:card"/>
  <meta content="Content cover image" name="twitter:image:alt"/>
  <meta content="Highly accurate protein structure prediction with AlphaFold" name="twitter:title"/>
  <meta content="Nature - AlphaFold predicts protein structures with an accuracy competitive with experimental structures in the majority of cases using a novel deep learning architecture." name="twitter:description"/>
  <meta content="https://media-springernature-com.proxy.lib.ohio-state.edu/full/springer-static/image/art%3A10.1038%2Fs41586-021-03819-2/MediaObjects/41586_2021_3819_Fig1_HTML.png" name="twitter:image"/>
  <meta content="https://www-nature-com.proxy.lib.ohio-state.edu/articles/s41586-021-03819-2" property="og:url"/>
  <meta content="article" property="og:type"/>
  <meta content="Nature" property="og:site_name"/>
  <meta content="Highly accurate protein structure prediction with AlphaFold - Nature" property="og:title"/>
  <meta content="AlphaFold predicts protein structures with an accuracy competitive with experimental structures in the majority of cases using a novel deep learning architecture." property="og:description"/>
  <meta content="https://media-springernature-com.proxy.lib.ohio-state.edu/m685/springer-static/image/art%3A10.1038%2Fs41586-021-03819-2/MediaObjects/41586_2021_3819_Fig1_HTML.png" property="og:image"/>
  <script>
   window.eligibleForRa21 = 'false';
  </script>
  <style type="text/css">
   .c-cookie-banner {
			background-color: #01324b;
			color: white;
			font-size: 1rem;
			position: fixed;
			bottom: 0;
			left: 0;
			right: 0;
			padding: 16px 0;
			font-family: sans-serif;
			z-index: 100002;
			text-align: center;
		}
		.c-cookie-banner__container {
			margin: 0 auto;
			max-width: 1280px;
			padding: 0 16px;
		}
		.c-cookie-banner p {
			margin-bottom: 8px;
		}
		.c-cookie-banner p:last-child {
			margin-bottom: 0;
		}	
		.c-cookie-banner__dismiss {
			background-color: transparent;
			border: 0;
			padding: 0;
			margin-left: 4px;
			color: inherit;
			text-decoration: underline;
			font-size: inherit;
		}
		.c-cookie-banner__dismiss:hover {
			text-decoration: none;
		}
  </style>
  <script async="" src="https://injections.readcube.com/nature/inject.ef8e25d3.js" type="text/javascript">
  </script>
  <link href="https://injections.readcube.com/styles/nature_checkout.7bec98da.css" rel="stylesheet" type="text/css"/>
  <script>
   window.dataLayer = window.dataLayer || [];
            window.dataLayer.push({
                recommendations: {
                    recommender: 'semantic',
                    model: 'specter',
                    policy_id: 'speedy-BootstrappedUCB',
                    timestamp: 1698030235,
                    embedded_user: 'null'
                }
            });
  </script>
 </head>
 <body class="article-page">
  <noscript>
   <iframe height="0" src="https://www.googletagmanager.com/ns.html?id=GTM-MRVXSHQ" style="display:none;visibility:hidden" width="0">
   </iframe>
  </noscript>
  <div class="position-relative cleared z-index-50 background-white" data-test="top-containers">
   <a class="c-skip-link" href="#content">
    Skip to main content
   </a>
   <div class="c-grade-c-banner u-hide">
    <div class="c-grade-c-banner__container">
     <p>
      Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain
            the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in
            Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles
            and JavaScript.
     </p>
    </div>
   </div>
   <div class="u-lazy-ad-wrapper u-mbs-0">
    <div class="deferred-placeholder" data-placeholder="/placeholder/v1/institutionalBanner?bpids=[bpids] #institutional-banner-container" data-replace="true">
    </div>
    <aside class="c-ad c-ad--728x90">
     <div class="c-ad__inner" data-container-type="banner-advert">
      <p class="c-ad__label">
       Advertisement
      </p>
      <div class="div-gpt-ad advert leaderboard js-ad text-center hide-print grade-c-hide" data-ad-type="top" data-gpt="" data-gpt-sizes="728x90" data-gpt-targeting="type=article;pos=top;artid=s41586-021-03819-2;doi=10.1038/s41586-021-03819-2;techmeta=119,139,141;subjmeta=114,1305,2266,2411,535,57,631;kwrd=Computational+biophysics,Machine+learning,Protein+structure+predictions,Structural+biology" data-gpt-unitpath="/285/nature.com/article" data-pa11y-ignore="" data-test="top-ad" id="div-gpt-ad-top-1">
       <noscript>
        <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/nature.com/article&amp;sz=728x90&amp;c=893380862&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41586-021-03819-2%26doi%3D10.1038/s41586-021-03819-2%26techmeta%3D119,139,141%26subjmeta%3D114,1305,2266,2411,535,57,631%26kwrd%3DComputational+biophysics,Machine+learning,Protein+structure+predictions,Structural+biology">
         <img alt="Advertisement" data-test="gpt-advert-fallback-img" height="90" src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/nature.com/article&amp;sz=728x90&amp;c=893380862&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41586-021-03819-2%26doi%3D10.1038/s41586-021-03819-2%26techmeta%3D119,139,141%26subjmeta%3D114,1305,2266,2411,535,57,631%26kwrd%3DComputational+biophysics,Machine+learning,Protein+structure+predictions,Structural+biology" width="728"/>
        </a>
       </noscript>
      </div>
     </div>
    </aside>
   </div>
   <header class="c-header" data-header="" data-track-component="nature-150-split-header" id="header" style="border-color:#000">
    <div class="c-header__row">
     <div class="c-header__container">
      <div class="c-header__split">
       <div class="c-header__logo-container">
        <a data-track="click" data-track-action="home" data-track-label="image" href="/">
         <picture class="c-header__logo">
          <source media="(min-width: 875px)" srcset="https://media-springernature-com.proxy.lib.ohio-state.edu/full/nature-cms/uploads/product/nature/header-86f1267ea01eccd46b530284be10585e.svg"/>
          <img alt="Nature" height="32" src="https://media-springernature-com.proxy.lib.ohio-state.edu/full/nature-cms/uploads/product/nature/header-86f1267ea01eccd46b530284be10585e.svg"/>
         </picture>
        </a>
       </div>
       <ul class="c-header__menu c-header__menu--global">
        <li class="c-header__item c-header__item--padding c-header__item--hide-md-max">
         <a class="c-header__link" data-test="siteindex-link" data-track="click" data-track-action="open nature research index" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/siteindex">
          <span>
           View all journals
          </span>
         </a>
        </li>
        <li class="c-header__item c-header__item--padding c-header__item--pipe">
         <a aria-expanded="false" aria-haspopup="true" class="c-header__link" data-header-expander="" data-test="search-link" data-track="click" data-track-action="open search tray" data-track-label="button" href="javascript:;" role="button">
          <span>
           Search
          </span>
          <svg aria-hidden="true" focusable="false" height="22" role="img" viewbox="0 0 18 18" width="22" xmlns="http://www.w3.org/2000/svg">
           <path d="M16.48 15.455c.283.282.29.749.007 1.032a.738.738 0 01-1.032-.007l-3.045-3.044a7 7 0 111.026-1.026zM8 14A6 6 0 108 2a6 6 0 000 12z">
           </path>
          </svg>
         </a>
         <div class="c-header__dropdown c-header__dropdown--full-width has-tethered u-js-hide" data-track-component="nature-150-split-header" hidden="" id="search-menu">
          <div class="c-header__container">
           <h2 class="c-header__visually-hidden">
            Search
           </h2>
           <form action="/search" autocomplete="off" class="c-header__search-form" data-test="inline-search" method="get" role="search">
            <label class="c-header__heading" for="keywords">
             Search articles by subject, keyword or author
            </label>
            <div class="c-header__search-layout c-header__search-layout--max-width">
             <div>
              <input class="c-header__input" id="keywords" name="q" required="" type="text" value=""/>
             </div>
             <div class="c-header__search-layout">
              <div>
               <label class="c-header__visually-hidden" for="results-from">
                Show results from
               </label>
               <select class="c-header__select" id="results-from" name="journal">
                <option selected="" value="">
                 All journals
                </option>
                <option value="nature">
                 This journal
                </option>
               </select>
              </div>
              <div>
               <button class="c-header__search-button" type="submit">
                Search
               </button>
              </div>
             </div>
            </div>
           </form>
           <div class="c-header__flush">
            <a class="c-header__link" data-track="click" data-track-action="advanced search" data-track-label="link" href="/search/advanced">
             Advanced search
            </a>
           </div>
           <h3 class="c-header__heading c-header__heading--keyline">
            Quick links
           </h3>
           <ul class="c-header__list">
            <li>
             <a class="c-header__link" data-track="click" data-track-action="explore articles by subject" data-track-label="link" href="/subjects">
              Explore articles by subject
             </a>
            </li>
            <li>
             <a class="c-header__link" data-track="click" data-track-action="find a job" data-track-label="link" href="/naturecareers">
              Find a job
             </a>
            </li>
            <li>
             <a class="c-header__link" data-track="click" data-track-action="guide to authors" data-track-label="link" href="/authors/index.html">
              Guide to authors
             </a>
            </li>
            <li>
             <a class="c-header__link" data-track="click" data-track-action="editorial policies" data-track-label="link" href="/authors/editorial_policies/">
              Editorial policies
             </a>
            </li>
           </ul>
          </div>
         </div>
        </li>
        <li class="c-header__item c-header__item--padding">
         <a class="c-header__link eds-c-header__link" href="https://idp-nature-com.proxy.lib.ohio-state.edu/auth/personal/springernature?redirect_uri=https://www-nature-com.proxy.lib.ohio-state.edu/articles/s41586-021-03819-2" id="identity-account-widget">
          Log in
         </a>
        </li>
       </ul>
      </div>
     </div>
    </div>
    <div class="c-header__row">
     <div class="c-header__container" data-test="navigation-row">
      <div class="c-header__split">
       <ul class="c-header__menu c-header__menu--journal">
        <li class="c-header__item c-header__item--dropdown-menu" data-test="explore-content-button">
         <a aria-expanded="false" aria-haspopup="true" class="c-header__link" data-header-expander="" data-test="menu-button--explore" data-track="click" data-track-action="open explore expander" data-track-label="button" href="javascript:;" role="button">
          <span>
           <span class="c-header__show-text">
            Explore
           </span>
           content
          </span>
          <svg aria-hidden="true" focusable="false" height="16" role="img" viewbox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg">
           <path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)">
           </path>
          </svg>
         </a>
         <nav aria-labelledby="Explore-content" class="c-header__dropdown has-tethered u-js-hide" data-test="Explore-content" data-track-component="nature-150-split-header" hidden="" id="explore">
          <div class="c-header__container">
           <h2 class="c-header__heading c-header__heading--js-hide" id="Explore-content">
            Explore content
           </h2>
           <ul class="c-header__list c-header__list--js-stack">
            <li class="c-header__item">
             <a class="c-header__link" data-test="explore-nav-item" data-track="click" data-track-action="research articles" data-track-label="link" href="/nature/research-articles">
              Research articles
             </a>
            </li>
            <li class="c-header__item">
             <a class="c-header__link" data-test="explore-nav-item" data-track="click" data-track-action="news" data-track-label="link" href="/news">
              News
             </a>
            </li>
            <li class="c-header__item">
             <a class="c-header__link" data-test="explore-nav-item" data-track="click" data-track-action="opinion" data-track-label="link" href="/opinion">
              Opinion
             </a>
            </li>
            <li class="c-header__item">
             <a class="c-header__link" data-test="explore-nav-item" data-track="click" data-track-action="research analysis" data-track-label="link" href="/research-analysis">
              Research Analysis
             </a>
            </li>
            <li class="c-header__item">
             <a class="c-header__link" data-test="explore-nav-item" data-track="click" data-track-action="careers" data-track-label="link" href="/careers">
              Careers
             </a>
            </li>
            <li class="c-header__item">
             <a class="c-header__link" data-test="explore-nav-item" data-track="click" data-track-action="books &amp; culture" data-track-label="link" href="/books-culture">
              Books &amp; Culture
             </a>
            </li>
            <li class="c-header__item">
             <a class="c-header__link" data-test="explore-nav-item" data-track="click" data-track-action="podcasts" data-track-label="link" href="/nature/podcasts">
              Podcasts
             </a>
            </li>
            <li class="c-header__item">
             <a class="c-header__link" data-test="explore-nav-item" data-track="click" data-track-action="videos" data-track-label="link" href="/nature/videos">
              Videos
             </a>
            </li>
            <li class="c-header__item">
             <a class="c-header__link" data-test="explore-nav-item" data-track="click" data-track-action="current issue" data-track-label="link" href="/nature/current-issue">
              Current issue
             </a>
            </li>
            <li class="c-header__item">
             <a class="c-header__link" data-test="explore-nav-item" data-track="click" data-track-action="browse issues" data-track-label="link" href="/nature/browse-issues">
              Browse issues
             </a>
            </li>
            <li class="c-header__item">
             <a class="c-header__link" data-test="explore-nav-item" data-track="click" data-track-action="collections" data-track-label="link" href="/nature/collections">
              Collections
             </a>
            </li>
            <li class="c-header__item">
             <a class="c-header__link" data-test="explore-nav-item" data-track="click" data-track-action="subjects" data-track-label="link" href="/nature/browse-subjects">
              Subjects
             </a>
            </li>
           </ul>
           <ul class="c-header__list c-header__list--js-stack">
            <li class="c-header__item">
             <a class="c-header__link" data-track="click" data-track-action="facebook" data-track-label="link" href="https://www.facebook.com/Nature">
              Follow us on Facebook
             </a>
            </li>
            <li class="c-header__item">
             <a class="c-header__link" data-track="click" data-track-action="twitter" data-track-label="link" href="https://twitter.com/nature">
              Follow us on Twitter
             </a>
            </li>
            <li class="c-header__item c-header__item--hide-lg">
             <a class="c-header__link" data-track="click" data-track-action="Sign up for alerts" data-track-external="" data-track-label="link (mobile dropdown)" href="https://www-nature-com.proxy.lib.ohio-state.edu/my-account/alerts/subscribe-journal?list-id=1" rel="nofollow">
              Sign up for alerts
              <svg aria-hidden="true" focusable="false" height="18" role="img" viewbox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg">
               <path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill="#fff">
               </path>
              </svg>
             </a>
            </li>
            <li class="c-header__item c-header__item--hide-lg">
             <a class="c-header__link" data-track="click" data-track-action="rss feed" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/nature.rss">
              <span>
               RSS feed
              </span>
             </a>
            </li>
           </ul>
          </div>
         </nav>
        </li>
        <li class="c-header__item c-header__item--dropdown-menu">
         <a aria-expanded="false" aria-haspopup="true" class="c-header__link" data-header-expander="" data-test="menu-button--about-the-journal" data-track="click" data-track-action="open about the journal expander" data-track-label="button" href="javascript:;" role="button">
          <span>
           About
           <span class="c-header__show-text">
            the journal
           </span>
          </span>
          <svg aria-hidden="true" focusable="false" height="16" role="img" viewbox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg">
           <path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)">
           </path>
          </svg>
         </a>
         <nav aria-labelledby="About-the-journal" class="c-header__dropdown has-tethered u-js-hide" data-test="about-the-journal" data-track-component="nature-150-split-header" hidden="" id="about-the-journal">
          <div class="c-header__container">
           <h2 class="c-header__heading c-header__heading--js-hide" id="About-the-journal">
            About the journal
           </h2>
           <ul class="c-header__list c-header__list--js-stack">
            <li class="c-header__item">
             <a class="c-header__link" data-track="click" data-track-action="journal staff" data-track-label="link" href="/nature/journal-staff">
              Journal Staff
             </a>
            </li>
            <li class="c-header__item">
             <a class="c-header__link" data-track="click" data-track-action="about the editors" data-track-label="link" href="/nature/editors">
              About the Editors
             </a>
            </li>
            <li class="c-header__item">
             <a class="c-header__link" data-track="click" data-track-action="journal information" data-track-label="link" href="/nature/journal-information">
              Journal Information
             </a>
            </li>
            <li class="c-header__item">
             <a class="c-header__link" data-track="click" data-track-action="our publishing models" data-track-label="link" href="/nature/our-publishing-models">
              Our publishing models
             </a>
            </li>
            <li class="c-header__item">
             <a class="c-header__link" data-track="click" data-track-action="editorial values statement" data-track-label="link" href="/nature/editorial-values-statement">
              Editorial Values Statement
             </a>
            </li>
            <li class="c-header__item">
             <a class="c-header__link" data-track="click" data-track-action="journal metrics" data-track-label="link" href="/nature/journal-impact">
              Journal Metrics
             </a>
            </li>
            <li class="c-header__item">
             <a class="c-header__link" data-track="click" data-track-action="awards" data-track-label="link" href="/nature/awards">
              Awards
             </a>
            </li>
            <li class="c-header__item">
             <a class="c-header__link" data-track="click" data-track-action="contact" data-track-label="link" href="/nature/contact">
              Contact
             </a>
            </li>
            <li class="c-header__item">
             <a class="c-header__link" data-track="click" data-track-action="editorial policies" data-track-label="link" href="/nature/editorial-policies">
              Editorial policies
             </a>
            </li>
            <li class="c-header__item">
             <a class="c-header__link" data-track="click" data-track-action="history of nature" data-track-label="link" href="/nature/history-of-nature">
              History of Nature
             </a>
            </li>
            <li class="c-header__item">
             <a class="c-header__link" data-track="click" data-track-action="send a news tip" data-track-label="link" href="/nature/send-a-news-tip">
              Send a news tip
             </a>
            </li>
           </ul>
          </div>
         </nav>
        </li>
        <li class="c-header__item c-header__item--dropdown-menu" data-test="publish-with-us-button">
         <a aria-expanded="false" aria-haspopup="true" class="c-header__link c-header__link--dropdown-menu" data-header-expander="" data-test="menu-button--publish" data-track="click" data-track-action="open publish with us expander" data-track-label="button" href="javascript:;" role="button">
          <span>
           Publish
           <span class="c-header__show-text">
            with us
           </span>
          </span>
          <svg aria-hidden="true" focusable="false" height="16" role="img" viewbox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg">
           <path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)">
           </path>
          </svg>
         </a>
         <nav aria-labelledby="Publish-with-us-label" class="c-header__dropdown has-tethered u-js-hide" data-test="publish-with-us" data-track-component="nature-150-split-header" hidden="" id="publish-with-us">
          <div class="c-header__container">
           <h2 class="c-header__heading c-header__heading--js-hide" id="Publish-with-us-label">
            Publish with us
           </h2>
           <ul class="c-header__list c-header__list--js-stack">
            <li class="c-header__item">
             <a class="c-header__link" data-track="click" data-track-action="for authors" data-track-label="link" href="/nature/for-authors">
              For Authors
             </a>
            </li>
            <li class="c-header__item">
             <a class="c-header__link" data-track="click" data-track-action="for referees" data-track-label="link" href="/nature/for-referees">
              For Referees
             </a>
            </li>
            <li class="c-header__item">
             <a class="c-header__link" data-test="nature-author-services" data-track="click" data-track-action="manuscript author services" data-track-label="link manuscript author services" href="https://authorservices-springernature-com.proxy.lib.ohio-state.edu/go/sn/?utm_source=For+Authors&amp;utm_medium=Website_Nature&amp;utm_campaign=Platform+Experimentation+2022&amp;utm_id=PE2022">
              Language editing services
             </a>
            </li>
            <li class="c-header__item c-header__item--keyline">
             <a class="c-header__link" data-track="click" data-track-action="submit manuscript" data-track-external="" data-track-label="link (publish with us dropdown menu)" href="https://mts-nature-nature-com.proxy.lib.ohio-state.edu/">
              Submit manuscript
              <svg aria-hidden="true" focusable="false" height="18" role="img" viewbox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg">
               <path d="m15 0c1.1045695 0 2 .8954305 2 2v5.5c0 .27614237-.2238576.5-.5.5s-.5-.22385763-.5-.5v-5.5c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-9v3c0 1.1045695-.8954305 2-2 2h-3v10c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h7.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-7.5c-1.1045695 0-2-.8954305-2-2v-10.17157288c0-.53043297.21071368-1.0391408.58578644-1.41421356l3.82842712-3.82842712c.37507276-.37507276.88378059-.58578644 1.41421356-.58578644zm-.5442863 8.18867991 3.3545404 3.35454039c.2508994.2508994.2538696.6596433.0035959.909917-.2429543.2429542-.6561449.2462671-.9065387-.0089489l-2.2609825-2.3045251.0010427 7.2231989c0 .3569916-.2898381.6371378-.6473715.6371378-.3470771 0-.6473715-.2852563-.6473715-.6371378l-.0010428-7.2231995-2.2611222 2.3046654c-.2531661.2580415-.6562868.2592444-.9065605.0089707-.24295423-.2429542-.24865597-.6576651.0036132-.9099343l3.3546673-3.35466731c.2509089-.25090888.6612706-.25227691.9135302-.00001728zm-.9557137-3.18867991c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm-8.5-3.587-3.587 3.587h2.587c.55228475 0 1-.44771525 1-1zm8.5 1.587c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill="#fff">
               </path>
              </svg>
             </a>
            </li>
           </ul>
          </div>
         </nav>
        </li>
       </ul>
       <ul class="c-header__menu c-header__menu--hide-lg-max">
        <li class="c-header__item">
         <a class="c-header__link" data-track="click" data-track-action="Sign up for alerts" data-track-external="" data-track-label="link (desktop site header)" href="https://www-nature-com.proxy.lib.ohio-state.edu/my-account/alerts/subscribe-journal?list-id=1" rel="nofollow">
          <span>
           Sign up for alerts
          </span>
          <svg aria-hidden="true" focusable="false" height="18" role="img" viewbox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg">
           <path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill="#222">
           </path>
          </svg>
         </a>
        </li>
        <li class="c-header__item c-header__item--pipe">
         <a class="c-header__link" data-track="click" data-track-action="rss feed" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/nature.rss">
          <span>
           RSS feed
          </span>
         </a>
        </li>
       </ul>
      </div>
     </div>
    </div>
   </header>
   <nav aria-label="breadcrumbs" class="u-mb-16">
    <div class="u-container">
     <ol class="c-breadcrumbs" itemscope="" itemtype="https://schema.org/BreadcrumbList">
      <li class="c-breadcrumbs__item" id="breadcrumb0" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem">
       <a class="c-breadcrumbs__link" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature" href="/" itemprop="item">
        <span itemprop="name">
         nature
        </span>
       </a>
       <meta content="1" itemprop="position"/>
       <svg aria-hidden="true" class="c-breadcrumbs__chevron" focusable="false" height="10" role="img" viewbox="0 0 10 10" width="10" xmlns="http://www.w3.org/2000/svg">
        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)">
        </path>
       </svg>
      </li>
      <li class="c-breadcrumbs__item" id="breadcrumb1" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem">
       <a class="c-breadcrumbs__link" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:articles" href="/nature/articles?type=article" itemprop="item">
        <span itemprop="name">
         articles
        </span>
       </a>
       <meta content="2" itemprop="position"/>
       <svg aria-hidden="true" class="c-breadcrumbs__chevron" focusable="false" height="10" role="img" viewbox="0 0 10 10" width="10" xmlns="http://www.w3.org/2000/svg">
        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)">
        </path>
       </svg>
      </li>
      <li class="c-breadcrumbs__item" id="breadcrumb2" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem">
       <span itemprop="name">
        article
       </span>
       <meta content="3" itemprop="position"/>
      </li>
     </ol>
    </div>
   </nav>
  </div>
  <div class="u-container u-mt-32 u-mb-32 u-clearfix" data-component="article-container" data-container-type="article" id="content">
   <main class="c-article-main-column u-float-left js-main-column" data-track-component="article body">
    <div aria-hidden="true" class="c-context-bar u-hide" data-context-bar="" data-context-bar-with-recommendations="" data-test="context-bar">
     <div class="c-context-bar__container u-container">
      <div class="c-context-bar__title">
       Highly accurate protein structure prediction with AlphaFold
      </div>
      <div class="c-pdf-download u-clear-both js-pdf-download">
       <a class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-draft-ignore="true" data-readcube-pdf-url="true" data-test="download-pdf" data-track="click" data-track-action="download pdf" data-track-external="" data-track-label="link" download="" href="/articles/s41586-021-03819-2.pdf">
        <span class="c-pdf-download__text">
         Download PDF
        </span>
        <svg aria-hidden="true" class="u-icon" focusable="false" height="16" width="16">
         <use xlink:href="#icon-download">
         </use>
        </svg>
       </a>
      </div>
     </div>
     <div id="recommendations">
      <div class="c-recommendations__container u-container u-display-none" data-component-recommendations="">
       <aside class="c-status-message c-status-message--success u-display-none" data-component-status-msg="">
        <svg aria-label="success:" class="c-status-message__icon" focusable="false" height="24" role="img" width="24">
         <use xlink:href="#icon-success">
         </use>
        </svg>
        <div class="c-status-message__message" id="success-message" tabindex="-1">
         Your content has downloaded
        </div>
       </aside>
       <div class="c-recommendations-header u-display-flex u-justify-content-space-between">
        <h2 class="c-recommendations-title" id="recommendation-heading">
         Similar content being viewed by others
        </h2>
        <button aria-label="Close" class="c-recommendations-close u-flex-static" data-track="click" data-track-action="close recommendations" type="button">
         <svg aria-hidden="true" class="u-icon" focusable="false" height="16" width="16">
          <use xlink:href="#icon-close">
          </use>
         </svg>
        </button>
       </div>
       <section aria-labelledby="recommendation-heading" aria-roledescription="carousel">
        <p class="u-visually-hidden">
         Slider with three content items shown per slide. Use the Previous and Next buttons to navigate the slides or the slide controller buttons at the end to navigate through each slide.
        </p>
        <div class="c-recommendations-list-container">
         <div class="c-recommendations-list">
          <div aria-label="Recommendation 1 of 9" aria-roledescription="slide" class="c-recommendations-list__item" role="group">
           <article class="u-full-height c-card c-card--flush" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
            <div class="c-card__layout u-full-height">
             <div class="c-card__image">
              <img alt="" src="https://media-springernature-com.proxy.lib.ohio-state.edu/w136h75/springer-static/image/art%3A10.1038%2Fs41586-021-03828-1/MediaObjects/41586_2021_3828_Fig1_HTML.png"/>
             </div>
             <div class="c-card__body u-display-flex u-flex-direction-column">
              <div class="c-recommendations-column-switch">
               <h3 class="c-card__title-recommendation u-sans-serif" itemprop="name headline">
                <a class="c-card__link" data-track="click" data-track-action="click recommendations - 1" data-track-label="10.1038/s41586-021-03828-1" href="https://doi-org.proxy.lib.ohio-state.edu/10.1038/s41586-021-03828-1" itemprop="url">
                 Highly accurate protein structure prediction for the human proteome
                </a>
               </h3>
               <div class="c-card__section c-meta">
                <span class="c-meta__item u-sans-serif">
                 Article
                </span>
                <span class="c-meta__item u-sans-serif">
                 22 July 2021
                </span>
               </div>
              </div>
              <p class="c-recommendations-authors u-hide-at-sm u-sans-serif">
               Kathryn Tunyasuvunakool, Jonas Adler, … Demis Hassabis
              </p>
             </div>
            </div>
           </article>
          </div>
          <div aria-label="Recommendation 2 of 9" aria-roledescription="slide" class="c-recommendations-list__item" role="group">
           <article class="u-full-height c-card c-card--flush" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
            <div class="c-card__layout u-full-height">
             <div class="c-card__image">
              <img alt="" src="https://media-springernature-com.proxy.lib.ohio-state.edu/w136h75/springer-static/image/art%3A10.1038%2Fs41467-023-41664-1/MediaObjects/41467_2023_41664_Fig1_HTML.png"/>
             </div>
             <div class="c-card__body u-display-flex u-flex-direction-column">
              <div class="c-recommendations-column-switch">
               <h3 class="c-card__title-recommendation u-sans-serif" itemprop="name headline">
                <a class="c-card__link" data-track="click" data-track-action="click recommendations - 2" data-track-label="10.1038/s41467-023-41664-1" href="https://doi-org.proxy.lib.ohio-state.edu/10.1038/s41467-023-41664-1" itemprop="url">
                 Accurate prediction of protein folding mechanisms by simple structure-based statistical mechanical models
                </a>
               </h3>
               <div class="c-card__section c-meta">
                <span class="c-meta__item u-sans-serif">
                 Article
                </span>
                <span class="c-meta__item u-sans-serif">
                 19 October 2023
                </span>
               </div>
              </div>
              <p class="c-recommendations-authors u-hide-at-sm u-sans-serif">
               Koji Ooka &amp; Munehito Arai
              </p>
             </div>
            </div>
           </article>
          </div>
          <div aria-label="Recommendation 3 of 9" aria-roledescription="slide" class="c-recommendations-list__item" role="group">
           <article class="u-full-height c-card c-card--flush" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
            <div class="c-card__layout u-full-height">
             <div class="c-card__image">
              <img alt="" src="https://media-springernature-com.proxy.lib.ohio-state.edu/w136h75/springer-static/image/art%3A10.1038%2Fs41586-019-1923-7/MediaObjects/41586_2019_1923_Fig1_HTML.png"/>
             </div>
             <div class="c-card__body u-display-flex u-flex-direction-column">
              <div class="c-recommendations-column-switch">
               <h3 class="c-card__title-recommendation u-sans-serif" itemprop="name headline">
                <a class="c-card__link" data-track="click" data-track-action="click recommendations - 3" data-track-label="10.1038/s41586-019-1923-7" href="https://doi-org.proxy.lib.ohio-state.edu/10.1038/s41586-019-1923-7" itemprop="url">
                 Improved protein structure prediction using potentials from deep learning
                </a>
               </h3>
               <div class="c-card__section c-meta">
                <span class="c-meta__item u-sans-serif">
                 Article
                </span>
                <span class="c-meta__item u-sans-serif">
                 15 January 2020
                </span>
               </div>
              </div>
              <p class="c-recommendations-authors u-hide-at-sm u-sans-serif">
               Andrew W. Senior, Richard Evans, … Demis Hassabis
              </p>
             </div>
            </div>
           </article>
          </div>
          <div aria-label="Recommendation 4 of 9" aria-roledescription="slide" class="c-recommendations-list__item" role="group">
           <article class="u-full-height c-card c-card--flush" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
            <div class="c-card__layout u-full-height">
             <div class="c-card__image">
              <img alt="" src="https://media-springernature-com.proxy.lib.ohio-state.edu/w136h75/springer-static/image/art%3A10.1038%2Fs41592-019-0598-1/MediaObjects/41592_2019_598_Fig1_HTML.png"/>
             </div>
             <div class="c-card__body u-display-flex u-flex-direction-column">
              <div class="c-recommendations-column-switch">
               <h3 class="c-card__title-recommendation u-sans-serif" itemprop="name headline">
                <a class="c-card__link" data-track="click" data-track-action="click recommendations - 4" data-track-label="10.1038/s41592-019-0598-1" href="https://doi-org.proxy.lib.ohio-state.edu/10.1038/s41592-019-0598-1" itemprop="url">
                 Unified rational protein engineering with sequence-based deep representation learning
                </a>
               </h3>
               <div class="c-card__section c-meta">
                <span class="c-meta__item u-sans-serif">
                 Article
                </span>
                <span class="c-meta__item u-sans-serif">
                 21 October 2019
                </span>
               </div>
              </div>
              <p class="c-recommendations-authors u-hide-at-sm u-sans-serif">
               Ethan C. Alley, Grigory Khimulya, … George M. Church
              </p>
             </div>
            </div>
           </article>
          </div>
          <div aria-label="Recommendation 5 of 9" aria-roledescription="slide" class="c-recommendations-list__item" role="group">
           <article class="u-full-height c-card c-card--flush" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
            <div class="c-card__layout u-full-height">
             <div class="c-card__image">
              <img alt="" src="https://media-springernature-com.proxy.lib.ohio-state.edu/w136h75/springer-static/image/art%3A10.1038%2Fs41598-019-40314-1/MediaObjects/41598_2019_40314_Fig1_HTML.png"/>
             </div>
             <div class="c-card__body u-display-flex u-flex-direction-column">
              <div class="c-recommendations-column-switch">
               <h3 class="c-card__title-recommendation u-sans-serif" itemprop="name headline">
                <a class="c-card__link" data-track="click" data-track-action="click recommendations - 5" data-track-label="10.1038/s41598-019-40314-1" href="https://doi-org.proxy.lib.ohio-state.edu/10.1038/s41598-019-40314-1" itemprop="url">
                 DESTINI: A deep-learning approach to contact-driven protein structure prediction
                </a>
               </h3>
               <div class="c-card__section c-meta">
                <span class="c-meta__item u-sans-serif">
                 Article
                </span>
                <span class="c-meta__item u-sans-serif">
                 05 March 2019
                </span>
               </div>
              </div>
              <p class="c-recommendations-authors u-hide-at-sm u-sans-serif">
               Mu Gao, Hongyi Zhou &amp; Jeffrey Skolnick
              </p>
             </div>
            </div>
           </article>
          </div>
          <div aria-label="Recommendation 6 of 9" aria-roledescription="slide" class="c-recommendations-list__item" role="group">
           <article class="u-full-height c-card c-card--flush" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
            <div class="c-card__layout u-full-height">
             <div class="c-card__image">
              <img alt="" src="https://media-springernature-com.proxy.lib.ohio-state.edu/w136h75/springer-static/image/art%3A10.1038%2Fs41596-021-00628-9/MediaObjects/41596_2021_628_Fig1_HTML.png"/>
             </div>
             <div class="c-card__body u-display-flex u-flex-direction-column">
              <div class="c-recommendations-column-switch">
               <h3 class="c-card__title-recommendation u-sans-serif" itemprop="name headline">
                <a class="c-card__link" data-track="click" data-track-action="click recommendations - 6" data-track-label="10.1038/s41596-021-00628-9" href="https://doi-org.proxy.lib.ohio-state.edu/10.1038/s41596-021-00628-9" itemprop="url">
                 The trRosetta server for fast and accurate protein structure prediction
                </a>
               </h3>
               <div class="c-card__section c-meta">
                <span class="c-meta__item u-sans-serif">
                 Article
                </span>
                <span class="c-meta__item u-sans-serif">
                 10 November 2021
                </span>
               </div>
              </div>
              <p class="c-recommendations-authors u-hide-at-sm u-sans-serif">
               Zongyang Du, Hong Su, … Jianyi Yang
              </p>
             </div>
            </div>
           </article>
          </div>
          <div aria-label="Recommendation 7 of 9" aria-roledescription="slide" class="c-recommendations-list__item" role="group">
           <article class="u-full-height c-card c-card--flush" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
            <div class="c-card__layout u-full-height">
             <div class="c-card__image">
              <img alt="" src="https://media-springernature-com.proxy.lib.ohio-state.edu/w136h75/springer-static/image/art%3A10.1038%2Fs41467-022-29268-7/MediaObjects/41467_2022_29268_Fig1_HTML.png"/>
             </div>
             <div class="c-card__body u-display-flex u-flex-direction-column">
              <div class="c-recommendations-column-switch">
               <h3 class="c-card__title-recommendation u-sans-serif" itemprop="name headline">
                <a class="c-card__link" data-track="click" data-track-action="click recommendations - 7" data-track-label="10.1038/s41467-022-29268-7" href="https://doi-org.proxy.lib.ohio-state.edu/10.1038/s41467-022-29268-7" itemprop="url">
                 Current progress and open challenges for applying deep learning across the biosciences
                </a>
               </h3>
               <div class="c-card__section c-meta">
                <span class="c-meta__item u-sans-serif">
                 Article
                </span>
                <span class="c-meta__item u-sans-serif">
                 01 April 2022
                </span>
               </div>
              </div>
              <p class="c-recommendations-authors u-hide-at-sm u-sans-serif">
               Nicolae Sapoval, Amirali Aghazadeh, … Todd J. Treangen
              </p>
             </div>
            </div>
           </article>
          </div>
          <div aria-label="Recommendation 8 of 9" aria-roledescription="slide" class="c-recommendations-list__item" role="group">
           <article class="u-full-height c-card c-card--flush" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
            <div class="c-card__layout u-full-height">
             <div class="c-card__image">
              <img alt="" src="https://media-springernature-com.proxy.lib.ohio-state.edu/w136h75/springer-static/image/art%3A10.1038%2Fs41467-019-11994-0/MediaObjects/41467_2019_11994_Fig1_HTML.png"/>
             </div>
             <div class="c-card__body u-display-flex u-flex-direction-column">
              <div class="c-recommendations-column-switch">
               <h3 class="c-card__title-recommendation u-sans-serif" itemprop="name headline">
                <a class="c-card__link" data-track="click" data-track-action="click recommendations - 8" data-track-label="10.1038/s41467-019-11994-0" href="https://doi-org.proxy.lib.ohio-state.edu/10.1038/s41467-019-11994-0" itemprop="url">
                 Deep learning extends de novo protein modelling coverage of genomes using iteratively predicted structural constraints
                </a>
               </h3>
               <div class="c-card__section c-meta">
                <span class="c-meta__item u-sans-serif">
                 Article
                </span>
                <span class="c-meta__item u-sans-serif">
                 04 September 2019
                </span>
               </div>
              </div>
              <p class="c-recommendations-authors u-hide-at-sm u-sans-serif">
               Joe G. Greener, Shaun M. Kandathil &amp; David T. Jones
              </p>
             </div>
            </div>
           </article>
          </div>
          <div aria-label="Recommendation 9 of 9" aria-roledescription="slide" class="c-recommendations-list__item" role="group">
           <article class="u-full-height c-card c-card--flush" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
            <div class="c-card__layout u-full-height">
             <div class="c-card__image">
              <img alt="" src="https://media-springernature-com.proxy.lib.ohio-state.edu/w136h75/springer-static/image/art%3A10.1038%2Fs41598-019-48786-x/MediaObjects/41598_2019_48786_Fig1_HTML.png"/>
             </div>
             <div class="c-card__body u-display-flex u-flex-direction-column">
              <div class="c-recommendations-column-switch">
               <h3 class="c-card__title-recommendation u-sans-serif" itemprop="name headline">
                <a class="c-card__link" data-track="click" data-track-action="click recommendations - 9" data-track-label="10.1038/s41598-019-48786-x" href="https://doi-org.proxy.lib.ohio-state.edu/10.1038/s41598-019-48786-x" itemprop="url">
                 Deeper Profiles and Cascaded Recurrent and Convolutional Neural Networks for state-of-the-art Protein Secondary Structure Prediction
                </a>
               </h3>
               <div class="c-card__section c-meta">
                <span class="c-meta__item u-sans-serif">
                 Article
                </span>
                <span class="c-meta__item u-sans-serif">
                 26 August 2019
                </span>
               </div>
              </div>
              <p class="c-recommendations-authors u-hide-at-sm u-sans-serif">
               Mirko Torrisi, Manaz Kaleel &amp; Gianluca Pollastri
              </p>
             </div>
            </div>
           </article>
          </div>
         </div>
        </div>
       </section>
      </div>
      <div class="js-greyout-page-background" data-component-grey-background="" style="display:none">
      </div>
     </div>
    </div>
    <article lang="en">
     <div class="c-pdf-button__container u-mb-16 u-hide-at-lg js-context-bar-sticky-point-mobile">
      <div class="c-pdf-container">
       <div class="c-pdf-download u-clear-both js-pdf-download">
        <a class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-draft-ignore="true" data-readcube-pdf-url="true" data-test="download-pdf" data-track="click" data-track-action="download pdf" data-track-external="" data-track-label="link" download="" href="/articles/s41586-021-03819-2.pdf">
         <span class="c-pdf-download__text">
          Download PDF
         </span>
         <svg aria-hidden="true" class="u-icon" focusable="false" height="16" width="16">
          <use xlink:href="#icon-download">
          </use>
         </svg>
        </a>
       </div>
      </div>
     </div>
     <div class="c-article-header">
      <header>
       <ul class="c-article-identifiers" data-test="article-identifier">
        <li class="c-article-identifiers__item" data-test="article-category">
         Article
        </li>
        <li class="c-article-identifiers__item">
         <a class="u-color-open-access" data-test="open-access" data-track="click" data-track-action="open access" data-track-label="link" href="https://www-springernature-com.proxy.lib.ohio-state.edu/gp/open-research/about/the-fundamentals-of-open-access-and-open-research">
          Open access
         </a>
        </li>
        <li class="c-article-identifiers__item">
         <a data-track="click" data-track-action="publication date" data-track-label="link" href="#article-info">
          Published:
          <time datetime="2021-07-15">
           15 July 2021
          </time>
         </a>
        </li>
       </ul>
       <h1 class="c-article-title" data-article-title="" data-test="article-title">
        Highly accurate protein structure prediction with AlphaFold
       </h1>
       <ul class="c-article-author-list c-article-author-list--long js-no-scroll" data-component-authors-activator="authors-list" data-test="authors-list">
        <li class="c-article-author-list__item">
         <a data-author-popup="auth-John-Jumper-Aff1" data-corresp-id="c1" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-John-Jumper-Aff1">
          John Jumper
          <svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16">
           <use xlink:href="#icon-email-new" xmlns:xlink="http://www.w3.org/1999/xlink">
           </use>
          </svg>
         </a>
         <span class="u-js-hide">
          <a class="js-orcid" href="http://orcid.org/0000-0001-6169-6580">
           <span class="u-visually-hidden">
            ORCID:
           </span>
           orcid.org/0000-0001-6169-6580
          </a>
         </span>
         <sup class="u-js-hide">
          <a href="#Aff1" tabindex="-1">
           1
          </a>
         </sup>
         <sup class="u-js-hide">
          <a href="#na1" tabindex="-1">
           na1
          </a>
         </sup>
         ,
        </li>
        <li class="c-article-author-list__item">
         <a data-author-popup="auth-Richard-Evans-Aff1" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Richard-Evans-Aff1">
          Richard Evans
         </a>
         <sup class="u-js-hide">
          <a href="#Aff1" tabindex="-1">
           1
          </a>
         </sup>
         <sup class="u-js-hide">
          <a href="#na1" tabindex="-1">
           na1
          </a>
         </sup>
         ,
        </li>
        <li class="c-article-author-list__item c-article-author-list__item--hide-small-screen">
         <a data-author-popup="auth-Alexander-Pritzel-Aff1" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Alexander-Pritzel-Aff1">
          Alexander Pritzel
         </a>
         <sup class="u-js-hide">
          <a href="#Aff1" tabindex="-1">
           1
          </a>
         </sup>
         <sup class="u-js-hide">
          <a href="#na1" tabindex="-1">
           na1
          </a>
         </sup>
         ,
        </li>
        <li class="c-article-author-list__item c-article-author-list__item--hide-small-screen">
         <a data-author-popup="auth-Tim-Green-Aff1" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Tim-Green-Aff1">
          Tim Green
         </a>
         <span class="u-js-hide">
          <a class="js-orcid" href="http://orcid.org/0000-0002-3227-1505">
           <span class="u-visually-hidden">
            ORCID:
           </span>
           orcid.org/0000-0002-3227-1505
          </a>
         </span>
         <sup class="u-js-hide">
          <a href="#Aff1" tabindex="-1">
           1
          </a>
         </sup>
         <sup class="u-js-hide">
          <a href="#na1" tabindex="-1">
           na1
          </a>
         </sup>
         ,
        </li>
        <li class="c-article-author-list__item c-article-author-list__item--hide-small-screen">
         <a data-author-popup="auth-Michael-Figurnov-Aff1" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Michael-Figurnov-Aff1">
          Michael Figurnov
         </a>
         <sup class="u-js-hide">
          <a href="#Aff1" tabindex="-1">
           1
          </a>
         </sup>
         <sup class="u-js-hide">
          <a href="#na1" tabindex="-1">
           na1
          </a>
         </sup>
         ,
        </li>
        <li class="c-article-author-list__item c-article-author-list__item--hide-small-screen">
         <a data-author-popup="auth-Olaf-Ronneberger-Aff1" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Olaf-Ronneberger-Aff1">
          Olaf Ronneberger
         </a>
         <sup class="u-js-hide">
          <a href="#Aff1" tabindex="-1">
           1
          </a>
         </sup>
         <sup class="u-js-hide">
          <a href="#na1" tabindex="-1">
           na1
          </a>
         </sup>
         ,
        </li>
        <li class="c-article-author-list__item c-article-author-list__item--hide-small-screen">
         <a data-author-popup="auth-Kathryn-Tunyasuvunakool-Aff1" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Kathryn-Tunyasuvunakool-Aff1">
          Kathryn Tunyasuvunakool
         </a>
         <sup class="u-js-hide">
          <a href="#Aff1" tabindex="-1">
           1
          </a>
         </sup>
         <sup class="u-js-hide">
          <a href="#na1" tabindex="-1">
           na1
          </a>
         </sup>
         ,
        </li>
        <li class="c-article-author-list__item c-article-author-list__item--hide-small-screen">
         <a data-author-popup="auth-Russ-Bates-Aff1" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Russ-Bates-Aff1">
          Russ Bates
         </a>
         <sup class="u-js-hide">
          <a href="#Aff1" tabindex="-1">
           1
          </a>
         </sup>
         <sup class="u-js-hide">
          <a href="#na1" tabindex="-1">
           na1
          </a>
         </sup>
         ,
        </li>
        <li class="c-article-author-list__item c-article-author-list__item--hide-small-screen">
         <a data-author-popup="auth-Augustin-__dek-Aff1" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Augustin-__dek-Aff1">
          Augustin Žídek
         </a>
         <sup class="u-js-hide">
          <a href="#Aff1" tabindex="-1">
           1
          </a>
         </sup>
         <sup class="u-js-hide">
          <a href="#na1" tabindex="-1">
           na1
          </a>
         </sup>
         ,
        </li>
        <li class="c-article-author-list__item c-article-author-list__item--hide-small-screen">
         <a data-author-popup="auth-Anna-Potapenko-Aff1" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Anna-Potapenko-Aff1">
          Anna Potapenko
         </a>
         <sup class="u-js-hide">
          <a href="#Aff1" tabindex="-1">
           1
          </a>
         </sup>
         <sup class="u-js-hide">
          <a href="#na1" tabindex="-1">
           na1
          </a>
         </sup>
         ,
        </li>
        <li class="c-article-author-list__item c-article-author-list__item--hide-small-screen">
         <a data-author-popup="auth-Alex-Bridgland-Aff1" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Alex-Bridgland-Aff1">
          Alex Bridgland
         </a>
         <sup class="u-js-hide">
          <a href="#Aff1" tabindex="-1">
           1
          </a>
         </sup>
         <sup class="u-js-hide">
          <a href="#na1" tabindex="-1">
           na1
          </a>
         </sup>
         ,
        </li>
        <li class="c-article-author-list__item c-article-author-list__item--hide-small-screen">
         <a data-author-popup="auth-Clemens-Meyer-Aff1" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Clemens-Meyer-Aff1">
          Clemens Meyer
         </a>
         <sup class="u-js-hide">
          <a href="#Aff1" tabindex="-1">
           1
          </a>
         </sup>
         <sup class="u-js-hide">
          <a href="#na1" tabindex="-1">
           na1
          </a>
         </sup>
         ,
        </li>
        <li class="c-article-author-list__item c-article-author-list__item--hide-small-screen">
         <a data-author-popup="auth-Simon_A__A_-Kohl-Aff1" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Simon_A__A_-Kohl-Aff1">
          Simon A. A. Kohl
         </a>
         <span class="u-js-hide">
          <a class="js-orcid" href="http://orcid.org/0000-0003-4271-4418">
           <span class="u-visually-hidden">
            ORCID:
           </span>
           orcid.org/0000-0003-4271-4418
          </a>
         </span>
         <sup class="u-js-hide">
          <a href="#Aff1" tabindex="-1">
           1
          </a>
         </sup>
         <sup class="u-js-hide">
          <a href="#na1" tabindex="-1">
           na1
          </a>
         </sup>
         ,
        </li>
        <li class="c-article-author-list__item c-article-author-list__item--hide-small-screen">
         <a data-author-popup="auth-Andrew_J_-Ballard-Aff1" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Andrew_J_-Ballard-Aff1">
          Andrew J. Ballard
         </a>
         <sup class="u-js-hide">
          <a href="#Aff1" tabindex="-1">
           1
          </a>
         </sup>
         <sup class="u-js-hide">
          <a href="#na1" tabindex="-1">
           na1
          </a>
         </sup>
         ,
        </li>
        <li class="c-article-author-list__item c-article-author-list__item--hide-small-screen">
         <a data-author-popup="auth-Andrew-Cowie-Aff1" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Andrew-Cowie-Aff1">
          Andrew Cowie
         </a>
         <sup class="u-js-hide">
          <a href="#Aff1" tabindex="-1">
           1
          </a>
         </sup>
         <sup class="u-js-hide">
          <a href="#na1" tabindex="-1">
           na1
          </a>
         </sup>
         ,
        </li>
        <li class="c-article-author-list__item c-article-author-list__item--hide-small-screen">
         <a data-author-popup="auth-Bernardino-Romera_Paredes-Aff1" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Bernardino-Romera_Paredes-Aff1">
          Bernardino Romera-Paredes
         </a>
         <sup class="u-js-hide">
          <a href="#Aff1" tabindex="-1">
           1
          </a>
         </sup>
         <sup class="u-js-hide">
          <a href="#na1" tabindex="-1">
           na1
          </a>
         </sup>
         ,
        </li>
        <li class="c-article-author-list__item c-article-author-list__item--hide-small-screen">
         <a data-author-popup="auth-Stanislav-Nikolov-Aff1" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Stanislav-Nikolov-Aff1">
          Stanislav Nikolov
         </a>
         <sup class="u-js-hide">
          <a href="#Aff1" tabindex="-1">
           1
          </a>
         </sup>
         <sup class="u-js-hide">
          <a href="#na1" tabindex="-1">
           na1
          </a>
         </sup>
         ,
        </li>
        <li class="c-article-author-list__item c-article-author-list__item--hide-small-screen">
         <a data-author-popup="auth-Rishub-Jain-Aff1" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Rishub-Jain-Aff1">
          Rishub Jain
         </a>
         <sup class="u-js-hide">
          <a href="#Aff1" tabindex="-1">
           1
          </a>
         </sup>
         <sup class="u-js-hide">
          <a href="#na1" tabindex="-1">
           na1
          </a>
         </sup>
         ,
        </li>
        <li class="c-article-author-list__item c-article-author-list__item--hide-small-screen">
         <a data-author-popup="auth-Jonas-Adler-Aff1" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Jonas-Adler-Aff1">
          Jonas Adler
         </a>
         <span class="u-js-hide">
          <a class="js-orcid" href="http://orcid.org/0000-0001-9928-3407">
           <span class="u-visually-hidden">
            ORCID:
           </span>
           orcid.org/0000-0001-9928-3407
          </a>
         </span>
         <sup class="u-js-hide">
          <a href="#Aff1" tabindex="-1">
           1
          </a>
         </sup>
         ,
        </li>
        <li class="c-article-author-list__item c-article-author-list__item--hide-small-screen">
         <a data-author-popup="auth-Trevor-Back-Aff1" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Trevor-Back-Aff1">
          Trevor Back
         </a>
         <sup class="u-js-hide">
          <a href="#Aff1" tabindex="-1">
           1
          </a>
         </sup>
         ,
        </li>
        <li class="c-article-author-list__item c-article-author-list__item--hide-small-screen">
         <a data-author-popup="auth-Stig-Petersen-Aff1" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Stig-Petersen-Aff1">
          Stig Petersen
         </a>
         <sup class="u-js-hide">
          <a href="#Aff1" tabindex="-1">
           1
          </a>
         </sup>
         ,
        </li>
        <li class="c-article-author-list__item c-article-author-list__item--hide-small-screen">
         <a data-author-popup="auth-David-Reiman-Aff1" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-David-Reiman-Aff1">
          David Reiman
         </a>
         <sup class="u-js-hide">
          <a href="#Aff1" tabindex="-1">
           1
          </a>
         </sup>
         ,
        </li>
        <li class="c-article-author-list__item c-article-author-list__item--hide-small-screen">
         <a data-author-popup="auth-Ellen-Clancy-Aff1" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Ellen-Clancy-Aff1">
          Ellen Clancy
         </a>
         <sup class="u-js-hide">
          <a href="#Aff1" tabindex="-1">
           1
          </a>
         </sup>
         ,
        </li>
        <li class="c-article-author-list__item c-article-author-list__item--hide-small-screen">
         <a data-author-popup="auth-Michal-Zielinski-Aff1" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Michal-Zielinski-Aff1">
          Michal Zielinski
         </a>
         <sup class="u-js-hide">
          <a href="#Aff1" tabindex="-1">
           1
          </a>
         </sup>
         ,
        </li>
        <li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen">
         <a data-author-popup="auth-Martin-Steinegger-Aff2-Aff3" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Martin-Steinegger-Aff2-Aff3">
          Martin Steinegger
         </a>
         <span class="u-js-hide">
          <a class="js-orcid" href="http://orcid.org/0000-0001-8781-9753">
           <span class="u-visually-hidden">
            ORCID:
           </span>
           orcid.org/0000-0001-8781-9753
          </a>
         </span>
         <sup class="u-js-hide">
          <a href="#Aff2" tabindex="-1">
           2
          </a>
          ,
          <a href="#Aff3" tabindex="-1">
           3
          </a>
         </sup>
         ,
        </li>
        <li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen">
         <a data-author-popup="auth-Michalina-Pacholska-Aff1" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Michalina-Pacholska-Aff1">
          Michalina Pacholska
         </a>
         <span class="u-js-hide">
          <a class="js-orcid" href="http://orcid.org/0000-0002-2160-6226">
           <span class="u-visually-hidden">
            ORCID:
           </span>
           orcid.org/0000-0002-2160-6226
          </a>
         </span>
         <sup class="u-js-hide">
          <a href="#Aff1" tabindex="-1">
           1
          </a>
         </sup>
         ,
        </li>
        <li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen">
         <a data-author-popup="auth-Tamas-Berghammer-Aff1" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Tamas-Berghammer-Aff1">
          Tamas Berghammer
         </a>
         <sup class="u-js-hide">
          <a href="#Aff1" tabindex="-1">
           1
          </a>
         </sup>
         ,
        </li>
        <li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen">
         <a data-author-popup="auth-Sebastian-Bodenstein-Aff1" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Sebastian-Bodenstein-Aff1">
          Sebastian Bodenstein
         </a>
         <sup class="u-js-hide">
          <a href="#Aff1" tabindex="-1">
           1
          </a>
         </sup>
         ,
        </li>
        <li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen">
         <a data-author-popup="auth-David-Silver-Aff1" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-David-Silver-Aff1">
          David Silver
         </a>
         <span class="u-js-hide">
          <a class="js-orcid" href="http://orcid.org/0000-0002-5197-2892">
           <span class="u-visually-hidden">
            ORCID:
           </span>
           orcid.org/0000-0002-5197-2892
          </a>
         </span>
         <sup class="u-js-hide">
          <a href="#Aff1" tabindex="-1">
           1
          </a>
         </sup>
         ,
        </li>
        <li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen">
         <a data-author-popup="auth-Oriol-Vinyals-Aff1" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Oriol-Vinyals-Aff1">
          Oriol Vinyals
         </a>
         <sup class="u-js-hide">
          <a href="#Aff1" tabindex="-1">
           1
          </a>
         </sup>
         ,
        </li>
        <li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen">
         <a data-author-popup="auth-Andrew_W_-Senior-Aff1" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Andrew_W_-Senior-Aff1">
          Andrew W. Senior
         </a>
         <span class="u-js-hide">
          <a class="js-orcid" href="http://orcid.org/0000-0002-2401-5691">
           <span class="u-visually-hidden">
            ORCID:
           </span>
           orcid.org/0000-0002-2401-5691
          </a>
         </span>
         <sup class="u-js-hide">
          <a href="#Aff1" tabindex="-1">
           1
          </a>
         </sup>
         ,
        </li>
        <li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen">
         <a data-author-popup="auth-Koray-Kavukcuoglu-Aff1" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Koray-Kavukcuoglu-Aff1">
          Koray Kavukcuoglu
         </a>
         <sup class="u-js-hide">
          <a href="#Aff1" tabindex="-1">
           1
          </a>
         </sup>
         ,
        </li>
        <li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen">
         <a data-author-popup="auth-Pushmeet-Kohli-Aff1" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Pushmeet-Kohli-Aff1">
          Pushmeet Kohli
         </a>
         <sup class="u-js-hide">
          <a href="#Aff1" tabindex="-1">
           1
          </a>
         </sup>
         &amp;
        </li>
        <li aria-label="Show all 34 authors for this article" class="c-article-author-list__show-more" title="Show all 34 authors for this article">
         …
        </li>
        <li class="c-article-author-list__item">
         <a data-author-popup="auth-Demis-Hassabis-Aff1" data-corresp-id="c2" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Demis-Hassabis-Aff1">
          Demis Hassabis
          <svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16">
           <use xlink:href="#icon-email-new" xmlns:xlink="http://www.w3.org/1999/xlink">
           </use>
          </svg>
         </a>
         <span class="u-js-hide">
          <a class="js-orcid" href="http://orcid.org/0000-0003-2812-9917">
           <span class="u-visually-hidden">
            ORCID:
           </span>
           orcid.org/0000-0003-2812-9917
          </a>
         </span>
         <sup class="u-js-hide">
          <a href="#Aff1" tabindex="-1">
           1
          </a>
         </sup>
         <sup class="u-js-hide">
          <a href="#na1" tabindex="-1">
           na1
          </a>
         </sup>
        </li>
       </ul>
       <button aria-expanded="false" class="c-article-author-list__button">
        <svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16">
         <use xlink:href="#icon-plus" xmlns:xlink="http://www.w3.org/1999/xlink">
         </use>
        </svg>
        <span>
         Show authors
        </span>
       </button>
       <p class="c-article-info-details" data-container-section="info">
        <a data-test="journal-link" data-track="click" data-track-action="journal homepage" data-track-category="article body" data-track-label="link" href="/">
         <i data-test="journal-title">
          Nature
         </i>
        </a>
        <b data-test="journal-volume">
         <span class="u-visually-hidden">
          volume
         </span>
         596
        </b>
        ,
        <span class="u-visually-hidden">
         pages
        </span>
        583–589 (
        <span data-test="article-publication-year">
         2021
        </span>
        )
        <a class="c-article-info-details__cite-as u-hide-print" data-track="click" data-track-action="cite this article" data-track-label="link" href="#citeas">
         Cite this article
        </a>
       </p>
       <div class="c-article-metrics-bar__wrapper u-clear-both">
        <ul class="c-article-metrics-bar u-list-reset">
         <li class="c-article-metrics-bar__item">
          <p class="c-article-metrics-bar__count">
           1.29m
           <span class="c-article-metrics-bar__label">
            Accesses
           </span>
          </p>
         </li>
         <li class="c-article-metrics-bar__item">
          <p class="c-article-metrics-bar__count">
           10k
           <span class="c-article-metrics-bar__label">
            Citations
           </span>
          </p>
         </li>
         <li class="c-article-metrics-bar__item">
          <p class="c-article-metrics-bar__count">
           3528
           <span class="c-article-metrics-bar__label">
            Altmetric
           </span>
          </p>
         </li>
         <li class="c-article-metrics-bar__item">
          <p class="c-article-metrics-bar__details">
           <a data-track="click" data-track-action="view metrics" data-track-label="link" href="/articles/s41586-021-03819-2/metrics" rel="nofollow">
            Metrics
            <span class="u-visually-hidden">
             details
            </span>
           </a>
          </p>
         </li>
        </ul>
       </div>
      </header>
     </div>
     <div class="c-article-body">
      <section aria-labelledby="Abs1" data-title="Abstract" lang="en">
       <div class="c-article-section" id="Abs1-section">
        <h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Abs1">
         Abstract
        </h2>
        <div class="c-article-section__content" id="Abs1-content">
         <p>
          Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort
          <sup>
           <a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR1" id="ref-link-section-d262236594e745" title="Thompson, M. C., Yeates, T. O. &amp; Rodriguez, J. A. Advances in methods for atomic resolution macromolecular structure determination. F1000Res. 9, 667 (2020).">
            1
           </a>
           ,
           <a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR2" id="ref-link-section-d262236594e745_1" title="Bai, X.-C., McMullan, G. &amp; Scheres, S. H. W. How cryo-EM is revolutionizing structural biology. Trends Biochem. Sci. 40, 49–57 (2015).">
            2
           </a>
           ,
           <a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR3" id="ref-link-section-d262236594e745_2" title="Jaskolski, M., Dauter, Z. &amp; Wlodawer, A. A brief history of macromolecular crystallography, illustrated by a family tree and its Nobel fruits. FEBS J. 281, 3985–4009 (2014).">
            3
           </a>
           ,
           <a aria-label="Reference 4" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR4" id="ref-link-section-d262236594e748" title="Wüthrich, K. The way to NMR structures of proteins. Nat. Struct. Biol. 8, 923–925 (2001).">
            4
           </a>
          </sup>
          , the structures of around 100,000 unique proteins have been determined
          <sup>
           <a aria-label="Reference 5" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR5" id="ref-link-section-d262236594e752" title="wwPDB Consortium. Protein Data Bank: the single global archive for 3D macromolecular structure data. Nucleic Acids Res. 47, D520–D528 (2018).">
            5
           </a>
          </sup>
          , but this represents a small fraction of the billions of known protein sequences
          <sup>
           <a aria-label="Reference 6" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR6" id="ref-link-section-d262236594e756" title="Mitchell, A. L. et al. MGnify: the microbiome analysis resource in 2020. Nucleic Acids Res. 48, D570–D578 (2020).">
            6
           </a>
           ,
           <a aria-label="Reference 7" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR7" id="ref-link-section-d262236594e759" title="Steinegger, M., Mirdita, M. &amp; Söding, J. Protein-level assembly increases protein sequence recovery from metagenomic samples manyfold. Nat. Methods 16, 603–606 (2019).">
            7
           </a>
          </sup>
          . Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’
          <sup>
           <a aria-label="Reference 8" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR8" id="ref-link-section-d262236594e763" title="Dill, K. A., Ozkan, S. B., Shell, M. S. &amp; Weikl, T. R. The protein folding problem. Annu. Rev. Biophys. 37, 289–316 (2008).">
            8
           </a>
          </sup>
          —has been an important open research problem for more than 50 years
          <sup>
           <a aria-label="Reference 9" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR9" id="ref-link-section-d262236594e767" title="Anfinsen, C. B. Principles that govern the folding of protein chains. Science 181, 223–230 (1973).">
            9
           </a>
          </sup>
          . Despite recent progress
          <sup>
           <a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR10" id="ref-link-section-d262236594e772" title="Senior, A. W. et al. Improved protein structure prediction using potentials from deep learning. Nature 577, 706–710 (2020).">
            10
           </a>
           ,
           <a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR11" id="ref-link-section-d262236594e772_1" title="Wang, S., Sun, S., Li, Z., Zhang, R. &amp; Xu, J. Accurate de novo prediction of protein contact map by ultra-deep learning model. PLOS Comput. Biol. 13, e1005324 (2017).">
            11
           </a>
           ,
           <a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR12" id="ref-link-section-d262236594e772_2" title="Zheng, W. et al. Deep-learning contact-map guided protein structure prediction in CASP13. Proteins 87, 1149–1164 (2019).">
            12
           </a>
           ,
           <a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR13" id="ref-link-section-d262236594e772_3" title="Abriata, L. A., Tamò, G. E. &amp; Dal Peraro, M. A further leap of improvement in tertiary structure prediction in CASP13 prompts new routes for future assessments. Proteins 87, 1100–1112 (2019).">
            13
           </a>
           ,
           <a aria-label="Reference 14" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR14" id="ref-link-section-d262236594e775" title="Pearce, R. &amp; Zhang, Y. Deep learning techniques have significantly impacted protein structure prediction and protein design. Curr. Opin. Struct. Biol. 68, 194–207 (2021).">
            14
           </a>
          </sup>
          , existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)
          <sup>
           <a aria-label="Reference 15" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR15" id="ref-link-section-d262236594e779" title="Moult, J., Fidelis, K., Kryshtafovych, A., Schwede, T. &amp; Topf, M. Critical assessment of techniques for protein structure prediction, fourteenth round. CASP 14 Abstract Book 
                  https://www.predictioncenter.org/casp14/doc/CASP14_Abstracts.pdf
                  
                 (2020).">
            15
           </a>
          </sup>
          , demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm.
         </p>
        </div>
       </div>
      </section>
      <noscript>
      </noscript>
      <div class="main-content">
       <section data-title="Main">
        <div class="c-article-section" id="Sec1-section">
         <h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec1">
          Main
         </h2>
         <div class="c-article-section__content" id="Sec1-content">
          <p>
           The development of computational methods to predict three-dimensional (3D) protein structures from the protein sequence has proceeded along two complementary paths that focus on either the physical interactions or the evolutionary history. The physical interaction programme heavily integrates our understanding of molecular driving forces into either thermodynamic or kinetic simulation of protein physics
           <sup>
            <a aria-label="Reference 16" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR16" id="ref-link-section-d262236594e805" title="Brini, E., Simmerling, C. &amp; Dill, K. Protein storytelling through physics. Science 370, eaaz3041 (2020).">
             16
            </a>
           </sup>
           or statistical approximations thereof
           <sup>
            <a aria-label="Reference 17" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR17" id="ref-link-section-d262236594e809" title="Sippl, M. J. Calculation of conformational ensembles from potentials of mean force. An approach to the knowledge-based prediction of local structures in globular proteins. J. Mol. Biol. 213, 859–883 (1990).">
             17
            </a>
           </sup>
           . Although theoretically very appealing, this approach has proved highly challenging for even moderate-sized proteins due to the computational intractability of molecular simulation, the context dependence of protein stability and the difficulty of producing sufficiently accurate models of protein physics. The evolutionary programme has provided an alternative in recent years, in which the constraints on protein structure are derived from bioinformatics analysis of the evolutionary history of proteins, homology to solved structures
           <sup>
            <a aria-label="Reference 18" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR18" id="ref-link-section-d262236594e813" title="Šali, A. &amp; Blundell, T. L. Comparative protein modelling by satisfaction of spatial restraints. J. Mol. Biol. 234, 779–815 (1993).">
             18
            </a>
            ,
            <a aria-label="Reference 19" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR19" id="ref-link-section-d262236594e816" title="Roy, A., Kucukural, A. &amp; Zhang, Y. I-TASSER: a unified platform for automated protein structure and function prediction. Nat. Protocols 5, 725–738 (2010).">
             19
            </a>
           </sup>
           and pairwise evolutionary correlations
           <sup>
            <a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR20" id="ref-link-section-d262236594e820" title="Altschuh, D., Lesk, A. M., Bloomer, A. C. &amp; Klug, A. Correlation of co-ordinated amino acid substitutions with function in viruses related to tobacco mosaic virus. J. Mol. Biol. 193, 693–707 (1987).">
             20
            </a>
            ,
            <a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR21" id="ref-link-section-d262236594e820_1" title="Shindyalov, I. N., Kolchanov, N. A. &amp; Sander, C. Can three-dimensional contacts in protein structures be predicted by analysis of correlated mutations? Protein Eng. 7, 349–358 (1994).">
             21
            </a>
            ,
            <a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR22" id="ref-link-section-d262236594e820_2" title="Weigt, M., White, R. A., Szurmant, H., Hoch, J. A. &amp; Hwa, T. Identification of direct residue contacts in protein–protein interaction by message passing. Proc. Natl Acad. Sci. USA 106, 67–72 (2009).">
             22
            </a>
            ,
            <a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR23" id="ref-link-section-d262236594e820_3" title="Marks, D. S. et al. Protein 3D structure computed from evolutionary sequence variation. PLoS ONE 6, e28766 (2011).">
             23
            </a>
            ,
            <a aria-label="Reference 24" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR24" id="ref-link-section-d262236594e823" title="Jones, D. T., Buchan, D. W. A., Cozzetto, D. &amp; Pontil, M. PSICOV: precise structural contact prediction using sparse inverse covariance estimation on large multiple sequence alignments. Bioinformatics 28, 184–190 (2012).">
             24
            </a>
           </sup>
           . This bioinformatics approach has benefited greatly from the steady growth of experimental protein structures deposited in the Protein Data Bank (PDB)
           <sup>
            <a aria-label="Reference 5" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR5" id="ref-link-section-d262236594e827" title="wwPDB Consortium. Protein Data Bank: the single global archive for 3D macromolecular structure data. Nucleic Acids Res. 47, D520–D528 (2018).">
             5
            </a>
           </sup>
           , the explosion of genomic sequencing and the rapid development of deep learning techniques to interpret these correlations. Despite these advances, contemporary physical and evolutionary-history-based approaches produce predictions that are far short of experimental accuracy in the majority of cases in which a close homologue has not been solved experimentally and this has limited their utility for many biological applications.
          </p>
          <p>
           In this study, we develop the first, to our knowledge, computational approach capable of predicting protein structures to near experimental accuracy in a majority of cases. The neural network AlphaFold that we developed was entered into the CASP14 assessment (May–July 2020; entered under the team name ‘AlphaFold2’ and a completely different model from our CASP13 AlphaFold system
           <sup>
            <a aria-label="Reference 10" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR10" id="ref-link-section-d262236594e834" title="Senior, A. W. et al. Improved protein structure prediction using potentials from deep learning. Nature 577, 706–710 (2020).">
             10
            </a>
           </sup>
           ). The CASP assessment is carried out biennially using recently solved structures that have not been deposited in the PDB or publicly disclosed so that it is a blind test for the participating methods, and has long served as the gold-standard assessment for the accuracy of structure prediction
           <sup>
            <a aria-label="Reference 25" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR25" id="ref-link-section-d262236594e838" title="Moult, J., Pedersen, J. T., Judson, R. &amp; Fidelis, K. A large-scale experiment to assess protein structure prediction methods. Proteins 23, ii–iv (1995).">
             25
            </a>
            ,
            <a aria-label="Reference 26" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR26" id="ref-link-section-d262236594e841" title="Kryshtafovych, A., Schwede, T., Topf, M., Fidelis, K. &amp; Moult, J. Critical assessment of methods of protein structure prediction (CASP)-round XIII. Proteins 87, 1011–1020 (2019).">
             26
            </a>
           </sup>
           .
          </p>
          <p>
           In CASP14, AlphaFold structures were vastly more accurate than competing methods. AlphaFold structures had a median backbone accuracy of 0.96 Å r.m.s.d.
           <sub>
            95
           </sub>
           (Cα root-mean-square deviation at 95% residue coverage) (95% confidence interval = 0.85–1.16 Å) whereas the next best performing method had a median backbone accuracy of 2.8 Å r.m.s.d.
           <sub>
            95
           </sub>
           (95% confidence interval = 2.7–4.0 Å) (measured on CASP domains; see Fig.
           <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41586-021-03819-2#Fig1">
            1a
           </a>
           for backbone accuracy and Supplementary Fig.
           <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41586-021-03819-2#MOESM1">
            14
           </a>
           for all-atom accuracy). As a comparison point for this accuracy, the width of a carbon atom is approximately 1.4 Å. In addition to very accurate domain structures (Fig.
           <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41586-021-03819-2#Fig1">
            1b
           </a>
           ), AlphaFold is able to produce highly accurate side chains (Fig.
           <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41586-021-03819-2#Fig1">
            1c
           </a>
           ) when the backbone is highly accurate and considerably improves over template-based methods even when strong templates are available. The all-atom accuracy of AlphaFold was 1.5 Å r.m.s.d.
           <sub>
            95
           </sub>
           (95% confidence interval = 1.2–1.6 Å) compared with the 3.5 Å r.m.s.d.
           <sub>
            95
           </sub>
           (95% confidence interval = 3.1–4.2 Å) of the best alternative method. Our methods are scalable to very long proteins with accurate domains and domain-packing (see Fig.
           <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41586-021-03819-2#Fig1">
            1d
           </a>
           for the prediction of a 2,180-residue protein with no structural homologues). Finally, the model is able to provide precise, per-residue estimates of its reliability that should enable the confident use of these predictions.
          </p>
          <div class="c-article-section__figure js-c-reading-companion-figures-item" data-container-section="figure" data-test="figure" data-title="AlphaFold produces highly accurate structures." id="figure-1">
           <figure>
            <figcaption>
             <b class="c-article-section__figure-caption" data-test="figure-caption-text" id="Fig1">
              Fig. 1: AlphaFold produces highly accurate structures.
             </b>
            </figcaption>
            <div class="c-article-section__figure-content">
             <div class="c-article-section__figure-item">
              <a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-action="view figure" data-track-label="image" href="/articles/s41586-021-03819-2/figures/1" rel="nofollow">
               <picture>
                <source srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-021-03819-2/MediaObjects/41586_2021_3819_Fig1_HTML.png?as=webp" type="image/webp"/>
                <img alt="figure 1" aria-describedby="Fig1" height="439" loading="lazy" src="//media-springernature-com.proxy.lib.ohio-state.edu/lw685/springer-static/image/art%3A10.1038%2Fs41586-021-03819-2/MediaObjects/41586_2021_3819_Fig1_HTML.png" width="685"/>
               </picture>
              </a>
             </div>
             <div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-1-desc">
              <p>
               <b>
                a
               </b>
               , The performance of AlphaFold on the CASP14 dataset (
               <i>
                n
               </i>
               = 87 protein domains) relative to the top-15 entries (out of 146 entries), group numbers correspond to the numbers assigned to entrants by CASP. Data are median and the 95% confidence interval of the median, estimated from 10,000 bootstrap samples.
               <b>
                b
               </b>
               , Our prediction of CASP14 target T1049 (PDB 6Y4F, blue) compared with the true (experimental) structure (green). Four residues in the C terminus of the crystal structure are
               <i>
                B
               </i>
               -factor outliers and are not depicted.
               <b>
                c
               </b>
               , CASP14 target T1056 (PDB 6YJ1). An example of a well-predicted zinc-binding site (AlphaFold has accurate side chains even though it does not explicitly predict the zinc ion).
               <b>
                d
               </b>
               , CASP target T1044 (PDB 6VR4)—a 2,180-residue single chain—was predicted with correct domain packing (the prediction was made after CASP using AlphaFold without intervention).
               <b>
                e
               </b>
               , Model architecture. Arrows show the information flow among the various components described in this paper. Array shapes are shown in parentheses with
               <i>
                s
               </i>
               , number of sequences (
               <i>
                N
               </i>
               <sub>
                seq
               </sub>
               in the main text);
               <i>
                r
               </i>
               , number of residues (
               <i>
                N
               </i>
               <sub>
                res
               </sub>
               in the main text);
               <i>
                c
               </i>
               , number of channels.
              </p>
             </div>
            </div>
            <div class="u-text-right u-hide-print">
             <a aria-label="Full size image figure 1" class="c-article__pill-button" data-test="article-link" data-track="click" data-track-action="view figure" data-track-dest="link:Figure1 Full size image" data-track-label="button" href="/articles/s41586-021-03819-2/figures/1" rel="nofollow">
              <span>
               Full size image
              </span>
              <svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16">
               <use xlink:href="#icon-chevron-right" xmlns:xlink="http://www.w3.org/1999/xlink">
               </use>
              </svg>
             </a>
            </div>
           </figure>
          </div>
          <p>
           We demonstrate in Fig.
           <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41586-021-03819-2#Fig2">
            2a
           </a>
           that the high accuracy that AlphaFold demonstrated in CASP14 extends to a large sample of recently released PDB structures; in this dataset, all structures were deposited in the PDB after our training data cut-off and are analysed as full chains (see
           <a data-track="click" data-track-action="section anchor" data-track-label="link" href="/articles/s41586-021-03819-2#Sec10">
            Methods
           </a>
           , Supplementary Fig.
           <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41586-021-03819-2#MOESM1">
            15
           </a>
           and Supplementary Table
           <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41586-021-03819-2#MOESM1">
            6
           </a>
           for more details). Furthermore, we observe high side-chain accuracy when the backbone prediction is accurate (Fig.
           <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41586-021-03819-2#Fig2">
            2b
           </a>
           ) and we show that our confidence measure, the predicted local-distance difference test (pLDDT), reliably predicts the Cα local-distance difference test (lDDT-Cα) accuracy of the corresponding prediction (Fig.
           <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41586-021-03819-2#Fig2">
            2c
           </a>
           ). We also find that the global superposition metric template modelling score (TM-score)
           <sup>
            <a aria-label="Reference 27" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR27" id="ref-link-section-d262236594e953" title="Zhang, Y. &amp; Skolnick, J. Scoring function for automated assessment of protein structure template quality. Proteins 57, 702–710 (2004).">
             27
            </a>
           </sup>
           can be accurately estimated (Fig.
           <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41586-021-03819-2#Fig2">
            2d
           </a>
           ). Overall, these analyses validate that the high accuracy and reliability of AlphaFold on CASP14 proteins also transfers to an uncurated collection of recent PDB submissions, as would be expected (see
           <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41586-021-03819-2#MOESM1">
            Supplementary Methods 1.15
           </a>
           and Supplementary Fig.
           <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41586-021-03819-2#MOESM1">
            11
           </a>
           for confirmation that this high accuracy extends to new folds).
          </p>
          <div class="c-article-section__figure js-c-reading-companion-figures-item" data-container-section="figure" data-test="figure" data-title="Accuracy of AlphaFold on recent PDB structures." id="figure-2">
           <figure>
            <figcaption>
             <b class="c-article-section__figure-caption" data-test="figure-caption-text" id="Fig2">
              Fig. 2: Accuracy of AlphaFold on recent PDB structures.
             </b>
            </figcaption>
            <div class="c-article-section__figure-content">
             <div class="c-article-section__figure-item">
              <a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-action="view figure" data-track-label="image" href="/articles/s41586-021-03819-2/figures/2" rel="nofollow">
               <picture>
                <source srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-021-03819-2/MediaObjects/41586_2021_3819_Fig2_HTML.png?as=webp" type="image/webp"/>
                <img alt="figure 2" aria-describedby="Fig2" height="1128" loading="lazy" src="//media-springernature-com.proxy.lib.ohio-state.edu/lw685/springer-static/image/art%3A10.1038%2Fs41586-021-03819-2/MediaObjects/41586_2021_3819_Fig2_HTML.png" width="685"/>
               </picture>
              </a>
             </div>
             <div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-2-desc">
              <p>
               The analysed structures are newer than any structure in the training set. Further filtering is applied to reduce redundancy (see
               <a data-track="click" data-track-action="section anchor" data-track-label="link" href="/articles/s41586-021-03819-2#Sec10">
                Methods
               </a>
               ).
               <b>
                a
               </b>
               , Histogram of backbone r.m.s.d. for full chains (Cα r.m.s.d. at 95% coverage). Error bars are 95% confidence intervals (Poisson). This dataset excludes proteins with a template (identified by hmmsearch) from the training set with more than 40% sequence identity covering more than 1% of the chain (
               <i>
                n
               </i>
               = 3,144 protein chains). The overall median is 1.46 Å (95% confidence interval = 1.40–1.56 Å). Note that this measure will be highly sensitive to domain packing and domain accuracy; a high r.m.s.d. is expected for some chains with uncertain packing or packing errors.
               <b>
                b
               </b>
               , Correlation between backbone accuracy and side-chain accuracy. Filtered to structures with any observed side chains and resolution better than 2.5 Å (
               <i>
                n
               </i>
               = 5,317 protein chains); side chains were further filtered to
               <i>
                B
               </i>
               -factor &lt;30 Å
               <sup>
                2
               </sup>
               . A rotamer is classified as correct if the predicted torsion angle is within 40°. Each point aggregates a range of lDDT-Cα, with a bin size of 2 units above 70 lDDT-Cα and 5 units otherwise. Points correspond to the mean accuracy; error bars are 95% confidence intervals (Student
               <i>
                t
               </i>
               -test) of the mean on a per-residue basis.
               <b>
                c
               </b>
               , Confidence score compared to the true accuracy on chains. Least-squares linear fit lDDT-Cα = 0.997 × pLDDT − 1.17 (Pearson’s
               <i>
                r
               </i>
               = 0.76).
               <i>
                n
               </i>
               = 10,795 protein chains. The shaded region of the linear fit represents a 95% confidence interval estimated from 10,000 bootstrap samples. In the companion paper
               <sup>
                <a aria-label="Reference 39" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR39" id="ref-link-section-d262236594e1012" title="Tunyasuvunakool, K. et al. Highly accurate protein structure prediction for the human proteome. Nature 
                  https://doi-org.proxy.lib.ohio-state.edu/10.1038/s41586-021-03828-1
                  
                 (2021).">
                 39
                </a>
               </sup>
               , additional quantification of the reliability of pLDDT as a confidence measure is provided.
               <b>
                d
               </b>
               , Correlation between pTM and full chain TM-score. Least-squares linear fit TM-score = 0.98 × pTM + 0.07 (Pearson’s
               <i>
                r
               </i>
               = 0.85).
               <i>
                n
               </i>
               = 10,795 protein chains. The shaded region of the linear fit represents a 95% confidence interval estimated from 10,000 bootstrap samples.
              </p>
             </div>
            </div>
            <div class="u-text-right u-hide-print">
             <a aria-label="Full size image figure 2" class="c-article__pill-button" data-test="article-link" data-track="click" data-track-action="view figure" data-track-dest="link:Figure2 Full size image" data-track-label="button" href="/articles/s41586-021-03819-2/figures/2" rel="nofollow">
              <span>
               Full size image
              </span>
              <svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16">
               <use xlink:href="#icon-chevron-right" xmlns:xlink="http://www.w3.org/1999/xlink">
               </use>
              </svg>
             </a>
            </div>
           </figure>
          </div>
         </div>
        </div>
       </section>
       <section data-title="The AlphaFold network">
        <div class="c-article-section" id="Sec2-section">
         <h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec2">
          The AlphaFold network
         </h2>
         <div class="c-article-section__content" id="Sec2-content">
          <p>
           AlphaFold greatly improves the accuracy of structure prediction by incorporating novel neural network architectures and training procedures based on the evolutionary, physical and geometric constraints of protein structures. In particular, we demonstrate a new architecture to jointly embed multiple sequence alignments (MSAs) and pairwise features, a new output representation and associated loss that enable accurate end-to-end structure prediction, a new equivariant attention architecture, use of intermediate losses to achieve iterative refinement of predictions, masked MSA loss to jointly train with the structure, learning from unlabelled protein sequences using self-distillation and self-estimates of accuracy.
          </p>
          <p>
           The AlphaFold network directly predicts the 3D coordinates of all heavy atoms for a given protein using the primary amino acid sequence and aligned sequences of homologues as inputs (Fig.
           <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41586-021-03819-2#Fig1">
            1e
           </a>
           ; see
           <a data-track="click" data-track-action="section anchor" data-track-label="link" href="/articles/s41586-021-03819-2#Sec10">
            Methods
           </a>
           for details of inputs including databases, MSA construction and use of templates). A description of the most important ideas and components is provided below. The full network architecture and training procedure are provided in the
           <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41586-021-03819-2#MOESM1">
            Supplementary Methods
           </a>
           .
          </p>
          <p>
           The network comprises two main stages. First, the trunk of the network processes the inputs through repeated layers of a novel neural network block that we term Evoformer to produce an
           <i>
            N
           </i>
           <sub>
            seq
           </sub>
           ×
           <i>
            N
           </i>
           <sub>
            res
           </sub>
           array (
           <i>
            N
           </i>
           <sub>
            seq
           </sub>
           , number of sequences;
           <i>
            N
           </i>
           <sub>
            res
           </sub>
           , number of residues) that represents a processed MSA and an
           <i>
            N
           </i>
           <sub>
            res
           </sub>
           ×
           <i>
            N
           </i>
           <sub>
            res
           </sub>
           array that represents residue pairs. The MSA representation is initialized with the raw MSA (although see
           <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41586-021-03819-2#MOESM1">
            Supplementary Methods 1.2.7
           </a>
           for details of handling very deep MSAs). The Evoformer blocks contain a number of attention-based and non-attention-based components. We show evidence in ‘Interpreting the neural network’ that a concrete structural hypothesis arises early within the Evoformer blocks and is continuously refined. The key innovations in the Evoformer block are new mechanisms to exchange information within the MSA and pair representations that enable direct reasoning about the spatial and evolutionary relationships.
          </p>
          <p>
           The trunk of the network is followed by the structure module that introduces an explicit 3D structure in the form of a rotation and translation for each residue of the protein (global rigid body frames). These representations are initialized in a trivial state with all rotations set to the identity and all positions set to the origin, but rapidly develop and refine a highly accurate protein structure with precise atomic details. Key innovations in this section of the network include breaking the chain structure to allow simultaneous local refinement of all parts of the structure, a novel equivariant transformer to allow the network to implicitly reason about the unrepresented side-chain atoms and a loss term that places substantial weight on the orientational correctness of the residues. Both within the structure module and throughout the whole network, we reinforce the notion of iterative refinement by repeatedly applying the final loss to outputs and then feeding the outputs recursively into the same modules. The iterative refinement using the whole network (which we term ‘recycling’ and is related to approaches in computer vision
           <sup>
            <a aria-label="Reference 28" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR28" id="ref-link-section-d262236594e1087" title="Tu, Z. &amp; Bai, X. Auto-context and its application to high-level vision tasks and 3D brain image segmentation. IEEE Trans. Pattern Anal. Mach. Intell. 32, 1744–1757 (2010).">
             28
            </a>
            ,
            <a aria-label="Reference 29" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR29" id="ref-link-section-d262236594e1090" title="Carreira, J., Agrawal, P., Fragkiadaki, K. &amp; Malik, J. Human pose estimation with iterative error feedback. In Proc. IEEE Conference on Computer Vision and Pattern Recognition 4733–4742 (2016).">
             29
            </a>
           </sup>
           ) contributes markedly to accuracy with minor extra training time (see
           <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41586-021-03819-2#MOESM1">
            Supplementary Methods 1.8
           </a>
           for details).
          </p>
         </div>
        </div>
       </section>
       <section data-title="Evoformer">
        <div class="c-article-section" id="Sec3-section">
         <h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec3">
          Evoformer
         </h2>
         <div class="c-article-section__content" id="Sec3-content">
          <p>
           The key principle of the building block of the network—named Evoformer (Figs.
           <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41586-021-03819-2#Fig1">
            1
           </a>
           e,
           <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41586-021-03819-2#Fig3">
            3a
           </a>
           )—is to view the prediction of protein structures as a graph inference problem in 3D space in which the edges of the graph are defined by residues in proximity. The elements of the pair representation encode information about the relation between the residues (Fig.
           <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41586-021-03819-2#Fig3">
            3b
           </a>
           ). The columns of the MSA representation encode the individual residues of the input sequence while the rows represent the sequences in which those residues appear. Within this framework, we define a number of update operations that are applied in each block in which the different update operations are applied in series.
          </p>
          <div class="c-article-section__figure js-c-reading-companion-figures-item" data-container-section="figure" data-test="figure" data-title="Architectural details." id="figure-3">
           <figure>
            <figcaption>
             <b class="c-article-section__figure-caption" data-test="figure-caption-text" id="Fig3">
              Fig. 3: Architectural details.
             </b>
            </figcaption>
            <div class="c-article-section__figure-content">
             <div class="c-article-section__figure-item">
              <a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-action="view figure" data-track-label="image" href="/articles/s41586-021-03819-2/figures/3" rel="nofollow">
               <picture>
                <source srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-021-03819-2/MediaObjects/41586_2021_3819_Fig3_HTML.png?as=webp" type="image/webp"/>
                <img alt="figure 3" aria-describedby="Fig3" height="513" loading="lazy" src="//media-springernature-com.proxy.lib.ohio-state.edu/lw685/springer-static/image/art%3A10.1038%2Fs41586-021-03819-2/MediaObjects/41586_2021_3819_Fig3_HTML.png" width="685"/>
               </picture>
              </a>
             </div>
             <div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-3-desc">
              <p>
               <b>
                a
               </b>
               , Evoformer block. Arrows show the information flow. The shape of the arrays is shown in parentheses.
               <b>
                b
               </b>
               , The pair representation interpreted as directed edges in a graph.
               <b>
                c
               </b>
               , Triangle multiplicative update and triangle self-attention. The circles represent residues. Entries in the pair representation are illustrated as directed edges and in each diagram, the edge being updated is
               <i>
                ij
               </i>
               .
               <b>
                d
               </b>
               , Structure module including Invariant point attention (IPA) module. The single representation is a copy of the first row of the MSA representation.
               <b>
                e
               </b>
               , Residue gas: a representation of each residue as one free-floating rigid body for the backbone (blue triangles) and
               <i>
                χ
               </i>
               angles for the side chains (green circles). The corresponding atomic structure is shown below.
               <b>
                f
               </b>
               , Frame aligned point error (FAPE). Green, predicted structure; grey, true structure; (
               <i>
                R
               </i>
               <sub>
                <i>
                 k
                </i>
                ,
               </sub>
               <b>
                t
               </b>
               <sub>
                <i>
                 k
                </i>
               </sub>
               ), frames;
               <b>
                x
               </b>
               <sub>
                i
               </sub>
               , atom positions.
              </p>
             </div>
            </div>
            <div class="u-text-right u-hide-print">
             <a aria-label="Full size image figure 3" class="c-article__pill-button" data-test="article-link" data-track="click" data-track-action="view figure" data-track-dest="link:Figure3 Full size image" data-track-label="button" href="/articles/s41586-021-03819-2/figures/3" rel="nofollow">
              <span>
               Full size image
              </span>
              <svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16">
               <use xlink:href="#icon-chevron-right" xmlns:xlink="http://www.w3.org/1999/xlink">
               </use>
              </svg>
             </a>
            </div>
           </figure>
          </div>
          <p>
           The MSA representation updates the pair representation through an element-wise outer product that is summed over the MSA sequence dimension. In contrast to previous work
           <sup>
            <a aria-label="Reference 30" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR30" id="ref-link-section-d262236594e1179" title="Mirabello, C. &amp; Wallner, B. rawMSA: end-to-end deep learning using raw multiple sequence alignments. PLoS ONE 14, e0220182 (2019).">
             30
            </a>
           </sup>
           , this operation is applied within every block rather than once in the network, which enables the continuous communication from the evolving MSA representation to the pair representation.
          </p>
          <p>
           Within the pair representation, there are two different update patterns. Both are inspired by the necessity of consistency of the pair representation—for a pairwise description of amino acids to be representable as a single 3D structure, many constraints must be satisfied including the triangle inequality on distances. On the basis of this intuition, we arrange the update operations on the pair representation in terms of triangles of edges involving three different nodes (Fig.
           <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41586-021-03819-2#Fig3">
            3c
           </a>
           ). In particular, we add an extra logit bias to axial attention
           <sup>
            <a aria-label="Reference 31" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR31" id="ref-link-section-d262236594e1189" title="Huang, Z. et al. CCNet: criss-cross attention for semantic segmentation. In Proc. IEEE/CVF International Conference on Computer Vision 603–612 (2019).">
             31
            </a>
           </sup>
           to include the ‘missing edge’ of the triangle and we define a non-attention update operation ‘triangle multiplicative update’ that uses two edges to update the missing third edge (see
           <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41586-021-03819-2#MOESM1">
            Supplementary Methods 1.6.5
           </a>
           for details). The triangle multiplicative update was developed originally as a more symmetric and cheaper replacement for the attention, and networks that use only the attention or multiplicative update are both able to produce high-accuracy structures. However, the combination of the two updates is more accurate.
          </p>
          <p>
           We also use a variant of axial attention within the MSA representation. During the per-sequence attention in the MSA, we project additional logits from the pair stack to bias the MSA attention. This closes the loop by providing information flow from the pair representation back into the MSA representation, ensuring that the overall Evoformer block is able to fully mix information between the pair and MSA representations and prepare for structure generation within the structure module.
          </p>
         </div>
        </div>
       </section>
       <section data-title="End-to-end structure prediction">
        <div class="c-article-section" id="Sec4-section">
         <h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec4">
          End-to-end structure prediction
         </h2>
         <div class="c-article-section__content" id="Sec4-content">
          <p>
           The structure module (Fig.
           <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41586-021-03819-2#Fig3">
            3d
           </a>
           ) operates on a concrete 3D backbone structure using the pair representation and the original sequence row (single representation) of the MSA representation from the trunk. The 3D backbone structure is represented as
           <i>
            N
           </i>
           <sub>
            res
           </sub>
           independent rotations and translations, each with respect to the global frame (residue gas) (Fig.
           <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41586-021-03819-2#Fig3">
            3e
           </a>
           ). These rotations and translations—representing the geometry of the N-Cα-C atoms—prioritize the orientation of the protein backbone so that the location of the side chain of each residue is highly constrained within that frame. Conversely, the peptide bond geometry is completely unconstrained and the network is observed to frequently violate the chain constraint during the application of the structure module as breaking this constraint enables the local refinement of all parts of the chain without solving complex loop closure problems. Satisfaction of the peptide bond geometry is encouraged during fine-tuning by a violation loss term. Exact enforcement of peptide bond geometry is only achieved in the post-prediction relaxation of the structure by gradient descent in the Amber
           <sup>
            <a aria-label="Reference 32" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR32" id="ref-link-section-d262236594e1217" title="Hornak, V. et al. Comparison of multiple Amber force fields and development of improved protein backbone parameters. Proteins 65, 712–725 (2006).">
             32
            </a>
           </sup>
           force field. Empirically, this final relaxation does not improve the accuracy of the model as measured by the global distance test (GDT)
           <sup>
            <a aria-label="Reference 33" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR33" id="ref-link-section-d262236594e1221" title="Zemla, A. LGA: a method for finding 3D similarities in protein structures. Nucleic Acids Res. 31, 3370–3374 (2003).">
             33
            </a>
           </sup>
           or lDDT-Cα
           <sup>
            <a aria-label="Reference 34" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR34" id="ref-link-section-d262236594e1226" title="Mariani, V., Biasini, M., Barbato, A. &amp; Schwede, T. lDDT: a local superposition-free score for comparing protein structures and models using distance difference tests. Bioinformatics 29, 2722–2728 (2013).">
             34
            </a>
           </sup>
           but does remove distracting stereochemical violations without the loss of accuracy.
          </p>
          <p>
           The residue gas representation is updated iteratively in two stages (Fig.
           <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41586-021-03819-2#Fig3">
            3d
           </a>
           ). First, a geometry-aware attention operation that we term ‘invariant point attention’ (IPA) is used to update an
           <i>
            N
           </i>
           <sub>
            res
           </sub>
           set of neural activations (single representation) without changing the 3D positions, then an equivariant update operation is performed on the residue gas using the updated activations. The IPA augments each of the usual attention queries, keys and values with 3D points that are produced in the local frame of each residue such that the final value is invariant to global rotations and translations (see
           <a data-track="click" data-track-action="section anchor" data-track-label="link" href="/articles/s41586-021-03819-2#Sec10">
            Methods
           </a>
           ‘IPA’ for details). The 3D queries and keys also impose a strong spatial/locality bias on the attention, which is well-suited to the iterative refinement of the protein structure. After each attention operation and element-wise transition block, the module computes an update to the rotation and translation of each backbone frame. The application of these updates within the local frame of each residue makes the overall attention and update block an equivariant operation on the residue gas.
          </p>
          <p>
           Predictions of side-chain
           <i>
            χ
           </i>
           angles as well as the final, per-residue accuracy of the structure (pLDDT) are computed with small per-residue networks on the final activations at the end of the network. The estimate of the TM-score (pTM) is obtained from a pairwise error prediction that is computed as a linear projection from the final pair representation. The final loss (which we term the frame-aligned point error (FAPE) (Fig.
           <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41586-021-03819-2#Fig3">
            3f
           </a>
           )) compares the predicted atom positions to the true positions under many different alignments. For each alignment, defined by aligning the predicted frame (
           <i>
            R
           </i>
           <sub>
            <i>
             k
            </i>
           </sub>
           ,
           <b>
            t
           </b>
           <sub>
            <i>
             k
            </i>
           </sub>
           ) to the corresponding true frame, we compute the distance of all predicted atom positions
           <b>
            x
           </b>
           <sub>
            <i>
             i
            </i>
           </sub>
           from the true atom positions. The resulting
           <i>
            N
           </i>
           <sub>
            frames
           </sub>
           ×
           <i>
            N
           </i>
           <sub>
            atoms
           </sub>
           distances are penalized with a clamped
           <i>
            L
           </i>
           <sup>
            1
           </sup>
           loss. This creates a strong bias for atoms to be correct relative to the local frame of each residue and hence correct with respect to its side-chain interactions, as well as providing the main source of chirality for AlphaFold (
           <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41586-021-03819-2#MOESM1">
            Supplementary Methods 1.9.3
           </a>
           and Supplementary Fig.
           <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41586-021-03819-2#MOESM1">
            9
           </a>
           ).
          </p>
         </div>
        </div>
       </section>
       <section data-title="Training with labelled and unlabelled data">
        <div class="c-article-section" id="Sec5-section">
         <h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec5">
          Training with labelled and unlabelled data
         </h2>
         <div class="c-article-section__content" id="Sec5-content">
          <p>
           The AlphaFold architecture is able to train to high accuracy using only supervised learning on PDB data, but we are able to enhance accuracy (Fig.
           <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41586-021-03819-2#Fig4">
            4a
           </a>
           ) using an approach similar to noisy student self-distillation
           <sup>
            <a aria-label="Reference 35" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR35" id="ref-link-section-d262236594e1301" title="Xie, Q., Luong, M.-T., Hovy, E. &amp; Le, Q. V. Self-training with noisy student improves imagenet classification. In Proc. IEEE/CVF Conference on Computer Vision and Pattern Recognition 10687–10698 (2020).">
             35
            </a>
           </sup>
           . In this procedure, we use a trained network to predict the structure of around 350,000 diverse sequences from Uniclust30
           <sup>
            <a aria-label="Reference 36" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR36" id="ref-link-section-d262236594e1305" title="Mirdita, M. et al. Uniclust databases of clustered and deeply annotated protein sequences and alignments. Nucleic Acids Res. 45, D170–D176 (2017).">
             36
            </a>
           </sup>
           and make a new dataset of predicted structures filtered to a high-confidence subset. We then train the same architecture again from scratch using a mixture of PDB data and this new dataset of predicted structures as the training data, in which the various training data augmentations such as cropping and MSA subsampling make it challenging for the network to recapitulate the previously predicted structures. This self-distillation procedure makes effective use of the unlabelled sequence data and considerably improves the accuracy of the resulting network.
          </p>
          <div class="c-article-section__figure js-c-reading-companion-figures-item" data-container-section="figure" data-test="figure" data-title="Interpreting the neural network." id="figure-4">
           <figure>
            <figcaption>
             <b class="c-article-section__figure-caption" data-test="figure-caption-text" id="Fig4">
              Fig. 4: Interpreting the neural network.
             </b>
            </figcaption>
            <div class="c-article-section__figure-content">
             <div class="c-article-section__figure-item">
              <a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-action="view figure" data-track-label="image" href="/articles/s41586-021-03819-2/figures/4" rel="nofollow">
               <picture>
                <source srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-021-03819-2/MediaObjects/41586_2021_3819_Fig4_HTML.png?as=webp" type="image/webp"/>
                <img alt="figure 4" aria-describedby="Fig4" height="897" loading="lazy" src="//media-springernature-com.proxy.lib.ohio-state.edu/lw685/springer-static/image/art%3A10.1038%2Fs41586-021-03819-2/MediaObjects/41586_2021_3819_Fig4_HTML.png" width="685"/>
               </picture>
              </a>
             </div>
             <div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-4-desc">
              <p>
               <b>
                a
               </b>
               , Ablation results on two target sets: the CASP14 set of domains (
               <i>
                n
               </i>
               = 87 protein domains) and the PDB test set of chains with template coverage of ≤30% at 30% identity (
               <i>
                n
               </i>
               = 2,261 protein chains). Domains are scored with GDT and chains are scored with lDDT-Cα. The ablations are reported as a difference compared with the average of the three baseline seeds. Means (points) and 95% bootstrap percentile intervals (error bars) are computed using bootstrap estimates of 10,000 samples.
               <b>
                b
               </b>
               , Domain GDT trajectory over 4 recycling iterations and 48 Evoformer blocks on CASP14 targets LmrP (T1024) and Orf8 (T1064) where D1 and D2 refer to the individual domains as defined by the CASP assessment. Both T1024 domains obtain the correct structure early in the network, whereas the structure of T1064 changes multiple times and requires nearly the full depth of the network to reach the final structure. Note, 48 Evoformer blocks comprise one recycling iteration.
              </p>
             </div>
            </div>
            <div class="u-text-right u-hide-print">
             <a aria-label="Full size image figure 4" class="c-article__pill-button" data-test="article-link" data-track="click" data-track-action="view figure" data-track-dest="link:Figure4 Full size image" data-track-label="button" href="/articles/s41586-021-03819-2/figures/4" rel="nofollow">
              <span>
               Full size image
              </span>
              <svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16">
               <use xlink:href="#icon-chevron-right" xmlns:xlink="http://www.w3.org/1999/xlink">
               </use>
              </svg>
             </a>
            </div>
           </figure>
          </div>
          <p>
           Additionally, we randomly mask out or mutate individual residues within the MSA and have a Bidirectional Encoder Representations from Transformers (BERT)-style
           <sup>
            <a aria-label="Reference 37" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR37" id="ref-link-section-d262236594e1343" title="Devlin, J., Chang, M.-W., Lee, K. &amp; Toutanova, K. BERT: pre-training of deep bidirectional transformers for language understanding. In Proc. 2019 Conference of the North American Chapter of the Association for Computational Linguistics: Human Language Technologies 1, 4171–4186 (2019).">
             37
            </a>
           </sup>
           objective to predict the masked elements of the MSA sequences. This objective encourages the network to learn to interpret phylogenetic and covariation relationships without hardcoding a particular correlation statistic into the features. The BERT objective is trained jointly with the normal PDB structure loss on the same training examples and is not pre-trained, in contrast to recent independent work
           <sup>
            <a aria-label="Reference 38" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR38" id="ref-link-section-d262236594e1347" title="Rao, R. et al. MSA transformer. In Proc. 38th International Conference on Machine Learning PMLR 139, 8844–8856 (2021).">
             38
            </a>
           </sup>
           .
          </p>
         </div>
        </div>
       </section>
       <section data-title="Interpreting the neural network">
        <div class="c-article-section" id="Sec6-section">
         <h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec6">
          Interpreting the neural network
         </h2>
         <div class="c-article-section__content" id="Sec6-content">
          <p>
           To understand how AlphaFold predicts protein structure, we trained a separate structure module for each of the 48 Evoformer blocks in the network while keeping all parameters of the main network frozen (
           <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41586-021-03819-2#MOESM1">
            Supplementary Methods 1.14
           </a>
           ). Including our recycling stages, this provides a trajectory of 192 intermediate structures—one per full Evoformer block—in which each intermediate represents the belief of the network of the most likely structure at that block. The resulting trajectories are surprisingly smooth after the first few blocks, showing that AlphaFold makes constant incremental improvements to the structure until it can no longer improve (see Fig.
           <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41586-021-03819-2#Fig4">
            4b
           </a>
           for a trajectory of accuracy). These trajectories also illustrate the role of network depth. For very challenging proteins such as ORF8 of SARS-CoV-2 (T1064), the network searches and rearranges secondary structure elements for many layers before settling on a good structure. For other proteins such as LmrP (T1024), the network finds the final structure within the first few layers. Structure trajectories of CASP14 targets T1024, T1044, T1064 and T1091 that demonstrate a clear iterative building process for a range of protein sizes and difficulties are shown in Supplementary Videos
           <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41586-021-03819-2#MOESM3">
            1
           </a>
           –
           <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41586-021-03819-2#MOESM6">
            4
           </a>
           . In
           <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41586-021-03819-2#MOESM1">
            Supplementary Methods 1.16
           </a>
           and Supplementary Figs.
           <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41586-021-03819-2#MOESM1">
            12
           </a>
           ,
           <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41586-021-03819-2#MOESM1">
            13
           </a>
           , we interpret the attention maps produced by AlphaFold layers.
          </p>
          <p>
           Figure
           <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41586-021-03819-2#Fig4">
            4a
           </a>
           contains detailed ablations of the components of AlphaFold that demonstrate that a variety of different mechanisms contribute to AlphaFold accuracy. Detailed descriptions of each ablation model, their training details, extended discussion of ablation results and the effect of MSA depth on each ablation are provided in
           <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41586-021-03819-2#MOESM1">
            Supplementary Methods 1.13
           </a>
           and Supplementary Fig.
           <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41586-021-03819-2#MOESM1">
            10
           </a>
           .
          </p>
         </div>
        </div>
       </section>
       <section data-title="MSA depth and cross-chain contacts">
        <div class="c-article-section" id="Sec7-section">
         <h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec7">
          MSA depth and cross-chain contacts
         </h2>
         <div class="c-article-section__content" id="Sec7-content">
          <p>
           Although AlphaFold has a high accuracy across the vast majority of deposited PDB structures, we note that there are still factors that affect accuracy or limit the applicability of the model. The model uses MSAs and the accuracy decreases substantially when the median alignment depth is less than around 30 sequences (see Fig.
           <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41586-021-03819-2#Fig5">
            5a
           </a>
           for details). We observe a threshold effect where improvements in MSA depth over around 100 sequences lead to small gains. We hypothesize that the MSA information is needed to coarsely find the correct structure within the early stages of the network, but refinement of that prediction into a high-accuracy model does not depend crucially on the MSA information. The other substantial limitation that we have observed is that AlphaFold is much weaker for proteins that have few intra-chain or homotypic contacts compared to the number of heterotypic contacts (further details are provided in a companion paper
           <sup>
            <a aria-label="Reference 39" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR39" id="ref-link-section-d262236594e1405" title="Tunyasuvunakool, K. et al. Highly accurate protein structure prediction for the human proteome. Nature 
                  https://doi-org.proxy.lib.ohio-state.edu/10.1038/s41586-021-03828-1
                  
                 (2021).">
             39
            </a>
           </sup>
           ). This typically occurs for bridging domains within larger complexes in which the shape of the protein is created almost entirely by interactions with other chains in the complex. Conversely, AlphaFold is often able to give high-accuracy predictions for homomers, even when the chains are substantially intertwined (Fig.
           <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41586-021-03819-2#Fig5">
            5b
           </a>
           ). We expect that the ideas of AlphaFold are readily applicable to predicting full hetero-complexes in a future system and that this will remove the difficulty with protein chains that have a large number of hetero-contacts.
          </p>
          <div class="c-article-section__figure js-c-reading-companion-figures-item" data-container-section="figure" data-test="figure" data-title="Effect of MSA depth and cross-chain contacts." id="figure-5">
           <figure>
            <figcaption>
             <b class="c-article-section__figure-caption" data-test="figure-caption-text" id="Fig5">
              Fig. 5: Effect of MSA depth and cross-chain contacts.
             </b>
            </figcaption>
            <div class="c-article-section__figure-content">
             <div class="c-article-section__figure-item">
              <a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-action="view figure" data-track-label="image" href="/articles/s41586-021-03819-2/figures/5" rel="nofollow">
               <picture>
                <source srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-021-03819-2/MediaObjects/41586_2021_3819_Fig5_HTML.png?as=webp" type="image/webp"/>
                <img alt="figure 5" aria-describedby="Fig5" height="243" loading="lazy" src="//media-springernature-com.proxy.lib.ohio-state.edu/lw685/springer-static/image/art%3A10.1038%2Fs41586-021-03819-2/MediaObjects/41586_2021_3819_Fig5_HTML.png" width="685"/>
               </picture>
              </a>
             </div>
             <div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-5-desc">
              <p>
               <b>
                a
               </b>
               , Backbone accuracy (lDDT-Cα) for the redundancy-reduced set of the PDB after our training data cut-off, restricting to proteins in which at most 25% of the long-range contacts are between different heteromer chains. We further consider two groups of proteins based on template coverage at 30% sequence identity: covering more than 60% of the chain (
               <i>
                n
               </i>
               = 6,743 protein chains) and covering less than 30% of the chain (
               <i>
                n
               </i>
               = 1,596 protein chains). MSA depth is computed by counting the number of non-gap residues for each position in the MSA (using the
               <i>
                N
               </i>
               <sub>
                eff
               </sub>
               weighting scheme; see
               <a data-track="click" data-track-action="section anchor" data-track-label="link" href="/articles/s41586-021-03819-2#Sec10">
                Methods
               </a>
               for details) and taking the median across residues. The curves are obtained through Gaussian kernel average smoothing (window size is 0.2 units in log
               <sub>
                10
               </sub>
               (
               <i>
                N
               </i>
               <sub>
                eff
               </sub>
               )); the shaded area is the 95% confidence interval estimated using bootstrap of 10,000 samples.
               <b>
                b
               </b>
               , An intertwined homotrimer (PDB 6SK0) is correctly predicted without input stoichiometry and only a weak template (blue is predicted and green is experimental).
              </p>
             </div>
            </div>
            <div class="u-text-right u-hide-print">
             <a aria-label="Full size image figure 5" class="c-article__pill-button" data-test="article-link" data-track="click" data-track-action="view figure" data-track-dest="link:Figure5 Full size image" data-track-label="button" href="/articles/s41586-021-03819-2/figures/5" rel="nofollow">
              <span>
               Full size image
              </span>
              <svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16">
               <use xlink:href="#icon-chevron-right" xmlns:xlink="http://www.w3.org/1999/xlink">
               </use>
              </svg>
             </a>
            </div>
           </figure>
          </div>
         </div>
        </div>
       </section>
       <section data-title="Related work">
        <div class="c-article-section" id="Sec8-section">
         <h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec8">
          Related work
         </h2>
         <div class="c-article-section__content" id="Sec8-content">
          <p>
           The prediction of protein structures has had a long and varied development, which is extensively covered in a number of reviews
           <sup>
            <a aria-label="Reference 14" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR14" id="ref-link-section-d262236594e1465" title="Pearce, R. &amp; Zhang, Y. Deep learning techniques have significantly impacted protein structure prediction and protein design. Curr. Opin. Struct. Biol. 68, 194–207 (2021).">
             14
            </a>
            ,
            <a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR40" id="ref-link-section-d262236594e1468" title="Kuhlman, B. &amp; Bradley, P. Advances in protein structure prediction and design. Nat. Rev. Mol. Cell Biol. 20, 681–697 (2019).">
             40
            </a>
            ,
            <a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR41" id="ref-link-section-d262236594e1468_1" title="Marks, D. S., Hopf, T. A. &amp; Sander, C. Protein structure prediction from sequence variation. Nat. Biotechnol. 30, 1072–1080 (2012).">
             41
            </a>
            ,
            <a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR42" id="ref-link-section-d262236594e1468_2" title="Qian, N. &amp; Sejnowski, T. J. Predicting the secondary structure of globular proteins using neural network models. J. Mol. Biol. 202, 865–884 (1988).">
             42
            </a>
            ,
            <a aria-label="Reference 43" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR43" id="ref-link-section-d262236594e1471" title="Fariselli, P., Olmea, O., Valencia, A. &amp; Casadio, R. Prediction of contact maps with neural networks and correlated mutations. Protein Eng. 14, 835–843 (2001).">
             43
            </a>
           </sup>
           . Despite the long history of applying neural networks to structure prediction
           <sup>
            <a aria-label="Reference 14" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR14" id="ref-link-section-d262236594e1475" title="Pearce, R. &amp; Zhang, Y. Deep learning techniques have significantly impacted protein structure prediction and protein design. Curr. Opin. Struct. Biol. 68, 194–207 (2021).">
             14
            </a>
            ,
            <a aria-label="Reference 42" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR42" id="ref-link-section-d262236594e1478" title="Qian, N. &amp; Sejnowski, T. J. Predicting the secondary structure of globular proteins using neural network models. J. Mol. Biol. 202, 865–884 (1988).">
             42
            </a>
            ,
            <a aria-label="Reference 43" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR43" id="ref-link-section-d262236594e1481" title="Fariselli, P., Olmea, O., Valencia, A. &amp; Casadio, R. Prediction of contact maps with neural networks and correlated mutations. Protein Eng. 14, 835–843 (2001).">
             43
            </a>
           </sup>
           , they have only recently come to improve structure prediction
           <sup>
            <a aria-label="Reference 10" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR10" id="ref-link-section-d262236594e1485" title="Senior, A. W. et al. Improved protein structure prediction using potentials from deep learning. Nature 577, 706–710 (2020).">
             10
            </a>
            ,
            <a aria-label="Reference 11" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR11" id="ref-link-section-d262236594e1488" title="Wang, S., Sun, S., Li, Z., Zhang, R. &amp; Xu, J. Accurate de novo prediction of protein contact map by ultra-deep learning model. PLOS Comput. Biol. 13, e1005324 (2017).">
             11
            </a>
            ,
            <a aria-label="Reference 44" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR44" id="ref-link-section-d262236594e1491" title="Yang, J. et al. Improved protein structure prediction using predicted interresidue orientations. Proc. Natl Acad. Sci. USA 117, 1496–1503 (2020).">
             44
            </a>
            ,
            <a aria-label="Reference 45" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR45" id="ref-link-section-d262236594e1494" title="Li, Y. et al. Deducing high-accuracy protein contact-maps from a triplet of coevolutionary matrices through deep residual convolutional networks. PLOS Comput. Biol. 17, e1008865 (2021).">
             45
            </a>
           </sup>
           . These approaches effectively leverage the rapid improvement in computer vision systems
           <sup>
            <a aria-label="Reference 46" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR46" id="ref-link-section-d262236594e1498" title="He, K., Zhang, X., Ren, S. &amp; Sun, J. Deep residual learning for image recognition. In Proc. IEEE Conference on Computer Vision and Pattern Recognition 770–778 (2016).">
             46
            </a>
           </sup>
           by treating the problem of protein structure prediction as converting an ‘image’ of evolutionary couplings
           <sup>
            <a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR22" id="ref-link-section-d262236594e1502" title="Weigt, M., White, R. A., Szurmant, H., Hoch, J. A. &amp; Hwa, T. Identification of direct residue contacts in protein–protein interaction by message passing. Proc. Natl Acad. Sci. USA 106, 67–72 (2009).">
             22
            </a>
            ,
            <a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR23" id="ref-link-section-d262236594e1502_1" title="Marks, D. S. et al. Protein 3D structure computed from evolutionary sequence variation. PLoS ONE 6, e28766 (2011).">
             23
            </a>
            ,
            <a aria-label="Reference 24" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR24" id="ref-link-section-d262236594e1505" title="Jones, D. T., Buchan, D. W. A., Cozzetto, D. &amp; Pontil, M. PSICOV: precise structural contact prediction using sparse inverse covariance estimation on large multiple sequence alignments. Bioinformatics 28, 184–190 (2012).">
             24
            </a>
           </sup>
           to an ‘image’ of the protein distance matrix and then integrating the distance predictions into a heuristic system that produces the final 3D coordinate prediction. A few recent studies have been developed to predict the 3D coordinates directly
           <sup>
            <a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR47" id="ref-link-section-d262236594e1510" title="AlQuraishi, M. End-to-end differentiable learning of protein structure. Cell Syst. 8, 292–301 (2019).">
             47
            </a>
            ,
            <a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR48" id="ref-link-section-d262236594e1510_1" title="Senior, A. W. et al. Protein structure prediction using multiple deep neural networks in the 13th Critical Assessment of Protein Structure Prediction (CASP13). Proteins 87, 1141–1148 (2019).">
             48
            </a>
            ,
            <a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR49" id="ref-link-section-d262236594e1510_2" title="Ingraham, J., Riesselman, A. J., Sander, C. &amp; Marks, D. S. Learning protein structure with a differentiable simulator. in Proc. International Conference on Learning Representations (2019).">
             49
            </a>
            ,
            <a aria-label="Reference 50" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR50" id="ref-link-section-d262236594e1513" title="Li, J. Universal transforming geometric network. Preprint at 
                  https://arxiv-org.proxy.lib.ohio-state.edu/abs/1908.00723
                  
                 (2019).">
             50
            </a>
           </sup>
           , but the accuracy of these approaches does not match traditional, hand-crafted structure prediction pipelines
           <sup>
            <a aria-label="Reference 51" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR51" id="ref-link-section-d262236594e1517" title="Xu, J., McPartlon, M. &amp; Li, J. Improved protein structure prediction by deep learning irrespective of co-evolution information. Nat. Mach. Intell. 3, 601–609 (2021).">
             51
            </a>
           </sup>
           . In parallel, the success of attention-based networks for language processing
           <sup>
            <a aria-label="Reference 52" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR52" id="ref-link-section-d262236594e1521" title="Vaswani, A. et al. Attention is all you need. In Advances in Neural Information Processing Systems 5998–6008 (2017).">
             52
            </a>
           </sup>
           and, more recently, computer vision
           <sup>
            <a aria-label="Reference 31" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR31" id="ref-link-section-d262236594e1525" title="Huang, Z. et al. CCNet: criss-cross attention for semantic segmentation. In Proc. IEEE/CVF International Conference on Computer Vision 603–612 (2019).">
             31
            </a>
            ,
            <a aria-label="Reference 53" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR53" id="ref-link-section-d262236594e1528" title="Wang, H. et al. Axial-deeplab: stand-alone axial-attention for panoptic segmentation. in European Conference on Computer Vision 108–126 (Springer, 2020).">
             53
            </a>
           </sup>
           has inspired the exploration of attention-based methods for interpreting protein sequences
           <sup>
            <a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR54" id="ref-link-section-d262236594e1532" title="Alley, E. C., Khimulya, G., Biswas, S., AlQuraishi, M. &amp; Church, G. M. Unified rational protein engineering with sequence-based deep representation learning. Nat. Methods 16, 1315–1322 (2019).">
             54
            </a>
            ,
            <a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR55" id="ref-link-section-d262236594e1532_1" title="Heinzinger, M. et al. Modeling aspects of the language of life through transfer-learning protein sequences. BMC Bioinformatics 20, 723 (2019).">
             55
            </a>
            ,
            <a aria-label="Reference 56" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR56" id="ref-link-section-d262236594e1535" title="Rives, A. et al. Biological structure and function emerge from scaling unsupervised learning to 250 million protein sequences. Proc. Natl Acad. Sci. USA 118, e2016239118 (2021).">
             56
            </a>
           </sup>
           .
          </p>
         </div>
        </div>
       </section>
       <section data-title="Discussion">
        <div class="c-article-section" id="Sec9-section">
         <h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec9">
          Discussion
         </h2>
         <div class="c-article-section__content" id="Sec9-content">
          <p>
           The methodology that we have taken in designing AlphaFold is a combination of the bioinformatics and physical approaches: we use a physical and geometric inductive bias to build components that learn from PDB data with minimal imposition of handcrafted features (for example, AlphaFold builds hydrogen bonds effectively without a hydrogen bond score function). This results in a network that learns far more efficiently from the limited data in the PDB but is able to cope with the complexity and variety of structural data.
          </p>
          <p>
           In particular, AlphaFold is able to handle missing the physical context and produce accurate models in challenging cases such as intertwined homomers or proteins that only fold in the presence of an unknown haem group. The ability to handle underspecified structural conditions is essential to learning from PDB structures as the PDB represents the full range of conditions in which structures have been solved. In general, AlphaFold is trained to produce the protein structure most likely to appear as part of a PDB structure. For example, in cases in which a particular stochiometry, ligand or ion is predictable from the sequence alone, AlphaFold is likely to produce a structure that respects those constraints implicitly.
          </p>
          <p>
           AlphaFold has already demonstrated its utility to the experimental community, both for molecular replacement
           <sup>
            <a aria-label="Reference 57" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR57" id="ref-link-section-d262236594e1553" title="Pereira, J. et al. High-accuracy protein structure prediction in CASP14. Proteins 
                  https://doi-org.proxy.lib.ohio-state.edu/10.1002/prot.26171
                  
                 (2021).">
             57
            </a>
           </sup>
           and for interpreting cryogenic electron microscopy maps
           <sup>
            <a aria-label="Reference 58" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR58" id="ref-link-section-d262236594e1557" title="Gupta, M. et al. CryoEM and AI reveal a structure of SARS-CoV-2 Nsp2, a multifunctional protein involved in key host processes. Preprint at 
                  https://doi-org.proxy.lib.ohio-state.edu/10.1101/2021.05.10.443524
                  
                 (2021).">
             58
            </a>
           </sup>
           . Moreover, because AlphaFold outputs protein coordinates directly, AlphaFold produces predictions in graphics processing unit (GPU) minutes to GPU hours depending on the length of the protein sequence (for example, around one GPU minute per model for 384 residues; see
           <a data-track="click" data-track-action="section anchor" data-track-label="link" href="/articles/s41586-021-03819-2#Sec10">
            Methods
           </a>
           for details). This opens up the exciting possibility of predicting structures at the proteome-scale and beyond—in a companion paper
           <sup>
            <a aria-label="Reference 39" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR39" id="ref-link-section-d262236594e1564" title="Tunyasuvunakool, K. et al. Highly accurate protein structure prediction for the human proteome. Nature 
                  https://doi-org.proxy.lib.ohio-state.edu/10.1038/s41586-021-03828-1
                  
                 (2021).">
             39
            </a>
           </sup>
           , we demonstrate the application of AlphaFold to the entire human proteome
           <sup>
            <a aria-label="Reference 39" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR39" id="ref-link-section-d262236594e1568" title="Tunyasuvunakool, K. et al. Highly accurate protein structure prediction for the human proteome. Nature 
                  https://doi-org.proxy.lib.ohio-state.edu/10.1038/s41586-021-03828-1
                  
                 (2021).">
             39
            </a>
           </sup>
           .
          </p>
          <p>
           The explosion in available genomic sequencing techniques and data has revolutionized bioinformatics but the intrinsic challenge of experimental structure determination has prevented a similar expansion in our structural knowledge. By developing an accurate protein structure prediction algorithm, coupled with existing large and well-curated structure and sequence databases assembled by the experimental community, we hope to accelerate the advancement of structural bioinformatics that can keep pace with the genomics revolution. We hope that AlphaFold—and computational approaches that apply its techniques for other biophysical problems—will become essential tools of modern biology.
          </p>
         </div>
        </div>
       </section>
       <section data-title="Methods">
        <div class="c-article-section" id="Sec10-section">
         <h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec10">
          Methods
         </h2>
         <div class="c-article-section__content" id="Sec10-content">
          <h3 class="c-article__sub-heading" id="Sec11">
           Full algorithm details
          </h3>
          <p>
           Extensive explanations of the components and their motivations are available in
           <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41586-021-03819-2#MOESM1">
            Supplementary Methods 1.1–1.10
           </a>
           , in addition, pseudocode is available in
           <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41586-021-03819-2#MOESM1">
            Supplementary Information Algorithms 1–32
           </a>
           , network diagrams in Supplementary Figs.
           <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41586-021-03819-2#MOESM1">
            1
           </a>
           –
           <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41586-021-03819-2#MOESM1">
            8
           </a>
           , input features in Supplementary Table
           <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41586-021-03819-2#MOESM1">
            1
           </a>
           and additional details are provided in Supplementary Tables
           <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41586-021-03819-2#MOESM1">
            2
           </a>
           ,
           <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41586-021-03819-2#MOESM1">
            3
           </a>
           . Training and inference details are provided in
           <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41586-021-03819-2#MOESM1">
            Supplementary Methods 1.11–1.12
           </a>
           and Supplementary Tables
           <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41586-021-03819-2#MOESM1">
            4
           </a>
           ,
           <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41586-021-03819-2#MOESM1">
            5
           </a>
           .
          </p>
          <h3 class="c-article__sub-heading c-article__sub-heading--divider" id="Sec12">
           IPA
          </h3>
          <p>
           The IPA module combines the pair representation, the single representation and the geometric representation to update the single representation (Supplementary Fig.
           <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41586-021-03819-2#MOESM1">
            8
           </a>
           ). Each of these representations contributes affinities to the shared attention weights and then uses these weights to map its values to the output. The IPA operates in 3D space. Each residue produces query points, key points and value points in its local frame. These points are projected into the global frame using the backbone frame of the residue in which they interact with each other. The resulting points are then projected back into the local frame. The affinity computation in the 3D space uses squared distances and the coordinate transformations ensure the invariance of this module with respect to the global frame (see
           <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41586-021-03819-2#MOESM1">
            Supplementary Methods 1.8.2
           </a>
           ‘Invariant point attention (IPA)’ for the algorithm, proof of invariance and a description of the full multi-head version). A related construction that uses classic geometric invariants to construct pairwise features in place of the learned 3D points has been applied to protein design
           <sup>
            <a aria-label="Reference 59" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR59" id="ref-link-section-d262236594e1632" title="Ingraham, J., Garg, V. K., Barzilay, R. &amp; Jaakkola, T. Generative models for graph-based protein design. in Proc. 33rd Conference on Neural Information Processing Systems (2019).">
             59
            </a>
           </sup>
           .
          </p>
          <p>
           In addition to the IPA, standard dot product attention is computed on the abstract single representation and a special attention on the pair representation. The pair representation augments both the logits and the values of the attention process, which is the primary way in which the pair representation controls the structure generation.
          </p>
          <h3 class="c-article__sub-heading c-article__sub-heading--divider" id="Sec13">
           Inputs and data sources
          </h3>
          <p>
           Inputs to the network are the primary sequence, sequences from evolutionarily related proteins in the form of a MSA created by standard tools including jackhmmer
           <sup>
            <a aria-label="Reference 60" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR60" id="ref-link-section-d262236594e1647" title="Johnson, L. S., Eddy, S. R. &amp; Portugaly, E. Hidden Markov model speed heuristic and iterative HMM search procedure. BMC Bioinformatics 11, 431 (2010).">
             60
            </a>
           </sup>
           and HHBlits
           <sup>
            <a aria-label="Reference 61" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR61" id="ref-link-section-d262236594e1651" title="Remmert, M., Biegert, A., Hauser, A. &amp; Söding, J. HHblits: lightning-fast iterative protein sequence searching by HMM-HMM alignment. Nat. Methods 9, 173–175 (2012).">
             61
            </a>
           </sup>
           , and 3D atom coordinates of a small number of homologous structures (templates) where available. For both the MSA and templates, the search processes are tuned for high recall; spurious matches will probably appear in the raw MSA but this matches the training condition of the network.
          </p>
          <p>
           One of the sequence databases used, Big Fantastic Database (BFD), was custom-made and released publicly (see ‘Data availability’) and was used by several CASP teams. BFD is one of the largest publicly available collections of protein families. It consists of 65,983,866 families represented as MSAs and hidden Markov models (HMMs) covering 2,204,359,010 protein sequences from reference databases, metagenomes and metatranscriptomes.
          </p>
          <p>
           BFD was built in three steps. First, 2,423,213,294 protein sequences were collected from UniProt (Swiss-Prot&amp;TrEMBL, 2017-11)
           <sup>
            <a aria-label="Reference 62" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR62" id="ref-link-section-d262236594e1661" title="The UniProt Consortium. UniProt: the universal protein knowledgebase in 2021. Nucleic Acids Res. 49, D480–D489 (2020).">
             62
            </a>
           </sup>
           , a soil reference protein catalogue and the marine eukaryotic reference catalogue
           <sup>
            <a aria-label="Reference 7" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR7" id="ref-link-section-d262236594e1665" title="Steinegger, M., Mirdita, M. &amp; Söding, J. Protein-level assembly increases protein sequence recovery from metagenomic samples manyfold. Nat. Methods 16, 603–606 (2019).">
             7
            </a>
           </sup>
           , and clustered to 30% sequence identity, while enforcing a 90% alignment coverage of the shorter sequences using MMseqs2/Linclust
           <sup>
            <a aria-label="Reference 63" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR63" id="ref-link-section-d262236594e1669" title="Steinegger, M. &amp; Söding, J. Clustering huge protein sequence sets in linear time. Nat. Commun. 9, 2542 (2018).">
             63
            </a>
           </sup>
           . This resulted in 345,159,030 clusters. For computational efficiency, we removed all clusters with less than three members, resulting in 61,083,719 clusters. Second, we added 166,510,624 representative protein sequences from Metaclust NR (2017-05; discarding all sequences shorter than 150 residues)
           <sup>
            <a aria-label="Reference 63" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR63" id="ref-link-section-d262236594e1673" title="Steinegger, M. &amp; Söding, J. Clustering huge protein sequence sets in linear time. Nat. Commun. 9, 2542 (2018).">
             63
            </a>
           </sup>
           by aligning them against the cluster representatives using MMseqs2
           <sup>
            <a aria-label="Reference 64" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR64" id="ref-link-section-d262236594e1677" title="Steinegger, M. &amp; Söding, J. MMseqs2 enables sensitive protein sequence searching for the analysis of massive data sets. Nat. Biotechnol. 35, 1026–1028 (2017).">
             64
            </a>
           </sup>
           . Sequences that fulfilled the sequence identity and coverage criteria were assigned to the best scoring cluster. The remaining 25,347,429 sequences that could not be assigned were clustered separately and added as new clusters, resulting in the final clustering. Third, for each of the clusters, we computed an MSA using FAMSA
           <sup>
            <a aria-label="Reference 65" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR65" id="ref-link-section-d262236594e1682" title="Deorowicz, S., Debudaj-Grabysz, A. &amp; Gudyś, A. FAMSA: fast and accurate multiple sequence alignment of huge protein families. Sci. Rep. 6, 33964 (2016).">
             65
            </a>
           </sup>
           and computed the HMMs following the Uniclust HH-suite database protocol
           <sup>
            <a aria-label="Reference 36" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR36" id="ref-link-section-d262236594e1686" title="Mirdita, M. et al. Uniclust databases of clustered and deeply annotated protein sequences and alignments. Nucleic Acids Res. 45, D170–D176 (2017).">
             36
            </a>
           </sup>
           .
          </p>
          <p>
           The following versions of public datasets were used in this study. Our models were trained on a copy of the PDB
           <sup>
            <a aria-label="Reference 5" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR5" id="ref-link-section-d262236594e1693" title="wwPDB Consortium. Protein Data Bank: the single global archive for 3D macromolecular structure data. Nucleic Acids Res. 47, D520–D528 (2018).">
             5
            </a>
           </sup>
           downloaded on 28 August 2019. For finding template structures at prediction time, we used a copy of the PDB downloaded on 14 May 2020, and the PDB70
           <sup>
            <a aria-label="Reference 66" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR66" id="ref-link-section-d262236594e1697" title="Steinegger, M. et al. HH-suite3 for fast remote homology detection and deep protein annotation. BMC Bioinformatics 20, 473 (2019).">
             66
            </a>
           </sup>
           clustering database downloaded on 13 May 2020. For MSA lookup at both training and prediction time, we used Uniref90
           <sup>
            <a aria-label="Reference 67" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR67" id="ref-link-section-d262236594e1701" title="Suzek, B. E., Wang, Y., Huang, H., McGarvey, P. B. &amp; Wu, C. H. UniRef clusters: a comprehensive and scalable alternative for improving sequence similarity searches. Bioinformatics 31, 926–932 (2015).">
             67
            </a>
           </sup>
           v.2020_01, BFD, Uniclust30
           <sup>
            <a aria-label="Reference 36" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR36" id="ref-link-section-d262236594e1705" title="Mirdita, M. et al. Uniclust databases of clustered and deeply annotated protein sequences and alignments. Nucleic Acids Res. 45, D170–D176 (2017).">
             36
            </a>
           </sup>
           v.2018_08 and MGnify
           <sup>
            <a aria-label="Reference 6" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR6" id="ref-link-section-d262236594e1709" title="Mitchell, A. L. et al. MGnify: the microbiome analysis resource in 2020. Nucleic Acids Res. 48, D570–D578 (2020).">
             6
            </a>
           </sup>
           v.2018_12. For sequence distillation, we used Uniclust30
           <sup>
            <a aria-label="Reference 36" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR36" id="ref-link-section-d262236594e1714" title="Mirdita, M. et al. Uniclust databases of clustered and deeply annotated protein sequences and alignments. Nucleic Acids Res. 45, D170–D176 (2017).">
             36
            </a>
           </sup>
           v.2018_08 to construct a distillation structure dataset. Full details are provided in
           <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41586-021-03819-2#MOESM1">
            Supplementary Methods 1.2
           </a>
           .
          </p>
          <p>
           For MSA search on BFD + Uniclust30, and template search against PDB70, we used HHBlits
           <sup>
            <a aria-label="Reference 61" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR61" id="ref-link-section-d262236594e1725" title="Remmert, M., Biegert, A., Hauser, A. &amp; Söding, J. HHblits: lightning-fast iterative protein sequence searching by HMM-HMM alignment. Nat. Methods 9, 173–175 (2012).">
             61
            </a>
           </sup>
           and HHSearch
           <sup>
            <a aria-label="Reference 66" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR66" id="ref-link-section-d262236594e1729" title="Steinegger, M. et al. HH-suite3 for fast remote homology detection and deep protein annotation. BMC Bioinformatics 20, 473 (2019).">
             66
            </a>
           </sup>
           from hh-suite v.3.0-beta.3 (version 14/07/2017). For MSA search on Uniref90 and clustered MGnify, we used jackhmmer from HMMER3
           <sup>
            <a aria-label="Reference 68" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR68" id="ref-link-section-d262236594e1733" title="Eddy, S. R. Accelerated profile HMM searches. PLOS Comput. Biol. 7, e1002195 (2011).">
             68
            </a>
           </sup>
           . For constrained relaxation of structures, we used OpenMM v.7.3.1
           <sup>
            <a aria-label="Reference 69" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR69" id="ref-link-section-d262236594e1737" title="Eastman, P. et al. OpenMM 7: rapid development of high performance algorithms for molecular dynamics. PLOS Comput. Biol. 13, e1005659 (2017).">
             69
            </a>
           </sup>
           with the Amber99sb force field
           <sup>
            <a aria-label="Reference 32" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR32" id="ref-link-section-d262236594e1741" title="Hornak, V. et al. Comparison of multiple Amber force fields and development of improved protein backbone parameters. Proteins 65, 712–725 (2006).">
             32
            </a>
           </sup>
           . For neural network construction, running and other analyses, we used TensorFlow
           <sup>
            <a aria-label="Reference 70" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR70" id="ref-link-section-d262236594e1746" title="Ashish, A. M. A. et al. TensorFlow: large-scale machine learning on heterogeneous systems. Preprint at 
                  https://arxiv-org.proxy.lib.ohio-state.edu/abs/1603.04467
                  
                 (2015).">
             70
            </a>
           </sup>
           , Sonnet
           <sup>
            <a aria-label="Reference 71" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR71" id="ref-link-section-d262236594e1750" title="Reynolds, M. et al. Open sourcing Sonnet – a new library for constructing neural networks. DeepMind 
                  https://deepmind.com/blog/open-sourcing-sonnet/
                  
                 (7 April 2017).">
             71
            </a>
           </sup>
           , NumPy
           <sup>
            <a aria-label="Reference 72" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR72" id="ref-link-section-d262236594e1754" title="Harris, C. R. et al. Array programming with NumPy. Nature 585, 357–362 (2020).">
             72
            </a>
           </sup>
           , Python
           <sup>
            <a aria-label="Reference 73" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR73" id="ref-link-section-d262236594e1758" title="Van Rossum, G. &amp; Drake, F. L. Python 3 Reference Manual (CreateSpace, 2009).">
             73
            </a>
           </sup>
           and Colab
           <sup>
            <a aria-label="Reference 74" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR74" id="ref-link-section-d262236594e1762" title="Bisong, E. in Building Machine Learning and Deep Learning Models on Google Cloud Platform: A Comprehensive Guide for Beginners 59–64 (Apress, 2019).">
             74
            </a>
           </sup>
           .
          </p>
          <p>
           To quantify the effect of the different sequence data sources, we re-ran the CASP14 proteins using the same models but varying how the MSA was constructed. Removing BFD reduced the mean accuracy by 0.4 GDT, removing Mgnify reduced the mean accuracy by 0.7 GDT, and removing both reduced the mean accuracy by 6.1 GDT. In each case, we found that most targets had very small changes in accuracy but a few outliers had very large (20+ GDT) differences. This is consistent with the results in Fig.
           <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41586-021-03819-2#Fig5">
            5a
           </a>
           in which the depth of the MSA is relatively unimportant until it approaches a threshold value of around 30 sequences when the MSA size effects become quite large. We observe mostly overlapping effects between inclusion of BFD and Mgnify, but having at least one of these metagenomics databases is very important for target classes that are poorly represented in UniRef, and having both was necessary to achieve full CASP accuracy.
          </p>
          <h3 class="c-article__sub-heading c-article__sub-heading--divider" id="Sec14">
           Training regimen
          </h3>
          <p>
           To train, we use structures from the PDB with a maximum release date of 30 April 2018. Chains are sampled in inverse proportion to cluster size of a 40% sequence identity clustering. We then randomly crop them to 256 residues and assemble into batches of size 128. We train the model on Tensor Processing Unit (TPU) v3 with a batch size of 1 per TPU core, hence the model uses 128 TPU v3 cores. The model is trained until convergence (around 10 million samples) and further fine-tuned using longer crops of 384 residues, larger MSA stack and reduced learning rate (see
           <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41586-021-03819-2#MOESM1">
            Supplementary Methods 1.11
           </a>
           for the exact configuration). The initial training stage takes approximately 1 week, and the fine-tuning stage takes approximately 4 additional days.
          </p>
          <p>
           The network is supervised by the FAPE loss and a number of auxiliary losses. First, the final pair representation is linearly projected to a binned distance distribution (distogram) prediction, scored with a cross-entropy loss. Second, we use random masking on the input MSAs and require the network to reconstruct the masked regions from the output MSA representation using a BERT-like loss
           <sup>
            <a aria-label="Reference 37" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR37" id="ref-link-section-d262236594e1786" title="Devlin, J., Chang, M.-W., Lee, K. &amp; Toutanova, K. BERT: pre-training of deep bidirectional transformers for language understanding. In Proc. 2019 Conference of the North American Chapter of the Association for Computational Linguistics: Human Language Technologies 1, 4171–4186 (2019).">
             37
            </a>
           </sup>
           . Third, the output single representations of the structure module are used to predict binned per-residue lDDT-Cα values. Finally, we use an auxiliary side-chain loss during training, and an auxiliary structure violation loss during fine-tuning. Detailed descriptions and weighting are provided in the
           <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41586-021-03819-2#MOESM1">
            Supplementary Information
           </a>
           .
          </p>
          <p>
           An initial model trained with the above objectives was used to make structure predictions for a Uniclust dataset of 355,993 sequences with the full MSAs. These predictions were then used to train a final model with identical hyperparameters, except for sampling examples 75% of the time from the Uniclust prediction set, with sub-sampled MSAs, and 25% of the time from the clustered PDB set.
          </p>
          <p>
           We train five different models using different random seeds, some with templates and some without, to encourage diversity in the predictions (see Supplementary Table
           <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41586-021-03819-2#MOESM1">
            5
           </a>
           and
           <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41586-021-03819-2#MOESM1">
            Supplementary Methods 1.12.1
           </a>
           for details). We also fine-tuned these models after CASP14 to add a pTM prediction objective (
           <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41586-021-03819-2#MOESM1">
            Supplementary Methods 1.9.7
           </a>
           ) and use the obtained models for Fig.
           <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41586-021-03819-2#Fig2">
            2d
           </a>
           .
          </p>
          <h3 class="c-article__sub-heading c-article__sub-heading--divider" id="Sec15">
           Inference regimen
          </h3>
          <p>
           We inference the five trained models and use the predicted confidence score to select the best model per target.
          </p>
          <p>
           Using our CASP14 configuration for AlphaFold, the trunk of the network is run multiple times with different random choices for the MSA cluster centres (see
           <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41586-021-03819-2#MOESM1">
            Supplementary Methods 1.11.2
           </a>
           for details of the ensembling procedure). The full time to make a structure prediction varies considerably depending on the length of the protein. Representative timings for the neural network using a single model on V100 GPU are 4.8 min with 256 residues, 9.2 min with 384 residues and 18 h at 2,500 residues. These timings are measured using our open-source code, and the open-source code is notably faster than the version we ran in CASP14 as we now use the XLA compiler
           <sup>
            <a aria-label="Reference 75" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR75" id="ref-link-section-d262236594e1826" title="TensorFlow. XLA: Optimizing Compiler for TensorFlow. 
                  https://www.tensorflow.org/xla
                  
                 (2018).">
             75
            </a>
           </sup>
           .
          </p>
          <p>
           Since CASP14, we have found that the accuracy of the network without ensembling is very close or equal to the accuracy with ensembling and we turn off ensembling for most inference. Without ensembling, the network is 8× faster and the representative timings for a single model are 0.6 min with 256 residues, 1.1 min with 384 residues and 2.1 h with 2,500 residues.
          </p>
          <p>
           Inferencing large proteins can easily exceed the memory of a single GPU. For a V100 with 16 GB of memory, we can predict the structure of proteins up to around 1,300 residues without ensembling and the 256- and 384-residue inference times are using the memory of a single GPU. The memory usage is approximately quadratic in the number of residues, so a 2,500-residue protein involves using unified memory so that we can greatly exceed the memory of a single V100. In our cloud setup, a single V100 is used for computation on a 2,500-residue protein but we requested four GPUs to have sufficient memory.
          </p>
          <p>
           Searching genetic sequence databases to prepare inputs and final relaxation of the structures take additional central processing unit (CPU) time but do not require a GPU or TPU.
          </p>
          <h3 class="c-article__sub-heading c-article__sub-heading--divider" id="Sec16">
           Metrics
          </h3>
          <p>
           The predicted structure is compared to the true structure from the PDB in terms of lDDT metric
           <sup>
            <a aria-label="Reference 34" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR34" id="ref-link-section-d262236594e1848" title="Mariani, V., Biasini, M., Barbato, A. &amp; Schwede, T. lDDT: a local superposition-free score for comparing protein structures and models using distance difference tests. Bioinformatics 29, 2722–2728 (2013).">
             34
            </a>
           </sup>
           , as this metric reports the domain accuracy without requiring a domain segmentation of chain structures. The distances are either computed between all heavy atoms (lDDT) or only the Cα atoms to measure the backbone accuracy (lDDT-Cα). As lDDT-Cα only focuses on the Cα atoms, it does not include the penalty for structural violations and clashes. Domain accuracies in CASP are reported as GDT
           <sup>
            <a aria-label="Reference 33" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR33" id="ref-link-section-d262236594e1852" title="Zemla, A. LGA: a method for finding 3D similarities in protein structures. Nucleic Acids Res. 31, 3370–3374 (2003).">
             33
            </a>
           </sup>
           and the TM-score
           <sup>
            <a aria-label="Reference 27" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR27" id="ref-link-section-d262236594e1856" title="Zhang, Y. &amp; Skolnick, J. Scoring function for automated assessment of protein structure template quality. Proteins 57, 702–710 (2004).">
             27
            </a>
           </sup>
           is used as a full chain global superposition metric.
          </p>
          <p>
           We also report accuracies using the r.m.s.d.
           <sub>
            95
           </sub>
           (Cα r.m.s.d. at 95% coverage). We perform five iterations of (1) a least-squares alignment of the predicted structure and the PDB structure on the currently chosen Cα atoms (using all Cα atoms in the first iteration); (2) selecting the 95% of Cα atoms with the lowest alignment error. The r.m.s.d. of the atoms chosen for the final iterations is the r.m.s.d.
           <sub>
            95
           </sub>
           . This metric is more robust to apparent errors that can originate from crystal structure artefacts, although in some cases the removed 5% of residues will contain genuine modelling errors.
          </p>
          <h3 class="c-article__sub-heading c-article__sub-heading--divider" id="Sec17">
           Test set of recent PDB sequences
          </h3>
          <p>
           For evaluation on recent PDB sequences (Figs.
           <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41586-021-03819-2#Fig2">
            2
           </a>
           a–d,
           <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41586-021-03819-2#Fig4">
            4
           </a>
           a,
           <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41586-021-03819-2#Fig5">
            5a
           </a>
           ), we used a copy of the PDB downloaded 15 February 2021. Structures were filtered to those with a release date after 30 April 2018 (the date limit for inclusion in the training set for AlphaFold). Chains were further filtered to remove sequences that consisted of a single amino acid as well as sequences with an ambiguous chemical component at any residue position. Exact duplicates were removed, with the chain with the most resolved Cα atoms used as the representative sequence. Subsequently, structures with less than 16 resolved residues, with unknown residues or solved by NMR methods were removed. As the PDB contains many near-duplicate sequences, the chain with the highest resolution was selected from each cluster in the PDB 40% sequence clustering of the data. Furthermore, we removed all sequences for which fewer than 80 amino acids had the alpha carbon resolved and removed chains with more than 1,400 residues. The final dataset contained 10,795 protein sequences.
          </p>
          <p>
           The procedure for filtering the recent PDB dataset based on prior template identity was as follows. Hmmsearch was run with default parameters against a copy of the PDB SEQRES fasta downloaded 15 February 2021. Template hits were accepted if the associated structure had a release date earlier than 30 April 2018. Each residue position in a query sequence was assigned the maximum identity of any template hit covering that position. Filtering then proceeded as described in the individual figure legends, based on a combination of maximum identity and sequence coverage.
          </p>
          <p>
           The MSA depth analysis was based on computing the normalized number of effective sequences (
           <i>
            N
           </i>
           <sub>
            eff
           </sub>
           ) for each position of a query sequence. Per-residue
           <i>
            N
           </i>
           <sub>
            eff
           </sub>
           values were obtained by counting the number of non-gap residues in the MSA for this position and weighting the sequences using the
           <i>
            N
           </i>
           <sub>
            eff
           </sub>
           scheme
           <sup>
            <a aria-label="Reference 76" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR76" id="ref-link-section-d262236594e1902" title="Wu, T., Hou, J., Adhikari, B. &amp; Cheng, J. Analysis of several key factors influencing deep learning-based inter-residue contact prediction. Bioinformatics 36, 1091–1098 (2020).">
             76
            </a>
           </sup>
           with a threshold of 80% sequence identity measured on the region that is non-gap in either sequence.
          </p>
          <h3 class="c-article__sub-heading c-article__sub-heading--divider" id="Sec18">
           Reporting summary
          </h3>
          <p>
           Further information on research design is available in the
           <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41586-021-03819-2#MOESM2">
            Nature Research Reporting Summary
           </a>
           linked to this paper.
          </p>
         </div>
        </div>
       </section>
      </div>
      <div>
       <section data-title="Data availability">
        <div class="c-article-section" id="data-availability-section">
         <h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="data-availability">
          Data availability
         </h2>
         <div class="c-article-section__content" id="data-availability-content">
          <p>
           All input data are freely available from public sources.
          </p>
          <p>
           Structures from the PDB were used for training and as templates (
           <a href="https://www.wwpdb.org/ftp/pdb-ftp-sites">
            https://www.wwpdb.org/ftp/pdb-ftp-sites
           </a>
           ; for the associated sequence data and 40% sequence clustering see also
           <a href="https://ftp.wwpdb.org/pub/pdb/derived_data/">
            https://ftp.wwpdb.org/pub/pdb/derived_data/
           </a>
           and
           <a href="https://cdn.rcsb.org/resources/sequence/clusters/bc-40.out">
            https://cdn.rcsb.org/resources/sequence/clusters/bc-40.out
           </a>
           ). Training used a version of the PDB downloaded 28 August 2019, while the CASP14 template search used a version downloaded 14 May 2020. The template search also used the PDB70 database, downloaded 13 May 2020 (
           <a href="https://wwwuser-gwdg-de.proxy.lib.ohio-state.edu/~compbiol/data/hhsuite/databases/hhsuite_dbs/">
            https://wwwuser-gwdg-de.proxy.lib.ohio-state.edu/~compbiol/data/hhsuite/databases/hhsuite_dbs/
           </a>
           ).
          </p>
          <p>
           We show experimental structures from the PDB with accession numbers
           <a href="http://doi-org.proxy.lib.ohio-state.edu/10.2210/pdb6Y4F/pdb">
            6Y4F
           </a>
           <sup>
            <a aria-label="Reference 77" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR77" id="ref-link-section-d262236594e2034" title="Jiang, W. et al. MrpH, a new class of metal-binding adhesin, requires zinc to mediate biofilm formation. PLoS Pathog. 16, e1008707 (2020).">
             77
            </a>
           </sup>
           ,
           <a href="http://doi-org.proxy.lib.ohio-state.edu/10.2210/pdb6YJ1/pdb">
            6YJ1
           </a>
           <sup>
            <a aria-label="Reference 78" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR78" id="ref-link-section-d262236594e2044" title="Dunne, M., Ernst, P., Sobieraj, A., Pluckthun, A. &amp; Loessner, M. J. The M23 peptidase domain of the Staphylococcal phage 2638A endolysin. PDB 
                  https://doi-org.proxy.lib.ohio-state.edu/10.2210/pdb6YJ1/pdb
                  
                 (2020).">
             78
            </a>
           </sup>
           ,
           <a href="http://doi-org.proxy.lib.ohio-state.edu/10.2210/pdb6VR4/pdb">
            6VR4
           </a>
           <sup>
            <a aria-label="Reference 79" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR79" id="ref-link-section-d262236594e2054" title="Drobysheva, A. V. et al. Structure and function of virion RNA polymerase of a crAss-like phage. Nature 589, 306–309 (2021).">
             79
            </a>
           </sup>
           ,
           <a href="http://doi-org.proxy.lib.ohio-state.edu/10.2210/pdb6SK0/pdb">
            6SK0
           </a>
           <sup>
            <a aria-label="Reference 80" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR80" id="ref-link-section-d262236594e2065" title="Flaugnatti, N. et al. Structural basis for loading and inhibition of a bacterial T6SS phospholipase effector by the VgrG spike. EMBO J. 39, e104129 (2020).">
             80
            </a>
           </sup>
           ,
           <a href="http://doi-org.proxy.lib.ohio-state.edu/10.2210/pdb6FES/pdb">
            6
            <span class="u-small-caps">
             FES
            </span>
           </a>
           <sup>
            <a aria-label="Reference 81" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR81" id="ref-link-section-d262236594e2078" title="ElGamacy, M. et al. An interface-driven design strategy yields a novel, corrugated protein architecture. ACS Synth. Biol. 7, 2226–2235 (2018).">
             81
            </a>
           </sup>
           ,
           <a href="http://doi-org.proxy.lib.ohio-state.edu/10.2210/pdb6W6W/pdb">
            6W6W
           </a>
           <sup>
            <a aria-label="Reference 82" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR82" id="ref-link-section-d262236594e2088" title="Lim, C. J. et al. The structure of human CST reveals a decameric assembly bound to telomeric DNA. Science 368, 1081–1085 (2020).">
             82
            </a>
           </sup>
           ,
           <a href="http://doi-org.proxy.lib.ohio-state.edu/10.2210/pdb6T1Z/pdb">
            6T1Z
           </a>
           <sup>
            <a aria-label="Reference 83" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR83" id="ref-link-section-d262236594e2098" title="Debruycker, V. et al. An embedded lipid in the multidrug transporter LmrP suggests a mechanism for polyspecificity. Nat. Struct. Mol. Biol. 27, 829–835 (2020).">
             83
            </a>
           </sup>
           and
           <a href="http://doi-org.proxy.lib.ohio-state.edu/10.2210/pdb7JTL/pdb">
            7JTL
           </a>
           <sup>
            <a aria-label="Reference 84" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41586-021-03819-2#ref-CR84" id="ref-link-section-d262236594e2109" title="Flower, T. G. et al. Structure of SARS-CoV-2 ORF8, a rapidly evolving immune evasion protein. Proc. Natl Acad. Sci. USA 118, e2021785118 (2021).">
             84
            </a>
           </sup>
           .
          </p>
          <p>
           For MSA lookup at both the training and prediction time, we used UniRef90 v.2020_01 (https://ftp.ebi.ac.uk/pub/databases/uniprot/previous_releases/release-2020_01/uniref/), BFD (
           <a href="https://bfd.mmseqs.com">
            https://bfd.mmseqs.com
           </a>
           ), Uniclust30 v.2018_08 (
           <a href="https://wwwuser-gwdg-de.proxy.lib.ohio-state.edu/~compbiol/uniclust/2018_08/">
            https://wwwuser-gwdg-de.proxy.lib.ohio-state.edu/~compbiol/uniclust/2018_08/
           </a>
           ) and MGnify clusters v.2018_12 (
           <a href="https://ftp.ebi.ac.uk/pub/databases/metagenomics/peptide_database/2018_12/">
            https://ftp.ebi.ac.uk/pub/databases/metagenomics/peptide_database/2018_12/
           </a>
           ). Uniclust30 v.2018_08 was also used as input for constructing a distillation structure dataset.
          </p>
         </div>
        </div>
       </section>
       <section data-title="Code availability">
        <div class="c-article-section" id="code-availability-section">
         <h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="code-availability">
          Code availability
         </h2>
         <div class="c-article-section__content" id="code-availability-content">
          <p>
           Source code for the AlphaFold model, trained weights and inference script are available under an open-source license at
           <a href="https://github.com/deepmind/alphafold">
            https://github.com/deepmind/alphafold
           </a>
           .
          </p>
          <p>
           Neural networks were developed with TensorFlow v.1 (
           <a href="https://github.com/tensorflow/tensorflow">
            https://github.com/tensorflow/tensorflow
           </a>
           ), Sonnet v.1 (
           <a href="https://github.com/deepmind/sonnet">
            https://github.com/deepmind/sonnet
           </a>
           ), JAX v.0.1.69 (
           <a href="https://github.com/google/jax/">
            https://github.com/google/jax/
           </a>
           ) and Haiku v.0.0.4 (
           <a href="https://github.com/deepmind/dm-haiku">
            https://github.com/deepmind/dm-haiku
           </a>
           ). The XLA compiler is bundled with JAX and does not have a separate version number.
          </p>
          <p>
           For MSA search on BFD+Uniclust30, and for template search against PDB70, we used HHBlits and HHSearch from hh-suite v.3.0-beta.3 release 14/07/2017 (
           <a href="https://github.com/soedinglab/hh-suite">
            https://github.com/soedinglab/hh-suite
           </a>
           ). For MSA search on UniRef90 and clustered MGnify, we used jackhmmer from HMMER v.3.3 (
           <a href="http://eddylab.org/software/hmmer/">
            http://eddylab.org/software/hmmer/
           </a>
           ). For constrained relaxation of structures, we used OpenMM v.7.3.1 (
           <a href="https://github.com/openmm/openmm">
            https://github.com/openmm/openmm
           </a>
           ) with the Amber99sb force field.
          </p>
          <p>
           Construction of BFD used MMseqs2 v.925AF (
           <a href="https://github.com/soedinglab/MMseqs2">
            https://github.com/soedinglab/MMseqs2
           </a>
           ) and FAMSA v.1.2.5 (
           <a href="https://github.com/refresh-bio/FAMSA">
            https://github.com/refresh-bio/FAMSA
           </a>
           ).
          </p>
          <p>
           Data analysis used Python v.3.6 (
           <a href="https://www.python.org/">
            https://www.python.org/
           </a>
           ), NumPy v.1.16.4 (
           <a href="https://github.com/numpy/numpy">
            https://github.com/numpy/numpy
           </a>
           ), SciPy v.1.2.1 (
           <a href="https://www.scipy.org/">
            https://www.scipy.org/
           </a>
           ), seaborn v.0.11.1 (
           <a href="https://github.com/mwaskom/seaborn">
            https://github.com/mwaskom/seaborn
           </a>
           ), Matplotlib v.3.3.4 (
           <a href="https://github.com/matplotlib/matplotlib">
            https://github.com/matplotlib/matplotlib
           </a>
           ), bokeh v.1.4.0 (
           <a href="https://github.com/bokeh/bokeh">
            https://github.com/bokeh/bokeh
           </a>
           ), pandas v.1.1.5 (
           <a href="https://github.com/pandas-dev/pandas">
            https://github.com/pandas-dev/pandas
           </a>
           ), plotnine v.0.8.0 (
           <a href="https://github.com/has2k1/plotnine">
            https://github.com/has2k1/plotnine
           </a>
           ), statsmodels v.0.12.2 (
           <a href="https://github.com/statsmodels/statsmodels">
            https://github.com/statsmodels/statsmodels
           </a>
           ) and Colab (
           <a href="https://research-google-com.proxy.lib.ohio-state.edu/colaboratory">
            https://research-google-com.proxy.lib.ohio-state.edu/colaboratory
           </a>
           ). TM-align v.20190822 (
           <a href="https://zhanglab-dcmb-med-umich-edu.proxy.lib.ohio-state.edu/TM-align/">
            https://zhanglab-dcmb-med-umich-edu.proxy.lib.ohio-state.edu/TM-align/
           </a>
           ) was used for computing TM-scores. Structure visualizations were created in Pymol v.2.3.0 (
           <a href="https://github.com/schrodinger/pymol-open-source">
            https://github.com/schrodinger/pymol-open-source
           </a>
           ).
          </p>
         </div>
        </div>
       </section>
       <div id="MagazineFulltextArticleBodySuffix">
        <section aria-labelledby="Bib1" data-title="References">
         <div class="c-article-section" id="Bib1-section">
          <h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Bib1">
           References
          </h2>
          <div class="c-article-section__content" id="Bib1-content">
           <div data-container-section="references">
            <ol class="c-article-references" data-track-component="outbound reference">
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="1.">
              <p class="c-article-references__text" id="ref-CR1">
               Thompson, M. C., Yeates, T. O. &amp; Rodriguez, J. A. Advances in methods for atomic resolution macromolecular structure determination.
               <i>
                F1000Res
               </i>
               .
               <b>
                9
               </b>
               , 667 (2020).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 1" data-doi="10.12688/f1000research.25097.1" data-track="click" data-track-action="article reference" data-track-label="10.12688/f1000research.25097.1" href="https://doi-org.proxy.lib.ohio-state.edu/10.12688%2Ff1000research.25097.1" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="CAS reference 1" data-track="click" data-track-action="cas reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXis1OgtrzK" rel="nofollow noopener">
                CAS
               </a>
               <a aria-label="Google Scholar reference 1" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Advances%20in%20methods%20for%20atomic%20resolution%20macromolecular%20structure%20determination&amp;journal=F1000Res.&amp;doi=10.12688%2Ff1000research.25097.1&amp;volume=9&amp;publication_year=2020&amp;author=Thompson%2CMC&amp;author=Yeates%2CTO&amp;author=Rodriguez%2CJA" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="2.">
              <p class="c-article-references__text" id="ref-CR2">
               Bai, X.-C., McMullan, G. &amp; Scheres, S. H. W. How cryo-EM is revolutionizing structural biology.
               <i>
                Trends Biochem. Sci
               </i>
               .
               <b>
                40
               </b>
               , 49–57 (2015).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 2" data-doi="10.1016/j.tibs.2014.10.005" data-track="click" data-track-action="article reference" data-track-label="10.1016/j.tibs.2014.10.005" href="https://doi-org.proxy.lib.ohio-state.edu/10.1016%2Fj.tibs.2014.10.005" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="CAS reference 2" data-track="click" data-track-action="cas reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BC2cXhvVCktLnK" rel="nofollow noopener">
                CAS
               </a>
               <a aria-label="PubMed reference 2" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25544475" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="Google Scholar reference 2" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=How%20cryo-EM%20is%20revolutionizing%20structural%20biology&amp;journal=Trends%20Biochem.%20Sci.&amp;doi=10.1016%2Fj.tibs.2014.10.005&amp;volume=40&amp;pages=49-57&amp;publication_year=2015&amp;author=Bai%2CX-C&amp;author=McMullan%2CG&amp;author=Scheres%2CSHW" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="3.">
              <p class="c-article-references__text" id="ref-CR3">
               Jaskolski, M., Dauter, Z. &amp; Wlodawer, A. A brief history of macromolecular crystallography, illustrated by a family tree and its Nobel fruits.
               <i>
                FEBS J
               </i>
               .
               <b>
                281
               </b>
               , 3985–4009 (2014).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 3" data-doi="10.1111/febs.12796" data-track="click" data-track-action="article reference" data-track-label="10.1111/febs.12796" href="https://doi-org.proxy.lib.ohio-state.edu/10.1111%2Ffebs.12796" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="CAS reference 3" data-track="click" data-track-action="cas reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BC2cXhsFKnsbnM" rel="nofollow noopener">
                CAS
               </a>
               <a aria-label="PubMed reference 3" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24698025" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="PubMed Central reference 3" data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC6309182" rel="nofollow noopener">
                PubMed Central
               </a>
               <a aria-label="Google Scholar reference 3" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20brief%20history%20of%20macromolecular%20crystallography%2C%20illustrated%20by%20a%20family%20tree%20and%20its%20Nobel%20fruits&amp;journal=FEBS%20J.&amp;doi=10.1111%2Ffebs.12796&amp;volume=281&amp;pages=3985-4009&amp;publication_year=2014&amp;author=Jaskolski%2CM&amp;author=Dauter%2CZ&amp;author=Wlodawer%2CA" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="4.">
              <p class="c-article-references__text" id="ref-CR4">
               Wüthrich, K. The way to NMR structures of proteins.
               <i>
                Nat. Struct. Biol
               </i>
               .
               <b>
                8
               </b>
               , 923–925 (2001).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 4" data-doi="10.1038/nsb1101-923" data-track="click" data-track-action="article reference" data-track-label="10.1038/nsb1101-923" href="https://doi-org.proxy.lib.ohio-state.edu/10.1038%2Fnsb1101-923" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="PubMed reference 4" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11685234" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="Google Scholar reference 4" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20way%20to%20NMR%20structures%20of%20proteins&amp;journal=Nat.%20Struct.%20Biol.&amp;doi=10.1038%2Fnsb1101-923&amp;volume=8&amp;pages=923-925&amp;publication_year=2001&amp;author=W%C3%BCthrich%2CK" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="5.">
              <p class="c-article-references__text" id="ref-CR5">
               wwPDB Consortium. Protein Data Bank: the single global archive for 3D macromolecular structure data.
               <i>
                Nucleic Acids Res
               </i>
               .
               <b>
                47
               </b>
               , D520–D528 (2018).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 5" data-doi="10.1093/nar/gky949" data-track="click" data-track-action="article reference" data-track-label="10.1093/nar/gky949" href="https://doi-org.proxy.lib.ohio-state.edu/10.1093%2Fnar%2Fgky949" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="Google Scholar reference 5" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Protein%20Data%20Bank%3A%20the%20single%20global%20archive%20for%203D%20macromolecular%20structure%20data&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgky949&amp;volume=47&amp;pages=D520-D528&amp;publication_year=2018" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="6.">
              <p class="c-article-references__text" id="ref-CR6">
               Mitchell, A. L. et al. MGnify: the microbiome analysis resource in 2020.
               <i>
                Nucleic Acids Res
               </i>
               .
               <b>
                48
               </b>
               , D570–D578 (2020).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="CAS reference 6" data-track="click" data-track-action="cas reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXhs1GltrjN" rel="nofollow noopener">
                CAS
               </a>
               <a aria-label="PubMed reference 6" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31696235" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="Google Scholar reference 6" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=MGnify%3A%20the%20microbiome%20analysis%20resource%20in%202020&amp;journal=Nucleic%20Acids%20Res.&amp;volume=48&amp;pages=D570-D578&amp;publication_year=2020&amp;author=Mitchell%2CAL" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="7.">
              <p class="c-article-references__text" id="ref-CR7">
               Steinegger, M., Mirdita, M. &amp; Söding, J. Protein-level assembly increases protein sequence recovery from metagenomic samples manyfold.
               <i>
                Nat. Methods
               </i>
               <b>
                16
               </b>
               , 603–606 (2019).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 7" data-doi="10.1038/s41592-019-0437-4" data-track="click" data-track-action="article reference" data-track-label="10.1038/s41592-019-0437-4" href="https://doi-org.proxy.lib.ohio-state.edu/10.1038%2Fs41592-019-0437-4" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="CAS reference 7" data-track="click" data-track-action="cas reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXht1eku7vJ" rel="nofollow noopener">
                CAS
               </a>
               <a aria-label="PubMed reference 7" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31235882" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="Google Scholar reference 7" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Protein-level%20assembly%20increases%20protein%20sequence%20recovery%20from%20metagenomic%20samples%20manyfold&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fs41592-019-0437-4&amp;volume=16&amp;pages=603-606&amp;publication_year=2019&amp;author=Steinegger%2CM&amp;author=Mirdita%2CM&amp;author=S%C3%B6ding%2CJ" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="8.">
              <p class="c-article-references__text" id="ref-CR8">
               Dill, K. A., Ozkan, S. B., Shell, M. S. &amp; Weikl, T. R. The protein folding problem.
               <i>
                Annu. Rev. Biophys
               </i>
               .
               <b>
                37
               </b>
               , 289–316 (2008).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 8" data-doi="10.1146/annurev.biophys.37.092707.153558" data-track="click" data-track-action="article reference" data-track-label="10.1146/annurev.biophys.37.092707.153558" href="https://doi-org.proxy.lib.ohio-state.edu/10.1146%2Fannurev.biophys.37.092707.153558" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="ADS reference 8" data-track="click" data-track-action="ads reference" data-track-label="link" href="http://adsabs.harvard.edu.proxy.lib.ohio-state.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=1988PhRvC..37..289D" rel="nofollow noopener">
                ADS
               </a>
               <a aria-label="CAS reference 8" data-track="click" data-track-action="cas reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BD1cXnsVGlurw%3D" rel="nofollow noopener">
                CAS
               </a>
               <a aria-label="PubMed reference 8" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18573083" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="PubMed Central reference 8" data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC2443096" rel="nofollow noopener">
                PubMed Central
               </a>
               <a aria-label="Google Scholar reference 8" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20protein%20folding%20problem&amp;journal=Annu.%20Rev.%20Biophys.&amp;doi=10.1146%2Fannurev.biophys.37.092707.153558&amp;volume=37&amp;pages=289-316&amp;publication_year=2008&amp;author=Dill%2CKA&amp;author=Ozkan%2CSB&amp;author=Shell%2CMS&amp;author=Weikl%2CTR" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="9.">
              <p class="c-article-references__text" id="ref-CR9">
               Anfinsen, C. B. Principles that govern the folding of protein chains.
               <i>
                Science
               </i>
               <b>
                181
               </b>
               , 223–230 (1973).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 9" data-doi="10.1126/science.181.4096.223" data-track="click" data-track-action="article reference" data-track-label="10.1126/science.181.4096.223" href="https://doi-org.proxy.lib.ohio-state.edu/10.1126%2Fscience.181.4096.223" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="ADS reference 9" data-track="click" data-track-action="ads reference" data-track-label="link" href="http://adsabs.harvard.edu.proxy.lib.ohio-state.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=1973Sci...181..223A" rel="nofollow noopener">
                ADS
               </a>
               <a aria-label="CAS reference 9" data-track="click" data-track-action="cas reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DyaE3sXkvVygtbc%3D" rel="nofollow noopener">
                CAS
               </a>
               <a aria-label="PubMed reference 9" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=4124164" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="Google Scholar reference 9" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Principles%20that%20govern%20the%20folding%20of%20protein%20chains&amp;journal=Science&amp;doi=10.1126%2Fscience.181.4096.223&amp;volume=181&amp;pages=223-230&amp;publication_year=1973&amp;author=Anfinsen%2CCB" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="10.">
              <p class="c-article-references__text" id="ref-CR10">
               Senior, A. W. et al. Improved protein structure prediction using potentials from deep learning.
               <i>
                Nature
               </i>
               <b>
                577
               </b>
               , 706–710 (2020).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 10" data-doi="10.1038/s41586-019-1923-7" data-track="click" data-track-action="article reference" data-track-label="10.1038/s41586-019-1923-7" href="https://doi-org.proxy.lib.ohio-state.edu/10.1038%2Fs41586-019-1923-7" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="ADS reference 10" data-track="click" data-track-action="ads reference" data-track-label="link" href="http://adsabs.harvard.edu.proxy.lib.ohio-state.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2020Natur.577..706S" rel="nofollow noopener">
                ADS
               </a>
               <a aria-label="CAS reference 10" data-track="click" data-track-action="cas reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXis1SisL0%3D" rel="nofollow noopener">
                CAS
               </a>
               <a aria-label="PubMed reference 10" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31942072" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="Google Scholar reference 10" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Improved%20protein%20structure%20prediction%20using%20potentials%20from%20deep%20learning&amp;journal=Nature&amp;doi=10.1038%2Fs41586-019-1923-7&amp;volume=577&amp;pages=706-710&amp;publication_year=2020&amp;author=Senior%2CAW" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="11.">
              <p class="c-article-references__text" id="ref-CR11">
               Wang, S., Sun, S., Li, Z., Zhang, R. &amp; Xu, J. Accurate de novo prediction of protein contact map by ultra-deep learning model.
               <i>
                PLOS Comput. Biol
               </i>
               .
               <b>
                13
               </b>
               , e1005324 (2017).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 11" data-doi="10.1371/journal.pcbi.1005324" data-track="click" data-track-action="article reference" data-track-label="10.1371/journal.pcbi.1005324" href="https://doi-org.proxy.lib.ohio-state.edu/10.1371%2Fjournal.pcbi.1005324" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="ADS reference 11" data-track="click" data-track-action="ads reference" data-track-label="link" href="http://adsabs.harvard.edu.proxy.lib.ohio-state.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2017PLSCB..13E5324W" rel="nofollow noopener">
                ADS
               </a>
               <a aria-label="PubMed reference 11" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28056090" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="PubMed Central reference 11" data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC5249242" rel="nofollow noopener">
                PubMed Central
               </a>
               <a aria-label="Google Scholar reference 11" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Accurate%20de%20novo%20prediction%20of%20protein%20contact%20map%20by%20ultra-deep%20learning%20model&amp;journal=PLOS%20Comput.%20Biol.&amp;doi=10.1371%2Fjournal.pcbi.1005324&amp;volume=13&amp;publication_year=2017&amp;author=Wang%2CS&amp;author=Sun%2CS&amp;author=Li%2CZ&amp;author=Zhang%2CR&amp;author=Xu%2CJ" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="12.">
              <p class="c-article-references__text" id="ref-CR12">
               Zheng, W. et al. Deep-learning contact-map guided protein structure prediction in CASP13.
               <i>
                Proteins
               </i>
               <b>
                87
               </b>
               , 1149–1164 (2019).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 12" data-doi="10.1002/prot.25792" data-track="click" data-track-action="article reference" data-track-label="10.1002/prot.25792" href="https://doi-org.proxy.lib.ohio-state.edu/10.1002%2Fprot.25792" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="CAS reference 12" data-track="click" data-track-action="cas reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXhsFKgsrnK" rel="nofollow noopener">
                CAS
               </a>
               <a aria-label="PubMed reference 12" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31365149" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="PubMed Central reference 12" data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC6851476" rel="nofollow noopener">
                PubMed Central
               </a>
               <a aria-label="Google Scholar reference 12" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Deep-learning%20contact-map%20guided%20protein%20structure%20prediction%20in%20CASP13&amp;journal=Proteins&amp;doi=10.1002%2Fprot.25792&amp;volume=87&amp;pages=1149-1164&amp;publication_year=2019&amp;author=Zheng%2CW" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="13.">
              <p class="c-article-references__text" id="ref-CR13">
               Abriata, L. A., Tamò, G. E. &amp; Dal Peraro, M. A further leap of improvement in tertiary structure prediction in CASP13 prompts new routes for future assessments.
               <i>
                Proteins
               </i>
               <b>
                87
               </b>
               , 1100–1112 (2019).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 13" data-doi="10.1002/prot.25787" data-track="click" data-track-action="article reference" data-track-label="10.1002/prot.25787" href="https://doi-org.proxy.lib.ohio-state.edu/10.1002%2Fprot.25787" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="CAS reference 13" data-track="click" data-track-action="cas reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXhsFaqs77K" rel="nofollow noopener">
                CAS
               </a>
               <a aria-label="PubMed reference 13" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31344267" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="Google Scholar reference 13" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20further%20leap%20of%20improvement%20in%20tertiary%20structure%20prediction%20in%20CASP13%20prompts%20new%20routes%20for%20future%20assessments&amp;journal=Proteins&amp;doi=10.1002%2Fprot.25787&amp;volume=87&amp;pages=1100-1112&amp;publication_year=2019&amp;author=Abriata%2CLA&amp;author=Tam%C3%B2%2CGE&amp;author=Dal%20Peraro%2CM" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="14.">
              <p class="c-article-references__text" id="ref-CR14">
               Pearce, R. &amp; Zhang, Y. Deep learning techniques have significantly impacted protein structure prediction and protein design.
               <i>
                Curr. Opin. Struct. Biol
               </i>
               .
               <b>
                68
               </b>
               , 194–207 (2021).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 14" data-doi="10.1016/j.sbi.2021.01.007" data-track="click" data-track-action="article reference" data-track-label="10.1016/j.sbi.2021.01.007" href="https://doi-org.proxy.lib.ohio-state.edu/10.1016%2Fj.sbi.2021.01.007" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="CAS reference 14" data-track="click" data-track-action="cas reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXks1Ghtr4%3D" rel="nofollow noopener">
                CAS
               </a>
               <a aria-label="PubMed reference 14" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33639355" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="PubMed Central reference 14" data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC8222070" rel="nofollow noopener">
                PubMed Central
               </a>
               <a aria-label="Google Scholar reference 14" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Deep%20learning%20techniques%20have%20significantly%20impacted%20protein%20structure%20prediction%20and%20protein%20design&amp;journal=Curr.%20Opin.%20Struct.%20Biol.&amp;doi=10.1016%2Fj.sbi.2021.01.007&amp;volume=68&amp;pages=194-207&amp;publication_year=2021&amp;author=Pearce%2CR&amp;author=Zhang%2CY" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="15.">
              <p class="c-article-references__text" id="ref-CR15">
               Moult, J., Fidelis, K., Kryshtafovych, A., Schwede, T. &amp; Topf, M. Critical assessment of techniques for protein structure prediction, fourteenth round.
               <i>
                CASP 14 Abstract Book
               </i>
               <a data-track="click" data-track-action="external reference" data-track-label="https://www.predictioncenter.org/casp14/doc/CASP14_Abstracts.pdf" href="https://www.predictioncenter.org/casp14/doc/CASP14_Abstracts.pdf">
                https://www.predictioncenter.org/casp14/doc/CASP14_Abstracts.pdf
               </a>
               (2020).
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="16.">
              <p class="c-article-references__text" id="ref-CR16">
               Brini, E., Simmerling, C. &amp; Dill, K. Protein storytelling through physics.
               <i>
                Science
               </i>
               <b>
                370
               </b>
               , eaaz3041 (2020).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 16" data-doi="10.1126/science.aaz3041" data-track="click" data-track-action="article reference" data-track-label="10.1126/science.aaz3041" href="https://doi-org.proxy.lib.ohio-state.edu/10.1126%2Fscience.aaz3041" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="CAS reference 16" data-track="click" data-track-action="cas reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXisVOmu7vF" rel="nofollow noopener">
                CAS
               </a>
               <a aria-label="PubMed reference 16" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33243857" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="PubMed Central reference 16" data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC7945008" rel="nofollow noopener">
                PubMed Central
               </a>
               <a aria-label="Google Scholar reference 16" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Protein%20storytelling%20through%20physics&amp;journal=Science&amp;doi=10.1126%2Fscience.aaz3041&amp;volume=370&amp;publication_year=2020&amp;author=Brini%2CE&amp;author=Simmerling%2CC&amp;author=Dill%2CK" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="17.">
              <p class="c-article-references__text" id="ref-CR17">
               Sippl, M. J. Calculation of conformational ensembles from potentials of mean force. An approach to the knowledge-based prediction of local structures in globular proteins.
               <i>
                J. Mol. Biol
               </i>
               .
               <b>
                213
               </b>
               , 859–883 (1990).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 17" data-doi="10.1016/S0022-2836(05)80269-4" data-track="click" data-track-action="article reference" data-track-label="10.1016/S0022-2836(05)80269-4" href="https://doi-org.proxy.lib.ohio-state.edu/10.1016%2FS0022-2836%2805%2980269-4" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="CAS reference 17" data-track="click" data-track-action="cas reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DyaK3cXkvFCgs7Y%3D" rel="nofollow noopener">
                CAS
               </a>
               <a aria-label="PubMed reference 17" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=2359125" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="Google Scholar reference 17" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Calculation%20of%20conformational%20ensembles%20from%20potentials%20of%20mean%20force.%20An%20approach%20to%20the%20knowledge-based%20prediction%20of%20local%20structures%20in%20globular%20proteins&amp;journal=J.%20Mol.%20Biol.&amp;doi=10.1016%2FS0022-2836%2805%2980269-4&amp;volume=213&amp;pages=859-883&amp;publication_year=1990&amp;author=Sippl%2CMJ" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="18.">
              <p class="c-article-references__text" id="ref-CR18">
               Šali, A. &amp; Blundell, T. L. Comparative protein modelling by satisfaction of spatial restraints.
               <i>
                J. Mol. Biol
               </i>
               .
               <b>
                234
               </b>
               , 779–815 (1993).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 18" data-doi="10.1006/jmbi.1993.1626" data-track="click" data-track-action="article reference" data-track-label="10.1006/jmbi.1993.1626" href="https://doi-org.proxy.lib.ohio-state.edu/10.1006%2Fjmbi.1993.1626" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="PubMed reference 18" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=8254673" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="Google Scholar reference 18" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Comparative%20protein%20modelling%20by%20satisfaction%20of%20spatial%20restraints&amp;journal=J.%20Mol.%20Biol.&amp;doi=10.1006%2Fjmbi.1993.1626&amp;volume=234&amp;pages=779-815&amp;publication_year=1993&amp;author=%C5%A0ali%2CA&amp;author=Blundell%2CTL" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="19.">
              <p class="c-article-references__text" id="ref-CR19">
               Roy, A., Kucukural, A. &amp; Zhang, Y. I-TASSER: a unified platform for automated protein structure and function prediction.
               <i>
                Nat. Protocols
               </i>
               <b>
                5
               </b>
               , 725–738 (2010).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 19" data-doi="10.1038/nprot.2010.5" data-track="click" data-track-action="article reference" data-track-label="10.1038/nprot.2010.5" href="https://doi-org.proxy.lib.ohio-state.edu/10.1038%2Fnprot.2010.5" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="CAS reference 19" data-track="click" data-track-action="cas reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BC3cXksVahs74%3D" rel="nofollow noopener">
                CAS
               </a>
               <a aria-label="PubMed reference 19" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20360767" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="Google Scholar reference 19" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=I-TASSER%3A%20a%20unified%20platform%20for%20automated%20protein%20structure%20and%20function%20prediction&amp;journal=Nat.%20Protocols&amp;doi=10.1038%2Fnprot.2010.5&amp;volume=5&amp;pages=725-738&amp;publication_year=2010&amp;author=Roy%2CA&amp;author=Kucukural%2CA&amp;author=Zhang%2CY" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="20.">
              <p class="c-article-references__text" id="ref-CR20">
               Altschuh, D., Lesk, A. M., Bloomer, A. C. &amp; Klug, A. Correlation of co-ordinated amino acid substitutions with function in viruses related to tobacco mosaic virus.
               <i>
                J. Mol. Biol
               </i>
               .
               <b>
                193
               </b>
               , 693–707 (1987).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 20" data-doi="10.1016/0022-2836(87)90352-4" data-track="click" data-track-action="article reference" data-track-label="10.1016/0022-2836(87)90352-4" href="https://doi-org.proxy.lib.ohio-state.edu/10.1016%2F0022-2836%2887%2990352-4" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="CAS reference 20" data-track="click" data-track-action="cas reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DyaL2sXitV2ksL8%3D" rel="nofollow noopener">
                CAS
               </a>
               <a aria-label="PubMed reference 20" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=3612789" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="Google Scholar reference 20" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Correlation%20of%20co-ordinated%20amino%20acid%20substitutions%20with%20function%20in%20viruses%20related%20to%20tobacco%20mosaic%20virus&amp;journal=J.%20Mol.%20Biol.&amp;doi=10.1016%2F0022-2836%2887%2990352-4&amp;volume=193&amp;pages=693-707&amp;publication_year=1987&amp;author=Altschuh%2CD&amp;author=Lesk%2CAM&amp;author=Bloomer%2CAC&amp;author=Klug%2CA" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="21.">
              <p class="c-article-references__text" id="ref-CR21">
               Shindyalov, I. N., Kolchanov, N. A. &amp; Sander, C. Can three-dimensional contacts in protein structures be predicted by analysis of correlated mutations?
               <i>
                Protein Eng
               </i>
               .
               <b>
                7
               </b>
               , 349–358 (1994).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 21" data-doi="10.1093/protein/7.3.349" data-track="click" data-track-action="article reference" data-track-label="10.1093/protein/7.3.349" href="https://doi-org.proxy.lib.ohio-state.edu/10.1093%2Fprotein%2F7.3.349" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="CAS reference 21" data-track="click" data-track-action="cas reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DyaK2cXitFWqtbs%3D" rel="nofollow noopener">
                CAS
               </a>
               <a aria-label="PubMed reference 21" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=8177884" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="Google Scholar reference 21" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Can%20three-dimensional%20contacts%20in%20protein%20structures%20be%20predicted%20by%20analysis%20of%20correlated%20mutations%3F&amp;journal=Protein%20Eng.&amp;doi=10.1093%2Fprotein%2F7.3.349&amp;volume=7&amp;pages=349-358&amp;publication_year=1994&amp;author=Shindyalov%2CIN&amp;author=Kolchanov%2CNA&amp;author=Sander%2CC" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="22.">
              <p class="c-article-references__text" id="ref-CR22">
               Weigt, M., White, R. A., Szurmant, H., Hoch, J. A. &amp; Hwa, T. Identification of direct residue contacts in protein–protein interaction by message passing.
               <i>
                Proc. Natl Acad. Sci. USA
               </i>
               <b>
                106
               </b>
               , 67–72 (2009).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 22" data-doi="10.1073/pnas.0805923106" data-track="click" data-track-action="article reference" data-track-label="10.1073/pnas.0805923106" href="https://doi-org.proxy.lib.ohio-state.edu/10.1073%2Fpnas.0805923106" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="ADS reference 22" data-track="click" data-track-action="ads reference" data-track-label="link" href="http://adsabs.harvard.edu.proxy.lib.ohio-state.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2009PNAS..106...67W" rel="nofollow noopener">
                ADS
               </a>
               <a aria-label="CAS reference 22" data-track="click" data-track-action="cas reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BD1MXltF2jug%3D%3D" rel="nofollow noopener">
                CAS
               </a>
               <a aria-label="PubMed reference 22" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19116270" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="Google Scholar reference 22" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Identification%20of%20direct%20residue%20contacts%20in%20protein%E2%80%93protein%20interaction%20by%20message%20passing&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.0805923106&amp;volume=106&amp;pages=67-72&amp;publication_year=2009&amp;author=Weigt%2CM&amp;author=White%2CRA&amp;author=Szurmant%2CH&amp;author=Hoch%2CJA&amp;author=Hwa%2CT" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="23.">
              <p class="c-article-references__text" id="ref-CR23">
               Marks, D. S. et al. Protein 3D structure computed from evolutionary sequence variation.
               <i>
                PLoS ONE
               </i>
               <b>
                6
               </b>
               , e28766 (2011).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 23" data-doi="10.1371/journal.pone.0028766" data-track="click" data-track-action="article reference" data-track-label="10.1371/journal.pone.0028766" href="https://doi-org.proxy.lib.ohio-state.edu/10.1371%2Fjournal.pone.0028766" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="ADS reference 23" data-track="click" data-track-action="ads reference" data-track-label="link" href="http://adsabs.harvard.edu.proxy.lib.ohio-state.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2011PLoSO...628766M" rel="nofollow noopener">
                ADS
               </a>
               <a aria-label="CAS reference 23" data-track="click" data-track-action="cas reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BC3MXhs1KhurnJ" rel="nofollow noopener">
                CAS
               </a>
               <a aria-label="PubMed reference 23" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22163331" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="PubMed Central reference 23" data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC3233603" rel="nofollow noopener">
                PubMed Central
               </a>
               <a aria-label="Google Scholar reference 23" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Protein%203D%20structure%20computed%20from%20evolutionary%20sequence%20variation&amp;journal=PLoS%20ONE&amp;doi=10.1371%2Fjournal.pone.0028766&amp;volume=6&amp;publication_year=2011&amp;author=Marks%2CDS" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="24.">
              <p class="c-article-references__text" id="ref-CR24">
               Jones, D. T., Buchan, D. W. A., Cozzetto, D. &amp; Pontil, M. PSICOV: precise structural contact prediction using sparse inverse covariance estimation on large multiple sequence alignments.
               <i>
                Bioinformatics
               </i>
               <b>
                28
               </b>
               , 184–190 (2012).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 24" data-doi="10.1093/bioinformatics/btr638" data-track="click" data-track-action="article reference" data-track-label="10.1093/bioinformatics/btr638" href="https://doi-org.proxy.lib.ohio-state.edu/10.1093%2Fbioinformatics%2Fbtr638" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="CAS reference 24" data-track="click" data-track-action="cas reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BC38Xht1agurg%3D" rel="nofollow noopener">
                CAS
               </a>
               <a aria-label="PubMed reference 24" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22101153" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="Google Scholar reference 24" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=PSICOV%3A%20precise%20structural%20contact%20prediction%20using%20sparse%20inverse%20covariance%20estimation%20on%20large%20multiple%20sequence%20alignments&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtr638&amp;volume=28&amp;pages=184-190&amp;publication_year=2012&amp;author=Jones%2CDT&amp;author=Buchan%2CDWA&amp;author=Cozzetto%2CD&amp;author=Pontil%2CM" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="25.">
              <p class="c-article-references__text" id="ref-CR25">
               Moult, J., Pedersen, J. T., Judson, R. &amp; Fidelis, K. A large-scale experiment to assess protein structure prediction methods.
               <i>
                Proteins
               </i>
               <b>
                23
               </b>
               , ii–iv (1995).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 25" data-doi="10.1002/prot.340230303" data-track="click" data-track-action="article reference" data-track-label="10.1002/prot.340230303" href="https://doi-org.proxy.lib.ohio-state.edu/10.1002%2Fprot.340230303" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="CAS reference 25" data-track="click" data-track-action="cas reference" data-track-label="link" href="/articles/cas-redirect/1:STN:280:DyaK287oslCntw%3D%3D" rel="nofollow noopener">
                CAS
               </a>
               <a aria-label="PubMed reference 25" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=8710822" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="Google Scholar reference 25" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20large-scale%20experiment%20to%20assess%20protein%20structure%20prediction%20methods&amp;journal=Proteins&amp;doi=10.1002%2Fprot.340230303&amp;volume=23&amp;pages=ii-iv&amp;publication_year=1995&amp;author=Moult%2CJ&amp;author=Pedersen%2CJT&amp;author=Judson%2CR&amp;author=Fidelis%2CK" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="26.">
              <p class="c-article-references__text" id="ref-CR26">
               Kryshtafovych, A., Schwede, T., Topf, M., Fidelis, K. &amp; Moult, J. Critical assessment of methods of protein structure prediction (CASP)-round XIII.
               <i>
                Proteins
               </i>
               <b>
                87
               </b>
               , 1011–1020 (2019).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 26" data-doi="10.1002/prot.25823" data-track="click" data-track-action="article reference" data-track-label="10.1002/prot.25823" href="https://doi-org.proxy.lib.ohio-state.edu/10.1002%2Fprot.25823" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="CAS reference 26" data-track="click" data-track-action="cas reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXitVSlt7%2FO" rel="nofollow noopener">
                CAS
               </a>
               <a aria-label="PubMed reference 26" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31589781" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="PubMed Central reference 26" data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC6927249" rel="nofollow noopener">
                PubMed Central
               </a>
               <a aria-label="Google Scholar reference 26" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Critical%20assessment%20of%20methods%20of%20protein%20structure%20prediction%20%28CASP%29-round%20XIII&amp;journal=Proteins&amp;doi=10.1002%2Fprot.25823&amp;volume=87&amp;pages=1011-1020&amp;publication_year=2019&amp;author=Kryshtafovych%2CA&amp;author=Schwede%2CT&amp;author=Topf%2CM&amp;author=Fidelis%2CK&amp;author=Moult%2CJ" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="27.">
              <p class="c-article-references__text" id="ref-CR27">
               Zhang, Y. &amp; Skolnick, J. Scoring function for automated assessment of protein structure template quality.
               <i>
                Proteins
               </i>
               <b>
                57
               </b>
               , 702–710 (2004).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 27" data-doi="10.1002/prot.20264" data-track="click" data-track-action="article reference" data-track-label="10.1002/prot.20264" href="https://doi-org.proxy.lib.ohio-state.edu/10.1002%2Fprot.20264" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="CAS reference 27" data-track="click" data-track-action="cas reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BD2cXhtVaqtLvI" rel="nofollow noopener">
                CAS
               </a>
               <a aria-label="PubMed reference 27" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15476259" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="Google Scholar reference 27" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Scoring%20function%20for%20automated%20assessment%20of%20protein%20structure%20template%20quality&amp;journal=Proteins&amp;doi=10.1002%2Fprot.20264&amp;volume=57&amp;pages=702-710&amp;publication_year=2004&amp;author=Zhang%2CY&amp;author=Skolnick%2CJ" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="28.">
              <p class="c-article-references__text" id="ref-CR28">
               Tu, Z. &amp; Bai, X. Auto-context and its application to high-level vision tasks and 3D brain image segmentation.
               <i>
                IEEE Trans. Pattern Anal. Mach. Intell
               </i>
               .
               <b>
                32
               </b>
               , 1744–1757 (2010).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 28" data-doi="10.1109/TPAMI.2009.186" data-track="click" data-track-action="article reference" data-track-label="10.1109/TPAMI.2009.186" href="https://doi-org.proxy.lib.ohio-state.edu/10.1109%2FTPAMI.2009.186" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="PubMed reference 28" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20724753" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="Google Scholar reference 28" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Auto-context%20and%20its%20application%20to%20high-level%20vision%20tasks%20and%203D%20brain%20image%20segmentation&amp;journal=IEEE%20Trans.%20Pattern%20Anal.%20Mach.%20Intell.&amp;doi=10.1109%2FTPAMI.2009.186&amp;volume=32&amp;pages=1744-1757&amp;publication_year=2010&amp;author=Tu%2CZ&amp;author=Bai%2CX" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="29.">
              <p class="c-article-references__text" id="ref-CR29">
               Carreira, J., Agrawal, P., Fragkiadaki, K. &amp; Malik, J. Human pose estimation with iterative error feedback. In
               <i>
                Proc. IEEE Conference on Computer Vision and Pattern Recognition
               </i>
               4733–4742 (2016).
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="30.">
              <p class="c-article-references__text" id="ref-CR30">
               Mirabello, C. &amp; Wallner, B. rawMSA: end-to-end deep learning using raw multiple sequence alignments.
               <i>
                PLoS ONE
               </i>
               <b>
                14
               </b>
               , e0220182 (2019).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 30" data-doi="10.1371/journal.pone.0220182" data-track="click" data-track-action="article reference" data-track-label="10.1371/journal.pone.0220182" href="https://doi-org.proxy.lib.ohio-state.edu/10.1371%2Fjournal.pone.0220182" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="CAS reference 30" data-track="click" data-track-action="cas reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXhvFamur7E" rel="nofollow noopener">
                CAS
               </a>
               <a aria-label="PubMed reference 30" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31415569" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="PubMed Central reference 30" data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC6695225" rel="nofollow noopener">
                PubMed Central
               </a>
               <a aria-label="Google Scholar reference 30" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=rawMSA%3A%20end-to-end%20deep%20learning%20using%20raw%20multiple%20sequence%20alignments&amp;journal=PLoS%20ONE&amp;doi=10.1371%2Fjournal.pone.0220182&amp;volume=14&amp;publication_year=2019&amp;author=Mirabello%2CC&amp;author=Wallner%2CB" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="31.">
              <p class="c-article-references__text" id="ref-CR31">
               Huang, Z. et al. CCNet: criss-cross attention for semantic segmentation. In
               <i>
                Proc. IEEE/CVF International Conference on Computer Vision
               </i>
               603–612 (2019).
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="32.">
              <p class="c-article-references__text" id="ref-CR32">
               Hornak, V. et al. Comparison of multiple Amber force fields and development of improved protein backbone parameters.
               <i>
                Proteins
               </i>
               <b>
                65
               </b>
               , 712–725 (2006).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 32" data-doi="10.1002/prot.21123" data-track="click" data-track-action="article reference" data-track-label="10.1002/prot.21123" href="https://doi-org.proxy.lib.ohio-state.edu/10.1002%2Fprot.21123" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="CAS reference 32" data-track="click" data-track-action="cas reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BD28XhtFWqt7fM" rel="nofollow noopener">
                CAS
               </a>
               <a aria-label="PubMed reference 32" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16981200" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="PubMed Central reference 32" data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC4805110" rel="nofollow noopener">
                PubMed Central
               </a>
               <a aria-label="Google Scholar reference 32" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Comparison%20of%20multiple%20Amber%20force%20fields%20and%20development%20of%20improved%20protein%20backbone%20parameters&amp;journal=Proteins&amp;doi=10.1002%2Fprot.21123&amp;volume=65&amp;pages=712-725&amp;publication_year=2006&amp;author=Hornak%2CV" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="33.">
              <p class="c-article-references__text" id="ref-CR33">
               Zemla, A. LGA: a method for finding 3D similarities in protein structures.
               <i>
                Nucleic Acids Res
               </i>
               .
               <b>
                31
               </b>
               , 3370–3374 (2003).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 33" data-doi="10.1093/nar/gkg571" data-track="click" data-track-action="article reference" data-track-label="10.1093/nar/gkg571" href="https://doi-org.proxy.lib.ohio-state.edu/10.1093%2Fnar%2Fgkg571" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="CAS reference 33" data-track="click" data-track-action="cas reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BD3sXltVWjtbk%3D" rel="nofollow noopener">
                CAS
               </a>
               <a aria-label="PubMed reference 33" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12824330" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="PubMed Central reference 33" data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC168977" rel="nofollow noopener">
                PubMed Central
               </a>
               <a aria-label="Google Scholar reference 33" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=LGA%3A%20a%20method%20for%20finding%203D%20similarities%20in%20protein%20structures&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgkg571&amp;volume=31&amp;pages=3370-3374&amp;publication_year=2003&amp;author=Zemla%2CA" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="34.">
              <p class="c-article-references__text" id="ref-CR34">
               Mariani, V., Biasini, M., Barbato, A. &amp; Schwede, T. lDDT: a local superposition-free score for comparing protein structures and models using distance difference tests.
               <i>
                Bioinformatics
               </i>
               <b>
                29
               </b>
               , 2722–2728 (2013).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 34" data-doi="10.1093/bioinformatics/btt473" data-track="click" data-track-action="article reference" data-track-label="10.1093/bioinformatics/btt473" href="https://doi-org.proxy.lib.ohio-state.edu/10.1093%2Fbioinformatics%2Fbtt473" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="CAS reference 34" data-track="click" data-track-action="cas reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BC3sXhs1CisrfK" rel="nofollow noopener">
                CAS
               </a>
               <a aria-label="PubMed reference 34" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23986568" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="PubMed Central reference 34" data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC3799472" rel="nofollow noopener">
                PubMed Central
               </a>
               <a aria-label="Google Scholar reference 34" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=lDDT%3A%20a%20local%20superposition-free%20score%20for%20comparing%20protein%20structures%20and%20models%20using%20distance%20difference%20tests&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtt473&amp;volume=29&amp;pages=2722-2728&amp;publication_year=2013&amp;author=Mariani%2CV&amp;author=Biasini%2CM&amp;author=Barbato%2CA&amp;author=Schwede%2CT" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="35.">
              <p class="c-article-references__text" id="ref-CR35">
               Xie, Q., Luong, M.-T., Hovy, E. &amp; Le, Q. V. Self-training with noisy student improves imagenet classification. In
               <i>
                Proc. IEEE/CVF Conference on Computer Vision and Pattern Recognition
               </i>
               10687–10698 (2020).
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="36.">
              <p class="c-article-references__text" id="ref-CR36">
               Mirdita, M. et al. Uniclust databases of clustered and deeply annotated protein sequences and alignments.
               <i>
                Nucleic Acids Res
               </i>
               .
               <b>
                45
               </b>
               , D170–D176 (2017).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 36" data-doi="10.1093/nar/gkw1081" data-track="click" data-track-action="article reference" data-track-label="10.1093/nar/gkw1081" href="https://doi-org.proxy.lib.ohio-state.edu/10.1093%2Fnar%2Fgkw1081" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="CAS reference 36" data-track="click" data-track-action="cas reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhslWgsb8%3D" rel="nofollow noopener">
                CAS
               </a>
               <a aria-label="PubMed reference 36" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27899574" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="Google Scholar reference 36" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Uniclust%20databases%20of%20clustered%20and%20deeply%20annotated%20protein%20sequences%20and%20alignments&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgkw1081&amp;volume=45&amp;pages=D170-D176&amp;publication_year=2017&amp;author=Mirdita%2CM" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="37.">
              <p class="c-article-references__text" id="ref-CR37">
               Devlin, J., Chang, M.-W., Lee, K. &amp; Toutanova, K. BERT: pre-training of deep bidirectional transformers for language understanding. In
               <i>
                Proc. 2019 Conference of the North American Chapter of the Association for Computational Linguistics: Human Language Technologies
               </i>
               1, 4171–4186 (2019).
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="38.">
              <p class="c-article-references__text" id="ref-CR38">
               Rao, R. et al. MSA transformer. In
               <i>
                Proc. 38th International Conference on Machine Learning
               </i>
               PMLR 139, 8844–8856 (2021).
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="39.">
              <p class="c-article-references__text" id="ref-CR39">
               Tunyasuvunakool, K. et al. Highly accurate protein structure prediction for the human proteome.
               <i>
                Nature
               </i>
               <a data-track="click" data-track-action="external reference" data-track-label="10.1038/s41586-021-03828-1" href="https://doi-org.proxy.lib.ohio-state.edu/10.1038/s41586-021-03828-1">
                https://doi-org.proxy.lib.ohio-state.edu/10.1038/s41586-021-03828-1
               </a>
               (2021).
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="40.">
              <p class="c-article-references__text" id="ref-CR40">
               Kuhlman, B. &amp; Bradley, P. Advances in protein structure prediction and design.
               <i>
                Nat. Rev. Mol. Cell Biol
               </i>
               .
               <b>
                20
               </b>
               , 681–697 (2019).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 40" data-doi="10.1038/s41580-019-0163-x" data-track="click" data-track-action="article reference" data-track-label="10.1038/s41580-019-0163-x" href="https://doi-org.proxy.lib.ohio-state.edu/10.1038%2Fs41580-019-0163-x" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="CAS reference 40" data-track="click" data-track-action="cas reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXhsFyksL7J" rel="nofollow noopener">
                CAS
               </a>
               <a aria-label="PubMed reference 40" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31417196" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="PubMed Central reference 40" data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC7032036" rel="nofollow noopener">
                PubMed Central
               </a>
               <a aria-label="Google Scholar reference 40" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Advances%20in%20protein%20structure%20prediction%20and%20design&amp;journal=Nat.%20Rev.%20Mol.%20Cell%20Biol.&amp;doi=10.1038%2Fs41580-019-0163-x&amp;volume=20&amp;pages=681-697&amp;publication_year=2019&amp;author=Kuhlman%2CB&amp;author=Bradley%2CP" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="41.">
              <p class="c-article-references__text" id="ref-CR41">
               Marks, D. S., Hopf, T. A. &amp; Sander, C. Protein structure prediction from sequence variation.
               <i>
                Nat. Biotechnol
               </i>
               .
               <b>
                30
               </b>
               , 1072–1080 (2012).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 41" data-doi="10.1038/nbt.2419" data-track="click" data-track-action="article reference" data-track-label="10.1038/nbt.2419" href="https://doi-org.proxy.lib.ohio-state.edu/10.1038%2Fnbt.2419" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="CAS reference 41" data-track="click" data-track-action="cas reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BC38Xhs1elt7bM" rel="nofollow noopener">
                CAS
               </a>
               <a aria-label="PubMed reference 41" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23138306" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="PubMed Central reference 41" data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC4319528" rel="nofollow noopener">
                PubMed Central
               </a>
               <a aria-label="Google Scholar reference 41" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Protein%20structure%20prediction%20from%20sequence%20variation&amp;journal=Nat.%20Biotechnol.&amp;doi=10.1038%2Fnbt.2419&amp;volume=30&amp;pages=1072-1080&amp;publication_year=2012&amp;author=Marks%2CDS&amp;author=Hopf%2CTA&amp;author=Sander%2CC" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="42.">
              <p class="c-article-references__text" id="ref-CR42">
               Qian, N. &amp; Sejnowski, T. J. Predicting the secondary structure of globular proteins using neural network models.
               <i>
                J. Mol. Biol
               </i>
               .
               <b>
                202
               </b>
               , 865–884 (1988).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 42" data-doi="10.1016/0022-2836(88)90564-5" data-track="click" data-track-action="article reference" data-track-label="10.1016/0022-2836(88)90564-5" href="https://doi-org.proxy.lib.ohio-state.edu/10.1016%2F0022-2836%2888%2990564-5" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="CAS reference 42" data-track="click" data-track-action="cas reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DyaL1MXhtlWksb0%3D" rel="nofollow noopener">
                CAS
               </a>
               <a aria-label="PubMed reference 42" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=3172241" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="Google Scholar reference 42" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Predicting%20the%20secondary%20structure%20of%20globular%20proteins%20using%20neural%20network%20models&amp;journal=J.%20Mol.%20Biol.&amp;doi=10.1016%2F0022-2836%2888%2990564-5&amp;volume=202&amp;pages=865-884&amp;publication_year=1988&amp;author=Qian%2CN&amp;author=Sejnowski%2CTJ" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="43.">
              <p class="c-article-references__text" id="ref-CR43">
               Fariselli, P., Olmea, O., Valencia, A. &amp; Casadio, R. Prediction of contact maps with neural networks and correlated mutations.
               <i>
                Protein Eng
               </i>
               .
               <b>
                14
               </b>
               , 835–843 (2001).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 43" data-doi="10.1093/protein/14.11.835" data-track="click" data-track-action="article reference" data-track-label="10.1093/protein/14.11.835" href="https://doi-org.proxy.lib.ohio-state.edu/10.1093%2Fprotein%2F14.11.835" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="CAS reference 43" data-track="click" data-track-action="cas reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BD38XjtVentA%3D%3D" rel="nofollow noopener">
                CAS
               </a>
               <a aria-label="PubMed reference 43" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11742102" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="Google Scholar reference 43" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Prediction%20of%20contact%20maps%20with%20neural%20networks%20and%20correlated%20mutations&amp;journal=Protein%20Eng.&amp;doi=10.1093%2Fprotein%2F14.11.835&amp;volume=14&amp;pages=835-843&amp;publication_year=2001&amp;author=Fariselli%2CP&amp;author=Olmea%2CO&amp;author=Valencia%2CA&amp;author=Casadio%2CR" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="44.">
              <p class="c-article-references__text" id="ref-CR44">
               Yang, J. et al. Improved protein structure prediction using predicted interresidue orientations.
               <i>
                Proc. Natl Acad. Sci. USA
               </i>
               <b>
                117
               </b>
               , 1496–1503 (2020).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 44" data-doi="10.1073/pnas.1914677117" data-track="click" data-track-action="article reference" data-track-label="10.1073/pnas.1914677117" href="https://doi-org.proxy.lib.ohio-state.edu/10.1073%2Fpnas.1914677117" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="ADS reference 44" data-track="click" data-track-action="ads reference" data-track-label="link" href="http://adsabs.harvard.edu.proxy.lib.ohio-state.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2020PNAS..117.1496Y" rel="nofollow noopener">
                ADS
               </a>
               <a aria-label="CAS reference 44" data-track="click" data-track-action="cas reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsFKrsLg%3D" rel="nofollow noopener">
                CAS
               </a>
               <a aria-label="PubMed reference 44" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31896580" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="PubMed Central reference 44" data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC6983395" rel="nofollow noopener">
                PubMed Central
               </a>
               <a aria-label="Google Scholar reference 44" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Improved%20protein%20structure%20prediction%20using%20predicted%20interresidue%20orientations&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.1914677117&amp;volume=117&amp;pages=1496-1503&amp;publication_year=2020&amp;author=Yang%2CJ" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="45.">
              <p class="c-article-references__text" id="ref-CR45">
               Li, Y. et al. Deducing high-accuracy protein contact-maps from a triplet of coevolutionary matrices through deep residual convolutional networks.
               <i>
                PLOS Comput. Biol
               </i>
               .
               <b>
                17
               </b>
               , e1008865 (2021).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 45" data-doi="10.1371/journal.pcbi.1008865" data-track="click" data-track-action="article reference" data-track-label="10.1371/journal.pcbi.1008865" href="https://doi-org.proxy.lib.ohio-state.edu/10.1371%2Fjournal.pcbi.1008865" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="ADS reference 45" data-track="click" data-track-action="ads reference" data-track-label="link" href="http://adsabs.harvard.edu.proxy.lib.ohio-state.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2021mwaw.book.....L" rel="nofollow noopener">
                ADS
               </a>
               <a aria-label="CAS reference 45" data-track="click" data-track-action="cas reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXosFSms78%3D" rel="nofollow noopener">
                CAS
               </a>
               <a aria-label="PubMed reference 45" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33770072" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="PubMed Central reference 45" data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC8026059" rel="nofollow noopener">
                PubMed Central
               </a>
               <a aria-label="Google Scholar reference 45" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Deducing%20high-accuracy%20protein%20contact-maps%20from%20a%20triplet%20of%20coevolutionary%20matrices%20through%20deep%20residual%20convolutional%20networks&amp;journal=PLOS%20Comput.%20Biol.&amp;doi=10.1371%2Fjournal.pcbi.1008865&amp;volume=17&amp;publication_year=2021&amp;author=Li%2CY" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="46.">
              <p class="c-article-references__text" id="ref-CR46">
               He, K., Zhang, X., Ren, S. &amp; Sun, J. Deep residual learning for image recognition. In
               <i>
                Proc. IEEE Conference on Computer Vision and Pattern Recognition
               </i>
               770–778 (2016).
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="47.">
              <p class="c-article-references__text" id="ref-CR47">
               AlQuraishi, M. End-to-end differentiable learning of protein structure.
               <i>
                Cell Syst
               </i>
               .
               <b>
                8
               </b>
               , 292–301 (2019).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 47" data-doi="10.1016/j.cels.2019.03.006" data-track="click" data-track-action="article reference" data-track-label="10.1016/j.cels.2019.03.006" href="https://doi-org.proxy.lib.ohio-state.edu/10.1016%2Fj.cels.2019.03.006" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="CAS reference 47" data-track="click" data-track-action="cas reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXosVyhtb0%3D" rel="nofollow noopener">
                CAS
               </a>
               <a aria-label="PubMed reference 47" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31005579" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="PubMed Central reference 47" data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC6513320" rel="nofollow noopener">
                PubMed Central
               </a>
               <a aria-label="Google Scholar reference 47" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=End-to-end%20differentiable%20learning%20of%20protein%20structure&amp;journal=Cell%20Syst.&amp;doi=10.1016%2Fj.cels.2019.03.006&amp;volume=8&amp;pages=292-301&amp;publication_year=2019&amp;author=AlQuraishi%2CM" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="48.">
              <p class="c-article-references__text" id="ref-CR48">
               Senior, A. W. et al. Protein structure prediction using multiple deep neural networks in the 13th Critical Assessment of Protein Structure Prediction (CASP13).
               <i>
                Proteins
               </i>
               <b>
                87
               </b>
               , 1141–1148 (2019).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 48" data-doi="10.1002/prot.25834" data-track="click" data-track-action="article reference" data-track-label="10.1002/prot.25834" href="https://doi-org.proxy.lib.ohio-state.edu/10.1002%2Fprot.25834" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="CAS reference 48" data-track="click" data-track-action="cas reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXitFartb%2FK" rel="nofollow noopener">
                CAS
               </a>
               <a aria-label="PubMed reference 48" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31602685" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="PubMed Central reference 48" data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC7079254" rel="nofollow noopener">
                PubMed Central
               </a>
               <a aria-label="Google Scholar reference 48" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Protein%20structure%20prediction%20using%20multiple%20deep%20neural%20networks%20in%20the%2013th%20Critical%20Assessment%20of%20Protein%20Structure%20Prediction%20%28CASP13%29&amp;journal=Proteins&amp;doi=10.1002%2Fprot.25834&amp;volume=87&amp;pages=1141-1148&amp;publication_year=2019&amp;author=Senior%2CAW" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="49.">
              <p class="c-article-references__text" id="ref-CR49">
               Ingraham, J., Riesselman, A. J., Sander, C. &amp; Marks, D. S. Learning protein structure with a differentiable simulator. in
               <i>
                Proc. International Conference on Learning Representations
               </i>
               (2019).
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="50.">
              <p class="c-article-references__text" id="ref-CR50">
               Li, J. Universal transforming geometric network. Preprint at
               <a data-track="click" data-track-action="external reference" data-track-label="https://arxiv-org.proxy.lib.ohio-state.edu/abs/1908.00723" href="https://arxiv-org.proxy.lib.ohio-state.edu/abs/1908.00723">
                https://arxiv-org.proxy.lib.ohio-state.edu/abs/1908.00723
               </a>
               (2019).
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="51.">
              <p class="c-article-references__text" id="ref-CR51">
               Xu, J., McPartlon, M. &amp; Li, J. Improved protein structure prediction by deep learning irrespective of co-evolution information.
               <i>
                Nat. Mach. Intell
               </i>
               .
               <b>
                3
               </b>
               , 601–609 (2021).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 51" data-doi="10.1038/s42256-021-00348-5" data-track="click" data-track-action="article reference" data-track-label="10.1038/s42256-021-00348-5" href="https://doi-org.proxy.lib.ohio-state.edu/10.1038%2Fs42256-021-00348-5" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="PubMed reference 51" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34368623" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="PubMed Central reference 51" data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC8340610" rel="nofollow noopener">
                PubMed Central
               </a>
               <a aria-label="Google Scholar reference 51" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Improved%20protein%20structure%20prediction%20by%20deep%20learning%20irrespective%20of%20co-evolution%20information&amp;journal=Nat.%20Mach.%20Intell.&amp;doi=10.1038%2Fs42256-021-00348-5&amp;volume=3&amp;pages=601-609&amp;publication_year=2021&amp;author=Xu%2CJ&amp;author=McPartlon%2CM&amp;author=Li%2CJ" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="52.">
              <p class="c-article-references__text" id="ref-CR52">
               Vaswani, A. et al. Attention is all you need. In
               <i>
                Advances in Neural Information Processing Systems
               </i>
               5998–6008 (2017).
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="53.">
              <p class="c-article-references__text" id="ref-CR53">
               Wang, H. et al. Axial-deeplab: stand-alone axial-attention for panoptic segmentation. in
               <i>
                European Conference on Computer Vision
               </i>
               108–126 (Springer, 2020).
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="54.">
              <p class="c-article-references__text" id="ref-CR54">
               Alley, E. C., Khimulya, G., Biswas, S., AlQuraishi, M. &amp; Church, G. M. Unified rational protein engineering with sequence-based deep representation learning.
               <i>
                Nat. Methods
               </i>
               <b>
                16
               </b>
               , 1315–1322 (2019).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 54" data-doi="10.1038/s41592-019-0598-1" data-track="click" data-track-action="article reference" data-track-label="10.1038/s41592-019-0598-1" href="https://doi-org.proxy.lib.ohio-state.edu/10.1038%2Fs41592-019-0598-1" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="CAS reference 54" data-track="click" data-track-action="cas reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXitVSlsbnJ" rel="nofollow noopener">
                CAS
               </a>
               <a aria-label="PubMed reference 54" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31636460" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="PubMed Central reference 54" data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC7067682" rel="nofollow noopener">
                PubMed Central
               </a>
               <a aria-label="Google Scholar reference 54" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Unified%20rational%20protein%20engineering%20with%20sequence-based%20deep%20representation%20learning&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fs41592-019-0598-1&amp;volume=16&amp;pages=1315-1322&amp;publication_year=2019&amp;author=Alley%2CEC&amp;author=Khimulya%2CG&amp;author=Biswas%2CS&amp;author=AlQuraishi%2CM&amp;author=Church%2CGM" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="55.">
              <p class="c-article-references__text" id="ref-CR55">
               Heinzinger, M. et al. Modeling aspects of the language of life through transfer-learning protein sequences.
               <i>
                BMC Bioinformatics
               </i>
               <b>
                20
               </b>
               , 723 (2019).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 55" data-doi="10.1186/s12859-019-3220-8" data-track="click" data-track-action="article reference" data-track-label="10.1186/s12859-019-3220-8" href="https://doi-org.proxy.lib.ohio-state.edu/10.1186%2Fs12859-019-3220-8" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="CAS reference 55" data-track="click" data-track-action="cas reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXisVGjsLbK" rel="nofollow noopener">
                CAS
               </a>
               <a aria-label="PubMed reference 55" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31847804" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="PubMed Central reference 55" data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC6918593" rel="nofollow noopener">
                PubMed Central
               </a>
               <a aria-label="Google Scholar reference 55" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Modeling%20aspects%20of%20the%20language%20of%20life%20through%20transfer-learning%20protein%20sequences&amp;journal=BMC%20Bioinformatics&amp;doi=10.1186%2Fs12859-019-3220-8&amp;volume=20&amp;publication_year=2019&amp;author=Heinzinger%2CM" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="56.">
              <p class="c-article-references__text" id="ref-CR56">
               Rives, A. et al. Biological structure and function emerge from scaling unsupervised learning to 250 million protein sequences.
               <i>
                Proc. Natl Acad. Sci. USA
               </i>
               <b>
                118
               </b>
               , e2016239118 (2021).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 56" data-doi="10.1073/pnas.2016239118" data-track="click" data-track-action="article reference" data-track-label="10.1073/pnas.2016239118" href="https://doi-org.proxy.lib.ohio-state.edu/10.1073%2Fpnas.2016239118" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="CAS reference 56" data-track="click" data-track-action="cas reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXovVantro%3D" rel="nofollow noopener">
                CAS
               </a>
               <a aria-label="PubMed reference 56" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33876751" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="PubMed Central reference 56" data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC8053943" rel="nofollow noopener">
                PubMed Central
               </a>
               <a aria-label="Google Scholar reference 56" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Biological%20structure%20and%20function%20emerge%20from%20scaling%20unsupervised%20learning%20to%20250%20million%20protein%20sequences&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.2016239118&amp;volume=118&amp;publication_year=2021&amp;author=Rives%2CA" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="57.">
              <p class="c-article-references__text" id="ref-CR57">
               Pereira, J. et al. High-accuracy protein structure prediction in CASP14.
               <i>
                Proteins
               </i>
               <a data-track="click" data-track-action="external reference" data-track-label="10.1002/prot.26171" href="https://doi-org.proxy.lib.ohio-state.edu/10.1002/prot.26171">
                https://doi-org.proxy.lib.ohio-state.edu/10.1002/prot.26171
               </a>
               (2021).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 57" data-doi="10.1002/prot.26171" data-track="click" data-track-action="article reference" data-track-label="10.1002/prot.26171" href="https://doi-org.proxy.lib.ohio-state.edu/10.1002%2Fprot.26171" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="PubMed reference 57" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34387010" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="PubMed Central reference 57" data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC8881082" rel="nofollow noopener">
                PubMed Central
               </a>
               <a aria-label="Google Scholar reference 57" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=High-accuracy%20protein%20structure%20prediction%20in%20CASP14&amp;journal=Proteins&amp;doi=10.1002%2Fprot.26171&amp;publication_year=2021&amp;author=Pereira%2CJ" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="58.">
              <p class="c-article-references__text" id="ref-CR58">
               Gupta, M. et al. CryoEM and AI reveal a structure of SARS-CoV-2 Nsp2, a multifunctional protein involved in key host processes. Preprint at
               <a data-track="click" data-track-action="external reference" data-track-label="10.1101/2021.05.10.443524" href="https://doi-org.proxy.lib.ohio-state.edu/10.1101/2021.05.10.443524">
                https://doi-org.proxy.lib.ohio-state.edu/10.1101/2021.05.10.443524
               </a>
               (2021).
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="59.">
              <p class="c-article-references__text" id="ref-CR59">
               Ingraham, J., Garg, V. K., Barzilay, R. &amp; Jaakkola, T. Generative models for graph-based protein design. in
               <i>
                Proc. 33rd Conference on Neural Information Processing Systems
               </i>
               (2019).
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="60.">
              <p class="c-article-references__text" id="ref-CR60">
               Johnson, L. S., Eddy, S. R. &amp; Portugaly, E. Hidden Markov model speed heuristic and iterative HMM search procedure.
               <i>
                BMC Bioinformatics
               </i>
               <b>
                11
               </b>
               , 431 (2010).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 60" data-doi="10.1186/1471-2105-11-431" data-track="click" data-track-action="article reference" data-track-label="10.1186/1471-2105-11-431" href="https://doi-org.proxy.lib.ohio-state.edu/10.1186%2F1471-2105-11-431" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="PubMed reference 60" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20718988" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="PubMed Central reference 60" data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC2931519" rel="nofollow noopener">
                PubMed Central
               </a>
               <a aria-label="Google Scholar reference 60" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Hidden%20Markov%20model%20speed%20heuristic%20and%20iterative%20HMM%20search%20procedure&amp;journal=BMC%20Bioinformatics&amp;doi=10.1186%2F1471-2105-11-431&amp;volume=11&amp;publication_year=2010&amp;author=Johnson%2CLS&amp;author=Eddy%2CSR&amp;author=Portugaly%2CE" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="61.">
              <p class="c-article-references__text" id="ref-CR61">
               Remmert, M., Biegert, A., Hauser, A. &amp; Söding, J. HHblits: lightning-fast iterative protein sequence searching by HMM-HMM alignment.
               <i>
                Nat. Methods
               </i>
               <b>
                9
               </b>
               , 173–175 (2012).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 61" data-doi="10.1038/nmeth.1818" data-track="click" data-track-action="article reference" data-track-label="10.1038/nmeth.1818" href="https://doi-org.proxy.lib.ohio-state.edu/10.1038%2Fnmeth.1818" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="CAS reference 61" data-track="click" data-track-action="cas reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BC3MXhs1OltbnO" rel="nofollow noopener">
                CAS
               </a>
               <a aria-label="Google Scholar reference 61" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=HHblits%3A%20lightning-fast%20iterative%20protein%20sequence%20searching%20by%20HMM-HMM%20alignment&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fnmeth.1818&amp;volume=9&amp;pages=173-175&amp;publication_year=2012&amp;author=Remmert%2CM&amp;author=Biegert%2CA&amp;author=Hauser%2CA&amp;author=S%C3%B6ding%2CJ" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="62.">
              <p class="c-article-references__text" id="ref-CR62">
               The UniProt Consortium. UniProt: the universal protein knowledgebase in 2021.
               <i>
                Nucleic Acids Res
               </i>
               .
               <b>
                49
               </b>
               , D480–D489 (2020).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 62" data-doi="10.1093/nar/gkaa1100" data-track="click" data-track-action="article reference" data-track-label="10.1093/nar/gkaa1100" href="https://doi-org.proxy.lib.ohio-state.edu/10.1093%2Fnar%2Fgkaa1100" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="Google Scholar reference 62" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=UniProt%3A%20the%20universal%20protein%20knowledgebase%20in%202021&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgkaa1100&amp;volume=49&amp;pages=D480-D489&amp;publication_year=2020" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="63.">
              <p class="c-article-references__text" id="ref-CR63">
               Steinegger, M. &amp; Söding, J. Clustering huge protein sequence sets in linear time.
               <i>
                Nat. Commun
               </i>
               .
               <b>
                9
               </b>
               , 2542 (2018).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 63" data-doi="10.1038/s41467-018-04964-5" data-track="click" data-track-action="article reference" data-track-label="10.1038/s41467-018-04964-5" href="https://doi-org.proxy.lib.ohio-state.edu/10.1038%2Fs41467-018-04964-5" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="ADS reference 63" data-track="click" data-track-action="ads reference" data-track-label="link" href="http://adsabs.harvard.edu.proxy.lib.ohio-state.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2018NatCo...9.2542S" rel="nofollow noopener">
                ADS
               </a>
               <a aria-label="PubMed reference 63" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29959318" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="PubMed Central reference 63" data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC6026198" rel="nofollow noopener">
                PubMed Central
               </a>
               <a aria-label="Google Scholar reference 63" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Clustering%20huge%20protein%20sequence%20sets%20in%20linear%20time&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-018-04964-5&amp;volume=9&amp;publication_year=2018&amp;author=Steinegger%2CM&amp;author=S%C3%B6ding%2CJ" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="64.">
              <p class="c-article-references__text" id="ref-CR64">
               Steinegger, M. &amp; Söding, J. MMseqs2 enables sensitive protein sequence searching for the analysis of massive data sets.
               <i>
                Nat. Biotechnol
               </i>
               .
               <b>
                35
               </b>
               , 1026–1028 (2017).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 64" data-doi="10.1038/nbt.3988" data-track="click" data-track-action="article reference" data-track-label="10.1038/nbt.3988" href="https://doi-org.proxy.lib.ohio-state.edu/10.1038%2Fnbt.3988" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="CAS reference 64" data-track="click" data-track-action="cas reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXhs1GqsLzE" rel="nofollow noopener">
                CAS
               </a>
               <a aria-label="PubMed reference 64" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29035372" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="Google Scholar reference 64" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=MMseqs2%20enables%20sensitive%20protein%20sequence%20searching%20for%20the%20analysis%20of%20massive%20data%20sets&amp;journal=Nat.%20Biotechnol.&amp;doi=10.1038%2Fnbt.3988&amp;volume=35&amp;pages=1026-1028&amp;publication_year=2017&amp;author=Steinegger%2CM&amp;author=S%C3%B6ding%2CJ" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="65.">
              <p class="c-article-references__text" id="ref-CR65">
               Deorowicz, S., Debudaj-Grabysz, A. &amp; Gudyś, A. FAMSA: fast and accurate multiple sequence alignment of huge protein families.
               <i>
                Sci. Rep
               </i>
               .
               <b>
                6
               </b>
               , 33964 (2016).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 65" data-doi="10.1038/srep33964" data-track="click" data-track-action="article reference" data-track-label="10.1038/srep33964" href="https://doi-org.proxy.lib.ohio-state.edu/10.1038%2Fsrep33964" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="ADS reference 65" data-track="click" data-track-action="ads reference" data-track-label="link" href="http://adsabs.harvard.edu.proxy.lib.ohio-state.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2016NatSR...633964D" rel="nofollow noopener">
                ADS
               </a>
               <a aria-label="CAS reference 65" data-track="click" data-track-action="cas reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XhsF2qs7fN" rel="nofollow noopener">
                CAS
               </a>
               <a aria-label="PubMed reference 65" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27670777" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="PubMed Central reference 65" data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC5037421" rel="nofollow noopener">
                PubMed Central
               </a>
               <a aria-label="Google Scholar reference 65" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=FAMSA%3A%20fast%20and%20accurate%20multiple%20sequence%20alignment%20of%20huge%20protein%20families&amp;journal=Sci.%20Rep.&amp;doi=10.1038%2Fsrep33964&amp;volume=6&amp;publication_year=2016&amp;author=Deorowicz%2CS&amp;author=Debudaj-Grabysz%2CA&amp;author=Gudy%C5%9B%2CA" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="66.">
              <p class="c-article-references__text" id="ref-CR66">
               Steinegger, M. et al. HH-suite3 for fast remote homology detection and deep protein annotation.
               <i>
                BMC Bioinformatics
               </i>
               <b>
                20
               </b>
               , 473 (2019).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 66" data-doi="10.1186/s12859-019-3019-7" data-track="click" data-track-action="article reference" data-track-label="10.1186/s12859-019-3019-7" href="https://doi-org.proxy.lib.ohio-state.edu/10.1186%2Fs12859-019-3019-7" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="PubMed reference 66" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31521110" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="PubMed Central reference 66" data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC6744700" rel="nofollow noopener">
                PubMed Central
               </a>
               <a aria-label="Google Scholar reference 66" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=HH-suite3%20for%20fast%20remote%20homology%20detection%20and%20deep%20protein%20annotation&amp;journal=BMC%20Bioinformatics&amp;doi=10.1186%2Fs12859-019-3019-7&amp;volume=20&amp;publication_year=2019&amp;author=Steinegger%2CM" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="67.">
              <p class="c-article-references__text" id="ref-CR67">
               Suzek, B. E., Wang, Y., Huang, H., McGarvey, P. B. &amp; Wu, C. H. UniRef clusters: a comprehensive and scalable alternative for improving sequence similarity searches.
               <i>
                Bioinformatics
               </i>
               <b>
                31
               </b>
               , 926–932 (2015).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 67" data-doi="10.1093/bioinformatics/btu739" data-track="click" data-track-action="article reference" data-track-label="10.1093/bioinformatics/btu739" href="https://doi-org.proxy.lib.ohio-state.edu/10.1093%2Fbioinformatics%2Fbtu739" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="CAS reference 67" data-track="click" data-track-action="cas reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BC28Xht1Gntb7F" rel="nofollow noopener">
                CAS
               </a>
               <a aria-label="PubMed reference 67" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25398609" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="Google Scholar reference 67" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=UniRef%20clusters%3A%20a%20comprehensive%20and%20scalable%20alternative%20for%20improving%20sequence%20similarity%20searches&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtu739&amp;volume=31&amp;pages=926-932&amp;publication_year=2015&amp;author=Suzek%2CBE&amp;author=Wang%2CY&amp;author=Huang%2CH&amp;author=McGarvey%2CPB&amp;author=Wu%2CCH" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="68.">
              <p class="c-article-references__text" id="ref-CR68">
               Eddy, S. R. Accelerated profile HMM searches.
               <i>
                PLOS Comput. Biol
               </i>
               .
               <b>
                7
               </b>
               , e1002195 (2011).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 68" data-doi="10.1371/journal.pcbi.1002195" data-track="click" data-track-action="article reference" data-track-label="10.1371/journal.pcbi.1002195" href="https://doi-org.proxy.lib.ohio-state.edu/10.1371%2Fjournal.pcbi.1002195" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="ADS reference 68" data-track="click" data-track-action="ads reference" data-track-label="link" href="http://adsabs.harvard.edu.proxy.lib.ohio-state.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2011PLSCB...7E2195E" rel="nofollow noopener">
                ADS
               </a>
               <a aria-label="MathSciNet reference 68" data-track="click" data-track-action="mathscinet reference" data-track-label="link" href="http://www-ams-org.proxy.lib.ohio-state.edu/mathscinet-getitem?mr=2859646" rel="nofollow noopener">
                MathSciNet
               </a>
               <a aria-label="CAS reference 68" data-track="click" data-track-action="cas reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BC3MXhsVCku7rL" rel="nofollow noopener">
                CAS
               </a>
               <a aria-label="PubMed reference 68" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22039361" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="PubMed Central reference 68" data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC3197634" rel="nofollow noopener">
                PubMed Central
               </a>
               <a aria-label="Google Scholar reference 68" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Accelerated%20profile%20HMM%20searches&amp;journal=PLOS%20Comput.%20Biol.&amp;doi=10.1371%2Fjournal.pcbi.1002195&amp;volume=7&amp;publication_year=2011&amp;author=Eddy%2CSR" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="69.">
              <p class="c-article-references__text" id="ref-CR69">
               Eastman, P. et al. OpenMM 7: rapid development of high performance algorithms for molecular dynamics.
               <i>
                PLOS Comput. Biol
               </i>
               .
               <b>
                13
               </b>
               , e1005659 (2017).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 69" data-doi="10.1371/journal.pcbi.1005659" data-track="click" data-track-action="article reference" data-track-label="10.1371/journal.pcbi.1005659" href="https://doi-org.proxy.lib.ohio-state.edu/10.1371%2Fjournal.pcbi.1005659" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="PubMed reference 69" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28746339" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="PubMed Central reference 69" data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC5549999" rel="nofollow noopener">
                PubMed Central
               </a>
               <a aria-label="Google Scholar reference 69" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=OpenMM%207%3A%20rapid%20development%20of%20high%20performance%20algorithms%20for%20molecular%20dynamics&amp;journal=PLOS%20Comput.%20Biol.&amp;doi=10.1371%2Fjournal.pcbi.1005659&amp;volume=13&amp;publication_year=2017&amp;author=Eastman%2CP" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="70.">
              <p class="c-article-references__text" id="ref-CR70">
               Ashish, A. M. A. et al. TensorFlow: large-scale machine learning on heterogeneous systems. Preprint at
               <a data-track="click" data-track-action="external reference" data-track-label="https://arxiv-org.proxy.lib.ohio-state.edu/abs/1603.04467" href="https://arxiv-org.proxy.lib.ohio-state.edu/abs/1603.04467">
                https://arxiv-org.proxy.lib.ohio-state.edu/abs/1603.04467
               </a>
               (2015).
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="71.">
              <p class="c-article-references__text" id="ref-CR71">
               Reynolds, M. et al. Open sourcing Sonnet – a new library for constructing neural networks.
               <i>
                DeepMind
               </i>
               <a data-track="click" data-track-action="external reference" data-track-label="https://deepmind.com/blog/open-sourcing-sonnet/" href="https://deepmind.com/blog/open-sourcing-sonnet/">
                https://deepmind.com/blog/open-sourcing-sonnet/
               </a>
               (7 April 2017).
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="72.">
              <p class="c-article-references__text" id="ref-CR72">
               Harris, C. R. et al. Array programming with NumPy.
               <i>
                Nature
               </i>
               <b>
                585
               </b>
               , 357–362 (2020).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 72" data-doi="10.1038/s41586-020-2649-2" data-track="click" data-track-action="article reference" data-track-label="10.1038/s41586-020-2649-2" href="https://doi-org.proxy.lib.ohio-state.edu/10.1038%2Fs41586-020-2649-2" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="ADS reference 72" data-track="click" data-track-action="ads reference" data-track-label="link" href="http://adsabs.harvard.edu.proxy.lib.ohio-state.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2020Natur.585..357H" rel="nofollow noopener">
                ADS
               </a>
               <a aria-label="CAS reference 72" data-track="click" data-track-action="cas reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXitlWmsbbN" rel="nofollow noopener">
                CAS
               </a>
               <a aria-label="PubMed reference 72" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32939066" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="PubMed Central reference 72" data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC7759461" rel="nofollow noopener">
                PubMed Central
               </a>
               <a aria-label="Google Scholar reference 72" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Array%20programming%20with%20NumPy&amp;journal=Nature&amp;doi=10.1038%2Fs41586-020-2649-2&amp;volume=585&amp;pages=357-362&amp;publication_year=2020&amp;author=Harris%2CCR" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="73.">
              <p class="c-article-references__text" id="ref-CR73">
               Van Rossum, G. &amp; Drake, F. L.
               <i>
                Python 3 Reference Manual
               </i>
               (CreateSpace, 2009).
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="74.">
              <p class="c-article-references__text" id="ref-CR74">
               Bisong, E. in
               <i>
                Building Machine Learning and Deep Learning Models on Google Cloud Platform: A Comprehensive Guide for Beginners
               </i>
               59–64 (Apress, 2019).
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="75.">
              <p class="c-article-references__text" id="ref-CR75">
               TensorFlow. XLA: Optimizing Compiler for TensorFlow.
               <a data-track="click" data-track-action="external reference" data-track-label="https://www.tensorflow.org/xla" href="https://www.tensorflow.org/xla">
                https://www.tensorflow.org/xla
               </a>
               (2018).
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="76.">
              <p class="c-article-references__text" id="ref-CR76">
               Wu, T., Hou, J., Adhikari, B. &amp; Cheng, J. Analysis of several key factors influencing deep learning-based inter-residue contact prediction.
               <i>
                Bioinformatics
               </i>
               <b>
                36
               </b>
               , 1091–1098 (2020).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 76" data-doi="10.1093/bioinformatics/btz679" data-track="click" data-track-action="article reference" data-track-label="10.1093/bioinformatics/btz679" href="https://doi-org.proxy.lib.ohio-state.edu/10.1093%2Fbioinformatics%2Fbtz679" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="CAS reference 76" data-track="click" data-track-action="cas reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXisVOrtbvJ" rel="nofollow noopener">
                CAS
               </a>
               <a aria-label="PubMed reference 76" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31504181" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="Google Scholar reference 76" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Analysis%20of%20several%20key%20factors%20influencing%20deep%20learning-based%20inter-residue%20contact%20prediction&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtz679&amp;volume=36&amp;pages=1091-1098&amp;publication_year=2020&amp;author=Wu%2CT&amp;author=Hou%2CJ&amp;author=Adhikari%2CB&amp;author=Cheng%2CJ" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="77.">
              <p class="c-article-references__text" id="ref-CR77">
               Jiang, W. et al. MrpH, a new class of metal-binding adhesin, requires zinc to mediate biofilm formation.
               <i>
                PLoS Pathog
               </i>
               .
               <b>
                16
               </b>
               , e1008707 (2020).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 77" data-doi="10.1371/journal.ppat.1008707" data-track="click" data-track-action="article reference" data-track-label="10.1371/journal.ppat.1008707" href="https://doi-org.proxy.lib.ohio-state.edu/10.1371%2Fjournal.ppat.1008707" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="CAS reference 77" data-track="click" data-track-action="cas reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXhs1Glt7fI" rel="nofollow noopener">
                CAS
               </a>
               <a aria-label="PubMed reference 77" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32780778" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="PubMed Central reference 77" data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC7444556" rel="nofollow noopener">
                PubMed Central
               </a>
               <a aria-label="Google Scholar reference 77" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=MrpH%2C%20a%20new%20class%20of%20metal-binding%20adhesin%2C%20requires%20zinc%20to%20mediate%20biofilm%20formation&amp;journal=PLoS%20Pathog.&amp;doi=10.1371%2Fjournal.ppat.1008707&amp;volume=16&amp;publication_year=2020&amp;author=Jiang%2CW" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="78.">
              <p class="c-article-references__text" id="ref-CR78">
               Dunne, M., Ernst, P., Sobieraj, A., Pluckthun, A. &amp; Loessner, M. J. The M23 peptidase domain of the Staphylococcal phage 2638A endolysin.
               <i>
                PDB
               </i>
               <a data-track="click" data-track-action="external reference" data-track-label="10.2210/pdb6YJ1/pdb" href="https://doi-org.proxy.lib.ohio-state.edu/10.2210/pdb6YJ1/pdb">
                https://doi-org.proxy.lib.ohio-state.edu/10.2210/pdb6YJ1/pdb
               </a>
               (2020).
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="79.">
              <p class="c-article-references__text" id="ref-CR79">
               Drobysheva, A. V. et al. Structure and function of virion RNA polymerase of a crAss-like phage.
               <i>
                Nature
               </i>
               <b>
                589
               </b>
               , 306–309 (2021).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 79" data-doi="10.1038/s41586-020-2921-5" data-track="click" data-track-action="article reference" data-track-label="10.1038/s41586-020-2921-5" href="https://doi-org.proxy.lib.ohio-state.edu/10.1038%2Fs41586-020-2921-5" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="ADS reference 79" data-track="click" data-track-action="ads reference" data-track-label="link" href="http://adsabs.harvard.edu.proxy.lib.ohio-state.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2021Natur.589..306D" rel="nofollow noopener">
                ADS
               </a>
               <a aria-label="CAS reference 79" data-track="click" data-track-action="cas reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXitlOgs7jI" rel="nofollow noopener">
                CAS
               </a>
               <a aria-label="PubMed reference 79" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33208949" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="Google Scholar reference 79" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Structure%20and%20function%20of%20virion%20RNA%20polymerase%20of%20a%20crAss-like%20phage&amp;journal=Nature&amp;doi=10.1038%2Fs41586-020-2921-5&amp;volume=589&amp;pages=306-309&amp;publication_year=2021&amp;author=Drobysheva%2CAV" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="80.">
              <p class="c-article-references__text" id="ref-CR80">
               Flaugnatti, N. et al. Structural basis for loading and inhibition of a bacterial T6SS phospholipase effector by the VgrG spike.
               <i>
                EMBO J
               </i>
               .
               <b>
                39
               </b>
               , e104129 (2020).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 80" data-doi="10.15252/embj.2019104129" data-track="click" data-track-action="article reference" data-track-label="10.15252/embj.2019104129" href="https://doi-org.proxy.lib.ohio-state.edu/10.15252%2Fembj.2019104129" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="CAS reference 80" data-track="click" data-track-action="cas reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXotFCnu7s%3D" rel="nofollow noopener">
                CAS
               </a>
               <a aria-label="PubMed reference 80" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32350888" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="PubMed Central reference 80" data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC7265238" rel="nofollow noopener">
                PubMed Central
               </a>
               <a aria-label="Google Scholar reference 80" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Structural%20basis%20for%20loading%20and%20inhibition%20of%20a%20bacterial%20T6SS%20phospholipase%20effector%20by%20the%20VgrG%20spike&amp;journal=EMBO%20J.&amp;doi=10.15252%2Fembj.2019104129&amp;volume=39&amp;publication_year=2020&amp;author=Flaugnatti%2CN" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="81.">
              <p class="c-article-references__text" id="ref-CR81">
               ElGamacy, M. et al. An interface-driven design strategy yields a novel, corrugated protein architecture.
               <i>
                ACS Synth. Biol
               </i>
               .
               <b>
                7
               </b>
               , 2226–2235 (2018).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 81" data-doi="10.1021/acssynbio.8b00224" data-track="click" data-track-action="article reference" data-track-label="10.1021/acssynbio.8b00224" href="https://doi-org.proxy.lib.ohio-state.edu/10.1021%2Facssynbio.8b00224" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="CAS reference 81" data-track="click" data-track-action="cas reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhsFyqtbnP" rel="nofollow noopener">
                CAS
               </a>
               <a aria-label="PubMed reference 81" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30148951" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="Google Scholar reference 81" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=An%20interface-driven%20design%20strategy%20yields%20a%20novel%2C%20corrugated%20protein%20architecture&amp;journal=ACS%20Synth.%20Biol.&amp;doi=10.1021%2Facssynbio.8b00224&amp;volume=7&amp;pages=2226-2235&amp;publication_year=2018&amp;author=ElGamacy%2CM" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="82.">
              <p class="c-article-references__text" id="ref-CR82">
               Lim, C. J. et al. The structure of human CST reveals a decameric assembly bound to telomeric DNA.
               <i>
                Science
               </i>
               <b>
                368
               </b>
               , 1081–1085 (2020).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 82" data-doi="10.1126/science.aaz9649" data-track="click" data-track-action="article reference" data-track-label="10.1126/science.aaz9649" href="https://doi-org.proxy.lib.ohio-state.edu/10.1126%2Fscience.aaz9649" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="ADS reference 82" data-track="click" data-track-action="ads reference" data-track-label="link" href="http://adsabs.harvard.edu.proxy.lib.ohio-state.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2020Sci...368.1081L" rel="nofollow noopener">
                ADS
               </a>
               <a aria-label="CAS reference 82" data-track="click" data-track-action="cas reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtFSqt7fI" rel="nofollow noopener">
                CAS
               </a>
               <a aria-label="PubMed reference 82" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32499435" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="PubMed Central reference 82" data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC7559292" rel="nofollow noopener">
                PubMed Central
               </a>
               <a aria-label="Google Scholar reference 82" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20structure%20of%20human%20CST%20reveals%20a%20decameric%20assembly%20bound%20to%20telomeric%20DNA&amp;journal=Science&amp;doi=10.1126%2Fscience.aaz9649&amp;volume=368&amp;pages=1081-1085&amp;publication_year=2020&amp;author=Lim%2CCJ" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="83.">
              <p class="c-article-references__text" id="ref-CR83">
               Debruycker, V. et al. An embedded lipid in the multidrug transporter LmrP suggests a mechanism for polyspecificity.
               <i>
                Nat. Struct. Mol. Biol
               </i>
               .
               <b>
                27
               </b>
               , 829–835 (2020).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 83" data-doi="10.1038/s41594-020-0464-y" data-track="click" data-track-action="article reference" data-track-label="10.1038/s41594-020-0464-y" href="https://doi-org.proxy.lib.ohio-state.edu/10.1038%2Fs41594-020-0464-y" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="CAS reference 83" data-track="click" data-track-action="cas reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsVKkt7bE" rel="nofollow noopener">
                CAS
               </a>
               <a aria-label="PubMed reference 83" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32719456" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="PubMed Central reference 83" data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC7951658" rel="nofollow noopener">
                PubMed Central
               </a>
               <a aria-label="Google Scholar reference 83" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=An%20embedded%20lipid%20in%20the%20multidrug%20transporter%20LmrP%20suggests%20a%20mechanism%20for%20polyspecificity&amp;journal=Nat.%20Struct.%20Mol.%20Biol.&amp;doi=10.1038%2Fs41594-020-0464-y&amp;volume=27&amp;pages=829-835&amp;publication_year=2020&amp;author=Debruycker%2CV" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
             <li class="c-article-references__item js-c-reading-companion-references-item" data-counter="84.">
              <p class="c-article-references__text" id="ref-CR84">
               Flower, T. G. et al. Structure of SARS-CoV-2 ORF8, a rapidly evolving immune evasion protein.
               <i>
                Proc. Natl Acad. Sci. USA
               </i>
               <b>
                118
               </b>
               , e2021785118 (2021).
              </p>
              <p class="c-article-references__links u-hide-print">
               <a aria-label="Article reference 84" data-doi="10.1073/pnas.2021785118" data-track="click" data-track-action="article reference" data-track-label="10.1073/pnas.2021785118" href="https://doi-org.proxy.lib.ohio-state.edu/10.1073%2Fpnas.2021785118" rel="nofollow noopener">
                Article
               </a>
               <a aria-label="CAS reference 84" data-track="click" data-track-action="cas reference" data-track-label="link" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXhsVCitb4%3D" rel="nofollow noopener">
                CAS
               </a>
               <a aria-label="PubMed reference 84" data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33361333" rel="nofollow noopener">
                PubMed
               </a>
               <a aria-label="Google Scholar reference 84" data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Structure%20of%20SARS-CoV-2%20ORF8%2C%20a%20rapidly%20evolving%20immune%20evasion%20protein&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.2021785118&amp;volume=118&amp;publication_year=2021&amp;author=Flower%2CTG" rel="nofollow noopener">
                Google Scholar
               </a>
              </p>
             </li>
            </ol>
            <p class="c-article-references__download u-hide-print">
             <a data-track="click" data-track-action="download citation references" data-track-label="link" href="https://citation-needed-springer-com.proxy.lib.ohio-state.edu/v2/references/10.1038/s41586-021-03819-2?format=refman&amp;flavour=references" rel="nofollow">
              Download references
              <svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16">
               <use xlink:href="#icon-download" xmlns:xlink="http://www.w3.org/1999/xlink">
               </use>
              </svg>
             </a>
            </p>
           </div>
          </div>
         </div>
        </section>
       </div>
       <section data-title="Acknowledgements">
        <div class="c-article-section" id="Ack1-section">
         <h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Ack1">
          Acknowledgements
         </h2>
         <div class="c-article-section__content" id="Ack1-content">
          <p>
           We thank A. Rrustemi, A. Gu, A. Guseynov, B. Hechtman, C. Beattie, C. Jones, C. Donner, E. Parisotto, E. Elsen, F. Popovici, G. Necula, H. Maclean, J. Menick, J. Kirkpatrick, J. Molloy, J. Yim, J. Stanway, K. Simonyan, L. Sifre, L. Martens, M. Johnson, M. O’Neill, N. Antropova, R. Hadsell, S. Blackwell, S. Das, S. Hou, S. Gouws, S. Wheelwright, T. Hennigan, T. Ward, Z. Wu, Ž. Avsec and the Research Platform Team for their contributions; M. Mirdita for his help with the datasets; M. Piovesan-Forster, A. Nelson and R. Kemp for their help managing the project; the JAX, TensorFlow and XLA teams for detailed support and enabling machine learning models of the complexity of AlphaFold; our colleagues at DeepMind, Google and Alphabet for their encouragement and support; and J. Moult and the CASP14 organizers, and the experimentalists whose structures enabled the assessment. M.S. acknowledges support from the National Research Foundation of Korea grant (2019R1A6A1A10073437, 2020M3A9G7103933) and the Creative-Pioneering Researchers Program through Seoul National University.
          </p>
         </div>
        </div>
       </section>
       <section aria-labelledby="author-information" data-title="Author information">
        <div class="c-article-section" id="author-information-section">
         <h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="author-information">
          Author information
         </h2>
         <div class="c-article-section__content" id="author-information-content">
          <span class="c-article-author-information__subtitle u-visually-hidden" id="author-notes">
           Author notes
          </span>
          <ol class="c-article-author-information__list">
           <li class="c-article-author-information__item" id="na1">
            <p>
             These authors contributed equally: John Jumper, Richard Evans, Alexander Pritzel, Tim Green, Michael Figurnov, Olaf Ronneberger, Kathryn Tunyasuvunakool, Russ Bates, Augustin Žídek, Anna Potapenko, Alex Bridgland, Clemens Meyer, Simon A. A. Kohl, Andrew J. Ballard, Andrew Cowie, Bernardino Romera-Paredes, Stanislav Nikolov, Rishub Jain, Demis Hassabis
            </p>
           </li>
          </ol>
          <h3 class="c-article__sub-heading" id="affiliations">
           Authors and Affiliations
          </h3>
          <ol class="c-article-author-affiliation__list">
           <li id="Aff1">
            <p class="c-article-author-affiliation__address">
             DeepMind, London, UK
            </p>
            <p class="c-article-author-affiliation__authors-list">
             John Jumper, Richard Evans, Alexander Pritzel, Tim Green, Michael Figurnov, Olaf Ronneberger, Kathryn Tunyasuvunakool, Russ Bates, Augustin Žídek, Anna Potapenko, Alex Bridgland, Clemens Meyer, Simon A. A. Kohl, Andrew J. Ballard, Andrew Cowie, Bernardino Romera-Paredes, Stanislav Nikolov, Rishub Jain, Jonas Adler, Trevor Back, Stig Petersen, David Reiman, Ellen Clancy, Michal Zielinski, Michalina Pacholska, Tamas Berghammer, Sebastian Bodenstein, David Silver, Oriol Vinyals, Andrew W. Senior, Koray Kavukcuoglu, Pushmeet Kohli &amp; Demis Hassabis
            </p>
           </li>
           <li id="Aff2">
            <p class="c-article-author-affiliation__address">
             School of Biological Sciences, Seoul National University, Seoul, South Korea
            </p>
            <p class="c-article-author-affiliation__authors-list">
             Martin Steinegger
            </p>
           </li>
           <li id="Aff3">
            <p class="c-article-author-affiliation__address">
             Artificial Intelligence Institute, Seoul National University, Seoul, South Korea
            </p>
            <p class="c-article-author-affiliation__authors-list">
             Martin Steinegger
            </p>
           </li>
          </ol>
          <div class="u-js-hide u-hide-print" data-test="author-info">
           <span class="c-article__sub-heading">
            Authors
           </span>
           <ol class="c-article-authors-search u-list-reset">
            <li id="auth-John-Jumper-Aff1">
             <span class="c-article-authors-search__title u-h3 js-search-name">
              John Jumper
             </span>
             <div class="c-article-authors-search__list">
              <div class="c-article-authors-search__item c-article-authors-search__list-item--left">
               <a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=John%20Jumper" rel="nofollow">
                View author publications
               </a>
              </div>
              <div class="c-article-authors-search__item c-article-authors-search__list-item--right">
               <p class="search-in-title-js c-article-authors-search__text">
                You can also search for this author in
                <span class="c-article-identifiers">
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=search&amp;term=John%20Jumper" rel="nofollow">
                  PubMed
                 </a>
                 <span class="u-hide">
                 </span>
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22John%20Jumper%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">
                  Google Scholar
                 </a>
                </span>
               </p>
              </div>
             </div>
            </li>
            <li id="auth-Richard-Evans-Aff1">
             <span class="c-article-authors-search__title u-h3 js-search-name">
              Richard Evans
             </span>
             <div class="c-article-authors-search__list">
              <div class="c-article-authors-search__item c-article-authors-search__list-item--left">
               <a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Richard%20Evans" rel="nofollow">
                View author publications
               </a>
              </div>
              <div class="c-article-authors-search__item c-article-authors-search__list-item--right">
               <p class="search-in-title-js c-article-authors-search__text">
                You can also search for this author in
                <span class="c-article-identifiers">
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=search&amp;term=Richard%20Evans" rel="nofollow">
                  PubMed
                 </a>
                 <span class="u-hide">
                 </span>
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Richard%20Evans%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">
                  Google Scholar
                 </a>
                </span>
               </p>
              </div>
             </div>
            </li>
            <li id="auth-Alexander-Pritzel-Aff1">
             <span class="c-article-authors-search__title u-h3 js-search-name">
              Alexander Pritzel
             </span>
             <div class="c-article-authors-search__list">
              <div class="c-article-authors-search__item c-article-authors-search__list-item--left">
               <a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Alexander%20Pritzel" rel="nofollow">
                View author publications
               </a>
              </div>
              <div class="c-article-authors-search__item c-article-authors-search__list-item--right">
               <p class="search-in-title-js c-article-authors-search__text">
                You can also search for this author in
                <span class="c-article-identifiers">
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=search&amp;term=Alexander%20Pritzel" rel="nofollow">
                  PubMed
                 </a>
                 <span class="u-hide">
                 </span>
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Alexander%20Pritzel%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">
                  Google Scholar
                 </a>
                </span>
               </p>
              </div>
             </div>
            </li>
            <li id="auth-Tim-Green-Aff1">
             <span class="c-article-authors-search__title u-h3 js-search-name">
              Tim Green
             </span>
             <div class="c-article-authors-search__list">
              <div class="c-article-authors-search__item c-article-authors-search__list-item--left">
               <a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Tim%20Green" rel="nofollow">
                View author publications
               </a>
              </div>
              <div class="c-article-authors-search__item c-article-authors-search__list-item--right">
               <p class="search-in-title-js c-article-authors-search__text">
                You can also search for this author in
                <span class="c-article-identifiers">
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=search&amp;term=Tim%20Green" rel="nofollow">
                  PubMed
                 </a>
                 <span class="u-hide">
                 </span>
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Tim%20Green%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">
                  Google Scholar
                 </a>
                </span>
               </p>
              </div>
             </div>
            </li>
            <li id="auth-Michael-Figurnov-Aff1">
             <span class="c-article-authors-search__title u-h3 js-search-name">
              Michael Figurnov
             </span>
             <div class="c-article-authors-search__list">
              <div class="c-article-authors-search__item c-article-authors-search__list-item--left">
               <a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Michael%20Figurnov" rel="nofollow">
                View author publications
               </a>
              </div>
              <div class="c-article-authors-search__item c-article-authors-search__list-item--right">
               <p class="search-in-title-js c-article-authors-search__text">
                You can also search for this author in
                <span class="c-article-identifiers">
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=search&amp;term=Michael%20Figurnov" rel="nofollow">
                  PubMed
                 </a>
                 <span class="u-hide">
                 </span>
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Michael%20Figurnov%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">
                  Google Scholar
                 </a>
                </span>
               </p>
              </div>
             </div>
            </li>
            <li id="auth-Olaf-Ronneberger-Aff1">
             <span class="c-article-authors-search__title u-h3 js-search-name">
              Olaf Ronneberger
             </span>
             <div class="c-article-authors-search__list">
              <div class="c-article-authors-search__item c-article-authors-search__list-item--left">
               <a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Olaf%20Ronneberger" rel="nofollow">
                View author publications
               </a>
              </div>
              <div class="c-article-authors-search__item c-article-authors-search__list-item--right">
               <p class="search-in-title-js c-article-authors-search__text">
                You can also search for this author in
                <span class="c-article-identifiers">
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=search&amp;term=Olaf%20Ronneberger" rel="nofollow">
                  PubMed
                 </a>
                 <span class="u-hide">
                 </span>
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Olaf%20Ronneberger%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">
                  Google Scholar
                 </a>
                </span>
               </p>
              </div>
             </div>
            </li>
            <li id="auth-Kathryn-Tunyasuvunakool-Aff1">
             <span class="c-article-authors-search__title u-h3 js-search-name">
              Kathryn Tunyasuvunakool
             </span>
             <div class="c-article-authors-search__list">
              <div class="c-article-authors-search__item c-article-authors-search__list-item--left">
               <a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Kathryn%20Tunyasuvunakool" rel="nofollow">
                View author publications
               </a>
              </div>
              <div class="c-article-authors-search__item c-article-authors-search__list-item--right">
               <p class="search-in-title-js c-article-authors-search__text">
                You can also search for this author in
                <span class="c-article-identifiers">
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=search&amp;term=Kathryn%20Tunyasuvunakool" rel="nofollow">
                  PubMed
                 </a>
                 <span class="u-hide">
                 </span>
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Kathryn%20Tunyasuvunakool%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">
                  Google Scholar
                 </a>
                </span>
               </p>
              </div>
             </div>
            </li>
            <li id="auth-Russ-Bates-Aff1">
             <span class="c-article-authors-search__title u-h3 js-search-name">
              Russ Bates
             </span>
             <div class="c-article-authors-search__list">
              <div class="c-article-authors-search__item c-article-authors-search__list-item--left">
               <a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Russ%20Bates" rel="nofollow">
                View author publications
               </a>
              </div>
              <div class="c-article-authors-search__item c-article-authors-search__list-item--right">
               <p class="search-in-title-js c-article-authors-search__text">
                You can also search for this author in
                <span class="c-article-identifiers">
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=search&amp;term=Russ%20Bates" rel="nofollow">
                  PubMed
                 </a>
                 <span class="u-hide">
                 </span>
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Russ%20Bates%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">
                  Google Scholar
                 </a>
                </span>
               </p>
              </div>
             </div>
            </li>
            <li id="auth-Augustin-__dek-Aff1">
             <span class="c-article-authors-search__title u-h3 js-search-name">
              Augustin Žídek
             </span>
             <div class="c-article-authors-search__list">
              <div class="c-article-authors-search__item c-article-authors-search__list-item--left">
               <a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Augustin%20%C5%BD%C3%ADdek" rel="nofollow">
                View author publications
               </a>
              </div>
              <div class="c-article-authors-search__item c-article-authors-search__list-item--right">
               <p class="search-in-title-js c-article-authors-search__text">
                You can also search for this author in
                <span class="c-article-identifiers">
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=search&amp;term=Augustin%20%C5%BD%C3%ADdek" rel="nofollow">
                  PubMed
                 </a>
                 <span class="u-hide">
                 </span>
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Augustin%20%C5%BD%C3%ADdek%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">
                  Google Scholar
                 </a>
                </span>
               </p>
              </div>
             </div>
            </li>
            <li id="auth-Anna-Potapenko-Aff1">
             <span class="c-article-authors-search__title u-h3 js-search-name">
              Anna Potapenko
             </span>
             <div class="c-article-authors-search__list">
              <div class="c-article-authors-search__item c-article-authors-search__list-item--left">
               <a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Anna%20Potapenko" rel="nofollow">
                View author publications
               </a>
              </div>
              <div class="c-article-authors-search__item c-article-authors-search__list-item--right">
               <p class="search-in-title-js c-article-authors-search__text">
                You can also search for this author in
                <span class="c-article-identifiers">
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=search&amp;term=Anna%20Potapenko" rel="nofollow">
                  PubMed
                 </a>
                 <span class="u-hide">
                 </span>
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Anna%20Potapenko%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">
                  Google Scholar
                 </a>
                </span>
               </p>
              </div>
             </div>
            </li>
            <li id="auth-Alex-Bridgland-Aff1">
             <span class="c-article-authors-search__title u-h3 js-search-name">
              Alex Bridgland
             </span>
             <div class="c-article-authors-search__list">
              <div class="c-article-authors-search__item c-article-authors-search__list-item--left">
               <a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Alex%20Bridgland" rel="nofollow">
                View author publications
               </a>
              </div>
              <div class="c-article-authors-search__item c-article-authors-search__list-item--right">
               <p class="search-in-title-js c-article-authors-search__text">
                You can also search for this author in
                <span class="c-article-identifiers">
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=search&amp;term=Alex%20Bridgland" rel="nofollow">
                  PubMed
                 </a>
                 <span class="u-hide">
                 </span>
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Alex%20Bridgland%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">
                  Google Scholar
                 </a>
                </span>
               </p>
              </div>
             </div>
            </li>
            <li id="auth-Clemens-Meyer-Aff1">
             <span class="c-article-authors-search__title u-h3 js-search-name">
              Clemens Meyer
             </span>
             <div class="c-article-authors-search__list">
              <div class="c-article-authors-search__item c-article-authors-search__list-item--left">
               <a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Clemens%20Meyer" rel="nofollow">
                View author publications
               </a>
              </div>
              <div class="c-article-authors-search__item c-article-authors-search__list-item--right">
               <p class="search-in-title-js c-article-authors-search__text">
                You can also search for this author in
                <span class="c-article-identifiers">
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=search&amp;term=Clemens%20Meyer" rel="nofollow">
                  PubMed
                 </a>
                 <span class="u-hide">
                 </span>
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Clemens%20Meyer%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">
                  Google Scholar
                 </a>
                </span>
               </p>
              </div>
             </div>
            </li>
            <li id="auth-Simon_A__A_-Kohl-Aff1">
             <span class="c-article-authors-search__title u-h3 js-search-name">
              Simon A. A. Kohl
             </span>
             <div class="c-article-authors-search__list">
              <div class="c-article-authors-search__item c-article-authors-search__list-item--left">
               <a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Simon%20A.%20A.%20Kohl" rel="nofollow">
                View author publications
               </a>
              </div>
              <div class="c-article-authors-search__item c-article-authors-search__list-item--right">
               <p class="search-in-title-js c-article-authors-search__text">
                You can also search for this author in
                <span class="c-article-identifiers">
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=search&amp;term=Simon%20A.%20A.%20Kohl" rel="nofollow">
                  PubMed
                 </a>
                 <span class="u-hide">
                 </span>
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Simon%20A.%20A.%20Kohl%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">
                  Google Scholar
                 </a>
                </span>
               </p>
              </div>
             </div>
            </li>
            <li id="auth-Andrew_J_-Ballard-Aff1">
             <span class="c-article-authors-search__title u-h3 js-search-name">
              Andrew J. Ballard
             </span>
             <div class="c-article-authors-search__list">
              <div class="c-article-authors-search__item c-article-authors-search__list-item--left">
               <a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Andrew%20J.%20Ballard" rel="nofollow">
                View author publications
               </a>
              </div>
              <div class="c-article-authors-search__item c-article-authors-search__list-item--right">
               <p class="search-in-title-js c-article-authors-search__text">
                You can also search for this author in
                <span class="c-article-identifiers">
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=search&amp;term=Andrew%20J.%20Ballard" rel="nofollow">
                  PubMed
                 </a>
                 <span class="u-hide">
                 </span>
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Andrew%20J.%20Ballard%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">
                  Google Scholar
                 </a>
                </span>
               </p>
              </div>
             </div>
            </li>
            <li id="auth-Andrew-Cowie-Aff1">
             <span class="c-article-authors-search__title u-h3 js-search-name">
              Andrew Cowie
             </span>
             <div class="c-article-authors-search__list">
              <div class="c-article-authors-search__item c-article-authors-search__list-item--left">
               <a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Andrew%20Cowie" rel="nofollow">
                View author publications
               </a>
              </div>
              <div class="c-article-authors-search__item c-article-authors-search__list-item--right">
               <p class="search-in-title-js c-article-authors-search__text">
                You can also search for this author in
                <span class="c-article-identifiers">
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=search&amp;term=Andrew%20Cowie" rel="nofollow">
                  PubMed
                 </a>
                 <span class="u-hide">
                 </span>
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Andrew%20Cowie%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">
                  Google Scholar
                 </a>
                </span>
               </p>
              </div>
             </div>
            </li>
            <li id="auth-Bernardino-Romera_Paredes-Aff1">
             <span class="c-article-authors-search__title u-h3 js-search-name">
              Bernardino Romera-Paredes
             </span>
             <div class="c-article-authors-search__list">
              <div class="c-article-authors-search__item c-article-authors-search__list-item--left">
               <a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Bernardino%20Romera-Paredes" rel="nofollow">
                View author publications
               </a>
              </div>
              <div class="c-article-authors-search__item c-article-authors-search__list-item--right">
               <p class="search-in-title-js c-article-authors-search__text">
                You can also search for this author in
                <span class="c-article-identifiers">
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=search&amp;term=Bernardino%20Romera-Paredes" rel="nofollow">
                  PubMed
                 </a>
                 <span class="u-hide">
                 </span>
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Bernardino%20Romera-Paredes%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">
                  Google Scholar
                 </a>
                </span>
               </p>
              </div>
             </div>
            </li>
            <li id="auth-Stanislav-Nikolov-Aff1">
             <span class="c-article-authors-search__title u-h3 js-search-name">
              Stanislav Nikolov
             </span>
             <div class="c-article-authors-search__list">
              <div class="c-article-authors-search__item c-article-authors-search__list-item--left">
               <a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Stanislav%20Nikolov" rel="nofollow">
                View author publications
               </a>
              </div>
              <div class="c-article-authors-search__item c-article-authors-search__list-item--right">
               <p class="search-in-title-js c-article-authors-search__text">
                You can also search for this author in
                <span class="c-article-identifiers">
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=search&amp;term=Stanislav%20Nikolov" rel="nofollow">
                  PubMed
                 </a>
                 <span class="u-hide">
                 </span>
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Stanislav%20Nikolov%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">
                  Google Scholar
                 </a>
                </span>
               </p>
              </div>
             </div>
            </li>
            <li id="auth-Rishub-Jain-Aff1">
             <span class="c-article-authors-search__title u-h3 js-search-name">
              Rishub Jain
             </span>
             <div class="c-article-authors-search__list">
              <div class="c-article-authors-search__item c-article-authors-search__list-item--left">
               <a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Rishub%20Jain" rel="nofollow">
                View author publications
               </a>
              </div>
              <div class="c-article-authors-search__item c-article-authors-search__list-item--right">
               <p class="search-in-title-js c-article-authors-search__text">
                You can also search for this author in
                <span class="c-article-identifiers">
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=search&amp;term=Rishub%20Jain" rel="nofollow">
                  PubMed
                 </a>
                 <span class="u-hide">
                 </span>
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Rishub%20Jain%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">
                  Google Scholar
                 </a>
                </span>
               </p>
              </div>
             </div>
            </li>
            <li id="auth-Jonas-Adler-Aff1">
             <span class="c-article-authors-search__title u-h3 js-search-name">
              Jonas Adler
             </span>
             <div class="c-article-authors-search__list">
              <div class="c-article-authors-search__item c-article-authors-search__list-item--left">
               <a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Jonas%20Adler" rel="nofollow">
                View author publications
               </a>
              </div>
              <div class="c-article-authors-search__item c-article-authors-search__list-item--right">
               <p class="search-in-title-js c-article-authors-search__text">
                You can also search for this author in
                <span class="c-article-identifiers">
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=search&amp;term=Jonas%20Adler" rel="nofollow">
                  PubMed
                 </a>
                 <span class="u-hide">
                 </span>
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Jonas%20Adler%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">
                  Google Scholar
                 </a>
                </span>
               </p>
              </div>
             </div>
            </li>
            <li id="auth-Trevor-Back-Aff1">
             <span class="c-article-authors-search__title u-h3 js-search-name">
              Trevor Back
             </span>
             <div class="c-article-authors-search__list">
              <div class="c-article-authors-search__item c-article-authors-search__list-item--left">
               <a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Trevor%20Back" rel="nofollow">
                View author publications
               </a>
              </div>
              <div class="c-article-authors-search__item c-article-authors-search__list-item--right">
               <p class="search-in-title-js c-article-authors-search__text">
                You can also search for this author in
                <span class="c-article-identifiers">
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=search&amp;term=Trevor%20Back" rel="nofollow">
                  PubMed
                 </a>
                 <span class="u-hide">
                 </span>
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Trevor%20Back%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">
                  Google Scholar
                 </a>
                </span>
               </p>
              </div>
             </div>
            </li>
            <li id="auth-Stig-Petersen-Aff1">
             <span class="c-article-authors-search__title u-h3 js-search-name">
              Stig Petersen
             </span>
             <div class="c-article-authors-search__list">
              <div class="c-article-authors-search__item c-article-authors-search__list-item--left">
               <a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Stig%20Petersen" rel="nofollow">
                View author publications
               </a>
              </div>
              <div class="c-article-authors-search__item c-article-authors-search__list-item--right">
               <p class="search-in-title-js c-article-authors-search__text">
                You can also search for this author in
                <span class="c-article-identifiers">
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=search&amp;term=Stig%20Petersen" rel="nofollow">
                  PubMed
                 </a>
                 <span class="u-hide">
                 </span>
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Stig%20Petersen%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">
                  Google Scholar
                 </a>
                </span>
               </p>
              </div>
             </div>
            </li>
            <li id="auth-David-Reiman-Aff1">
             <span class="c-article-authors-search__title u-h3 js-search-name">
              David Reiman
             </span>
             <div class="c-article-authors-search__list">
              <div class="c-article-authors-search__item c-article-authors-search__list-item--left">
               <a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=David%20Reiman" rel="nofollow">
                View author publications
               </a>
              </div>
              <div class="c-article-authors-search__item c-article-authors-search__list-item--right">
               <p class="search-in-title-js c-article-authors-search__text">
                You can also search for this author in
                <span class="c-article-identifiers">
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=search&amp;term=David%20Reiman" rel="nofollow">
                  PubMed
                 </a>
                 <span class="u-hide">
                 </span>
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22David%20Reiman%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">
                  Google Scholar
                 </a>
                </span>
               </p>
              </div>
             </div>
            </li>
            <li id="auth-Ellen-Clancy-Aff1">
             <span class="c-article-authors-search__title u-h3 js-search-name">
              Ellen Clancy
             </span>
             <div class="c-article-authors-search__list">
              <div class="c-article-authors-search__item c-article-authors-search__list-item--left">
               <a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Ellen%20Clancy" rel="nofollow">
                View author publications
               </a>
              </div>
              <div class="c-article-authors-search__item c-article-authors-search__list-item--right">
               <p class="search-in-title-js c-article-authors-search__text">
                You can also search for this author in
                <span class="c-article-identifiers">
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=search&amp;term=Ellen%20Clancy" rel="nofollow">
                  PubMed
                 </a>
                 <span class="u-hide">
                 </span>
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Ellen%20Clancy%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">
                  Google Scholar
                 </a>
                </span>
               </p>
              </div>
             </div>
            </li>
            <li id="auth-Michal-Zielinski-Aff1">
             <span class="c-article-authors-search__title u-h3 js-search-name">
              Michal Zielinski
             </span>
             <div class="c-article-authors-search__list">
              <div class="c-article-authors-search__item c-article-authors-search__list-item--left">
               <a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Michal%20Zielinski" rel="nofollow">
                View author publications
               </a>
              </div>
              <div class="c-article-authors-search__item c-article-authors-search__list-item--right">
               <p class="search-in-title-js c-article-authors-search__text">
                You can also search for this author in
                <span class="c-article-identifiers">
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=search&amp;term=Michal%20Zielinski" rel="nofollow">
                  PubMed
                 </a>
                 <span class="u-hide">
                 </span>
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Michal%20Zielinski%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">
                  Google Scholar
                 </a>
                </span>
               </p>
              </div>
             </div>
            </li>
            <li id="auth-Martin-Steinegger-Aff2-Aff3">
             <span class="c-article-authors-search__title u-h3 js-search-name">
              Martin Steinegger
             </span>
             <div class="c-article-authors-search__list">
              <div class="c-article-authors-search__item c-article-authors-search__list-item--left">
               <a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Martin%20Steinegger" rel="nofollow">
                View author publications
               </a>
              </div>
              <div class="c-article-authors-search__item c-article-authors-search__list-item--right">
               <p class="search-in-title-js c-article-authors-search__text">
                You can also search for this author in
                <span class="c-article-identifiers">
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=search&amp;term=Martin%20Steinegger" rel="nofollow">
                  PubMed
                 </a>
                 <span class="u-hide">
                 </span>
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Martin%20Steinegger%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">
                  Google Scholar
                 </a>
                </span>
               </p>
              </div>
             </div>
            </li>
            <li id="auth-Michalina-Pacholska-Aff1">
             <span class="c-article-authors-search__title u-h3 js-search-name">
              Michalina Pacholska
             </span>
             <div class="c-article-authors-search__list">
              <div class="c-article-authors-search__item c-article-authors-search__list-item--left">
               <a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Michalina%20Pacholska" rel="nofollow">
                View author publications
               </a>
              </div>
              <div class="c-article-authors-search__item c-article-authors-search__list-item--right">
               <p class="search-in-title-js c-article-authors-search__text">
                You can also search for this author in
                <span class="c-article-identifiers">
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=search&amp;term=Michalina%20Pacholska" rel="nofollow">
                  PubMed
                 </a>
                 <span class="u-hide">
                 </span>
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Michalina%20Pacholska%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">
                  Google Scholar
                 </a>
                </span>
               </p>
              </div>
             </div>
            </li>
            <li id="auth-Tamas-Berghammer-Aff1">
             <span class="c-article-authors-search__title u-h3 js-search-name">
              Tamas Berghammer
             </span>
             <div class="c-article-authors-search__list">
              <div class="c-article-authors-search__item c-article-authors-search__list-item--left">
               <a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Tamas%20Berghammer" rel="nofollow">
                View author publications
               </a>
              </div>
              <div class="c-article-authors-search__item c-article-authors-search__list-item--right">
               <p class="search-in-title-js c-article-authors-search__text">
                You can also search for this author in
                <span class="c-article-identifiers">
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=search&amp;term=Tamas%20Berghammer" rel="nofollow">
                  PubMed
                 </a>
                 <span class="u-hide">
                 </span>
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Tamas%20Berghammer%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">
                  Google Scholar
                 </a>
                </span>
               </p>
              </div>
             </div>
            </li>
            <li id="auth-Sebastian-Bodenstein-Aff1">
             <span class="c-article-authors-search__title u-h3 js-search-name">
              Sebastian Bodenstein
             </span>
             <div class="c-article-authors-search__list">
              <div class="c-article-authors-search__item c-article-authors-search__list-item--left">
               <a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Sebastian%20Bodenstein" rel="nofollow">
                View author publications
               </a>
              </div>
              <div class="c-article-authors-search__item c-article-authors-search__list-item--right">
               <p class="search-in-title-js c-article-authors-search__text">
                You can also search for this author in
                <span class="c-article-identifiers">
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=search&amp;term=Sebastian%20Bodenstein" rel="nofollow">
                  PubMed
                 </a>
                 <span class="u-hide">
                 </span>
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Sebastian%20Bodenstein%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">
                  Google Scholar
                 </a>
                </span>
               </p>
              </div>
             </div>
            </li>
            <li id="auth-David-Silver-Aff1">
             <span class="c-article-authors-search__title u-h3 js-search-name">
              David Silver
             </span>
             <div class="c-article-authors-search__list">
              <div class="c-article-authors-search__item c-article-authors-search__list-item--left">
               <a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=David%20Silver" rel="nofollow">
                View author publications
               </a>
              </div>
              <div class="c-article-authors-search__item c-article-authors-search__list-item--right">
               <p class="search-in-title-js c-article-authors-search__text">
                You can also search for this author in
                <span class="c-article-identifiers">
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=search&amp;term=David%20Silver" rel="nofollow">
                  PubMed
                 </a>
                 <span class="u-hide">
                 </span>
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22David%20Silver%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">
                  Google Scholar
                 </a>
                </span>
               </p>
              </div>
             </div>
            </li>
            <li id="auth-Oriol-Vinyals-Aff1">
             <span class="c-article-authors-search__title u-h3 js-search-name">
              Oriol Vinyals
             </span>
             <div class="c-article-authors-search__list">
              <div class="c-article-authors-search__item c-article-authors-search__list-item--left">
               <a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Oriol%20Vinyals" rel="nofollow">
                View author publications
               </a>
              </div>
              <div class="c-article-authors-search__item c-article-authors-search__list-item--right">
               <p class="search-in-title-js c-article-authors-search__text">
                You can also search for this author in
                <span class="c-article-identifiers">
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=search&amp;term=Oriol%20Vinyals" rel="nofollow">
                  PubMed
                 </a>
                 <span class="u-hide">
                 </span>
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Oriol%20Vinyals%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">
                  Google Scholar
                 </a>
                </span>
               </p>
              </div>
             </div>
            </li>
            <li id="auth-Andrew_W_-Senior-Aff1">
             <span class="c-article-authors-search__title u-h3 js-search-name">
              Andrew W. Senior
             </span>
             <div class="c-article-authors-search__list">
              <div class="c-article-authors-search__item c-article-authors-search__list-item--left">
               <a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Andrew%20W.%20Senior" rel="nofollow">
                View author publications
               </a>
              </div>
              <div class="c-article-authors-search__item c-article-authors-search__list-item--right">
               <p class="search-in-title-js c-article-authors-search__text">
                You can also search for this author in
                <span class="c-article-identifiers">
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=search&amp;term=Andrew%20W.%20Senior" rel="nofollow">
                  PubMed
                 </a>
                 <span class="u-hide">
                 </span>
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Andrew%20W.%20Senior%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">
                  Google Scholar
                 </a>
                </span>
               </p>
              </div>
             </div>
            </li>
            <li id="auth-Koray-Kavukcuoglu-Aff1">
             <span class="c-article-authors-search__title u-h3 js-search-name">
              Koray Kavukcuoglu
             </span>
             <div class="c-article-authors-search__list">
              <div class="c-article-authors-search__item c-article-authors-search__list-item--left">
               <a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Koray%20Kavukcuoglu" rel="nofollow">
                View author publications
               </a>
              </div>
              <div class="c-article-authors-search__item c-article-authors-search__list-item--right">
               <p class="search-in-title-js c-article-authors-search__text">
                You can also search for this author in
                <span class="c-article-identifiers">
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=search&amp;term=Koray%20Kavukcuoglu" rel="nofollow">
                  PubMed
                 </a>
                 <span class="u-hide">
                 </span>
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Koray%20Kavukcuoglu%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">
                  Google Scholar
                 </a>
                </span>
               </p>
              </div>
             </div>
            </li>
            <li id="auth-Pushmeet-Kohli-Aff1">
             <span class="c-article-authors-search__title u-h3 js-search-name">
              Pushmeet Kohli
             </span>
             <div class="c-article-authors-search__list">
              <div class="c-article-authors-search__item c-article-authors-search__list-item--left">
               <a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Pushmeet%20Kohli" rel="nofollow">
                View author publications
               </a>
              </div>
              <div class="c-article-authors-search__item c-article-authors-search__list-item--right">
               <p class="search-in-title-js c-article-authors-search__text">
                You can also search for this author in
                <span class="c-article-identifiers">
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=search&amp;term=Pushmeet%20Kohli" rel="nofollow">
                  PubMed
                 </a>
                 <span class="u-hide">
                 </span>
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Pushmeet%20Kohli%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">
                  Google Scholar
                 </a>
                </span>
               </p>
              </div>
             </div>
            </li>
            <li id="auth-Demis-Hassabis-Aff1">
             <span class="c-article-authors-search__title u-h3 js-search-name">
              Demis Hassabis
             </span>
             <div class="c-article-authors-search__list">
              <div class="c-article-authors-search__item c-article-authors-search__list-item--left">
               <a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Demis%20Hassabis" rel="nofollow">
                View author publications
               </a>
              </div>
              <div class="c-article-authors-search__item c-article-authors-search__list-item--right">
               <p class="search-in-title-js c-article-authors-search__text">
                You can also search for this author in
                <span class="c-article-identifiers">
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=search&amp;term=Demis%20Hassabis" rel="nofollow">
                  PubMed
                 </a>
                 <span class="u-hide">
                 </span>
                 <a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Demis%20Hassabis%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">
                  Google Scholar
                 </a>
                </span>
               </p>
              </div>
             </div>
            </li>
           </ol>
          </div>
          <h3 class="c-article__sub-heading" id="contributions">
           Contributions
          </h3>
          <p>
           J.J. and D.H. led the research. J.J., R.E., A. Pritzel, M.F., O.R., R.B., A. Potapenko, S.A.A.K., B.R.-P., J.A., M.P., T. Berghammer and O.V. developed the neural network architecture and training. T.G., A.Ž., K.T., R.B., A.B., R.E., A.J.B., A.C., S.N., R.J., D.R., M.Z. and S.B. developed the data, analytics and inference systems. D.H., K.K., P.K., C.M. and E.C. managed the research. T.G. led the technical platform. P.K., A.W.S., K.K., O.V., D.S., S.P. and T. Back contributed technical advice and ideas. M.S. created the BFD genomics database and provided technical assistance on HHBlits. D.H., R.E., A.W.S. and K.K. conceived the AlphaFold project. J.J., R.E. and A.W.S. conceived the end-to-end approach. J.J., A. Pritzel, O.R., A. Potapenko, R.E., M.F., T.G., K.T., C.M. and D.H. wrote the paper.
          </p>
          <h3 class="c-article__sub-heading" id="corresponding-author">
           Corresponding authors
          </h3>
          <p id="corresponding-author-list">
           Correspondence to
           <a href="mailto:jumper@deepmind.com" id="corresp-c1">
            John Jumper
           </a>
           or
           <a href="mailto:dhcontact@deepmind.com" id="corresp-c2">
            Demis Hassabis
           </a>
           .
          </p>
         </div>
        </div>
       </section>
       <section data-title="Ethics declarations">
        <div class="c-article-section" id="ethics-section">
         <h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="ethics">
          Ethics declarations
         </h2>
         <div class="c-article-section__content" id="ethics-content">
          <h3 class="c-article__sub-heading" id="FPar1">
           Competing interests
          </h3>
          <p>
           J.J., R.E., A. Pritzel, T.G., M.F., O.R., R.B., A.B., S.A.A.K., D.R. and A.W.S. have filed non-provisional patent applications 16/701,070 and PCT/EP2020/084238, and provisional patent applications 63/107,362, 63/118,917, 63/118,918, 63/118,921 and 63/118,919, each in the name of DeepMind Technologies Limited, each pending, relating to machine learning for predicting protein structures. The other authors declare no competing interests.
          </p>
         </div>
        </div>
       </section>
       <section data-title="Additional information">
        <div class="c-article-section" id="additional-information-section">
         <h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="additional-information">
          Additional information
         </h2>
         <div class="c-article-section__content" id="additional-information-content">
          <p>
           <b>
            Peer review information
           </b>
           <i>
            Nature
           </i>
           thanks Mohammed AlQuraishi, Charlotte Deane and Yang Zhang for their contribution to the peer review of this work.
          </p>
          <p>
           <b>
            Publisher’s note
           </b>
           Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.
          </p>
         </div>
        </div>
       </section>
       <section data-title="Supplementary information">
        <div class="c-article-section" id="Sec20-section">
         <h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec20">
          Supplementary information
         </h2>
         <div class="c-article-section__content" id="Sec20-content">
          <div data-test="supplementary-info">
           <div class="c-article-figshare-container" data-test="figshare-container" id="figshareContainer">
           </div>
           <div class="c-article-supplementary__item" data-test="supp-item" id="MOESM1">
            <h3 class="c-article-supplementary__title u-h3">
             <a class="print-link" data-supp-info-image="" data-test="supp-info-link" data-track="click" data-track-action="view supplementary info" data-track-label="supplementary information" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-021-03819-2/MediaObjects/41586_2021_3819_MOESM1_ESM.pdf">
              Supplementary Information
             </a>
            </h3>
            <div class="c-article-supplementary__description" data-component="thumbnail-container">
             <p>
              Description of the method details of the AlphaFold system, model, and analysis, including data pipeline, datasets, model blocks, loss functions, training and inference details, and ablations. Includes Supplementary Methods, Supplementary Figures, Supplementary Tables and Supplementary Algorithms.
             </p>
            </div>
           </div>
           <div class="c-article-supplementary__item" data-test="supp-item" id="MOESM2">
            <h3 class="c-article-supplementary__title u-h3">
             <a class="print-link" data-supp-info-image="" data-test="supp-info-link" data-track="click" data-track-action="view supplementary info" data-track-label="reporting summary" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-021-03819-2/MediaObjects/41586_2021_3819_MOESM2_ESM.pdf">
              Reporting Summary
             </a>
            </h3>
           </div>
           <div class="c-article-supplementary__item" data-test="supp-item" id="MOESM3">
            <h3 class="c-article-supplementary__title u-h3">
             <a class="print-link" data-supp-info-image="" data-test="supp-info-link" data-track="click" data-track-action="view supplementary info" data-track-label="supplementary video 1" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-021-03819-2/MediaObjects/41586_2021_3819_MOESM3_ESM.mp4">
              Supplementary Video 1
             </a>
            </h3>
            <div class="c-article-supplementary__description" data-component="thumbnail-container">
             <p>
              Video of the intermediate structure trajectory of the CASP14 target T1024 (LmrP) A two-domain target (408 residues). Both domains are folded early, while their packing is adjusted for a longer time.
             </p>
            </div>
           </div>
           <div class="c-article-supplementary__item" data-test="supp-item" id="MOESM4">
            <h3 class="c-article-supplementary__title u-h3">
             <a class="print-link" data-supp-info-image="" data-test="supp-info-link" data-track="click" data-track-action="view supplementary info" data-track-label="supplementary video 2" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-021-03819-2/MediaObjects/41586_2021_3819_MOESM4_ESM.mp4">
              Supplementary Video 2
             </a>
            </h3>
            <div class="c-article-supplementary__description" data-component="thumbnail-container">
             <p>
              Video of the intermediate structure trajectory of the CASP14 target T1044 (RNA polymerase of crAss-like phage). A large protein (2180 residues), with multiple domains. Some domains are folded quickly, while others take a considerable amount of time to fold.
             </p>
            </div>
           </div>
           <div class="c-article-supplementary__item" data-test="supp-item" id="MOESM5">
            <h3 class="c-article-supplementary__title u-h3">
             <a class="print-link" data-supp-info-image="" data-test="supp-info-link" data-track="click" data-track-action="view supplementary info" data-track-label="supplementary video 3" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-021-03819-2/MediaObjects/41586_2021_3819_MOESM5_ESM.mp4">
              Supplementary Video 3
             </a>
            </h3>
            <div class="c-article-supplementary__description" data-component="thumbnail-container">
             <p>
              Video of the intermediate structure trajectory of the CASP14 target T1064 (Orf8). A very difficult single-domain target (106 residues) that takes the entire depth of the network to fold.
             </p>
            </div>
           </div>
           <div class="c-article-supplementary__item" data-test="supp-item" id="MOESM6">
            <h3 class="c-article-supplementary__title u-h3">
             <a class="print-link" data-supp-info-image="" data-test="supp-info-link" data-track="click" data-track-action="view supplementary info" data-track-label="supplementary video 4" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-021-03819-2/MediaObjects/41586_2021_3819_MOESM6_ESM.mp4">
              Supplementary Video 4
             </a>
            </h3>
            <div class="c-article-supplementary__description" data-component="thumbnail-container">
             <p>
              Video of the intermediate structure trajectory of the CASP14 target T1091. A multi-domain target (863 residues). Individual domains’ structure is determined early, while the domain packing evolves throughout the network. The network is exploring unphysical configurations throughout the process, resulting in long ‘strings’ in the visualization.
             </p>
            </div>
           </div>
          </div>
         </div>
        </div>
       </section>
       <section data-title="Rights and permissions">
        <div class="c-article-section" id="rightslink-section">
         <h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="rightslink">
          Rights and permissions
         </h2>
         <div class="c-article-section__content" id="rightslink-content">
          <p>
           <b>
            Open Access
           </b>
           This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit
           <a href="http://creativecommons.org/licenses/by/4.0/" rel="license">
            http://creativecommons.org/licenses/by/4.0/
           </a>
           .
          </p>
          <p class="c-article-rights">
           <a data-track="click" data-track-action="view rights and permissions" data-track-label="link" href="https://s100.copyright.com/AppDispatchServlet?title=Highly%20accurate%20protein%20structure%20prediction%20with%20AlphaFold&amp;author=John%20Jumper%20et%20al&amp;contentID=10.1038%2Fs41586-021-03819-2&amp;copyright=The%20Author%28s%29&amp;publication=0028-0836&amp;publicationDate=2021-07-15&amp;publisherName=SpringerNature&amp;orderBeanReset=true&amp;oa=CC%20BY">
            Reprints and Permissions
           </a>
          </p>
         </div>
        </div>
       </section>
       <section aria-labelledby="article-info" data-title="About this article">
        <div class="c-article-section" id="article-info-section">
         <h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="article-info">
          About this article
         </h2>
         <div class="c-article-section__content" id="article-info-content">
          <div class="c-bibliographic-information">
           <div class="u-hide-print c-bibliographic-information__column c-bibliographic-information__column--border">
            <a data-crossmark="10.1038/s41586-021-03819-2" data-test="crossmark" data-track="click" data-track-action="Click Crossmark" data-track-label="link" href="https://crossmark-crossref-org.proxy.lib.ohio-state.edu/dialog/?doi=10.1038/s41586-021-03819-2" rel="noopener" target="_blank">
             <img alt="Check for updates. Verify currency and authenticity via CrossMark" height="81" src="data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>" width="57"/>
            </a>
           </div>
           <div class="c-bibliographic-information__column">
            <h3 class="c-article__sub-heading" id="citeas">
             Cite this article
            </h3>
            <p class="c-bibliographic-information__citation">
             Jumper, J., Evans, R., Pritzel, A.
             <i>
              et al.
             </i>
             Highly accurate protein structure prediction with AlphaFold.
             <i>
              Nature
             </i>
             <b>
              596
             </b>
             , 583–589 (2021). https://doi-org.proxy.lib.ohio-state.edu/10.1038/s41586-021-03819-2
            </p>
            <p class="c-bibliographic-information__download-citation u-hide-print">
             <a data-test="citation-link" data-track="click" data-track-action="download article citation" data-track-external="" data-track-label="link" href="https://citation-needed-springer-com.proxy.lib.ohio-state.edu/v2/references/10.1038/s41586-021-03819-2?format=refman&amp;flavour=citation" rel="nofollow">
              Download citation
              <svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16">
               <use xlink:href="#icon-download" xmlns:xlink="http://www.w3.org/1999/xlink">
               </use>
              </svg>
             </a>
            </p>
            <ul class="c-bibliographic-information__list" data-test="publication-history">
             <li class="c-bibliographic-information__list-item">
              <p>
               Received
               <span class="u-hide">
                :
               </span>
               <span class="c-bibliographic-information__value">
                <time datetime="2021-05-11">
                 11 May 2021
                </time>
               </span>
              </p>
             </li>
             <li class="c-bibliographic-information__list-item">
              <p>
               Accepted
               <span class="u-hide">
                :
               </span>
               <span class="c-bibliographic-information__value">
                <time datetime="2021-07-12">
                 12 July 2021
                </time>
               </span>
              </p>
             </li>
             <li class="c-bibliographic-information__list-item">
              <p>
               Published
               <span class="u-hide">
                :
               </span>
               <span class="c-bibliographic-information__value">
                <time datetime="2021-07-15">
                 15 July 2021
                </time>
               </span>
              </p>
             </li>
             <li class="c-bibliographic-information__list-item">
              <p>
               Issue Date
               <span class="u-hide">
                :
               </span>
               <span class="c-bibliographic-information__value">
                <time datetime="2021-08-26">
                 26 August 2021
                </time>
               </span>
              </p>
             </li>
             <li class="c-bibliographic-information__list-item c-bibliographic-information__list-item--full-width">
              <p>
               <abbr title="Digital Object Identifier">
                DOI
               </abbr>
               <span class="u-hide">
                :
               </span>
               <span class="c-bibliographic-information__value">
                https://doi-org.proxy.lib.ohio-state.edu/10.1038/s41586-021-03819-2
               </span>
              </p>
             </li>
            </ul>
            <div data-component="share-box">
             <div class="c-article-share-box u-display-block">
              <h3 class="c-article__sub-heading">
               Share this article
              </h3>
              <p class="c-article-share-box__description">
               Anyone you share the following link with will be able to read this content:
              </p>
              <button class="js-get-share-url c-article-share-box__button" data-track="click" data-track-action="get shareable link" data-track-external="" data-track-label="button" id="get-share-url" type="button">
               Get shareable link
              </button>
              <div class="js-no-share-url-container u-display-none" hidden="">
               <p class="js-c-article-share-box__no-sharelink-info c-article-share-box__no-sharelink-info">
                Sorry, a shareable link is not currently available for this article.
               </p>
              </div>
              <div class="js-share-url-container u-display-none" hidden="">
               <p class="js-share-url c-article-share-box__only-read-input" data-track="click" data-track-action="select share url" data-track-label="button" id="share-url">
               </p>
               <button class="js-copy-share-url c-article-share-box__button--link-like" data-track="click" data-track-action="copy share url" data-track-external="" data-track-label="button" id="copy-share-url" type="button">
                Copy to clipboard
               </button>
              </div>
              <p class="js-c-article-share-box__additional-info c-article-share-box__additional-info">
               Provided by the Springer Nature SharedIt content-sharing initiative
              </p>
             </div>
            </div>
            <div data-component="article-info-list">
             <h3 class="c-article__sub-heading">
              Subjects
             </h3>
             <ul class="c-article-subject-list">
              <li class="c-article-subject-list__subject">
               <a data-track="click" data-track-action="view subject" data-track-label="link" href="/subjects/computational-biophysics">
                Computational biophysics
               </a>
              </li>
              <li class="c-article-subject-list__subject">
               <a data-track="click" data-track-action="view subject" data-track-label="link" href="/subjects/machine-learning">
                Machine learning
               </a>
              </li>
              <li class="c-article-subject-list__subject">
               <a data-track="click" data-track-action="view subject" data-track-label="link" href="/subjects/protein-structure-predictions">
                Protein structure predictions
               </a>
              </li>
              <li class="c-article-subject-list__subject">
               <a data-track="click" data-track-action="view subject" data-track-label="link" href="/subjects/structural-biology">
                Structural biology
               </a>
              </li>
             </ul>
            </div>
           </div>
          </div>
         </div>
        </div>
       </section>
      </div>
      <section>
       <div class="c-article-section js-article-section" id="further-reading-section">
        <h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="further-reading">
         This article is cited by
        </h2>
        <div class="c-article-section__content js-collapsible-section" id="further-reading-content">
         <ul class="c-article-further-reading__list" id="further-reading-list">
          <li class="c-article-further-reading__item js-ref-item">
           <h3 class="c-article-further-reading__title">
            <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:Identification of histidine kinase inhibitors through screening of natural compounds to combat mastitis caused by Streptococcus agalactiae in dairy cattle" href="https://doi-org.proxy.lib.ohio-state.edu/10.1186/s13036-023-00378-0">
             Identification of histidine kinase inhibitors through screening of natural compounds to combat mastitis caused by Streptococcus agalactiae in dairy cattle
            </a>
           </h3>
           <ul class="c-author-list c-author-list--compact u-sans-serif u-mb-4 u-mt-auto" data-test="author-list">
            <li>
             Rajesh Kumar Pathak
            </li>
            <li>
             Jun-Mo Kim
            </li>
           </ul>
           <p class="c-article-further-reading__journal-title">
            <i>
             Journal of Biological Engineering
            </i>
            (2023)
           </p>
          </li>
         </ul>
        </div>
       </div>
      </section>
      <section data-title="Comments">
       <div class="c-article-section" id="article-comments-section">
        <h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="article-comments">
         Comments
        </h2>
        <div class="c-article-section__content" id="article-comments-content">
         <p>
          By submitting a comment you agree to abide by our
          <a href="/info/tandc.html">
           Terms
          </a>
          and
          <a href="/info/community-guidelines.html">
           Community Guidelines
          </a>
          . If you find something abusive or that does not comply with our terms or guidelines please flag it as inappropriate.
         </p>
        </div>
       </div>
      </section>
      <div id="inject-comments">
       <div class="placeholder" data-disqus-placeholder="/platform/disqus?doi=10.1038/s41586-021-03819-2 #article-comments-container" data-replace="true">
       </div>
      </div>
     </div>
    </article>
   </main>
   <aside aria-label="Article navigation" class="c-article-extras u-hide-print" data-component-reading-companion="" data-container-type="reading-companion" data-track-component="reading companion">
    <div class="js-context-bar-sticky-point-desktop">
     <div class="c-pdf-download u-clear-both js-pdf-download">
      <a class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-draft-ignore="true" data-readcube-pdf-url="true" data-test="download-pdf" data-track="click" data-track-action="download pdf" data-track-external="" data-track-label="link" download="" href="/articles/s41586-021-03819-2.pdf">
       <span class="c-pdf-download__text">
        Download PDF
       </span>
       <svg aria-hidden="true" class="u-icon" focusable="false" height="16" width="16">
        <use xlink:href="#icon-download">
        </use>
       </svg>
      </a>
     </div>
    </div>
    <div class="c-article-associated-content__container">
     <section>
      <h2 class="c-article-associated-content__title u-mb-24">
       Associated Content
      </h2>
      <div class="c-article-associated-content__collection nature outlook u-mb-24">
       <section>
        <p class="c-article-associated-content__collection-label u-sans-serif u-text-bold u-mb-8">
         Nature Outlook
        </p>
        <h3 class="c-article-associated-content__collection-title u-h3 u-mb-8">
         <a class="u-link-inherit" data-test="collection-link" data-track="click" data-track-action="view nature outlook" data-track-category="associated content" data-track-label="outlook" href="https://www-nature-com.proxy.lib.ohio-state.edu/collections/dgebebiadj">
          Robotics and artificial intelligence
         </a>
        </h3>
       </section>
      </div>
      <div class="c-article-associated-content__collection focus u-mb-24">
       <section>
        <p class="c-article-associated-content__collection-label u-sans-serif u-text-bold u-mb-8">
         Focus
        </p>
        <h3 class="c-article-associated-content__collection-title u-h3 u-mb-8">
         <a class="u-link-inherit" data-test="collection-link" data-track="click" data-track-action="view focus" data-track-category="associated content" data-track-label="focus" href="https://www-nature-com.proxy.lib.ohio-state.edu/collections/dfejabhghd">
          Method of the Year 2021: Protein structure prediction
         </a>
        </h3>
       </section>
      </div>
      <div class="u-full-height u-mb-24">
       <article class="u-full-height c-card c-card--flush">
        <div class="c-card__layout u-full-height">
         <div class="c-card__body">
          <h3 class="c-card__title">
           <a class="c-card__link u-link-inherit" data-track="click" data-track-action="view article" data-track-category="associated content" data-track-label="research" href="/articles/s41586-021-03828-1">
            Highly accurate protein structure prediction for the human proteome
           </a>
          </h3>
          <ul class="c-author-list c-author-list--compact c-author-list--truncated" data-test="author-list">
           <li>
            Kathryn Tunyasuvunakool
           </li>
           <li>
            Jonas Adler
           </li>
           <li>
            Demis Hassabis
           </li>
          </ul>
          <div class="c-card__section c-meta">
           <span class="c-meta__item">
            Nature
           </span>
           <span class="c-meta__item" data-test="article.type">
            <span class="c-meta__type">
             Article
            </span>
           </span>
           <span class="c-meta__item" data-test="open-access">
            <span class="u-color-open-access">
             Open Access
            </span>
           </span>
           <time class="c-meta__item" datetime="2021-07-22">
            22 Jul 2021
           </time>
          </div>
         </div>
        </div>
       </article>
      </div>
      <div class="u-full-height u-mb-24">
       <article class="u-full-height c-card c-card--flush">
        <div class="c-card__layout u-full-height">
         <div class="c-card__body">
          <h3 class="c-card__title">
           <a class="c-card__link u-link-inherit" data-track="click" data-track-action="view article" data-track-category="associated content" data-track-label="news_and_views" href="/articles/d41586-021-02265-4">
            Protein-structure prediction revolutionized
           </a>
          </h3>
          <ul class="c-author-list c-author-list--compact" data-test="author-list">
           <li>
            Mohammed AlQuraishi
           </li>
          </ul>
          <div class="c-card__section c-meta">
           <span class="c-meta__item">
            Nature
           </span>
           <span class="c-meta__item" data-test="article.type">
            <span class="c-meta__type">
             News &amp; Views
            </span>
           </span>
           <time class="c-meta__item" datetime="2021-08-23">
            23 Aug 2021
           </time>
          </div>
         </div>
        </div>
       </article>
      </div>
      <div class="u-full-height u-mb-24">
       <article class="u-full-height c-card c-card--flush">
        <div class="c-card__layout u-full-height">
         <div class="c-card__body">
          <h3 class="c-card__title">
           <a class="c-card__link u-link-inherit" data-track="click" data-track-action="view article" data-track-category="associated content" data-track-label="research_highlights" href="/articles/s41580-023-00671-2">
            Lasker Award for AlphaFold
           </a>
          </h3>
          <ul class="c-author-list c-author-list--compact" data-test="author-list">
           <li>
            Paulina Strzyz
           </li>
          </ul>
          <div class="c-card__section c-meta">
           <span class="c-meta__item">
            Nature Reviews Molecular Cell Biology
           </span>
           <span class="c-meta__item" data-test="article.type">
            <span class="c-meta__type">
             Research Highlight
            </span>
           </span>
           <time class="c-meta__item" datetime="2023-09-26">
            26 Sept 2023
           </time>
          </div>
         </div>
        </div>
       </article>
      </div>
     </section>
    </div>
    <script>
     window.dataLayer = window.dataLayer || [];
            window.dataLayer[0] = window.dataLayer[0] || {};
            window.dataLayer[0].content = window.dataLayer[0].content || {};
            window.dataLayer[0].content.associatedContentTypes = "outlook;focus;research;news_and_views;research_highlights";
    </script>
    <div class="c-reading-companion">
     <div class="c-reading-companion__sticky" data-component="reading-companion-sticky" data-test="reading-companion-sticky" style="top: 40px;">
      <ul class="c-reading-companion__tabs" role="tablist">
       <li role="presentation">
        <button aria-controls="tabpanel-sections" aria-selected="true" class="c-reading-companion__tab c-reading-companion__tab--active" data-tab-target="sections" data-track="click" data-track-action="sections tab" data-track-label="tab" id="tab-sections" role="tab">
         Sections
        </button>
       </li>
       <li role="presentation">
        <button aria-controls="tabpanel-figures" aria-selected="false" class="c-reading-companion__tab" data-tab-target="figures" data-track="click" data-track-action="figures tab" data-track-label="tab" id="tab-figures" role="tab" tabindex="-1">
         Figures
        </button>
       </li>
       <li role="presentation">
        <button aria-controls="tabpanel-references" aria-selected="false" class="c-reading-companion__tab" data-tab-target="references" data-track="click" data-track-action="references tab" data-track-label="tab" id="tab-references" role="tab" tabindex="-1">
         References
        </button>
       </li>
      </ul>
      <div aria-labelledby="tab-sections" class="c-reading-companion__panel c-reading-companion__sections c-reading-companion__panel--active" id="tabpanel-sections" role="tabpanel">
       <div class="c-reading-companion__scroll-pane" style="max-height: none;">
        <ul class="c-reading-companion__sections-list">
         <li class="c-reading-companion__section-item" id="rc-sec-Abs1">
          <a data-track="click" data-track-action="section anchor" data-track-label="link:Abstract" href="#Abs1">
           Abstract
          </a>
         </li>
         <li class="c-reading-companion__section-item" id="rc-sec-Sec1">
          <a data-track="click" data-track-action="section anchor" data-track-label="link:Main" href="#Sec1">
           Main
          </a>
         </li>
         <li class="c-reading-companion__section-item" id="rc-sec-Sec2">
          <a data-track="click" data-track-action="section anchor" data-track-label="link:The AlphaFold network" href="#Sec2">
           The AlphaFold network
          </a>
         </li>
         <li class="c-reading-companion__section-item" id="rc-sec-Sec3">
          <a data-track="click" data-track-action="section anchor" data-track-label="link:Evoformer" href="#Sec3">
           Evoformer
          </a>
         </li>
         <li class="c-reading-companion__section-item" id="rc-sec-Sec4">
          <a data-track="click" data-track-action="section anchor" data-track-label="link:Endtoend structure prediction" href="#Sec4">
           End-to-end structure prediction
          </a>
         </li>
         <li class="c-reading-companion__section-item" id="rc-sec-Sec5">
          <a data-track="click" data-track-action="section anchor" data-track-label="link:Training with labelled and unlabelled data" href="#Sec5">
           Training with labelled and unlabelled data
          </a>
         </li>
         <li class="c-reading-companion__section-item" id="rc-sec-Sec6">
          <a data-track="click" data-track-action="section anchor" data-track-label="link:Interpreting the neural network" href="#Sec6">
           Interpreting the neural network
          </a>
         </li>
         <li class="c-reading-companion__section-item" id="rc-sec-Sec7">
          <a data-track="click" data-track-action="section anchor" data-track-label="link:MSA depth and crosschain contacts" href="#Sec7">
           MSA depth and cross-chain contacts
          </a>
         </li>
         <li class="c-reading-companion__section-item" id="rc-sec-Sec8">
          <a data-track="click" data-track-action="section anchor" data-track-label="link:Related work" href="#Sec8">
           Related work
          </a>
         </li>
         <li class="c-reading-companion__section-item" id="rc-sec-Sec9">
          <a data-track="click" data-track-action="section anchor" data-track-label="link:Discussion" href="#Sec9">
           Discussion
          </a>
         </li>
         <li class="c-reading-companion__section-item" id="rc-sec-Sec10">
          <a data-track="click" data-track-action="section anchor" data-track-label="link:Methods" href="#Sec10">
           Methods
          </a>
         </li>
         <li class="c-reading-companion__section-item" id="rc-sec-data-availability">
          <a data-track="click" data-track-action="section anchor" data-track-label="link:Data availability" href="#data-availability">
           Data availability
          </a>
         </li>
         <li class="c-reading-companion__section-item" id="rc-sec-code-availability">
          <a data-track="click" data-track-action="section anchor" data-track-label="link:Code availability" href="#code-availability">
           Code availability
          </a>
         </li>
         <li class="c-reading-companion__section-item" id="rc-sec-Bib1">
          <a data-track="click" data-track-action="section anchor" data-track-label="link:References" href="#Bib1">
           References
          </a>
         </li>
         <li class="c-reading-companion__section-item" id="rc-sec-Ack1">
          <a data-track="click" data-track-action="section anchor" data-track-label="link:Acknowledgements" href="#Ack1">
           Acknowledgements
          </a>
         </li>
         <li class="c-reading-companion__section-item" id="rc-sec-author-information">
          <a data-track="click" data-track-action="section anchor" data-track-label="link:Author information" href="#author-information">
           Author information
          </a>
         </li>
         <li class="c-reading-companion__section-item" id="rc-sec-ethics">
          <a data-track="click" data-track-action="section anchor" data-track-label="link:Ethics declarations" href="#ethics">
           Ethics declarations
          </a>
         </li>
         <li class="c-reading-companion__section-item" id="rc-sec-additional-information">
          <a data-track="click" data-track-action="section anchor" data-track-label="link:Additional information" href="#additional-information">
           Additional information
          </a>
         </li>
         <li class="c-reading-companion__section-item" id="rc-sec-Sec20">
          <a data-track="click" data-track-action="section anchor" data-track-label="link:Supplementary information" href="#Sec20">
           Supplementary information
          </a>
         </li>
         <li class="c-reading-companion__section-item" id="rc-sec-rightslink">
          <a data-track="click" data-track-action="section anchor" data-track-label="link:Rights and permissions" href="#rightslink">
           Rights and permissions
          </a>
         </li>
         <li class="c-reading-companion__section-item" id="rc-sec-article-info">
          <a data-track="click" data-track-action="section anchor" data-track-label="link:About this article" href="#article-info">
           About this article
          </a>
         </li>
         <li class="c-reading-companion__section-item" id="rc-sec-further-reading">
          <a data-track="click" data-track-action="section anchor" data-track-label="link:This article is cited by" href="#further-reading">
           This article is cited by
          </a>
         </li>
         <li class="c-reading-companion__section-item" id="rc-sec-article-comments">
          <a data-track="click" data-track-action="section anchor" data-track-label="link:Comments" href="#article-comments">
           Comments
          </a>
         </li>
        </ul>
       </div>
       <div class="u-lazy-ad-wrapper u-mt-16 u-show" data-component-mpu="">
        <div class="c-ad c-ad--300x250">
         <div class="c-ad__inner">
          <p class="c-ad__label">
           Advertisement
          </p>
          <div class="div-gpt-ad advert medium-rectangle js-ad text-center hide-print grade-c-hide" data-ad-type="right" data-gpt="" data-gpt-sizes="300x250" data-gpt-targeting="type=article;pos=right;artid=s41586-021-03819-2;doi=10.1038/s41586-021-03819-2;techmeta=119,139,141;subjmeta=114,1305,2266,2411,535,57,631;kwrd=Computational+biophysics,Machine+learning,Protein+structure+predictions,Structural+biology" data-gpt-unitpath="/285/nature.com/article" data-pa11y-ignore="" data-test="right-ad" id="div-gpt-ad-right-2">
           <noscript>
            <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/nature.com/article&amp;sz=300x250&amp;c=1575262724&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41586-021-03819-2%26doi%3D10.1038/s41586-021-03819-2%26techmeta%3D119,139,141%26subjmeta%3D114,1305,2266,2411,535,57,631%26kwrd%3DComputational+biophysics,Machine+learning,Protein+structure+predictions,Structural+biology">
             <img alt="Advertisement" data-test="gpt-advert-fallback-img" height="250" src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/nature.com/article&amp;sz=300x250&amp;c=1575262724&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41586-021-03819-2%26doi%3D10.1038/s41586-021-03819-2%26techmeta%3D119,139,141%26subjmeta%3D114,1305,2266,2411,535,57,631%26kwrd%3DComputational+biophysics,Machine+learning,Protein+structure+predictions,Structural+biology" width="300"/>
            </a>
           </noscript>
          </div>
         </div>
        </div>
       </div>
      </div>
      <div aria-labelledby="tab-figures" class="c-reading-companion__panel c-reading-companion__figures c-reading-companion__panel--full-width" id="tabpanel-figures" role="tabpanel">
       <div class="c-reading-companion__scroll-pane">
        <ul class="c-reading-companion__figures-list">
         <li class="c-reading-companion__figure-item">
          <figure>
           <figcaption>
            <b class="c-reading-companion__figure-title u-h4" id="rc-Fig1">
             Fig. 1: AlphaFold produces highly accurate structures.
            </b>
           </figcaption>
           <picture>
            <source data-srcset="https://media-springernature-com.proxy.lib.ohio-state.edu/lw685/springer-static/image/art%3A10.1038%2Fs41586-021-03819-2/MediaObjects/41586_2021_3819_Fig1_HTML.png?"/>
            <img alt="figure 1" aria-describedby="rc-Fig1" data-src="https://media-springernature-com.proxy.lib.ohio-state.edu/lw685/springer-static/image/art%3A10.1038%2Fs41586-021-03819-2/MediaObjects/41586_2021_3819_Fig1_HTML.png"/>
           </picture>
           <p class="c-reading-companion__figure-links">
            <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="#Fig1">
             View in article
            </a>
            <a class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/s41586-021-03819-2/figures/1" rel="nofollow">
             Full size image
             <svg class="u-icon" height="16" width="16">
              <use href="#icon-chevron-right">
              </use>
             </svg>
            </a>
           </p>
          </figure>
         </li>
         <li class="c-reading-companion__figure-item">
          <figure>
           <figcaption>
            <b class="c-reading-companion__figure-title u-h4" id="rc-Fig2">
             Fig. 2: Accuracy of AlphaFold on recent PDB structures.
            </b>
           </figcaption>
           <picture>
            <source data-srcset="https://media-springernature-com.proxy.lib.ohio-state.edu/lw685/springer-static/image/art%3A10.1038%2Fs41586-021-03819-2/MediaObjects/41586_2021_3819_Fig2_HTML.png?"/>
            <img alt="figure 2" aria-describedby="rc-Fig2" data-src="https://media-springernature-com.proxy.lib.ohio-state.edu/lw685/springer-static/image/art%3A10.1038%2Fs41586-021-03819-2/MediaObjects/41586_2021_3819_Fig2_HTML.png"/>
           </picture>
           <p class="c-reading-companion__figure-links">
            <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="#Fig2">
             View in article
            </a>
            <a class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/s41586-021-03819-2/figures/2" rel="nofollow">
             Full size image
             <svg class="u-icon" height="16" width="16">
              <use href="#icon-chevron-right">
              </use>
             </svg>
            </a>
           </p>
          </figure>
         </li>
         <li class="c-reading-companion__figure-item">
          <figure>
           <figcaption>
            <b class="c-reading-companion__figure-title u-h4" id="rc-Fig3">
             Fig. 3: Architectural details.
            </b>
           </figcaption>
           <picture>
            <source data-srcset="https://media-springernature-com.proxy.lib.ohio-state.edu/lw685/springer-static/image/art%3A10.1038%2Fs41586-021-03819-2/MediaObjects/41586_2021_3819_Fig3_HTML.png?"/>
            <img alt="figure 3" aria-describedby="rc-Fig3" data-src="https://media-springernature-com.proxy.lib.ohio-state.edu/lw685/springer-static/image/art%3A10.1038%2Fs41586-021-03819-2/MediaObjects/41586_2021_3819_Fig3_HTML.png"/>
           </picture>
           <p class="c-reading-companion__figure-links">
            <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="#Fig3">
             View in article
            </a>
            <a class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/s41586-021-03819-2/figures/3" rel="nofollow">
             Full size image
             <svg class="u-icon" height="16" width="16">
              <use href="#icon-chevron-right">
              </use>
             </svg>
            </a>
           </p>
          </figure>
         </li>
         <li class="c-reading-companion__figure-item">
          <figure>
           <figcaption>
            <b class="c-reading-companion__figure-title u-h4" id="rc-Fig4">
             Fig. 4: Interpreting the neural network.
            </b>
           </figcaption>
           <picture>
            <source data-srcset="https://media-springernature-com.proxy.lib.ohio-state.edu/lw685/springer-static/image/art%3A10.1038%2Fs41586-021-03819-2/MediaObjects/41586_2021_3819_Fig4_HTML.png?"/>
            <img alt="figure 4" aria-describedby="rc-Fig4" data-src="https://media-springernature-com.proxy.lib.ohio-state.edu/lw685/springer-static/image/art%3A10.1038%2Fs41586-021-03819-2/MediaObjects/41586_2021_3819_Fig4_HTML.png"/>
           </picture>
           <p class="c-reading-companion__figure-links">
            <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="#Fig4">
             View in article
            </a>
            <a class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/s41586-021-03819-2/figures/4" rel="nofollow">
             Full size image
             <svg class="u-icon" height="16" width="16">
              <use href="#icon-chevron-right">
              </use>
             </svg>
            </a>
           </p>
          </figure>
         </li>
         <li class="c-reading-companion__figure-item">
          <figure>
           <figcaption>
            <b class="c-reading-companion__figure-title u-h4" id="rc-Fig5">
             Fig. 5: Effect of MSA depth and cross-chain contacts.
            </b>
           </figcaption>
           <picture>
            <source data-srcset="https://media-springernature-com.proxy.lib.ohio-state.edu/lw685/springer-static/image/art%3A10.1038%2Fs41586-021-03819-2/MediaObjects/41586_2021_3819_Fig5_HTML.png?"/>
            <img alt="figure 5" aria-describedby="rc-Fig5" data-src="https://media-springernature-com.proxy.lib.ohio-state.edu/lw685/springer-static/image/art%3A10.1038%2Fs41586-021-03819-2/MediaObjects/41586_2021_3819_Fig5_HTML.png"/>
           </picture>
           <p class="c-reading-companion__figure-links">
            <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="#Fig5">
             View in article
            </a>
            <a class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/s41586-021-03819-2/figures/5" rel="nofollow">
             Full size image
             <svg class="u-icon" height="16" width="16">
              <use href="#icon-chevron-right">
              </use>
             </svg>
            </a>
           </p>
          </figure>
         </li>
        </ul>
       </div>
      </div>
      <div aria-labelledby="tab-references" class="c-reading-companion__panel c-reading-companion__references c-reading-companion__panel--full-width" id="tabpanel-references" role="tabpanel">
       <div class="c-reading-companion__scroll-pane">
        <ol class="c-reading-companion__references-list c-reading-companion__references-list--numeric">
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR1">
           Thompson, M. C., Yeates, T. O. &amp; Rodriguez, J. A. Advances in methods for atomic resolution macromolecular structure determination.
           <i>
            F1000Res
           </i>
           .
           <b>
            9
           </b>
           , 667 (2020).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.12688/f1000research.25097.1" href="https://doi-org.proxy.lib.ohio-state.edu/10.12688%2Ff1000research.25097.1">
            Article
           </a>
           <a data-track="click" data-track-action="cas reference" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/cas-redirect/1:CAS:528:DC%2BB3cXis1OgtrzK">
            CAS
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Advances%20in%20methods%20for%20atomic%20resolution%20macromolecular%20structure%20determination&amp;journal=F1000Res.&amp;doi=10.12688%2Ff1000research.25097.1&amp;volume=9&amp;publication_year=2020&amp;author=Thompson%2CMC&amp;author=Yeates%2CTO&amp;author=Rodriguez%2CJA">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR2">
           Bai, X.-C., McMullan, G. &amp; Scheres, S. H. W. How cryo-EM is revolutionizing structural biology.
           <i>
            Trends Biochem. Sci
           </i>
           .
           <b>
            40
           </b>
           , 49–57 (2015).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1016/j.tibs.2014.10.005" href="https://doi-org.proxy.lib.ohio-state.edu/10.1016%2Fj.tibs.2014.10.005">
            Article
           </a>
           <a data-track="click" data-track-action="cas reference" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/cas-redirect/1:CAS:528:DC%2BC2cXhvVCktLnK">
            CAS
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25544475">
            PubMed
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=How%20cryo-EM%20is%20revolutionizing%20structural%20biology&amp;journal=Trends%20Biochem.%20Sci.&amp;doi=10.1016%2Fj.tibs.2014.10.005&amp;volume=40&amp;pages=49-57&amp;publication_year=2015&amp;author=Bai%2CX-C&amp;author=McMullan%2CG&amp;author=Scheres%2CSHW">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR3">
           Jaskolski, M., Dauter, Z. &amp; Wlodawer, A. A brief history of macromolecular crystallography, illustrated by a family tree and its Nobel fruits.
           <i>
            FEBS J
           </i>
           .
           <b>
            281
           </b>
           , 3985–4009 (2014).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1111/febs.12796" href="https://doi-org.proxy.lib.ohio-state.edu/10.1111%2Ffebs.12796">
            Article
           </a>
           <a data-track="click" data-track-action="cas reference" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/cas-redirect/1:CAS:528:DC%2BC2cXhsFKnsbnM">
            CAS
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24698025">
            PubMed
           </a>
           <a data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC6309182">
            PubMed Central
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20brief%20history%20of%20macromolecular%20crystallography%2C%20illustrated%20by%20a%20family%20tree%20and%20its%20Nobel%20fruits&amp;journal=FEBS%20J.&amp;doi=10.1111%2Ffebs.12796&amp;volume=281&amp;pages=3985-4009&amp;publication_year=2014&amp;author=Jaskolski%2CM&amp;author=Dauter%2CZ&amp;author=Wlodawer%2CA">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR4">
           Wüthrich, K. The way to NMR structures of proteins.
           <i>
            Nat. Struct. Biol
           </i>
           .
           <b>
            8
           </b>
           , 923–925 (2001).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1038/nsb1101-923" href="https://doi-org.proxy.lib.ohio-state.edu/10.1038%2Fnsb1101-923">
            Article
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11685234">
            PubMed
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20way%20to%20NMR%20structures%20of%20proteins&amp;journal=Nat.%20Struct.%20Biol.&amp;doi=10.1038%2Fnsb1101-923&amp;volume=8&amp;pages=923-925&amp;publication_year=2001&amp;author=W%C3%BCthrich%2CK">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR5">
           wwPDB Consortium. Protein Data Bank: the single global archive for 3D macromolecular structure data.
           <i>
            Nucleic Acids Res
           </i>
           .
           <b>
            47
           </b>
           , D520–D528 (2018).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1093/nar/gky949" href="https://doi-org.proxy.lib.ohio-state.edu/10.1093%2Fnar%2Fgky949">
            Article
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Protein%20Data%20Bank%3A%20the%20single%20global%20archive%20for%203D%20macromolecular%20structure%20data&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgky949&amp;volume=47&amp;pages=D520-D528&amp;publication_year=2018">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR6">
           Mitchell, A. L. et al. MGnify: the microbiome analysis resource in 2020.
           <i>
            Nucleic Acids Res
           </i>
           .
           <b>
            48
           </b>
           , D570–D578 (2020).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="cas reference" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/cas-redirect/1:CAS:528:DC%2BB3cXhs1GltrjN">
            CAS
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31696235">
            PubMed
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=MGnify%3A%20the%20microbiome%20analysis%20resource%20in%202020&amp;journal=Nucleic%20Acids%20Res.&amp;volume=48&amp;pages=D570-D578&amp;publication_year=2020&amp;author=Mitchell%2CAL">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR7">
           Steinegger, M., Mirdita, M. &amp; Söding, J. Protein-level assembly increases protein sequence recovery from metagenomic samples manyfold.
           <i>
            Nat. Methods
           </i>
           <b>
            16
           </b>
           , 603–606 (2019).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1038/s41592-019-0437-4" href="https://doi-org.proxy.lib.ohio-state.edu/10.1038%2Fs41592-019-0437-4">
            Article
           </a>
           <a data-track="click" data-track-action="cas reference" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/cas-redirect/1:CAS:528:DC%2BC1MXht1eku7vJ">
            CAS
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31235882">
            PubMed
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Protein-level%20assembly%20increases%20protein%20sequence%20recovery%20from%20metagenomic%20samples%20manyfold&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fs41592-019-0437-4&amp;volume=16&amp;pages=603-606&amp;publication_year=2019&amp;author=Steinegger%2CM&amp;author=Mirdita%2CM&amp;author=S%C3%B6ding%2CJ">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR8">
           Dill, K. A., Ozkan, S. B., Shell, M. S. &amp; Weikl, T. R. The protein folding problem.
           <i>
            Annu. Rev. Biophys
           </i>
           .
           <b>
            37
           </b>
           , 289–316 (2008).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1146/annurev.biophys.37.092707.153558" href="https://doi-org.proxy.lib.ohio-state.edu/10.1146%2Fannurev.biophys.37.092707.153558">
            Article
           </a>
           <a data-track="click" data-track-action="ads reference" data-track-label="link" href="http://adsabs.harvard.edu.proxy.lib.ohio-state.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=1988PhRvC..37..289D">
            ADS
           </a>
           <a data-track="click" data-track-action="cas reference" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/cas-redirect/1:CAS:528:DC%2BD1cXnsVGlurw%3D">
            CAS
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18573083">
            PubMed
           </a>
           <a data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC2443096">
            PubMed Central
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20protein%20folding%20problem&amp;journal=Annu.%20Rev.%20Biophys.&amp;doi=10.1146%2Fannurev.biophys.37.092707.153558&amp;volume=37&amp;pages=289-316&amp;publication_year=2008&amp;author=Dill%2CKA&amp;author=Ozkan%2CSB&amp;author=Shell%2CMS&amp;author=Weikl%2CTR">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR9">
           Anfinsen, C. B. Principles that govern the folding of protein chains.
           <i>
            Science
           </i>
           <b>
            181
           </b>
           , 223–230 (1973).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1126/science.181.4096.223" href="https://doi-org.proxy.lib.ohio-state.edu/10.1126%2Fscience.181.4096.223">
            Article
           </a>
           <a data-track="click" data-track-action="ads reference" data-track-label="link" href="http://adsabs.harvard.edu.proxy.lib.ohio-state.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=1973Sci...181..223A">
            ADS
           </a>
           <a data-track="click" data-track-action="cas reference" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/cas-redirect/1:CAS:528:DyaE3sXkvVygtbc%3D">
            CAS
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=4124164">
            PubMed
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Principles%20that%20govern%20the%20folding%20of%20protein%20chains&amp;journal=Science&amp;doi=10.1126%2Fscience.181.4096.223&amp;volume=181&amp;pages=223-230&amp;publication_year=1973&amp;author=Anfinsen%2CCB">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR10">
           Senior, A. W. et al. Improved protein structure prediction using potentials from deep learning.
           <i>
            Nature
           </i>
           <b>
            577
           </b>
           , 706–710 (2020).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1038/s41586-019-1923-7" href="https://doi-org.proxy.lib.ohio-state.edu/10.1038%2Fs41586-019-1923-7">
            Article
           </a>
           <a data-track="click" data-track-action="ads reference" data-track-label="link" href="http://adsabs.harvard.edu.proxy.lib.ohio-state.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2020Natur.577..706S">
            ADS
           </a>
           <a data-track="click" data-track-action="cas reference" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/cas-redirect/1:CAS:528:DC%2BB3cXis1SisL0%3D">
            CAS
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31942072">
            PubMed
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Improved%20protein%20structure%20prediction%20using%20potentials%20from%20deep%20learning&amp;journal=Nature&amp;doi=10.1038%2Fs41586-019-1923-7&amp;volume=577&amp;pages=706-710&amp;publication_year=2020&amp;author=Senior%2CAW">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR11">
           Wang, S., Sun, S., Li, Z., Zhang, R. &amp; Xu, J. Accurate de novo prediction of protein contact map by ultra-deep learning model.
           <i>
            PLOS Comput. Biol
           </i>
           .
           <b>
            13
           </b>
           , e1005324 (2017).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1371/journal.pcbi.1005324" href="https://doi-org.proxy.lib.ohio-state.edu/10.1371%2Fjournal.pcbi.1005324">
            Article
           </a>
           <a data-track="click" data-track-action="ads reference" data-track-label="link" href="http://adsabs.harvard.edu.proxy.lib.ohio-state.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2017PLSCB..13E5324W">
            ADS
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28056090">
            PubMed
           </a>
           <a data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC5249242">
            PubMed Central
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Accurate%20de%20novo%20prediction%20of%20protein%20contact%20map%20by%20ultra-deep%20learning%20model&amp;journal=PLOS%20Comput.%20Biol.&amp;doi=10.1371%2Fjournal.pcbi.1005324&amp;volume=13&amp;publication_year=2017&amp;author=Wang%2CS&amp;author=Sun%2CS&amp;author=Li%2CZ&amp;author=Zhang%2CR&amp;author=Xu%2CJ">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR12">
           Zheng, W. et al. Deep-learning contact-map guided protein structure prediction in CASP13.
           <i>
            Proteins
           </i>
           <b>
            87
           </b>
           , 1149–1164 (2019).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1002/prot.25792" href="https://doi-org.proxy.lib.ohio-state.edu/10.1002%2Fprot.25792">
            Article
           </a>
           <a data-track="click" data-track-action="cas reference" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/cas-redirect/1:CAS:528:DC%2BC1MXhsFKgsrnK">
            CAS
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31365149">
            PubMed
           </a>
           <a data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC6851476">
            PubMed Central
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Deep-learning%20contact-map%20guided%20protein%20structure%20prediction%20in%20CASP13&amp;journal=Proteins&amp;doi=10.1002%2Fprot.25792&amp;volume=87&amp;pages=1149-1164&amp;publication_year=2019&amp;author=Zheng%2CW">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR13">
           Abriata, L. A., Tamò, G. E. &amp; Dal Peraro, M. A further leap of improvement in tertiary structure prediction in CASP13 prompts new routes for future assessments.
           <i>
            Proteins
           </i>
           <b>
            87
           </b>
           , 1100–1112 (2019).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1002/prot.25787" href="https://doi-org.proxy.lib.ohio-state.edu/10.1002%2Fprot.25787">
            Article
           </a>
           <a data-track="click" data-track-action="cas reference" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/cas-redirect/1:CAS:528:DC%2BC1MXhsFaqs77K">
            CAS
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31344267">
            PubMed
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20further%20leap%20of%20improvement%20in%20tertiary%20structure%20prediction%20in%20CASP13%20prompts%20new%20routes%20for%20future%20assessments&amp;journal=Proteins&amp;doi=10.1002%2Fprot.25787&amp;volume=87&amp;pages=1100-1112&amp;publication_year=2019&amp;author=Abriata%2CLA&amp;author=Tam%C3%B2%2CGE&amp;author=Dal%20Peraro%2CM">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR14">
           Pearce, R. &amp; Zhang, Y. Deep learning techniques have significantly impacted protein structure prediction and protein design.
           <i>
            Curr. Opin. Struct. Biol
           </i>
           .
           <b>
            68
           </b>
           , 194–207 (2021).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1016/j.sbi.2021.01.007" href="https://doi-org.proxy.lib.ohio-state.edu/10.1016%2Fj.sbi.2021.01.007">
            Article
           </a>
           <a data-track="click" data-track-action="cas reference" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/cas-redirect/1:CAS:528:DC%2BB3MXks1Ghtr4%3D">
            CAS
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33639355">
            PubMed
           </a>
           <a data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC8222070">
            PubMed Central
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Deep%20learning%20techniques%20have%20significantly%20impacted%20protein%20structure%20prediction%20and%20protein%20design&amp;journal=Curr.%20Opin.%20Struct.%20Biol.&amp;doi=10.1016%2Fj.sbi.2021.01.007&amp;volume=68&amp;pages=194-207&amp;publication_year=2021&amp;author=Pearce%2CR&amp;author=Zhang%2CY">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR15">
           Moult, J., Fidelis, K., Kryshtafovych, A., Schwede, T. &amp; Topf, M. Critical assessment of techniques for protein structure prediction, fourteenth round.
           <i>
            CASP 14 Abstract Book
           </i>
           <a data-track="click" data-track-action="external reference" data-track-label="https://www.predictioncenter.org/casp14/doc/CASP14_Abstracts.pdf" href="https://www.predictioncenter.org/casp14/doc/CASP14_Abstracts.pdf">
            https://www.predictioncenter.org/casp14/doc/CASP14_Abstracts.pdf
           </a>
           (2020).
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR16">
           Brini, E., Simmerling, C. &amp; Dill, K. Protein storytelling through physics.
           <i>
            Science
           </i>
           <b>
            370
           </b>
           , eaaz3041 (2020).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1126/science.aaz3041" href="https://doi-org.proxy.lib.ohio-state.edu/10.1126%2Fscience.aaz3041">
            Article
           </a>
           <a data-track="click" data-track-action="cas reference" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/cas-redirect/1:CAS:528:DC%2BB3cXisVOmu7vF">
            CAS
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33243857">
            PubMed
           </a>
           <a data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC7945008">
            PubMed Central
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Protein%20storytelling%20through%20physics&amp;journal=Science&amp;doi=10.1126%2Fscience.aaz3041&amp;volume=370&amp;publication_year=2020&amp;author=Brini%2CE&amp;author=Simmerling%2CC&amp;author=Dill%2CK">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR17">
           Sippl, M. J. Calculation of conformational ensembles from potentials of mean force. An approach to the knowledge-based prediction of local structures in globular proteins.
           <i>
            J. Mol. Biol
           </i>
           .
           <b>
            213
           </b>
           , 859–883 (1990).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1016/S0022-2836(05)80269-4" href="https://doi-org.proxy.lib.ohio-state.edu/10.1016%2FS0022-2836%2805%2980269-4">
            Article
           </a>
           <a data-track="click" data-track-action="cas reference" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/cas-redirect/1:CAS:528:DyaK3cXkvFCgs7Y%3D">
            CAS
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=2359125">
            PubMed
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Calculation%20of%20conformational%20ensembles%20from%20potentials%20of%20mean%20force.%20An%20approach%20to%20the%20knowledge-based%20prediction%20of%20local%20structures%20in%20globular%20proteins&amp;journal=J.%20Mol.%20Biol.&amp;doi=10.1016%2FS0022-2836%2805%2980269-4&amp;volume=213&amp;pages=859-883&amp;publication_year=1990&amp;author=Sippl%2CMJ">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR18">
           Šali, A. &amp; Blundell, T. L. Comparative protein modelling by satisfaction of spatial restraints.
           <i>
            J. Mol. Biol
           </i>
           .
           <b>
            234
           </b>
           , 779–815 (1993).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1006/jmbi.1993.1626" href="https://doi-org.proxy.lib.ohio-state.edu/10.1006%2Fjmbi.1993.1626">
            Article
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=8254673">
            PubMed
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Comparative%20protein%20modelling%20by%20satisfaction%20of%20spatial%20restraints&amp;journal=J.%20Mol.%20Biol.&amp;doi=10.1006%2Fjmbi.1993.1626&amp;volume=234&amp;pages=779-815&amp;publication_year=1993&amp;author=%C5%A0ali%2CA&amp;author=Blundell%2CTL">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR19">
           Roy, A., Kucukural, A. &amp; Zhang, Y. I-TASSER: a unified platform for automated protein structure and function prediction.
           <i>
            Nat. Protocols
           </i>
           <b>
            5
           </b>
           , 725–738 (2010).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1038/nprot.2010.5" href="https://doi-org.proxy.lib.ohio-state.edu/10.1038%2Fnprot.2010.5">
            Article
           </a>
           <a data-track="click" data-track-action="cas reference" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/cas-redirect/1:CAS:528:DC%2BC3cXksVahs74%3D">
            CAS
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20360767">
            PubMed
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=I-TASSER%3A%20a%20unified%20platform%20for%20automated%20protein%20structure%20and%20function%20prediction&amp;journal=Nat.%20Protocols&amp;doi=10.1038%2Fnprot.2010.5&amp;volume=5&amp;pages=725-738&amp;publication_year=2010&amp;author=Roy%2CA&amp;author=Kucukural%2CA&amp;author=Zhang%2CY">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR20">
           Altschuh, D., Lesk, A. M., Bloomer, A. C. &amp; Klug, A. Correlation of co-ordinated amino acid substitutions with function in viruses related to tobacco mosaic virus.
           <i>
            J. Mol. Biol
           </i>
           .
           <b>
            193
           </b>
           , 693–707 (1987).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1016/0022-2836(87)90352-4" href="https://doi-org.proxy.lib.ohio-state.edu/10.1016%2F0022-2836%2887%2990352-4">
            Article
           </a>
           <a data-track="click" data-track-action="cas reference" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/cas-redirect/1:CAS:528:DyaL2sXitV2ksL8%3D">
            CAS
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=3612789">
            PubMed
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Correlation%20of%20co-ordinated%20amino%20acid%20substitutions%20with%20function%20in%20viruses%20related%20to%20tobacco%20mosaic%20virus&amp;journal=J.%20Mol.%20Biol.&amp;doi=10.1016%2F0022-2836%2887%2990352-4&amp;volume=193&amp;pages=693-707&amp;publication_year=1987&amp;author=Altschuh%2CD&amp;author=Lesk%2CAM&amp;author=Bloomer%2CAC&amp;author=Klug%2CA">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR21">
           Shindyalov, I. N., Kolchanov, N. A. &amp; Sander, C. Can three-dimensional contacts in protein structures be predicted by analysis of correlated mutations?
           <i>
            Protein Eng
           </i>
           .
           <b>
            7
           </b>
           , 349–358 (1994).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1093/protein/7.3.349" href="https://doi-org.proxy.lib.ohio-state.edu/10.1093%2Fprotein%2F7.3.349">
            Article
           </a>
           <a data-track="click" data-track-action="cas reference" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/cas-redirect/1:CAS:528:DyaK2cXitFWqtbs%3D">
            CAS
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=8177884">
            PubMed
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Can%20three-dimensional%20contacts%20in%20protein%20structures%20be%20predicted%20by%20analysis%20of%20correlated%20mutations%3F&amp;journal=Protein%20Eng.&amp;doi=10.1093%2Fprotein%2F7.3.349&amp;volume=7&amp;pages=349-358&amp;publication_year=1994&amp;author=Shindyalov%2CIN&amp;author=Kolchanov%2CNA&amp;author=Sander%2CC">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR22">
           Weigt, M., White, R. A., Szurmant, H., Hoch, J. A. &amp; Hwa, T. Identification of direct residue contacts in protein–protein interaction by message passing.
           <i>
            Proc. Natl Acad. Sci. USA
           </i>
           <b>
            106
           </b>
           , 67–72 (2009).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1073/pnas.0805923106" href="https://doi-org.proxy.lib.ohio-state.edu/10.1073%2Fpnas.0805923106">
            Article
           </a>
           <a data-track="click" data-track-action="ads reference" data-track-label="link" href="http://adsabs.harvard.edu.proxy.lib.ohio-state.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2009PNAS..106...67W">
            ADS
           </a>
           <a data-track="click" data-track-action="cas reference" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/cas-redirect/1:CAS:528:DC%2BD1MXltF2jug%3D%3D">
            CAS
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19116270">
            PubMed
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Identification%20of%20direct%20residue%20contacts%20in%20protein%E2%80%93protein%20interaction%20by%20message%20passing&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.0805923106&amp;volume=106&amp;pages=67-72&amp;publication_year=2009&amp;author=Weigt%2CM&amp;author=White%2CRA&amp;author=Szurmant%2CH&amp;author=Hoch%2CJA&amp;author=Hwa%2CT">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR23">
           Marks, D. S. et al. Protein 3D structure computed from evolutionary sequence variation.
           <i>
            PLoS ONE
           </i>
           <b>
            6
           </b>
           , e28766 (2011).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1371/journal.pone.0028766" href="https://doi-org.proxy.lib.ohio-state.edu/10.1371%2Fjournal.pone.0028766">
            Article
           </a>
           <a data-track="click" data-track-action="ads reference" data-track-label="link" href="http://adsabs.harvard.edu.proxy.lib.ohio-state.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2011PLoSO...628766M">
            ADS
           </a>
           <a data-track="click" data-track-action="cas reference" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/cas-redirect/1:CAS:528:DC%2BC3MXhs1KhurnJ">
            CAS
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22163331">
            PubMed
           </a>
           <a data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC3233603">
            PubMed Central
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Protein%203D%20structure%20computed%20from%20evolutionary%20sequence%20variation&amp;journal=PLoS%20ONE&amp;doi=10.1371%2Fjournal.pone.0028766&amp;volume=6&amp;publication_year=2011&amp;author=Marks%2CDS">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR24">
           Jones, D. T., Buchan, D. W. A., Cozzetto, D. &amp; Pontil, M. PSICOV: precise structural contact prediction using sparse inverse covariance estimation on large multiple sequence alignments.
           <i>
            Bioinformatics
           </i>
           <b>
            28
           </b>
           , 184–190 (2012).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1093/bioinformatics/btr638" href="https://doi-org.proxy.lib.ohio-state.edu/10.1093%2Fbioinformatics%2Fbtr638">
            Article
           </a>
           <a data-track="click" data-track-action="cas reference" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/cas-redirect/1:CAS:528:DC%2BC38Xht1agurg%3D">
            CAS
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22101153">
            PubMed
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=PSICOV%3A%20precise%20structural%20contact%20prediction%20using%20sparse%20inverse%20covariance%20estimation%20on%20large%20multiple%20sequence%20alignments&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtr638&amp;volume=28&amp;pages=184-190&amp;publication_year=2012&amp;author=Jones%2CDT&amp;author=Buchan%2CDWA&amp;author=Cozzetto%2CD&amp;author=Pontil%2CM">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR25">
           Moult, J., Pedersen, J. T., Judson, R. &amp; Fidelis, K. A large-scale experiment to assess protein structure prediction methods.
           <i>
            Proteins
           </i>
           <b>
            23
           </b>
           , ii–iv (1995).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1002/prot.340230303" href="https://doi-org.proxy.lib.ohio-state.edu/10.1002%2Fprot.340230303">
            Article
           </a>
           <a data-track="click" data-track-action="cas reference" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/cas-redirect/1:STN:280:DyaK287oslCntw%3D%3D">
            CAS
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=8710822">
            PubMed
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20large-scale%20experiment%20to%20assess%20protein%20structure%20prediction%20methods&amp;journal=Proteins&amp;doi=10.1002%2Fprot.340230303&amp;volume=23&amp;pages=ii-iv&amp;publication_year=1995&amp;author=Moult%2CJ&amp;author=Pedersen%2CJT&amp;author=Judson%2CR&amp;author=Fidelis%2CK">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR26">
           Kryshtafovych, A., Schwede, T., Topf, M., Fidelis, K. &amp; Moult, J. Critical assessment of methods of protein structure prediction (CASP)-round XIII.
           <i>
            Proteins
           </i>
           <b>
            87
           </b>
           , 1011–1020 (2019).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1002/prot.25823" href="https://doi-org.proxy.lib.ohio-state.edu/10.1002%2Fprot.25823">
            Article
           </a>
           <a data-track="click" data-track-action="cas reference" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/cas-redirect/1:CAS:528:DC%2BC1MXitVSlt7%2FO">
            CAS
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31589781">
            PubMed
           </a>
           <a data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC6927249">
            PubMed Central
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Critical%20assessment%20of%20methods%20of%20protein%20structure%20prediction%20%28CASP%29-round%20XIII&amp;journal=Proteins&amp;doi=10.1002%2Fprot.25823&amp;volume=87&amp;pages=1011-1020&amp;publication_year=2019&amp;author=Kryshtafovych%2CA&amp;author=Schwede%2CT&amp;author=Topf%2CM&amp;author=Fidelis%2CK&amp;author=Moult%2CJ">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR27">
           Zhang, Y. &amp; Skolnick, J. Scoring function for automated assessment of protein structure template quality.
           <i>
            Proteins
           </i>
           <b>
            57
           </b>
           , 702–710 (2004).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1002/prot.20264" href="https://doi-org.proxy.lib.ohio-state.edu/10.1002%2Fprot.20264">
            Article
           </a>
           <a data-track="click" data-track-action="cas reference" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/cas-redirect/1:CAS:528:DC%2BD2cXhtVaqtLvI">
            CAS
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15476259">
            PubMed
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Scoring%20function%20for%20automated%20assessment%20of%20protein%20structure%20template%20quality&amp;journal=Proteins&amp;doi=10.1002%2Fprot.20264&amp;volume=57&amp;pages=702-710&amp;publication_year=2004&amp;author=Zhang%2CY&amp;author=Skolnick%2CJ">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR28">
           Tu, Z. &amp; Bai, X. Auto-context and its application to high-level vision tasks and 3D brain image segmentation.
           <i>
            IEEE Trans. Pattern Anal. Mach. Intell
           </i>
           .
           <b>
            32
           </b>
           , 1744–1757 (2010).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1109/TPAMI.2009.186" href="https://doi-org.proxy.lib.ohio-state.edu/10.1109%2FTPAMI.2009.186">
            Article
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20724753">
            PubMed
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Auto-context%20and%20its%20application%20to%20high-level%20vision%20tasks%20and%203D%20brain%20image%20segmentation&amp;journal=IEEE%20Trans.%20Pattern%20Anal.%20Mach.%20Intell.&amp;doi=10.1109%2FTPAMI.2009.186&amp;volume=32&amp;pages=1744-1757&amp;publication_year=2010&amp;author=Tu%2CZ&amp;author=Bai%2CX">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR29">
           Carreira, J., Agrawal, P., Fragkiadaki, K. &amp; Malik, J. Human pose estimation with iterative error feedback. In
           <i>
            Proc. IEEE Conference on Computer Vision and Pattern Recognition
           </i>
           4733–4742 (2016).
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR30">
           Mirabello, C. &amp; Wallner, B. rawMSA: end-to-end deep learning using raw multiple sequence alignments.
           <i>
            PLoS ONE
           </i>
           <b>
            14
           </b>
           , e0220182 (2019).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1371/journal.pone.0220182" href="https://doi-org.proxy.lib.ohio-state.edu/10.1371%2Fjournal.pone.0220182">
            Article
           </a>
           <a data-track="click" data-track-action="cas reference" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/cas-redirect/1:CAS:528:DC%2BC1MXhvFamur7E">
            CAS
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31415569">
            PubMed
           </a>
           <a data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC6695225">
            PubMed Central
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=rawMSA%3A%20end-to-end%20deep%20learning%20using%20raw%20multiple%20sequence%20alignments&amp;journal=PLoS%20ONE&amp;doi=10.1371%2Fjournal.pone.0220182&amp;volume=14&amp;publication_year=2019&amp;author=Mirabello%2CC&amp;author=Wallner%2CB">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR31">
           Huang, Z. et al. CCNet: criss-cross attention for semantic segmentation. In
           <i>
            Proc. IEEE/CVF International Conference on Computer Vision
           </i>
           603–612 (2019).
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR32">
           Hornak, V. et al. Comparison of multiple Amber force fields and development of improved protein backbone parameters.
           <i>
            Proteins
           </i>
           <b>
            65
           </b>
           , 712–725 (2006).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1002/prot.21123" href="https://doi-org.proxy.lib.ohio-state.edu/10.1002%2Fprot.21123">
            Article
           </a>
           <a data-track="click" data-track-action="cas reference" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/cas-redirect/1:CAS:528:DC%2BD28XhtFWqt7fM">
            CAS
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16981200">
            PubMed
           </a>
           <a data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC4805110">
            PubMed Central
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Comparison%20of%20multiple%20Amber%20force%20fields%20and%20development%20of%20improved%20protein%20backbone%20parameters&amp;journal=Proteins&amp;doi=10.1002%2Fprot.21123&amp;volume=65&amp;pages=712-725&amp;publication_year=2006&amp;author=Hornak%2CV">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR33">
           Zemla, A. LGA: a method for finding 3D similarities in protein structures.
           <i>
            Nucleic Acids Res
           </i>
           .
           <b>
            31
           </b>
           , 3370–3374 (2003).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1093/nar/gkg571" href="https://doi-org.proxy.lib.ohio-state.edu/10.1093%2Fnar%2Fgkg571">
            Article
           </a>
           <a data-track="click" data-track-action="cas reference" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/cas-redirect/1:CAS:528:DC%2BD3sXltVWjtbk%3D">
            CAS
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12824330">
            PubMed
           </a>
           <a data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC168977">
            PubMed Central
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=LGA%3A%20a%20method%20for%20finding%203D%20similarities%20in%20protein%20structures&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgkg571&amp;volume=31&amp;pages=3370-3374&amp;publication_year=2003&amp;author=Zemla%2CA">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR34">
           Mariani, V., Biasini, M., Barbato, A. &amp; Schwede, T. lDDT: a local superposition-free score for comparing protein structures and models using distance difference tests.
           <i>
            Bioinformatics
           </i>
           <b>
            29
           </b>
           , 2722–2728 (2013).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1093/bioinformatics/btt473" href="https://doi-org.proxy.lib.ohio-state.edu/10.1093%2Fbioinformatics%2Fbtt473">
            Article
           </a>
           <a data-track="click" data-track-action="cas reference" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/cas-redirect/1:CAS:528:DC%2BC3sXhs1CisrfK">
            CAS
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23986568">
            PubMed
           </a>
           <a data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC3799472">
            PubMed Central
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=lDDT%3A%20a%20local%20superposition-free%20score%20for%20comparing%20protein%20structures%20and%20models%20using%20distance%20difference%20tests&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtt473&amp;volume=29&amp;pages=2722-2728&amp;publication_year=2013&amp;author=Mariani%2CV&amp;author=Biasini%2CM&amp;author=Barbato%2CA&amp;author=Schwede%2CT">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR35">
           Xie, Q., Luong, M.-T., Hovy, E. &amp; Le, Q. V. Self-training with noisy student improves imagenet classification. In
           <i>
            Proc. IEEE/CVF Conference on Computer Vision and Pattern Recognition
           </i>
           10687–10698 (2020).
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR36">
           Mirdita, M. et al. Uniclust databases of clustered and deeply annotated protein sequences and alignments.
           <i>
            Nucleic Acids Res
           </i>
           .
           <b>
            45
           </b>
           , D170–D176 (2017).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1093/nar/gkw1081" href="https://doi-org.proxy.lib.ohio-state.edu/10.1093%2Fnar%2Fgkw1081">
            Article
           </a>
           <a data-track="click" data-track-action="cas reference" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/cas-redirect/1:CAS:528:DC%2BC1cXhslWgsb8%3D">
            CAS
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27899574">
            PubMed
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Uniclust%20databases%20of%20clustered%20and%20deeply%20annotated%20protein%20sequences%20and%20alignments&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgkw1081&amp;volume=45&amp;pages=D170-D176&amp;publication_year=2017&amp;author=Mirdita%2CM">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR37">
           Devlin, J., Chang, M.-W., Lee, K. &amp; Toutanova, K. BERT: pre-training of deep bidirectional transformers for language understanding. In
           <i>
            Proc. 2019 Conference of the North American Chapter of the Association for Computational Linguistics: Human Language Technologies
           </i>
           1, 4171–4186 (2019).
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR38">
           Rao, R. et al. MSA transformer. In
           <i>
            Proc. 38th International Conference on Machine Learning
           </i>
           PMLR 139, 8844–8856 (2021).
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR39">
           Tunyasuvunakool, K. et al. Highly accurate protein structure prediction for the human proteome.
           <i>
            Nature
           </i>
           <a data-track="click" data-track-action="external reference" data-track-label="10.1038/s41586-021-03828-1" href="https://doi-org.proxy.lib.ohio-state.edu/10.1038/s41586-021-03828-1">
            https://doi-org.proxy.lib.ohio-state.edu/10.1038/s41586-021-03828-1
           </a>
           (2021).
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR40">
           Kuhlman, B. &amp; Bradley, P. Advances in protein structure prediction and design.
           <i>
            Nat. Rev. Mol. Cell Biol
           </i>
           .
           <b>
            20
           </b>
           , 681–697 (2019).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1038/s41580-019-0163-x" href="https://doi-org.proxy.lib.ohio-state.edu/10.1038%2Fs41580-019-0163-x">
            Article
           </a>
           <a data-track="click" data-track-action="cas reference" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/cas-redirect/1:CAS:528:DC%2BC1MXhsFyksL7J">
            CAS
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31417196">
            PubMed
           </a>
           <a data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC7032036">
            PubMed Central
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Advances%20in%20protein%20structure%20prediction%20and%20design&amp;journal=Nat.%20Rev.%20Mol.%20Cell%20Biol.&amp;doi=10.1038%2Fs41580-019-0163-x&amp;volume=20&amp;pages=681-697&amp;publication_year=2019&amp;author=Kuhlman%2CB&amp;author=Bradley%2CP">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR41">
           Marks, D. S., Hopf, T. A. &amp; Sander, C. Protein structure prediction from sequence variation.
           <i>
            Nat. Biotechnol
           </i>
           .
           <b>
            30
           </b>
           , 1072–1080 (2012).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1038/nbt.2419" href="https://doi-org.proxy.lib.ohio-state.edu/10.1038%2Fnbt.2419">
            Article
           </a>
           <a data-track="click" data-track-action="cas reference" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/cas-redirect/1:CAS:528:DC%2BC38Xhs1elt7bM">
            CAS
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23138306">
            PubMed
           </a>
           <a data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC4319528">
            PubMed Central
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Protein%20structure%20prediction%20from%20sequence%20variation&amp;journal=Nat.%20Biotechnol.&amp;doi=10.1038%2Fnbt.2419&amp;volume=30&amp;pages=1072-1080&amp;publication_year=2012&amp;author=Marks%2CDS&amp;author=Hopf%2CTA&amp;author=Sander%2CC">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR42">
           Qian, N. &amp; Sejnowski, T. J. Predicting the secondary structure of globular proteins using neural network models.
           <i>
            J. Mol. Biol
           </i>
           .
           <b>
            202
           </b>
           , 865–884 (1988).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1016/0022-2836(88)90564-5" href="https://doi-org.proxy.lib.ohio-state.edu/10.1016%2F0022-2836%2888%2990564-5">
            Article
           </a>
           <a data-track="click" data-track-action="cas reference" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/cas-redirect/1:CAS:528:DyaL1MXhtlWksb0%3D">
            CAS
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=3172241">
            PubMed
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Predicting%20the%20secondary%20structure%20of%20globular%20proteins%20using%20neural%20network%20models&amp;journal=J.%20Mol.%20Biol.&amp;doi=10.1016%2F0022-2836%2888%2990564-5&amp;volume=202&amp;pages=865-884&amp;publication_year=1988&amp;author=Qian%2CN&amp;author=Sejnowski%2CTJ">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR43">
           Fariselli, P., Olmea, O., Valencia, A. &amp; Casadio, R. Prediction of contact maps with neural networks and correlated mutations.
           <i>
            Protein Eng
           </i>
           .
           <b>
            14
           </b>
           , 835–843 (2001).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1093/protein/14.11.835" href="https://doi-org.proxy.lib.ohio-state.edu/10.1093%2Fprotein%2F14.11.835">
            Article
           </a>
           <a data-track="click" data-track-action="cas reference" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/cas-redirect/1:CAS:528:DC%2BD38XjtVentA%3D%3D">
            CAS
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11742102">
            PubMed
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Prediction%20of%20contact%20maps%20with%20neural%20networks%20and%20correlated%20mutations&amp;journal=Protein%20Eng.&amp;doi=10.1093%2Fprotein%2F14.11.835&amp;volume=14&amp;pages=835-843&amp;publication_year=2001&amp;author=Fariselli%2CP&amp;author=Olmea%2CO&amp;author=Valencia%2CA&amp;author=Casadio%2CR">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR44">
           Yang, J. et al. Improved protein structure prediction using predicted interresidue orientations.
           <i>
            Proc. Natl Acad. Sci. USA
           </i>
           <b>
            117
           </b>
           , 1496–1503 (2020).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1073/pnas.1914677117" href="https://doi-org.proxy.lib.ohio-state.edu/10.1073%2Fpnas.1914677117">
            Article
           </a>
           <a data-track="click" data-track-action="ads reference" data-track-label="link" href="http://adsabs.harvard.edu.proxy.lib.ohio-state.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2020PNAS..117.1496Y">
            ADS
           </a>
           <a data-track="click" data-track-action="cas reference" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsFKrsLg%3D">
            CAS
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31896580">
            PubMed
           </a>
           <a data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC6983395">
            PubMed Central
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Improved%20protein%20structure%20prediction%20using%20predicted%20interresidue%20orientations&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.1914677117&amp;volume=117&amp;pages=1496-1503&amp;publication_year=2020&amp;author=Yang%2CJ">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR45">
           Li, Y. et al. Deducing high-accuracy protein contact-maps from a triplet of coevolutionary matrices through deep residual convolutional networks.
           <i>
            PLOS Comput. Biol
           </i>
           .
           <b>
            17
           </b>
           , e1008865 (2021).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1371/journal.pcbi.1008865" href="https://doi-org.proxy.lib.ohio-state.edu/10.1371%2Fjournal.pcbi.1008865">
            Article
           </a>
           <a data-track="click" data-track-action="ads reference" data-track-label="link" href="http://adsabs.harvard.edu.proxy.lib.ohio-state.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2021mwaw.book.....L">
            ADS
           </a>
           <a data-track="click" data-track-action="cas reference" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/cas-redirect/1:CAS:528:DC%2BB3MXosFSms78%3D">
            CAS
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33770072">
            PubMed
           </a>
           <a data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC8026059">
            PubMed Central
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Deducing%20high-accuracy%20protein%20contact-maps%20from%20a%20triplet%20of%20coevolutionary%20matrices%20through%20deep%20residual%20convolutional%20networks&amp;journal=PLOS%20Comput.%20Biol.&amp;doi=10.1371%2Fjournal.pcbi.1008865&amp;volume=17&amp;publication_year=2021&amp;author=Li%2CY">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR46">
           He, K., Zhang, X., Ren, S. &amp; Sun, J. Deep residual learning for image recognition. In
           <i>
            Proc. IEEE Conference on Computer Vision and Pattern Recognition
           </i>
           770–778 (2016).
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR47">
           AlQuraishi, M. End-to-end differentiable learning of protein structure.
           <i>
            Cell Syst
           </i>
           .
           <b>
            8
           </b>
           , 292–301 (2019).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1016/j.cels.2019.03.006" href="https://doi-org.proxy.lib.ohio-state.edu/10.1016%2Fj.cels.2019.03.006">
            Article
           </a>
           <a data-track="click" data-track-action="cas reference" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/cas-redirect/1:CAS:528:DC%2BC1MXosVyhtb0%3D">
            CAS
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31005579">
            PubMed
           </a>
           <a data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC6513320">
            PubMed Central
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=End-to-end%20differentiable%20learning%20of%20protein%20structure&amp;journal=Cell%20Syst.&amp;doi=10.1016%2Fj.cels.2019.03.006&amp;volume=8&amp;pages=292-301&amp;publication_year=2019&amp;author=AlQuraishi%2CM">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR48">
           Senior, A. W. et al. Protein structure prediction using multiple deep neural networks in the 13th Critical Assessment of Protein Structure Prediction (CASP13).
           <i>
            Proteins
           </i>
           <b>
            87
           </b>
           , 1141–1148 (2019).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1002/prot.25834" href="https://doi-org.proxy.lib.ohio-state.edu/10.1002%2Fprot.25834">
            Article
           </a>
           <a data-track="click" data-track-action="cas reference" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/cas-redirect/1:CAS:528:DC%2BC1MXitFartb%2FK">
            CAS
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31602685">
            PubMed
           </a>
           <a data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC7079254">
            PubMed Central
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Protein%20structure%20prediction%20using%20multiple%20deep%20neural%20networks%20in%20the%2013th%20Critical%20Assessment%20of%20Protein%20Structure%20Prediction%20%28CASP13%29&amp;journal=Proteins&amp;doi=10.1002%2Fprot.25834&amp;volume=87&amp;pages=1141-1148&amp;publication_year=2019&amp;author=Senior%2CAW">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR49">
           Ingraham, J., Riesselman, A. J., Sander, C. &amp; Marks, D. S. Learning protein structure with a differentiable simulator. in
           <i>
            Proc. International Conference on Learning Representations
           </i>
           (2019).
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR50">
           Li, J. Universal transforming geometric network. Preprint at
           <a data-track="click" data-track-action="external reference" data-track-label="https://arxiv-org.proxy.lib.ohio-state.edu/abs/1908.00723" href="https://arxiv-org.proxy.lib.ohio-state.edu/abs/1908.00723">
            https://arxiv-org.proxy.lib.ohio-state.edu/abs/1908.00723
           </a>
           (2019).
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR51">
           Xu, J., McPartlon, M. &amp; Li, J. Improved protein structure prediction by deep learning irrespective of co-evolution information.
           <i>
            Nat. Mach. Intell
           </i>
           .
           <b>
            3
           </b>
           , 601–609 (2021).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1038/s42256-021-00348-5" href="https://doi-org.proxy.lib.ohio-state.edu/10.1038%2Fs42256-021-00348-5">
            Article
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34368623">
            PubMed
           </a>
           <a data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC8340610">
            PubMed Central
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Improved%20protein%20structure%20prediction%20by%20deep%20learning%20irrespective%20of%20co-evolution%20information&amp;journal=Nat.%20Mach.%20Intell.&amp;doi=10.1038%2Fs42256-021-00348-5&amp;volume=3&amp;pages=601-609&amp;publication_year=2021&amp;author=Xu%2CJ&amp;author=McPartlon%2CM&amp;author=Li%2CJ">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR52">
           Vaswani, A. et al. Attention is all you need. In
           <i>
            Advances in Neural Information Processing Systems
           </i>
           5998–6008 (2017).
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR53">
           Wang, H. et al. Axial-deeplab: stand-alone axial-attention for panoptic segmentation. in
           <i>
            European Conference on Computer Vision
           </i>
           108–126 (Springer, 2020).
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR54">
           Alley, E. C., Khimulya, G., Biswas, S., AlQuraishi, M. &amp; Church, G. M. Unified rational protein engineering with sequence-based deep representation learning.
           <i>
            Nat. Methods
           </i>
           <b>
            16
           </b>
           , 1315–1322 (2019).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1038/s41592-019-0598-1" href="https://doi-org.proxy.lib.ohio-state.edu/10.1038%2Fs41592-019-0598-1">
            Article
           </a>
           <a data-track="click" data-track-action="cas reference" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/cas-redirect/1:CAS:528:DC%2BC1MXitVSlsbnJ">
            CAS
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31636460">
            PubMed
           </a>
           <a data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC7067682">
            PubMed Central
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Unified%20rational%20protein%20engineering%20with%20sequence-based%20deep%20representation%20learning&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fs41592-019-0598-1&amp;volume=16&amp;pages=1315-1322&amp;publication_year=2019&amp;author=Alley%2CEC&amp;author=Khimulya%2CG&amp;author=Biswas%2CS&amp;author=AlQuraishi%2CM&amp;author=Church%2CGM">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR55">
           Heinzinger, M. et al. Modeling aspects of the language of life through transfer-learning protein sequences.
           <i>
            BMC Bioinformatics
           </i>
           <b>
            20
           </b>
           , 723 (2019).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1186/s12859-019-3220-8" href="https://doi-org.proxy.lib.ohio-state.edu/10.1186%2Fs12859-019-3220-8">
            Article
           </a>
           <a data-track="click" data-track-action="cas reference" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/cas-redirect/1:CAS:528:DC%2BC1MXisVGjsLbK">
            CAS
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31847804">
            PubMed
           </a>
           <a data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC6918593">
            PubMed Central
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Modeling%20aspects%20of%20the%20language%20of%20life%20through%20transfer-learning%20protein%20sequences&amp;journal=BMC%20Bioinformatics&amp;doi=10.1186%2Fs12859-019-3220-8&amp;volume=20&amp;publication_year=2019&amp;author=Heinzinger%2CM">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR56">
           Rives, A. et al. Biological structure and function emerge from scaling unsupervised learning to 250 million protein sequences.
           <i>
            Proc. Natl Acad. Sci. USA
           </i>
           <b>
            118
           </b>
           , e2016239118 (2021).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1073/pnas.2016239118" href="https://doi-org.proxy.lib.ohio-state.edu/10.1073%2Fpnas.2016239118">
            Article
           </a>
           <a data-track="click" data-track-action="cas reference" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/cas-redirect/1:CAS:528:DC%2BB3MXovVantro%3D">
            CAS
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33876751">
            PubMed
           </a>
           <a data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC8053943">
            PubMed Central
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Biological%20structure%20and%20function%20emerge%20from%20scaling%20unsupervised%20learning%20to%20250%20million%20protein%20sequences&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.2016239118&amp;volume=118&amp;publication_year=2021&amp;author=Rives%2CA">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR57">
           Pereira, J. et al. High-accuracy protein structure prediction in CASP14.
           <i>
            Proteins
           </i>
           <a data-track="click" data-track-action="external reference" data-track-label="10.1002/prot.26171" href="https://doi-org.proxy.lib.ohio-state.edu/10.1002/prot.26171">
            https://doi-org.proxy.lib.ohio-state.edu/10.1002/prot.26171
           </a>
           (2021).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1002/prot.26171" href="https://doi-org.proxy.lib.ohio-state.edu/10.1002%2Fprot.26171">
            Article
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34387010">
            PubMed
           </a>
           <a data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC8881082">
            PubMed Central
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=High-accuracy%20protein%20structure%20prediction%20in%20CASP14&amp;journal=Proteins&amp;doi=10.1002%2Fprot.26171&amp;publication_year=2021&amp;author=Pereira%2CJ">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR58">
           Gupta, M. et al. CryoEM and AI reveal a structure of SARS-CoV-2 Nsp2, a multifunctional protein involved in key host processes. Preprint at
           <a data-track="click" data-track-action="external reference" data-track-label="10.1101/2021.05.10.443524" href="https://doi-org.proxy.lib.ohio-state.edu/10.1101/2021.05.10.443524">
            https://doi-org.proxy.lib.ohio-state.edu/10.1101/2021.05.10.443524
           </a>
           (2021).
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR59">
           Ingraham, J., Garg, V. K., Barzilay, R. &amp; Jaakkola, T. Generative models for graph-based protein design. in
           <i>
            Proc. 33rd Conference on Neural Information Processing Systems
           </i>
           (2019).
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR60">
           Johnson, L. S., Eddy, S. R. &amp; Portugaly, E. Hidden Markov model speed heuristic and iterative HMM search procedure.
           <i>
            BMC Bioinformatics
           </i>
           <b>
            11
           </b>
           , 431 (2010).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1186/1471-2105-11-431" href="https://doi-org.proxy.lib.ohio-state.edu/10.1186%2F1471-2105-11-431">
            Article
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20718988">
            PubMed
           </a>
           <a data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC2931519">
            PubMed Central
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Hidden%20Markov%20model%20speed%20heuristic%20and%20iterative%20HMM%20search%20procedure&amp;journal=BMC%20Bioinformatics&amp;doi=10.1186%2F1471-2105-11-431&amp;volume=11&amp;publication_year=2010&amp;author=Johnson%2CLS&amp;author=Eddy%2CSR&amp;author=Portugaly%2CE">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR61">
           Remmert, M., Biegert, A., Hauser, A. &amp; Söding, J. HHblits: lightning-fast iterative protein sequence searching by HMM-HMM alignment.
           <i>
            Nat. Methods
           </i>
           <b>
            9
           </b>
           , 173–175 (2012).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1038/nmeth.1818" href="https://doi-org.proxy.lib.ohio-state.edu/10.1038%2Fnmeth.1818">
            Article
           </a>
           <a data-track="click" data-track-action="cas reference" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/cas-redirect/1:CAS:528:DC%2BC3MXhs1OltbnO">
            CAS
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=HHblits%3A%20lightning-fast%20iterative%20protein%20sequence%20searching%20by%20HMM-HMM%20alignment&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fnmeth.1818&amp;volume=9&amp;pages=173-175&amp;publication_year=2012&amp;author=Remmert%2CM&amp;author=Biegert%2CA&amp;author=Hauser%2CA&amp;author=S%C3%B6ding%2CJ">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR62">
           The UniProt Consortium. UniProt: the universal protein knowledgebase in 2021.
           <i>
            Nucleic Acids Res
           </i>
           .
           <b>
            49
           </b>
           , D480–D489 (2020).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1093/nar/gkaa1100" href="https://doi-org.proxy.lib.ohio-state.edu/10.1093%2Fnar%2Fgkaa1100">
            Article
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=UniProt%3A%20the%20universal%20protein%20knowledgebase%20in%202021&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgkaa1100&amp;volume=49&amp;pages=D480-D489&amp;publication_year=2020">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR63">
           Steinegger, M. &amp; Söding, J. Clustering huge protein sequence sets in linear time.
           <i>
            Nat. Commun
           </i>
           .
           <b>
            9
           </b>
           , 2542 (2018).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1038/s41467-018-04964-5" href="https://doi-org.proxy.lib.ohio-state.edu/10.1038%2Fs41467-018-04964-5">
            Article
           </a>
           <a data-track="click" data-track-action="ads reference" data-track-label="link" href="http://adsabs.harvard.edu.proxy.lib.ohio-state.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2018NatCo...9.2542S">
            ADS
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29959318">
            PubMed
           </a>
           <a data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC6026198">
            PubMed Central
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Clustering%20huge%20protein%20sequence%20sets%20in%20linear%20time&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-018-04964-5&amp;volume=9&amp;publication_year=2018&amp;author=Steinegger%2CM&amp;author=S%C3%B6ding%2CJ">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR64">
           Steinegger, M. &amp; Söding, J. MMseqs2 enables sensitive protein sequence searching for the analysis of massive data sets.
           <i>
            Nat. Biotechnol
           </i>
           .
           <b>
            35
           </b>
           , 1026–1028 (2017).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1038/nbt.3988" href="https://doi-org.proxy.lib.ohio-state.edu/10.1038%2Fnbt.3988">
            Article
           </a>
           <a data-track="click" data-track-action="cas reference" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/cas-redirect/1:CAS:528:DC%2BC2sXhs1GqsLzE">
            CAS
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29035372">
            PubMed
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=MMseqs2%20enables%20sensitive%20protein%20sequence%20searching%20for%20the%20analysis%20of%20massive%20data%20sets&amp;journal=Nat.%20Biotechnol.&amp;doi=10.1038%2Fnbt.3988&amp;volume=35&amp;pages=1026-1028&amp;publication_year=2017&amp;author=Steinegger%2CM&amp;author=S%C3%B6ding%2CJ">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR65">
           Deorowicz, S., Debudaj-Grabysz, A. &amp; Gudyś, A. FAMSA: fast and accurate multiple sequence alignment of huge protein families.
           <i>
            Sci. Rep
           </i>
           .
           <b>
            6
           </b>
           , 33964 (2016).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1038/srep33964" href="https://doi-org.proxy.lib.ohio-state.edu/10.1038%2Fsrep33964">
            Article
           </a>
           <a data-track="click" data-track-action="ads reference" data-track-label="link" href="http://adsabs.harvard.edu.proxy.lib.ohio-state.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2016NatSR...633964D">
            ADS
           </a>
           <a data-track="click" data-track-action="cas reference" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/cas-redirect/1:CAS:528:DC%2BC28XhsF2qs7fN">
            CAS
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27670777">
            PubMed
           </a>
           <a data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC5037421">
            PubMed Central
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=FAMSA%3A%20fast%20and%20accurate%20multiple%20sequence%20alignment%20of%20huge%20protein%20families&amp;journal=Sci.%20Rep.&amp;doi=10.1038%2Fsrep33964&amp;volume=6&amp;publication_year=2016&amp;author=Deorowicz%2CS&amp;author=Debudaj-Grabysz%2CA&amp;author=Gudy%C5%9B%2CA">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR66">
           Steinegger, M. et al. HH-suite3 for fast remote homology detection and deep protein annotation.
           <i>
            BMC Bioinformatics
           </i>
           <b>
            20
           </b>
           , 473 (2019).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1186/s12859-019-3019-7" href="https://doi-org.proxy.lib.ohio-state.edu/10.1186%2Fs12859-019-3019-7">
            Article
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31521110">
            PubMed
           </a>
           <a data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC6744700">
            PubMed Central
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=HH-suite3%20for%20fast%20remote%20homology%20detection%20and%20deep%20protein%20annotation&amp;journal=BMC%20Bioinformatics&amp;doi=10.1186%2Fs12859-019-3019-7&amp;volume=20&amp;publication_year=2019&amp;author=Steinegger%2CM">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR67">
           Suzek, B. E., Wang, Y., Huang, H., McGarvey, P. B. &amp; Wu, C. H. UniRef clusters: a comprehensive and scalable alternative for improving sequence similarity searches.
           <i>
            Bioinformatics
           </i>
           <b>
            31
           </b>
           , 926–932 (2015).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1093/bioinformatics/btu739" href="https://doi-org.proxy.lib.ohio-state.edu/10.1093%2Fbioinformatics%2Fbtu739">
            Article
           </a>
           <a data-track="click" data-track-action="cas reference" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/cas-redirect/1:CAS:528:DC%2BC28Xht1Gntb7F">
            CAS
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25398609">
            PubMed
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=UniRef%20clusters%3A%20a%20comprehensive%20and%20scalable%20alternative%20for%20improving%20sequence%20similarity%20searches&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtu739&amp;volume=31&amp;pages=926-932&amp;publication_year=2015&amp;author=Suzek%2CBE&amp;author=Wang%2CY&amp;author=Huang%2CH&amp;author=McGarvey%2CPB&amp;author=Wu%2CCH">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR68">
           Eddy, S. R. Accelerated profile HMM searches.
           <i>
            PLOS Comput. Biol
           </i>
           .
           <b>
            7
           </b>
           , e1002195 (2011).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1371/journal.pcbi.1002195" href="https://doi-org.proxy.lib.ohio-state.edu/10.1371%2Fjournal.pcbi.1002195">
            Article
           </a>
           <a data-track="click" data-track-action="ads reference" data-track-label="link" href="http://adsabs.harvard.edu.proxy.lib.ohio-state.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2011PLSCB...7E2195E">
            ADS
           </a>
           <a data-track="click" data-track-action="mathscinet reference" data-track-label="link" href="http://www-ams-org.proxy.lib.ohio-state.edu/mathscinet-getitem?mr=2859646">
            MathSciNet
           </a>
           <a data-track="click" data-track-action="cas reference" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/cas-redirect/1:CAS:528:DC%2BC3MXhsVCku7rL">
            CAS
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22039361">
            PubMed
           </a>
           <a data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC3197634">
            PubMed Central
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Accelerated%20profile%20HMM%20searches&amp;journal=PLOS%20Comput.%20Biol.&amp;doi=10.1371%2Fjournal.pcbi.1002195&amp;volume=7&amp;publication_year=2011&amp;author=Eddy%2CSR">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR69">
           Eastman, P. et al. OpenMM 7: rapid development of high performance algorithms for molecular dynamics.
           <i>
            PLOS Comput. Biol
           </i>
           .
           <b>
            13
           </b>
           , e1005659 (2017).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1371/journal.pcbi.1005659" href="https://doi-org.proxy.lib.ohio-state.edu/10.1371%2Fjournal.pcbi.1005659">
            Article
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28746339">
            PubMed
           </a>
           <a data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC5549999">
            PubMed Central
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=OpenMM%207%3A%20rapid%20development%20of%20high%20performance%20algorithms%20for%20molecular%20dynamics&amp;journal=PLOS%20Comput.%20Biol.&amp;doi=10.1371%2Fjournal.pcbi.1005659&amp;volume=13&amp;publication_year=2017&amp;author=Eastman%2CP">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR70">
           Ashish, A. M. A. et al. TensorFlow: large-scale machine learning on heterogeneous systems. Preprint at
           <a data-track="click" data-track-action="external reference" data-track-label="https://arxiv-org.proxy.lib.ohio-state.edu/abs/1603.04467" href="https://arxiv-org.proxy.lib.ohio-state.edu/abs/1603.04467">
            https://arxiv-org.proxy.lib.ohio-state.edu/abs/1603.04467
           </a>
           (2015).
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR71">
           Reynolds, M. et al. Open sourcing Sonnet – a new library for constructing neural networks.
           <i>
            DeepMind
           </i>
           <a data-track="click" data-track-action="external reference" data-track-label="https://deepmind.com/blog/open-sourcing-sonnet/" href="https://deepmind.com/blog/open-sourcing-sonnet/">
            https://deepmind.com/blog/open-sourcing-sonnet/
           </a>
           (7 April 2017).
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR72">
           Harris, C. R. et al. Array programming with NumPy.
           <i>
            Nature
           </i>
           <b>
            585
           </b>
           , 357–362 (2020).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1038/s41586-020-2649-2" href="https://doi-org.proxy.lib.ohio-state.edu/10.1038%2Fs41586-020-2649-2">
            Article
           </a>
           <a data-track="click" data-track-action="ads reference" data-track-label="link" href="http://adsabs.harvard.edu.proxy.lib.ohio-state.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2020Natur.585..357H">
            ADS
           </a>
           <a data-track="click" data-track-action="cas reference" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/cas-redirect/1:CAS:528:DC%2BB3cXitlWmsbbN">
            CAS
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32939066">
            PubMed
           </a>
           <a data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC7759461">
            PubMed Central
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Array%20programming%20with%20NumPy&amp;journal=Nature&amp;doi=10.1038%2Fs41586-020-2649-2&amp;volume=585&amp;pages=357-362&amp;publication_year=2020&amp;author=Harris%2CCR">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR73">
           Van Rossum, G. &amp; Drake, F. L.
           <i>
            Python 3 Reference Manual
           </i>
           (CreateSpace, 2009).
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR74">
           Bisong, E. in
           <i>
            Building Machine Learning and Deep Learning Models on Google Cloud Platform: A Comprehensive Guide for Beginners
           </i>
           59–64 (Apress, 2019).
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR75">
           TensorFlow. XLA: Optimizing Compiler for TensorFlow.
           <a data-track="click" data-track-action="external reference" data-track-label="https://www.tensorflow.org/xla" href="https://www.tensorflow.org/xla">
            https://www.tensorflow.org/xla
           </a>
           (2018).
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR76">
           Wu, T., Hou, J., Adhikari, B. &amp; Cheng, J. Analysis of several key factors influencing deep learning-based inter-residue contact prediction.
           <i>
            Bioinformatics
           </i>
           <b>
            36
           </b>
           , 1091–1098 (2020).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1093/bioinformatics/btz679" href="https://doi-org.proxy.lib.ohio-state.edu/10.1093%2Fbioinformatics%2Fbtz679">
            Article
           </a>
           <a data-track="click" data-track-action="cas reference" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/cas-redirect/1:CAS:528:DC%2BB3cXisVOrtbvJ">
            CAS
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31504181">
            PubMed
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Analysis%20of%20several%20key%20factors%20influencing%20deep%20learning-based%20inter-residue%20contact%20prediction&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtz679&amp;volume=36&amp;pages=1091-1098&amp;publication_year=2020&amp;author=Wu%2CT&amp;author=Hou%2CJ&amp;author=Adhikari%2CB&amp;author=Cheng%2CJ">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR77">
           Jiang, W. et al. MrpH, a new class of metal-binding adhesin, requires zinc to mediate biofilm formation.
           <i>
            PLoS Pathog
           </i>
           .
           <b>
            16
           </b>
           , e1008707 (2020).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1371/journal.ppat.1008707" href="https://doi-org.proxy.lib.ohio-state.edu/10.1371%2Fjournal.ppat.1008707">
            Article
           </a>
           <a data-track="click" data-track-action="cas reference" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/cas-redirect/1:CAS:528:DC%2BB3cXhs1Glt7fI">
            CAS
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32780778">
            PubMed
           </a>
           <a data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC7444556">
            PubMed Central
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=MrpH%2C%20a%20new%20class%20of%20metal-binding%20adhesin%2C%20requires%20zinc%20to%20mediate%20biofilm%20formation&amp;journal=PLoS%20Pathog.&amp;doi=10.1371%2Fjournal.ppat.1008707&amp;volume=16&amp;publication_year=2020&amp;author=Jiang%2CW">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR78">
           Dunne, M., Ernst, P., Sobieraj, A., Pluckthun, A. &amp; Loessner, M. J. The M23 peptidase domain of the Staphylococcal phage 2638A endolysin.
           <i>
            PDB
           </i>
           <a data-track="click" data-track-action="external reference" data-track-label="10.2210/pdb6YJ1/pdb" href="https://doi-org.proxy.lib.ohio-state.edu/10.2210/pdb6YJ1/pdb">
            https://doi-org.proxy.lib.ohio-state.edu/10.2210/pdb6YJ1/pdb
           </a>
           (2020).
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR79">
           Drobysheva, A. V. et al. Structure and function of virion RNA polymerase of a crAss-like phage.
           <i>
            Nature
           </i>
           <b>
            589
           </b>
           , 306–309 (2021).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1038/s41586-020-2921-5" href="https://doi-org.proxy.lib.ohio-state.edu/10.1038%2Fs41586-020-2921-5">
            Article
           </a>
           <a data-track="click" data-track-action="ads reference" data-track-label="link" href="http://adsabs.harvard.edu.proxy.lib.ohio-state.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2021Natur.589..306D">
            ADS
           </a>
           <a data-track="click" data-track-action="cas reference" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/cas-redirect/1:CAS:528:DC%2BB3cXitlOgs7jI">
            CAS
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33208949">
            PubMed
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Structure%20and%20function%20of%20virion%20RNA%20polymerase%20of%20a%20crAss-like%20phage&amp;journal=Nature&amp;doi=10.1038%2Fs41586-020-2921-5&amp;volume=589&amp;pages=306-309&amp;publication_year=2021&amp;author=Drobysheva%2CAV">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR80">
           Flaugnatti, N. et al. Structural basis for loading and inhibition of a bacterial T6SS phospholipase effector by the VgrG spike.
           <i>
            EMBO J
           </i>
           .
           <b>
            39
           </b>
           , e104129 (2020).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.15252/embj.2019104129" href="https://doi-org.proxy.lib.ohio-state.edu/10.15252%2Fembj.2019104129">
            Article
           </a>
           <a data-track="click" data-track-action="cas reference" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/cas-redirect/1:CAS:528:DC%2BB3cXotFCnu7s%3D">
            CAS
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32350888">
            PubMed
           </a>
           <a data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC7265238">
            PubMed Central
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Structural%20basis%20for%20loading%20and%20inhibition%20of%20a%20bacterial%20T6SS%20phospholipase%20effector%20by%20the%20VgrG%20spike&amp;journal=EMBO%20J.&amp;doi=10.15252%2Fembj.2019104129&amp;volume=39&amp;publication_year=2020&amp;author=Flaugnatti%2CN">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR81">
           ElGamacy, M. et al. An interface-driven design strategy yields a novel, corrugated protein architecture.
           <i>
            ACS Synth. Biol
           </i>
           .
           <b>
            7
           </b>
           , 2226–2235 (2018).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1021/acssynbio.8b00224" href="https://doi-org.proxy.lib.ohio-state.edu/10.1021%2Facssynbio.8b00224">
            Article
           </a>
           <a data-track="click" data-track-action="cas reference" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/cas-redirect/1:CAS:528:DC%2BC1cXhsFyqtbnP">
            CAS
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30148951">
            PubMed
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=An%20interface-driven%20design%20strategy%20yields%20a%20novel%2C%20corrugated%20protein%20architecture&amp;journal=ACS%20Synth.%20Biol.&amp;doi=10.1021%2Facssynbio.8b00224&amp;volume=7&amp;pages=2226-2235&amp;publication_year=2018&amp;author=ElGamacy%2CM">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR82">
           Lim, C. J. et al. The structure of human CST reveals a decameric assembly bound to telomeric DNA.
           <i>
            Science
           </i>
           <b>
            368
           </b>
           , 1081–1085 (2020).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1126/science.aaz9649" href="https://doi-org.proxy.lib.ohio-state.edu/10.1126%2Fscience.aaz9649">
            Article
           </a>
           <a data-track="click" data-track-action="ads reference" data-track-label="link" href="http://adsabs.harvard.edu.proxy.lib.ohio-state.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2020Sci...368.1081L">
            ADS
           </a>
           <a data-track="click" data-track-action="cas reference" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtFSqt7fI">
            CAS
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32499435">
            PubMed
           </a>
           <a data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC7559292">
            PubMed Central
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20structure%20of%20human%20CST%20reveals%20a%20decameric%20assembly%20bound%20to%20telomeric%20DNA&amp;journal=Science&amp;doi=10.1126%2Fscience.aaz9649&amp;volume=368&amp;pages=1081-1085&amp;publication_year=2020&amp;author=Lim%2CCJ">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR83">
           Debruycker, V. et al. An embedded lipid in the multidrug transporter LmrP suggests a mechanism for polyspecificity.
           <i>
            Nat. Struct. Mol. Biol
           </i>
           .
           <b>
            27
           </b>
           , 829–835 (2020).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1038/s41594-020-0464-y" href="https://doi-org.proxy.lib.ohio-state.edu/10.1038%2Fs41594-020-0464-y">
            Article
           </a>
           <a data-track="click" data-track-action="cas reference" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsVKkt7bE">
            CAS
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32719456">
            PubMed
           </a>
           <a data-track="click" data-track-action="pubmed central reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/pmc/articles/PMC7951658">
            PubMed Central
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=An%20embedded%20lipid%20in%20the%20multidrug%20transporter%20LmrP%20suggests%20a%20mechanism%20for%20polyspecificity&amp;journal=Nat.%20Struct.%20Mol.%20Biol.&amp;doi=10.1038%2Fs41594-020-0464-y&amp;volume=27&amp;pages=829-835&amp;publication_year=2020&amp;author=Debruycker%2CV">
            Google Scholar
           </a>
          </p>
         </li>
         <li class="c-reading-companion__reference-item">
          <p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR84">
           Flower, T. G. et al. Structure of SARS-CoV-2 ORF8, a rapidly evolving immune evasion protein.
           <i>
            Proc. Natl Acad. Sci. USA
           </i>
           <b>
            118
           </b>
           , e2021785118 (2021).
          </p>
          <p class="c-reading-companion__reference-links">
           <a data-track="click" data-track-action="article reference" data-track-label="10.1073/pnas.2021785118" href="https://doi-org.proxy.lib.ohio-state.edu/10.1073%2Fpnas.2021785118">
            Article
           </a>
           <a data-track="click" data-track-action="cas reference" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/articles/cas-redirect/1:CAS:528:DC%2BB3MXhsVCitb4%3D">
            CAS
           </a>
           <a data-track="click" data-track-action="pubmed reference" data-track-label="link" href="http://www-ncbi-nlm-nih-gov.proxy.lib.ohio-state.edu/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33361333">
            PubMed
           </a>
           <a data-track="click" data-track-action="google scholar reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Structure%20of%20SARS-CoV-2%20ORF8%2C%20a%20rapidly%20evolving%20immune%20evasion%20protein&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.2021785118&amp;volume=118&amp;publication_year=2021&amp;author=Flower%2CTG">
            Google Scholar
           </a>
          </p>
         </li>
        </ol>
       </div>
      </div>
     </div>
    </div>
   </aside>
  </div>
  <footer class="composite-layer" itemscope="" itemtype="http://schema.org/Periodical">
   <meta content="Springer Nature" itemprop="publisher"/>
   <div class="u-mt-16 u-mb-16">
    <div class="u-container">
     <div class="u-display-flex u-flex-wrap u-justify-content-space-between">
      <p class="c-meta u-ma-0 u-flex-shrink">
       <span class="c-meta__item">
        Nature (
        <i>
         Nature
        </i>
        )
       </span>
       <span class="c-meta__item">
        <abbr title="International Standard Serial Number">
         ISSN
        </abbr>
        <span itemprop="onlineIssn">
         1476-4687
        </span>
        (online)
       </span>
       <span class="c-meta__item">
        <abbr title="International Standard Serial Number">
         ISSN
        </abbr>
        <span itemprop="printIssn">
         0028-0836
        </span>
        (print)
       </span>
      </p>
     </div>
    </div>
   </div>
   <div class="c-footer">
    <div class="u-hide-print" data-track-component="footer">
     <h2 class="u-visually-hidden">
      nature.com sitemap
     </h2>
     <div class="c-footer__container">
      <div class="c-footer__grid c-footer__group--separator">
       <div class="c-footer__group">
        <h3 class="c-footer__heading u-mt-0">
         About Nature Portfolio
        </h3>
        <ul class="c-footer__list">
         <li class="c-footer__item">
          <a class="c-footer__link" data-track="click" data-track-action="about us" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/npg_/company_info/index.html">
           About us
          </a>
         </li>
         <li class="c-footer__item">
          <a class="c-footer__link" data-track="click" data-track-action="press releases" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/npg_/press_room/press_releases.html">
           Press releases
          </a>
         </li>
         <li class="c-footer__item">
          <a class="c-footer__link" data-track="click" data-track-action="press office" data-track-label="link" href="https://press-nature-com.proxy.lib.ohio-state.edu/">
           Press office
          </a>
         </li>
         <li class="c-footer__item">
          <a class="c-footer__link" data-track="click" data-track-action="contact us" data-track-label="link" href="https://support-nature-com.proxy.lib.ohio-state.edu/support/home">
           Contact us
          </a>
         </li>
        </ul>
       </div>
       <div class="c-footer__group">
        <h3 class="c-footer__heading u-mt-0">
         Discover content
        </h3>
        <ul class="c-footer__list">
         <li class="c-footer__item">
          <a class="c-footer__link" data-track="click" data-track-action="journals a-z" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/siteindex">
           Journals A-Z
          </a>
         </li>
         <li class="c-footer__item">
          <a class="c-footer__link" data-track="click" data-track-action="article by subject" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/subjects/">
           Articles by subject
          </a>
         </li>
         <li class="c-footer__item">
          <a class="c-footer__link" data-track="click" data-track-action="nano" data-track-label="link" href="https://nano-nature-com.proxy.lib.ohio-state.edu/">
           Nano
          </a>
         </li>
         <li class="c-footer__item">
          <a class="c-footer__link" data-track="click" data-track-action="protocol exchange" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/protocolexchange/">
           Protocol Exchange
          </a>
         </li>
         <li class="c-footer__item">
          <a class="c-footer__link" data-track="click" data-track-action="nature index" data-track-label="link" href="https://www.natureindex.com/">
           Nature Index
          </a>
         </li>
        </ul>
       </div>
       <div class="c-footer__group">
        <h3 class="c-footer__heading u-mt-0">
         Publishing policies
        </h3>
        <ul class="c-footer__list">
         <li class="c-footer__item">
          <a class="c-footer__link" data-track="click" data-track-action="Nature portfolio policies" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/authors/editorial_policies/">
           Nature portfolio policies
          </a>
         </li>
         <li class="c-footer__item">
          <a class="c-footer__link" data-track="click" data-track-action="open access" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/nature-research/open-access">
           Open access
          </a>
         </li>
        </ul>
       </div>
       <div class="c-footer__group">
        <h3 class="c-footer__heading u-mt-0">
         Author &amp; Researcher services
        </h3>
        <ul class="c-footer__list">
         <li class="c-footer__item">
          <a class="c-footer__link" data-track="click" data-track-action="reprints and permissions" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/reprints/">
           Reprints &amp; permissions
          </a>
         </li>
         <li class="c-footer__item">
          <a class="c-footer__link" data-track="click" data-track-action="data research service" data-track-label="link" href="https://www-springernature-com.proxy.lib.ohio-state.edu/gp/authors/research-data">
           Research data
          </a>
         </li>
         <li class="c-footer__item">
          <a class="c-footer__link" data-track="click" data-track-action="language editing" data-track-label="link" href="https://authorservices-springernature-com.proxy.lib.ohio-state.edu/language-editing/">
           Language editing
          </a>
         </li>
         <li class="c-footer__item">
          <a class="c-footer__link" data-track="click" data-track-action="scientific editing" data-track-label="link" href="https://authorservices-springernature-com.proxy.lib.ohio-state.edu/scientific-editing/">
           Scientific editing
          </a>
         </li>
         <li class="c-footer__item">
          <a class="c-footer__link" data-track="click" data-track-action="nature masterclasses" data-track-label="link" href="https://masterclasses-nature-com.proxy.lib.ohio-state.edu/">
           Nature Masterclasses
          </a>
         </li>
         <li class="c-footer__item">
          <a class="c-footer__link" data-track="click" data-track-action="live expert trainer-led workshops" data-track-label="link" href="https://masterclasses-nature-com.proxy.lib.ohio-state.edu/live-expert-trainer-led/23649702">
           Live Expert Trainer-led workshops
          </a>
         </li>
         <li class="c-footer__item">
          <a class="c-footer__link" data-track="click" data-track-action="research solutions" data-track-label="link" href="https://solutions-springernature-com.proxy.lib.ohio-state.edu/">
           Research Solutions
          </a>
         </li>
        </ul>
       </div>
       <div class="c-footer__group">
        <h3 class="c-footer__heading u-mt-0">
         Libraries &amp; institutions
        </h3>
        <ul class="c-footer__list">
         <li class="c-footer__item">
          <a class="c-footer__link" data-track="click" data-track-action="librarian service and tools" data-track-label="link" href="https://www-springernature-com.proxy.lib.ohio-state.edu/gp/librarians/tools-services">
           Librarian service &amp; tools
          </a>
         </li>
         <li class="c-footer__item">
          <a class="c-footer__link" data-track="click" data-track-action="librarian portal" data-track-label="link" href="https://www-springernature-com.proxy.lib.ohio-state.edu/gp/librarians/manage-your-account/librarianportal">
           Librarian portal
          </a>
         </li>
         <li class="c-footer__item">
          <a class="c-footer__link" data-track="click" data-track-action="open research" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/openresearch/about-open-access/information-for-institutions/">
           Open research
          </a>
         </li>
         <li class="c-footer__item">
          <a class="c-footer__link" data-track="click" data-track-action="Recommend to library" data-track-label="link" href="https://www-springernature-com.proxy.lib.ohio-state.edu/gp/librarians/recommend-to-your-library">
           Recommend to library
          </a>
         </li>
        </ul>
       </div>
       <div class="c-footer__group">
        <h3 class="c-footer__heading u-mt-0">
         Advertising &amp; partnerships
        </h3>
        <ul class="c-footer__list">
         <li class="c-footer__item">
          <a class="c-footer__link" data-track="click" data-track-action="advertising" data-track-label="link" href="https://partnerships-nature-com.proxy.lib.ohio-state.edu/product/digital-advertising/">
           Advertising
          </a>
         </li>
         <li class="c-footer__item">
          <a class="c-footer__link" data-track="click" data-track-action="partnerships and services" data-track-label="link" href="https://partnerships-nature-com.proxy.lib.ohio-state.edu/">
           Partnerships &amp; Services
          </a>
         </li>
         <li class="c-footer__item">
          <a class="c-footer__link" data-track="click" data-track-action="media kits" data-track-label="link" href="https://partnerships-nature-com.proxy.lib.ohio-state.edu/media-kits/">
           Media kits
          </a>
         </li>
         <li class="c-footer__item">
          <a class="c-footer__link" data-track-action="branded content" data-track-label="link" href="https://partnerships-nature-com.proxy.lib.ohio-state.edu/product/branded-content-native-advertising/">
           Branded
                        content
          </a>
         </li>
        </ul>
       </div>
       <div class="c-footer__group">
        <h3 class="c-footer__heading u-mt-0">
         Career development
        </h3>
        <ul class="c-footer__list">
         <li class="c-footer__item">
          <a class="c-footer__link" data-track="click" data-track-action="nature careers" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/naturecareers">
           Nature Careers
          </a>
         </li>
         <li class="c-footer__item">
          <a class="c-footer__link" data-track="click" data-track-action="nature conferences" data-track-label="link" href="https://conferences-nature-com.proxy.lib.ohio-state.edu">
           Nature
           <span class="u-visually-hidden">
           </span>
           Conferences
          </a>
         </li>
         <li class="c-footer__item">
          <a class="c-footer__link" data-track="click" data-track-action="nature events" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/natureevents/">
           Nature
           <span class="u-visually-hidden">
           </span>
           events
          </a>
         </li>
        </ul>
       </div>
       <div class="c-footer__group">
        <h3 class="c-footer__heading u-mt-0">
         Regional websites
        </h3>
        <ul class="c-footer__list">
         <li class="c-footer__item">
          <a class="c-footer__link" data-track="click" data-track-action="nature africa" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/natafrica">
           Nature Africa
          </a>
         </li>
         <li class="c-footer__item">
          <a class="c-footer__link" data-track="click" data-track-action="nature china" data-track-label="link" href="http://www.naturechina.com">
           Nature China
          </a>
         </li>
         <li class="c-footer__item">
          <a class="c-footer__link" data-track="click" data-track-action="nature india" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/nindia">
           Nature India
          </a>
         </li>
         <li class="c-footer__item">
          <a class="c-footer__link" data-track="click" data-track-action="nature Italy" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/natitaly">
           Nature Italy
          </a>
         </li>
         <li class="c-footer__item">
          <a class="c-footer__link" data-track="click" data-track-action="nature japan" data-track-label="link" href="https://www-natureasia-com.proxy.lib.ohio-state.edu/ja-jp/">
           Nature Japan
          </a>
         </li>
         <li class="c-footer__item">
          <a class="c-footer__link" data-track="click" data-track-action="nature korea" data-track-label="link" href="https://www-natureasia-com.proxy.lib.ohio-state.edu/ko-kr/">
           Nature Korea
          </a>
         </li>
         <li class="c-footer__item">
          <a class="c-footer__link" data-track="click" data-track-action="nature middle east" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/nmiddleeast/">
           Nature Middle East
          </a>
         </li>
        </ul>
       </div>
      </div>
     </div>
     <div class="c-footer__container">
      <ul class="c-footer__links">
       <li class="c-footer__item">
        <a class="c-footer__link" data-track="click" data-track-action="privacy policy" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/info/privacy">
         Privacy
                Policy
        </a>
       </li>
       <li class="c-footer__item">
        <a class="c-footer__link" data-track="click" data-track-action="use of cookies" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/info/cookies">
         Use
                of cookies
        </a>
       </li>
       <li class="c-footer__item">
        <button class="optanon-toggle-display c-footer__link" data-cc-action="preferences" data-track="click" data-track-action="manage cookies" data-track-label="link" onclick="javascript:;">
         Your privacy choices/Manage cookies
        </button>
       </li>
       <li class="c-footer__item">
        <a class="c-footer__link" data-track="click" data-track-action="legal notice" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/info/legal-notice">
         Legal
                notice
        </a>
       </li>
       <li class="c-footer__item">
        <a class="c-footer__link" data-track="click" data-track-action="accessibility statement" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/info/accessibility-statement">
         Accessibility
                statement
        </a>
       </li>
       <li class="c-footer__item">
        <a class="c-footer__link" data-track="click" data-track-action="terms and conditions" data-track-label="link" href="https://www-nature-com.proxy.lib.ohio-state.edu/info/terms-and-conditions">
         Terms &amp; Conditions
        </a>
       </li>
       <li class="c-footer__item">
        <a class="c-footer__link" data-track="click" data-track-action="california privacy statement" data-track-label="link" href="https://www-springernature-com.proxy.lib.ohio-state.edu/ccpa">
         Your US state privacy rights
        </a>
       </li>
      </ul>
     </div>
    </div>
    <div class="c-footer__container">
     <a class="c-footer__link" href="https://www-springernature-com.proxy.lib.ohio-state.edu/">
      <img alt="Springer Nature" height="20" loading="lazy" src="/static/images/logos/sn-logo-white-ea63208b81.svg" width="200"/>
     </a>
     <p class="c-footer__legal" data-test="copyright">
      © 2023 Springer Nature Limited
     </p>
    </div>
   </div>
   <div aria-hidden="true" class="u-visually-hidden">
    <!--?xml version="1.0" encoding="UTF-8"?-->
    <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
     <defs>
      <path d="M0 .74h56.72v55.24H0z" id="a">
      </path>
     </defs>
     <symbol id="icon-access" viewbox="0 0 18 18">
      <path d="m14 8c.5522847 0 1 .44771525 1 1v7h2.5c.2761424 0 .5.2238576.5.5v1.5h-18v-1.5c0-.2761424.22385763-.5.5-.5h2.5v-7c0-.55228475.44771525-1 1-1s1 .44771525 1 1v6.9996556h8v-6.9996556c0-.55228475.4477153-1 1-1zm-8 0 2 1v5l-2 1zm6 0v7l-2-1v-5zm-2.42653766-7.59857636 7.03554716 4.92488299c.4162533.29137735.5174853.86502537.226108 1.28127873-.1721584.24594054-.4534847.39241464-.7536934.39241464h-14.16284822c-.50810197 0-.92-.41189803-.92-.92 0-.30020869.1464741-.58153499.39241464-.75369337l7.03554714-4.92488299c.34432015-.2410241.80260453-.2410241 1.14692468 0zm-.57346234 2.03988748-3.65526982 2.55868888h7.31053962z" fill-rule="evenodd">
      </path>
     </symbol>
     <symbol id="icon-account" viewbox="0 0 18 18">
      <path d="m10.2379028 16.9048051c1.3083556-.2032362 2.5118471-.7235183 3.5294683-1.4798399-.8731327-2.5141501-2.0638925-3.935978-3.7673711-4.3188248v-1.27684611c1.1651924-.41183641 2-1.52307546 2-2.82929429 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.30621883.83480763 2.41745788 2 2.82929429v1.27684611c-1.70347856.3828468-2.89423845 1.8046747-3.76737114 4.3188248 1.01762123.7563216 2.22111275 1.2766037 3.52946833 1.4798399.40563808.0629726.81921174.0951949 1.23790281.0951949s.83226473-.0322223 1.2379028-.0951949zm4.3421782-2.1721994c1.4927655-1.4532925 2.419919-3.484675 2.419919-5.7326057 0-4.418278-3.581722-8-8-8s-8 3.581722-8 8c0 2.2479307.92715352 4.2793132 2.41991895 5.7326057.75688473-2.0164459 1.83949951-3.6071894 3.48926591-4.3218837-1.14534283-.70360829-1.90918486-1.96796271-1.90918486-3.410722 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.44275929-.763842 2.70711371-1.9091849 3.410722 1.6497664.7146943 2.7323812 2.3054378 3.4892659 4.3218837zm-5.580081 3.2673943c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd">
      </path>
     </symbol>
     <symbol id="icon-alert" viewbox="0 0 18 18">
      <path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill-rule="evenodd">
      </path>
     </symbol>
     <symbol id="icon-arrow-broad" viewbox="0 0 16 16">
      <path d="m6.10307866 2.97190702v7.69043288l2.44965196-2.44676915c.38776071-.38730439 1.0088052-.39493524 1.38498697-.01919617.38609051.38563612.38643641 1.01053024-.00013864 1.39665039l-4.12239817 4.11754683c-.38616704.3857126-1.01187344.3861062-1.39846576-.0000311l-4.12258206-4.11773056c-.38618426-.38572979-.39254614-1.00476697-.01636437-1.38050605.38609047-.38563611 1.01018509-.38751562 1.4012233.00306241l2.44985644 2.4469734v-8.67638639c0-.54139983.43698413-.98042709.98493125-.98159081l7.89910522-.0043627c.5451687 0 .9871152.44142642.9871152.98595351s-.4419465.98595351-.9871152.98595351z" fill-rule="evenodd" transform="matrix(-1 0 0 -1 14 15)">
      </path>
     </symbol>
     <symbol id="icon-arrow-down" viewbox="0 0 16 16">
      <path d="m3.28337502 11.5302405 4.03074001 4.176208c.37758093.3912076.98937525.3916069 1.367372-.0000316l4.03091977-4.1763942c.3775978-.3912252.3838182-1.0190815.0160006-1.4001736-.3775061-.39113013-.9877245-.39303641-1.3700683.003106l-2.39538585 2.4818345v-11.6147896l-.00649339-.11662112c-.055753-.49733869-.46370161-.88337888-.95867408-.88337888-.49497246 0-.90292107.38604019-.95867408.88337888l-.00649338.11662112v11.6147896l-2.39518594-2.4816273c-.37913917-.39282218-.98637524-.40056175-1.35419292-.0194697-.37750607.3911302-.37784433 1.0249269.00013556 1.4165479z" fill-rule="evenodd">
      </path>
     </symbol>
     <symbol id="icon-arrow-left" viewbox="0 0 16 16">
      <path d="m4.46975946 3.28337502-4.17620792 4.03074001c-.39120768.37758093-.39160691.98937525.0000316 1.367372l4.1763942 4.03091977c.39122514.3775978 1.01908149.3838182 1.40017357.0160006.39113012-.3775061.3930364-.9877245-.00310603-1.3700683l-2.48183446-2.39538585h11.61478958l.1166211-.00649339c.4973387-.055753.8833789-.46370161.8833789-.95867408 0-.49497246-.3860402-.90292107-.8833789-.95867408l-.1166211-.00649338h-11.61478958l2.4816273-2.39518594c.39282216-.37913917.40056173-.98637524.01946965-1.35419292-.39113012-.37750607-1.02492687-.37784433-1.41654791.00013556z" fill-rule="evenodd">
      </path>
     </symbol>
     <symbol id="icon-arrow-right" viewbox="0 0 16 16">
      <path d="m11.5302405 12.716625 4.176208-4.03074003c.3912076-.37758093.3916069-.98937525-.0000316-1.367372l-4.1763942-4.03091981c-.3912252-.37759778-1.0190815-.38381821-1.4001736-.01600053-.39113013.37750607-.39303641.98772445.003106 1.37006824l2.4818345 2.39538588h-11.6147896l-.11662112.00649339c-.49733869.055753-.88337888.46370161-.88337888.95867408 0 .49497246.38604019.90292107.88337888.95867408l.11662112.00649338h11.6147896l-2.4816273 2.39518592c-.39282218.3791392-.40056175.9863753-.0194697 1.3541929.3911302.3775061 1.0249269.3778444 1.4165479-.0001355z" fill-rule="evenodd">
      </path>
     </symbol>
     <symbol id="icon-arrow-sub" viewbox="0 0 16 16">
      <path d="m7.89692134 4.97190702v7.69043288l-2.44965196-2.4467692c-.38776071-.38730434-1.0088052-.39493519-1.38498697-.0191961-.38609047.3856361-.38643643 1.0105302.00013864 1.3966504l4.12239817 4.1175468c.38616704.3857126 1.01187344.3861062 1.39846576-.0000311l4.12258202-4.1177306c.3861843-.3857298.3925462-1.0047669.0163644-1.380506-.3860905-.38563612-1.0101851-.38751563-1.4012233.0030624l-2.44985643 2.4469734v-8.67638639c0-.54139983-.43698413-.98042709-.98493125-.98159081l-7.89910525-.0043627c-.54516866 0-.98711517.44142642-.98711517.98595351s.44194651.98595351.98711517.98595351z" fill-rule="evenodd">
      </path>
     </symbol>
     <symbol id="icon-arrow-up" viewbox="0 0 16 16">
      <path d="m12.716625 4.46975946-4.03074003-4.17620792c-.37758093-.39120768-.98937525-.39160691-1.367372.0000316l-4.03091981 4.1763942c-.37759778.39122514-.38381821 1.01908149-.01600053 1.40017357.37750607.39113012.98772445.3930364 1.37006824-.00310603l2.39538588-2.48183446v11.61478958l.00649339.1166211c.055753.4973387.46370161.8833789.95867408.8833789.49497246 0 .90292107-.3860402.95867408-.8833789l.00649338-.1166211v-11.61478958l2.39518592 2.4816273c.3791392.39282216.9863753.40056173 1.3541929.01946965.3775061-.39113012.3778444-1.02492687-.0001355-1.41654791z" fill-rule="evenodd">
      </path>
     </symbol>
     <symbol id="icon-article" viewbox="0 0 18 18">
      <path d="m13 15v-12.9906311c0-.0073595-.0019884-.0093689.0014977-.0093689l-11.00158888.00087166v13.00506804c0 .5482678.44615281.9940603.99415146.9940603h10.27350412c-.1701701-.2941734-.2675644-.6357129-.2675644-1zm-12 .0059397v-13.00506804c0-.5562408.44704472-1.00087166.99850233-1.00087166h11.00299537c.5510129 0 .9985023.45190985.9985023 1.0093689v2.9906311h3v9.9914698c0 1.1065798-.8927712 2.0085302-1.9940603 2.0085302h-12.01187942c-1.09954652 0-1.99406028-.8927712-1.99406028-1.9940603zm13-9.0059397v9c0 .5522847.4477153 1 1 1s1-.4477153 1-1v-9zm-10-2h7v4h-7zm1 1v2h5v-2zm-1 4h7v1h-7zm0 2h7v1h-7zm0 2h7v1h-7z" fill-rule="evenodd">
      </path>
     </symbol>
     <symbol id="icon-audio" viewbox="0 0 18 18">
      <path d="m13.0957477 13.5588459c-.195279.1937043-.5119137.193729-.7072234.0000551-.1953098-.193674-.1953346-.5077061-.0000556-.7014104 1.0251004-1.0168342 1.6108711-2.3905226 1.6108711-3.85745208 0-1.46604976-.5850634-2.83898246-1.6090736-3.85566829-.1951894-.19379323-.1950192-.50782531.0003802-.70141028.1953993-.19358497.512034-.19341614.7072234.00037709 1.2094886 1.20083761 1.901635 2.8250555 1.901635 4.55670148 0 1.73268608-.6929822 3.35779608-1.9037571 4.55880738zm2.1233994 2.1025159c-.195234.193749-.5118687.1938462-.7072235.0002171-.1953548-.1936292-.1954528-.5076613-.0002189-.7014104 1.5832215-1.5711805 2.4881302-3.6939808 2.4881302-5.96012998 0-2.26581266-.9046382-4.3883241-2.487443-5.95944795-.1952117-.19377107-.1950777-.50780316.0002993-.70141031s.5120117-.19347426.7072234.00029682c1.7683321 1.75528196 2.7800854 4.12911258 2.7800854 6.66056144 0 2.53182498-1.0120556 4.90597838-2.7808529 6.66132328zm-14.21898205-3.6854911c-.5523759 0-1.00016505-.4441085-1.00016505-.991944v-3.96777631c0-.54783558.44778915-.99194407 1.00016505-.99194407h2.0003301l5.41965617-3.8393633c.44948677-.31842296 1.07413994-.21516983 1.39520191.23062232.12116339.16823446.18629727.36981184.18629727.57655577v12.01603479c0 .5478356-.44778914.9919441-1.00016505.9919441-.20845738 0-.41170538-.0645985-.58133413-.184766l-5.41965617-3.8393633zm0-.991944h2.32084805l5.68047235 4.0241292v-12.01603479l-5.68047235 4.02412928h-2.32084805z" fill-rule="evenodd">
      </path>
     </symbol>
     <symbol id="icon-block" viewbox="0 0 24 24">
      <path d="m0 0h24v24h-24z" fill-rule="evenodd">
      </path>
     </symbol>
     <symbol id="icon-book" viewbox="0 0 18 18">
      <path d="m4 13v-11h1v11h11v-11h-13c-.55228475 0-1 .44771525-1 1v10.2675644c.29417337-.1701701.63571286-.2675644 1-.2675644zm12 1h-13c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1h13zm0 3h-13c-1.1045695 0-2-.8954305-2-2v-12c0-1.1045695.8954305-2 2-2h13c.5522847 0 1 .44771525 1 1v14c0 .5522847-.4477153 1-1 1zm-8.5-13h6c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1 2h4c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-4c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd">
      </path>
     </symbol>
     <symbol id="icon-broad" viewbox="0 0 24 24">
      <path d="m9.18274226 7.81v7.7999954l2.48162734-2.4816273c.3928221-.3928221 1.0219731-.4005617 1.4030652-.0194696.3911301.3911301.3914806 1.0249268-.0001404 1.4165479l-4.17620796 4.1762079c-.39120769.3912077-1.02508144.3916069-1.41671995-.0000316l-4.1763942-4.1763942c-.39122514-.3912251-.39767006-1.0190815-.01657798-1.4001736.39113012-.3911301 1.02337106-.3930364 1.41951349.0031061l2.48183446 2.4818344v-8.7999954c0-.54911294.4426881-.99439484.99778758-.99557515l8.00221246-.00442485c.5522847 0 1 .44771525 1 1s-.4477153 1-1 1z" fill-rule="evenodd" transform="matrix(-1 0 0 -1 20.182742 24.805206)">
      </path>
     </symbol>
     <symbol id="icon-calendar" viewbox="0 0 18 18">
      <path d="m12.5 0c.2761424 0 .5.21505737.5.49047852v.50952148h2c1.1072288 0 2 .89451376 2 2v12c0 1.1072288-.8945138 2-2 2h-12c-1.1072288 0-2-.8945138-2-2v-12c0-1.1072288.89451376-2 2-2h1v1h-1c-.55393837 0-1 .44579254-1 1v3h14v-3c0-.55393837-.4457925-1-1-1h-2v1.50952148c0 .27088381-.2319336.49047852-.5.49047852-.2761424 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.2319336-.49047852.5-.49047852zm3.5 7h-14v8c0 .5539384.44579254 1 1 1h12c.5539384 0 1-.4457925 1-1zm-11 6v1h-1v-1zm3 0v1h-1v-1zm3 0v1h-1v-1zm-6-2v1h-1v-1zm3 0v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-3-2v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-5.5-9c.27614237 0 .5.21505737.5.49047852v.50952148h5v1h-5v1.50952148c0 .27088381-.23193359.49047852-.5.49047852-.27614237 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.23193359-.49047852.5-.49047852z" fill-rule="evenodd">
      </path>
     </symbol>
     <symbol id="icon-cart" viewbox="0 0 18 18">
      <path d="m5 14c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm10 0c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm-10 1c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1 1-.4477153 1-1-.44771525-1-1-1zm10 0c-.5522847 0-1 .4477153-1 1s.4477153 1 1 1 1-.4477153 1-1-.4477153-1-1-1zm-12.82032249-15c.47691417 0 .88746157.33678127.98070211.80449199l.23823144 1.19501025 13.36277974.00045554c.5522847.00001882.9999659.44774934.9999659 1.00004222 0 .07084994-.0075361.14150708-.022474.2107727l-1.2908094 5.98534344c-.1007861.46742419-.5432548.80388386-1.0571651.80388386h-10.24805106c-.59173366 0-1.07142857.4477153-1.07142857 1 0 .5128358.41361449.9355072.94647737.9932723l.1249512.0067277h10.35933776c.2749512 0 .4979349.2228539.4979349.4978051 0 .2749417-.2227336.4978951-.4976753.4980063l-10.35959736.0041886c-1.18346732 0-2.14285714-.8954305-2.14285714-2 0-.6625717.34520317-1.24989198.87690425-1.61383592l-1.63768102-8.19004794c-.01312273-.06561364-.01950005-.131011-.0196107-.19547395l-1.71961253-.00064219c-.27614237 0-.5-.22385762-.5-.5 0-.27614237.22385763-.5.5-.5zm14.53193359 2.99950224h-13.11300004l1.20580469 6.02530174c.11024034-.0163252.22327998-.02480398.33844139-.02480398h10.27064786z">
      </path>
     </symbol>
     <symbol id="icon-chevron-less" viewbox="0 0 10 10">
      <path d="m5.58578644 4-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 -1 -1 0 9 9)">
      </path>
     </symbol>
     <symbol id="icon-chevron-more" viewbox="0 0 10 10">
      <path d="m5.58578644 6-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4.00000002c-.39052429.3905243-1.02368927.3905243-1.41421356 0s-.39052429-1.02368929 0-1.41421358z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 1)">
      </path>
     </symbol>
     <symbol id="icon-chevron-right" viewbox="0 0 10 10">
      <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)">
      </path>
     </symbol>
     <symbol id="icon-circle-fill" viewbox="0 0 16 16">
      <path d="m8 14c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z" fill-rule="evenodd">
      </path>
     </symbol>
     <symbol id="icon-circle" viewbox="0 0 16 16">
      <path d="m8 12c2.209139 0 4-1.790861 4-4s-1.790861-4-4-4-4 1.790861-4 4 1.790861 4 4 4zm0 2c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z" fill-rule="evenodd">
      </path>
     </symbol>
     <symbol id="icon-citation" viewbox="0 0 18 18">
      <path d="m8.63593473 5.99995183c2.20913897 0 3.99999997 1.79084375 3.99999997 3.99996146 0 1.40730761-.7267788 2.64486871-1.8254829 3.35783281 1.6240224.6764218 2.8754442 2.0093871 3.4610603 3.6412466l-1.0763845.000006c-.5310008-1.2078237-1.5108121-2.1940153-2.7691712-2.7181346l-.79002167-.329052v-1.023992l.63016577-.4089232c.8482885-.5504661 1.3698342-1.4895187 1.3698342-2.51898361 0-1.65683828-1.3431457-2.99996146-2.99999997-2.99996146-1.65685425 0-3 1.34312318-3 2.99996146 0 1.02946491.52154569 1.96851751 1.36983419 2.51898361l.63016581.4089232v1.023992l-.79002171.329052c-1.25835905.5241193-2.23817037 1.5103109-2.76917113 2.7181346l-1.07638453-.000006c.58561612-1.6318595 1.8370379-2.9648248 3.46106024-3.6412466-1.09870405-.7129641-1.82548287-1.9505252-1.82548287-3.35783281 0-2.20911771 1.790861-3.99996146 4-3.99996146zm7.36897597-4.99995183c1.1018574 0 1.9950893.89353404 1.9950893 2.00274083v5.994422c0 1.10608317-.8926228 2.00274087-1.9950893 2.00274087l-3.0049107-.0009037v-1l3.0049107.00091329c.5490631 0 .9950893-.44783123.9950893-1.00275046v-5.994422c0-.55646537-.4450595-1.00275046-.9950893-1.00275046h-14.00982141c-.54906309 0-.99508929.44783123-.99508929 1.00275046v5.9971821c0 .66666024.33333333.99999036 1 .99999036l2-.00091329v1l-2 .0009037c-1 0-2-.99999041-2-1.99998077v-5.9971821c0-1.10608322.8926228-2.00274083 1.99508929-2.00274083zm-8.5049107 2.9999711c.27614237 0 .5.22385547.5.5 0 .2761349-.22385763.5-.5.5h-4c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm3 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-1c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm4 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238651-.5-.5 0-.27614453.2238576-.5.5-.5z" fill-rule="evenodd">
      </path>
     </symbol>
     <symbol id="icon-close" viewbox="0 0 16 16">
      <path d="m2.29679575 12.2772478c-.39658757.3965876-.39438847 1.0328109-.00062148 1.4265779.39651227.3965123 1.03246768.3934888 1.42657791-.0006214l4.27724782-4.27724787 4.2772478 4.27724787c.3965876.3965875 1.0328109.3943884 1.4265779.0006214.3965123-.3965122.3934888-1.0324677-.0006214-1.4265779l-4.27724787-4.2772478 4.27724787-4.27724782c.3965875-.39658757.3943884-1.03281091.0006214-1.42657791-.3965122-.39651226-1.0324677-.39348875-1.4265779.00062148l-4.2772478 4.27724782-4.27724782-4.27724782c-.39658757-.39658757-1.03281091-.39438847-1.42657791-.00062148-.39651226.39651227-.39348875 1.03246768.00062148 1.42657791l4.27724782 4.27724782z" fill-rule="evenodd">
      </path>
     </symbol>
     <symbol id="icon-collections" viewbox="0 0 18 18">
      <path d="m15 4c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2h1c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-1v-1zm-4-3c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2v-9c0-1.1045695.8954305-2 2-2zm0 1h-8c-.51283584 0-.93550716.38604019-.99327227.88337887l-.00672773.11662113v9c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227zm-1.5 7c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd">
      </path>
     </symbol>
     <symbol id="icon-compare" viewbox="0 0 18 18">
      <path d="m12 3c3.3137085 0 6 2.6862915 6 6s-2.6862915 6-6 6c-1.0928452 0-2.11744941-.2921742-2.99996061-.8026704-.88181407.5102749-1.90678042.8026704-3.00003939.8026704-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6c1.09325897 0 2.11822532.29239547 3.00096303.80325037.88158756-.51107621 1.90619177-.80325037 2.99903697-.80325037zm-6 1c-2.76142375 0-5 2.23857625-5 5 0 2.7614237 2.23857625 5 5 5 .74397391 0 1.44999672-.162488 2.08451611-.4539116-1.27652344-1.1000812-2.08451611-2.7287264-2.08451611-4.5460884s.80799267-3.44600721 2.08434391-4.5463015c-.63434719-.29121054-1.34037-.4536985-2.08434391-.4536985zm6 0c-.7439739 0-1.4499967.16248796-2.08451611.45391156 1.27652341 1.10008123 2.08451611 2.72872644 2.08451611 4.54608844s-.8079927 3.4460072-2.08434391 4.5463015c.63434721.2912105 1.34037001.4536985 2.08434391.4536985 2.7614237 0 5-2.2385763 5-5 0-2.76142375-2.2385763-5-5-5zm-1.4162763 7.0005324h-3.16744736c.15614659.3572676.35283837.6927622.58425872 1.0006671h1.99892988c.23142036-.3079049.42811216-.6433995.58425876-1.0006671zm.4162763-2.0005324h-4c0 .34288501.0345146.67770871.10025909 1.0011864h3.79948181c.0657445-.32347769.1002591-.65830139.1002591-1.0011864zm-.4158423-1.99953894h-3.16831543c-.13859957.31730812-.24521946.651783-.31578599.99935097h3.79988742c-.0705665-.34756797-.1771864-.68204285-.315786-.99935097zm-1.58295822-1.999926-.08316107.06199199c-.34550042.27081213-.65446126.58611297-.91825862.93727862h2.00044041c-.28418626-.37830727-.6207872-.71499149-.99902072-.99927061z" fill-rule="evenodd">
      </path>
     </symbol>
     <symbol id="icon-download-file" viewbox="0 0 18 18">
      <path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.5046024 4c.27614237 0 .5.21637201.5.49209595v6.14827645l1.7462789-1.77990922c.1933927-.1971171.5125222-.19455839.7001689-.0069117.1932998.19329992.1910058.50899492-.0027774.70277812l-2.59089271 2.5908927c-.19483374.1948337-.51177825.1937771-.70556873-.0000133l-2.59099079-2.5909908c-.19484111-.1948411-.19043735-.5151448-.00279066-.70279146.19329987-.19329987.50465175-.19237083.70018565.00692852l1.74638684 1.78001764v-6.14827695c0-.27177709.23193359-.49209595.5-.49209595z" fill-rule="evenodd">
      </path>
     </symbol>
     <symbol id="icon-download" viewbox="0 0 16 16">
      <path d="m12.9975267 12.999368c.5467123 0 1.0024733.4478567 1.0024733 1.000316 0 .5563109-.4488226 1.000316-1.0024733 1.000316h-9.99505341c-.54671233 0-1.00247329-.4478567-1.00247329-1.000316 0-.5563109.44882258-1.000316 1.00247329-1.000316zm-4.9975267-11.999368c.55228475 0 1 .44497754 1 .99589209v6.80214418l2.4816273-2.48241149c.3928222-.39294628 1.0219732-.4006883 1.4030652-.01947579.3911302.39125371.3914806 1.02525073-.0001404 1.41699553l-4.17620792 4.17752758c-.39120769.3913313-1.02508144.3917306-1.41671995-.0000316l-4.17639421-4.17771394c-.39122513-.39134876-.39767006-1.01940351-.01657797-1.40061601.39113012-.39125372 1.02337105-.3931606 1.41951349.00310701l2.48183446 2.48261871v-6.80214418c0-.55001601.44386482-.99589209 1-.99589209z" fill-rule="evenodd">
      </path>
     </symbol>
     <symbol id="icon-editors" viewbox="0 0 18 18">
      <path d="m8.72592184 2.54588137c-.48811714-.34391207-1.08343326-.54588137-1.72592184-.54588137-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400182l-.79002171.32905522c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274v.9009805h-1v-.9009805c0-2.5479714 1.54557359-4.79153984 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4 1.09079823 0 2.07961816.43662103 2.80122451 1.1446278-.37707584.09278571-.7373238.22835063-1.07530267.40125357zm-2.72592184 14.45411863h-1v-.9009805c0-2.5479714 1.54557359-4.7915398 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.40732121-.7267788 2.64489414-1.8254829 3.3578652 2.2799093.9496145 3.8254829 3.1931829 3.8254829 5.7411543v.9009805h-1v-.9009805c0-2.1155483-1.2760206-4.0125067-3.2099783-4.8180274l-.7900217-.3290552v-1.02400184l.6301658-.40892721c.8482885-.55047139 1.3698342-1.489533 1.3698342-2.51900785 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400184l-.79002171.3290552c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274z" fill-rule="evenodd">
      </path>
     </symbol>
     <symbol id="icon-email" viewbox="0 0 18 18">
      <path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-.0049107 2.55749512v1.44250488l-7 4-7-4v-1.44250488l7 4z" fill-rule="evenodd">
      </path>
     </symbol>
     <symbol id="icon-error" viewbox="0 0 18 18">
      <path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm2.8630343 4.71100931-2.8630343 2.86303426-2.86303426-2.86303426c-.39658757-.39658757-1.03281091-.39438847-1.4265779-.00062147-.39651227.39651226-.39348876 1.03246767.00062147 1.4265779l2.86303426 2.86303426-2.86303426 2.8630343c-.39658757.3965875-.39438847 1.0328109-.00062147 1.4265779.39651226.3965122 1.03246767.3934887 1.4265779-.0006215l2.86303426-2.8630343 2.8630343 2.8630343c.3965875.3965876 1.0328109.3943885 1.4265779.0006215.3965122-.3965123.3934887-1.0324677-.0006215-1.4265779l-2.8630343-2.8630343 2.8630343-2.86303426c.3965876-.39658757.3943885-1.03281091.0006215-1.4265779-.3965123-.39651227-1.0324677-.39348876-1.4265779.00062147z" fill-rule="evenodd">
      </path>
     </symbol>
     <symbol id="icon-ethics" viewbox="0 0 18 18">
      <path d="m6.76384967 1.41421356.83301651-.8330165c.77492941-.77492941 2.03133823-.77492941 2.80626762 0l.8330165.8330165c.3750728.37507276.8837806.58578644 1.4142136.58578644h1.3496361c1.1045695 0 2 .8954305 2 2v1.34963611c0 .53043298.2107137 1.03914081.5857864 1.41421356l.8330165.83301651c.7749295.77492941.7749295 2.03133823 0 2.80626762l-.8330165.8330165c-.3750727.3750728-.5857864.8837806-.5857864 1.4142136v1.3496361c0 1.1045695-.8954305 2-2 2h-1.3496361c-.530433 0-1.0391408.2107137-1.4142136.5857864l-.8330165.8330165c-.77492939.7749295-2.03133821.7749295-2.80626762 0l-.83301651-.8330165c-.37507275-.3750727-.88378058-.5857864-1.41421356-.5857864h-1.34963611c-1.1045695 0-2-.8954305-2-2v-1.3496361c0-.530433-.21071368-1.0391408-.58578644-1.4142136l-.8330165-.8330165c-.77492941-.77492939-.77492941-2.03133821 0-2.80626762l.8330165-.83301651c.37507276-.37507275.58578644-.88378058.58578644-1.41421356v-1.34963611c0-1.1045695.8954305-2 2-2h1.34963611c.53043298 0 1.03914081-.21071368 1.41421356-.58578644zm-1.41421356 1.58578644h-1.34963611c-.55228475 0-1 .44771525-1 1v1.34963611c0 .79564947-.31607052 1.55871121-.87867966 2.12132034l-.8330165.83301651c-.38440512.38440512-.38440512 1.00764896 0 1.39205408l.8330165.83301646c.56260914.5626092.87867966 1.3256709.87867966 2.1213204v1.3496361c0 .5522847.44771525 1 1 1h1.34963611c.79564947 0 1.55871121.3160705 2.12132034.8786797l.83301651.8330165c.38440512.3844051 1.00764896.3844051 1.39205408 0l.83301646-.8330165c.5626092-.5626092 1.3256709-.8786797 2.1213204-.8786797h1.3496361c.5522847 0 1-.4477153 1-1v-1.3496361c0-.7956495.3160705-1.5587112.8786797-2.1213204l.8330165-.83301646c.3844051-.38440512.3844051-1.00764896 0-1.39205408l-.8330165-.83301651c-.5626092-.56260913-.8786797-1.32567087-.8786797-2.12132034v-1.34963611c0-.55228475-.4477153-1-1-1h-1.3496361c-.7956495 0-1.5587112-.31607052-2.1213204-.87867966l-.83301646-.8330165c-.38440512-.38440512-1.00764896-.38440512-1.39205408 0l-.83301651.8330165c-.56260913.56260914-1.32567087.87867966-2.12132034.87867966zm3.58698944 11.4960218c-.02081224.002155-.04199226.0030286-.06345763.002542-.98766446-.0223875-1.93408568-.3063547-2.75885125-.8155622-.23496767-.1450683-.30784554-.4531483-.16277726-.688116.14506827-.2349677.45314827-.3078455.68811595-.1627773.67447084.4164161 1.44758575.6483839 2.25617384.6667123.01759529.0003988.03495764.0017019.05204365.0038639.01713363-.0017748.03452416-.0026845.05212715-.0026845 2.4852814 0 4.5-2.0147186 4.5-4.5 0-1.04888973-.3593547-2.04134635-1.0074477-2.83787157-.1742817-.21419731-.1419238-.5291218.0722736-.70340353.2141973-.17428173.5291218-.14192375.7034035.07227357.7919032.97327203 1.2317706 2.18808682 1.2317706 3.46900153 0 3.0375661-2.4624339 5.5-5.5 5.5-.02146768 0-.04261937-.0013529-.06337445-.0039782zm1.57975095-10.78419583c.2654788.07599731.419084.35281842.3430867.61829728-.0759973.26547885-.3528185.419084-.6182973.3430867-.37560116-.10752146-.76586237-.16587951-1.15568824-.17249193-2.5587807-.00064534-4.58547766 2.00216524-4.58547766 4.49928198 0 .62691557.12797645 1.23496.37274865 1.7964426.11035133.2531347-.0053975.5477984-.25853224.6581497-.25313473.1103514-.54779841-.0053975-.65814974-.2585322-.29947131-.6869568-.45606667-1.43097603-.45606667-2.1960601 0-3.05211432 2.47714695-5.50006595 5.59399617-5.49921198.48576182.00815502.96289603.0795037 1.42238033.21103795zm-1.9766658 6.41091303 2.69835-2.94655317c.1788432-.21040373.4943901-.23598862.7047939-.05714545.2104037.17884318.2359886.49439014.0571454.70479387l-3.01637681 3.34277395c-.18039088.1999106-.48669547.2210637-.69285412.0478478l-1.93095347-1.62240047c-.21213845-.17678204-.24080048-.49206439-.06401844-.70420284.17678204-.21213844.49206439-.24080048.70420284-.06401844z" fill-rule="evenodd">
      </path>
     </symbol>
     <symbol id="icon-expand">
      <path d="M7.498 11.918a.997.997 0 0 0-.003-1.411.995.995 0 0 0-1.412-.003l-4.102 4.102v-3.51A1 1 0 0 0 .98 10.09.992.992 0 0 0 0 11.092V17c0 .554.448 1.002 1.002 1.002h5.907c.554 0 1.002-.45 1.002-1.003 0-.539-.45-.978-1.006-.978h-3.51zm3.005-5.835a.997.997 0 0 0 .003 1.412.995.995 0 0 0 1.411.003l4.103-4.103v3.51a1 1 0 0 0 1.001 1.006A.992.992 0 0 0 18 6.91V1.002A1 1 0 0 0 17 0h-5.907a1.003 1.003 0 0 0-1.002 1.003c0 .539.45.978 1.006.978h3.51z" fill-rule="evenodd">
      </path>
     </symbol>
     <symbol id="icon-explore" viewbox="0 0 18 18">
      <path d="m9 17c4.418278 0 8-3.581722 8-8s-3.581722-8-8-8-8 3.581722-8 8 3.581722 8 8 8zm0 1c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9zm0-2.5c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5c2.969509 0 5.400504-2.3575119 5.497023-5.31714844.0090007-.27599565.2400359-.49243782.5160315-.48343711.2759957.0090007.4924378.2400359.4834371.51603155-.114093 3.4985237-2.9869632 6.284554-6.4964916 6.284554zm-.29090657-12.99359748c.27587424-.01216621.50937715.20161139.52154336.47748563.01216621.27587423-.20161139.50937715-.47748563.52154336-2.93195733.12930094-5.25315116 2.54886451-5.25315116 5.49456849 0 .27614237-.22385763.5-.5.5s-.5-.22385763-.5-.5c0-3.48142406 2.74307146-6.34074398 6.20909343-6.49359748zm1.13784138 8.04763908-1.2004882-1.20048821c-.19526215-.19526215-.19526215-.51184463 0-.70710678s.51184463-.19526215.70710678 0l1.20048821 1.2004882 1.6006509-4.00162734-4.50670359 1.80268144-1.80268144 4.50670359zm4.10281269-6.50378907-2.6692597 6.67314927c-.1016411.2541026-.3029834.4554449-.557086.557086l-6.67314927 2.6692597 2.66925969-6.67314926c.10164107-.25410266.30298336-.45544495.55708602-.55708602z" fill-rule="evenodd">
      </path>
     </symbol>
     <symbol id="icon-filter" viewbox="0 0 16 16">
      <path d="m14.9738641 0c.5667192 0 1.0261359.4477136 1.0261359 1 0 .24221858-.0902161.47620768-.2538899.65849851l-5.6938314 6.34147206v5.49997973c0 .3147562-.1520673.6111434-.4104543.7999971l-2.05227171 1.4999945c-.45337535.3313696-1.09655869.2418269-1.4365902-.1999993-.13321514-.1730955-.20522717-.3836284-.20522717-.5999978v-6.99997423l-5.69383133-6.34147206c-.3731872-.41563511-.32996891-1.0473954.09653074-1.41107611.18705584-.15950448.42716133-.2474224.67571519-.2474224zm-5.9218641 8.5h-2.105v6.491l.01238459.0070843.02053271.0015705.01955278-.0070558 2.0532976-1.4990996zm-8.02585008-7.5-.01564945.00240169 5.83249953 6.49759831h2.313l5.836-6.499z">
      </path>
     </symbol>
     <symbol id="icon-home" viewbox="0 0 18 18">
      <path d="m9 5-6 6v5h4v-4h4v4h4v-5zm7 6.5857864v4.4142136c0 .5522847-.4477153 1-1 1h-5v-4h-2v4h-5c-.55228475 0-1-.4477153-1-1v-4.4142136c-.25592232 0-.51184464-.097631-.70710678-.2928932l-.58578644-.5857864c-.39052429-.3905243-.39052429-1.02368929 0-1.41421358l8.29289322-8.29289322 8.2928932 8.29289322c.3905243.39052429.3905243 1.02368928 0 1.41421358l-.5857864.5857864c-.1952622.1952622-.4511845.2928932-.7071068.2928932zm-7-9.17157284-7.58578644 7.58578644.58578644.5857864 7-6.99999996 7 6.99999996.5857864-.5857864z" fill-rule="evenodd">
      </path>
     </symbol>
     <symbol id="icon-image" viewbox="0 0 18 18">
      <path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm-3.49645283 10.1752453-3.89407257 6.7495552c.11705545.048464.24538859.0751995.37998328.0751995h10.60290092l-2.4329715-4.2154691-1.57494129 2.7288098zm8.49779013 6.8247547c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v13.98991071l4.50814957-7.81026689 3.08089884 5.33809539 1.57494129-2.7288097 3.5875735 6.2159812zm-3.0059397-11c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm0 1c-.5522847 0-1 .44771525-1 1s.4477153 1 1 1 1-.44771525 1-1-.4477153-1-1-1z" fill-rule="evenodd">
      </path>
     </symbol>
     <symbol id="icon-info" viewbox="0 0 18 18">
      <path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z" fill-rule="evenodd">
      </path>
     </symbol>
     <symbol id="icon-institution" viewbox="0 0 18 18">
      <path d="m7 16.9998189v-2.0003623h4v2.0003623h2v-3.0005434h-8v3.0005434zm-3-10.00181122h-1.52632364c-.27614237 0-.5-.22389817-.5-.50009056 0-.13995446.05863589-.27350497.16166338-.36820841l1.23156713-1.13206327h-2.36690687v12.00217346h3v-2.0003623h-3v-1.0001811h3v-1.0001811h1v-4.00072448h-1zm10 0v2.00036224h-1v4.00072448h1v1.0001811h3v1.0001811h-3v2.0003623h3v-12.00217346h-2.3695309l1.2315671 1.13206327c.2033191.186892.2166633.50325042.0298051.70660631-.0946863.10304615-.2282126.16169266-.3681417.16169266zm3-3.00054336c.5522847 0 1 .44779634 1 1.00018112v13.00235456h-18v-13.00235456c0-.55238478.44771525-1.00018112 1-1.00018112h3.45499992l4.20535144-3.86558216c.19129876-.17584288.48537447-.17584288.67667324 0l4.2053514 3.86558216zm-4 3.00054336h-8v1.00018112h8zm-2 6.00108672h1v-4.00072448h-1zm-1 0v-4.00072448h-2v4.00072448zm-3 0v-4.00072448h-1v4.00072448zm8-4.00072448c.5522847 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.4477153-1.00018112 1-1.00018112zm-12 0c.55228475 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.44771525-1.00018112 1-1.00018112zm5.99868798-7.81907007-5.24205601 4.81852671h10.48411203zm.00131202 3.81834559c-.55228475 0-1-.44779634-1-1.00018112s.44771525-1.00018112 1-1.00018112 1 .44779634 1 1.00018112-.44771525 1.00018112-1 1.00018112zm-1 11.00199236v1.0001811h2v-1.0001811z" fill-rule="evenodd">
      </path>
     </symbol>
     <symbol id="icon-location" viewbox="0 0 18 18">
      <path d="m9.39521328 16.2688008c.79596342-.7770119 1.59208152-1.6299956 2.33285652-2.5295081 1.4020032-1.7024324 2.4323601-3.3624519 2.9354918-4.871847.2228715-.66861448.3364384-1.29323246.3364384-1.8674457 0-3.3137085-2.6862915-6-6-6-3.36356866 0-6 2.60156856-6 6 0 .57421324.11356691 1.19883122.3364384 1.8674457.50313169 1.5093951 1.53348863 3.1694146 2.93549184 4.871847.74077492.8995125 1.53689309 1.7524962 2.33285648 2.5295081.13694479.1336842.26895677.2602648.39521328.3793207.12625651-.1190559.25826849-.2456365.39521328-.3793207zm-.39521328 1.7311992s-7-6-7-11c0-4 3.13400675-7 7-7 3.8659932 0 7 3.13400675 7 7 0 5-7 11-7 11zm0-8c-1.65685425 0-3-1.34314575-3-3s1.34314575-3 3-3c1.6568542 0 3 1.34314575 3 3s-1.3431458 3-3 3zm0-1c1.1045695 0 2-.8954305 2-2s-.8954305-2-2-2-2 .8954305-2 2 .8954305 2 2 2z" fill-rule="evenodd">
      </path>
     </symbol>
     <symbol id="icon-minus" viewbox="0 0 16 16">
      <path d="m2.00087166 7h11.99825664c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-11.99825664c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd">
      </path>
     </symbol>
     <symbol id="icon-newsletter" viewbox="0 0 18 18">
      <path d="m9 11.8482489 2-1.1428571v-1.7053918h-4v1.7053918zm-3-1.7142857v-2.1339632h6v2.1339632l3-1.71428574v-6.41967746h-12v6.41967746zm10-5.3839632 1.5299989.95624934c.2923814.18273835.4700011.50320827.4700011.8479983v8.44575236c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-8.44575236c0-.34479003.1776197-.66525995.47000106-.8479983l1.52999894-.95624934v-2.75c0-.55228475.44771525-1 1-1h12c.5522847 0 1 .44771525 1 1zm0 1.17924764v3.07075236l-7 4-7-4v-3.07075236l-1 .625v8.44575236c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-8.44575236zm-10-1.92924764h6v1h-6zm-1 2h8v1h-8z" fill-rule="evenodd">
      </path>
     </symbol>
     <symbol id="icon-orcid" viewbox="0 0 18 18">
      <path d="m9 1c4.418278 0 8 3.581722 8 8s-3.581722 8-8 8-8-3.581722-8-8 3.581722-8 8-8zm-2.90107518 5.2732337h-1.41865256v7.1712107h1.41865256zm4.55867178.02508949h-2.99247027v7.14612121h2.91062487c.7673039 0 1.4476365-.1483432 2.0410182-.445034s1.0511995-.7152915 1.3734671-1.2558144c.3222677-.540523.4833991-1.1603247.4833991-1.85942385 0-.68545815-.1602789-1.30270225-.4808414-1.85175082-.3205625-.54904856-.7707074-.97532211-1.3504481-1.27883343-.5797408-.30351132-1.2413173-.45526471-1.9847495-.45526471zm-.1892674 1.07933542c.7877654 0 1.4143875.22336734 1.8798852.67010873.4654977.44674138.698243 1.05546001.698243 1.82617415 0 .74343221-.2310402 1.34447791-.6931277 1.80315511-.4620874.4586773-1.0750688.6880124-1.8389625.6880124h-1.46810075v-4.98745039zm-5.08652545-3.71099194c-.21825533 0-.410525.08444276-.57681478.25333081-.16628977.16888806-.24943341.36245684-.24943341.58071218 0 .22345188.08314364.41961891.24943341.58850696.16628978.16888806.35855945.25333082.57681478.25333082.233845 0 .43390938-.08314364.60019916-.24943342.16628978-.16628977.24943342-.36375592.24943342-.59240436 0-.233845-.08314364-.43131115-.24943342-.59240437s-.36635416-.24163862-.60019916-.24163862z" fill-rule="evenodd">
      </path>
     </symbol>
     <symbol id="icon-plus" viewbox="0 0 16 16">
      <path d="m2.00087166 7h4.99912834v-4.99912834c0-.55276616.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v4.99912834h4.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-4.9991283v4.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-4.9991283h-4.99912834c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd">
      </path>
     </symbol>
     <symbol id="icon-print" viewbox="0 0 18 18">
      <path d="m16.0049107 5h-14.00982141c-.54941618 0-.99508929.4467783-.99508929.99961498v6.00077002c0 .5570958.44271433.999615.99508929.999615h1.00491071v-3h12v3h1.0049107c.5494162 0 .9950893-.4467783.9950893-.999615v-6.00077002c0-.55709576-.4427143-.99961498-.9950893-.99961498zm-2.0049107-1v-2.00208688c0-.54777062-.4519464-.99791312-1.0085302-.99791312h-7.9829396c-.55661731 0-1.0085302.44910695-1.0085302.99791312v2.00208688zm1 10v2.0018986c0 1.103521-.9019504 1.9981014-2.0085302 1.9981014h-7.9829396c-1.1092806 0-2.0085302-.8867064-2.0085302-1.9981014v-2.0018986h-1.00491071c-1.10185739 0-1.99508929-.8874333-1.99508929-1.999615v-6.00077002c0-1.10435686.8926228-1.99961498 1.99508929-1.99961498h1.00491071v-2.00208688c0-1.10341695.90195036-1.99791312 2.0085302-1.99791312h7.9829396c1.1092806 0 2.0085302.89826062 2.0085302 1.99791312v2.00208688h1.0049107c1.1018574 0 1.9950893.88743329 1.9950893 1.99961498v6.00077002c0 1.1043569-.8926228 1.999615-1.9950893 1.999615zm-1-3h-10v5.0018986c0 .5546075.44702548.9981014 1.0085302.9981014h7.9829396c.5565964 0 1.0085302-.4491701 1.0085302-.9981014zm-9 1h8v1h-8zm0 2h5v1h-5zm9-5c-.5522847 0-1-.44771525-1-1s.4477153-1 1-1 1 .44771525 1 1-.4477153 1-1 1z" fill-rule="evenodd">
      </path>
     </symbol>
     <symbol id="icon-search" viewbox="0 0 22 22">
      <path d="M21.697 20.261a1.028 1.028 0 01.01 1.448 1.034 1.034 0 01-1.448-.01l-4.267-4.267A9.812 9.811 0 010 9.812a9.812 9.811 0 1117.43 6.182zM9.812 18.222A8.41 8.41 0 109.81 1.403a8.41 8.41 0 000 16.82z" fill-rule="evenodd">
      </path>
     </symbol>
     <symbol id="icon-social-facebook" viewbox="0 0 24 24">
      <path d="m6.00368507 20c-1.10660471 0-2.00368507-.8945138-2.00368507-1.9940603v-12.01187942c0-1.10128908.89451376-1.99406028 1.99406028-1.99406028h12.01187942c1.1012891 0 1.9940603.89451376 1.9940603 1.99406028v12.01187942c0 1.1012891-.88679 1.9940603-2.0032184 1.9940603h-2.9570132v-6.1960818h2.0797387l.3114113-2.414723h-2.39115v-1.54164807c0-.69911803.1941355-1.1755439 1.1966615-1.1755439l1.2786739-.00055875v-2.15974763l-.2339477-.02492088c-.3441234-.03134957-.9500153-.07025255-1.6293054-.07025255-1.8435726 0-3.1057323 1.12531866-3.1057323 3.19187953v1.78079225h-2.0850778v2.414723h2.0850778v6.1960818z" fill-rule="evenodd">
      </path>
     </symbol>
     <symbol id="icon-social-twitter" viewbox="0 0 24 24">
      <path d="m18.8767135 6.87445248c.7638174-.46908424 1.351611-1.21167363 1.6250764-2.09636345-.7135248.43394112-1.50406.74870123-2.3464594.91677702-.6695189-.73342162-1.6297913-1.19486605-2.6922204-1.19486605-2.0399895 0-3.6933555 1.69603749-3.6933555 3.78628909 0 .29642457.0314329.58673729.0942985.8617704-3.06469922-.15890802-5.78835241-1.66547825-7.60988389-3.9574208-.3174714.56076194-.49978171 1.21167363-.49978171 1.90536824 0 1.31404706.65223085 2.47224203 1.64236444 3.15218497-.60350999-.0198635-1.17401554-.1925232-1.67222562-.47366811v.04583885c0 1.83355406 1.27302891 3.36609966 2.96411421 3.71294696-.31118484.0886217-.63651445.1329326-.97441718.1329326-.2357461 0-.47149219-.0229194-.69466516-.0672303.47149219 1.5065703 1.83253297 2.6036468 3.44975116 2.632678-1.2651707 1.0160946-2.85724264 1.6196394-4.5891906 1.6196394-.29861172 0-.59093688-.0152796-.88011875-.0504227 1.63450624 1.0726291 3.57548241 1.6990934 5.66104951 1.6990934 6.79263079 0 10.50641749-5.7711113 10.50641749-10.7751859l-.0094298-.48894775c.7229547-.53478659 1.3516109-1.20250585 1.8419628-1.96190282-.6632323.30100846-1.3751855.50422736-2.1217148.59590507z" fill-rule="evenodd">
      </path>
     </symbol>
     <symbol id="icon-social-youtube" viewbox="0 0 24 24">
      <path d="m10.1415 14.3973208-.0005625-5.19318431 4.863375 2.60554491zm9.963-7.92753362c-.6845625-.73643756-1.4518125-.73990314-1.803375-.7826454-2.518875-.18714178-6.2971875-.18714178-6.2971875-.18714178-.007875 0-3.7861875 0-6.3050625.18714178-.352125.04274226-1.1188125.04620784-1.8039375.7826454-.5394375.56084773-.7149375 1.8344515-.7149375 1.8344515s-.18 1.49597903-.18 2.99138042v1.4024082c0 1.495979.18 2.9913804.18 2.9913804s.1755 1.2736038.7149375 1.8344515c.685125.7364376 1.5845625.7133337 1.9850625.7901542 1.44.1420891 6.12.1859866 6.12.1859866s3.78225-.005776 6.301125-.1929178c.3515625-.0433198 1.1188125-.0467854 1.803375-.783223.5394375-.5608477.7155-1.8344515.7155-1.8344515s.18-1.4954014.18-2.9913804v-1.4024082c0-1.49540139-.18-2.99138042-.18-2.99138042s-.1760625-1.27360377-.7155-1.8344515z" fill-rule="evenodd">
      </path>
     </symbol>
     <symbol id="icon-subject-medicine" viewbox="0 0 18 18">
      <path d="m12.5 8h-6.5c-1.65685425 0-3 1.34314575-3 3v1c0 1.6568542 1.34314575 3 3 3h1v-2h-.5c-.82842712 0-1.5-.6715729-1.5-1.5s.67157288-1.5 1.5-1.5h1.5 2 1 2c1.6568542 0 3-1.34314575 3-3v-1c0-1.65685425-1.3431458-3-3-3h-2v2h1.5c.8284271 0 1.5.67157288 1.5 1.5s-.6715729 1.5-1.5 1.5zm-5.5-1v-1h-3.5c-1.38071187 0-2.5-1.11928813-2.5-2.5s1.11928813-2.5 2.5-2.5h1.02786405c.46573528 0 .92507448.10843528 1.34164078.31671843l1.13382424.56691212c.06026365-1.05041141.93116291-1.88363055 1.99667093-1.88363055 1.1045695 0 2 .8954305 2 2h2c2.209139 0 4 1.790861 4 4v1c0 2.209139-1.790861 4-4 4h-2v1h2c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2h-2c0 1.1045695-.8954305 2-2 2s-2-.8954305-2-2h-1c-2.209139 0-4-1.790861-4-4v-1c0-2.209139 1.790861-4 4-4zm0-2v-2.05652691c-.14564246-.03538148-.28733393-.08714006-.42229124-.15461871l-1.15541752-.57770876c-.27771087-.13885544-.583937-.21114562-.89442719-.21114562h-1.02786405c-.82842712 0-1.5.67157288-1.5 1.5s.67157288 1.5 1.5 1.5zm4 1v1h1.5c.2761424 0 .5-.22385763.5-.5s-.2238576-.5-.5-.5zm-1 1v-5c0-.55228475-.44771525-1-1-1s-1 .44771525-1 1v5zm-2 4v5c0 .5522847.44771525 1 1 1s1-.4477153 1-1v-5zm3 2v2h2c.5522847 0 1-.4477153 1-1s-.4477153-1-1-1zm-4-1v-1h-.5c-.27614237 0-.5.2238576-.5.5s.22385763.5.5.5zm-3.5-9h1c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd">
      </path>
     </symbol>
     <symbol id="icon-success" viewbox="0 0 18 18">
      <path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm3.4860198 4.98163161-4.71802968 5.50657859-2.62834168-2.02300024c-.42862421-.36730544-1.06564993-.30775346-1.42283677.13301307-.35718685.44076653-.29927542 1.0958383.12934879 1.46314377l3.40735508 2.7323063c.42215801.3385221 1.03700951.2798252 1.38749189-.1324571l5.38450527-6.33394549c.3613513-.43716226.3096573-1.09278382-.115462-1.46437175-.4251192-.37158792-1.0626796-.31842941-1.4240309.11873285z" fill-rule="evenodd">
      </path>
     </symbol>
     <symbol id="icon-table" viewbox="0 0 18 18">
      <path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587l-4.0059107-.001.001.001h-1l-.001-.001h-5l.001.001h-1l-.001-.001-3.00391071.001c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm-11.0059107 5h-3.999v6.9941413c0 .5572961.44630695 1.0058587.99508929 1.0058587h3.00391071zm6 0h-5v8h5zm5.0059107-4h-4.0059107v3h5.001v1h-5.001v7.999l4.0059107.001c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-12.5049107 9c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.22385763-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1.499-5h-5v3h5zm-6 0h-3.00391071c-.54871518 0-.99508929.44887827-.99508929 1.00585866v1.99414134h3.999z" fill-rule="evenodd">
      </path>
     </symbol>
     <symbol id="icon-tick-circle" viewbox="0 0 24 24">
      <path d="m12 2c5.5228475 0 10 4.4771525 10 10s-4.4771525 10-10 10-10-4.4771525-10-10 4.4771525-10 10-10zm0 1c-4.97056275 0-9 4.02943725-9 9 0 4.9705627 4.02943725 9 9 9 4.9705627 0 9-4.0294373 9-9 0-4.97056275-4.0294373-9-9-9zm4.2199868 5.36606669c.3613514-.43716226.9989118-.49032077 1.424031-.11873285s.4768133 1.02720949.115462 1.46437175l-6.093335 6.94397871c-.3622945.4128716-.9897871.4562317-1.4054264.0971157l-3.89719065-3.3672071c-.42862421-.3673054-.48653564-1.0223772-.1293488-1.4631437s.99421256-.5003185 1.42283677-.1330131l3.11097438 2.6987741z" fill-rule="evenodd">
      </path>
     </symbol>
     <symbol id="icon-tick" viewbox="0 0 16 16">
      <path d="m6.76799012 9.21106946-3.1109744-2.58349728c-.42862421-.35161617-1.06564993-.29460792-1.42283677.12733148s-.29927541 1.04903009.1293488 1.40064626l3.91576307 3.23873978c.41034319.3393961 1.01467563.2976897 1.37450571-.0948578l6.10568327-6.660841c.3613513-.41848908.3096572-1.04610608-.115462-1.4018218-.4251192-.35571573-1.0626796-.30482786-1.424031.11366122z" fill-rule="evenodd">
      </path>
     </symbol>
     <symbol id="icon-update" viewbox="0 0 18 18">
      <path d="m1 13v1c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-1h-1v-10h-14v10zm16-1h1v2c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-2h1v-9c0-.55228475.44771525-1 1-1h14c.5522847 0 1 .44771525 1 1zm-1 0v1h-4.5857864l-1 1h-2.82842716l-1-1h-4.58578644v-1h5l1 1h2l1-1zm-13-8h12v7h-12zm1 1v5h10v-5zm1 1h4v1h-4zm0 2h4v1h-4z" fill-rule="evenodd">
      </path>
     </symbol>
     <symbol id="icon-upload" viewbox="0 0 18 18">
      <path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.85576936 4.14572769c.19483374-.19483375.51177826-.19377714.70556874.00001334l2.59099082 2.59099079c.1948411.19484112.1904373.51514474.0027906.70279143-.1932998.19329987-.5046517.19237083-.7001856-.00692852l-1.74638687-1.7800176v6.14827687c0 .2717771-.23193359.492096-.5.492096-.27614237 0-.5-.216372-.5-.492096v-6.14827641l-1.74627892 1.77990922c-.1933927.1971171-.51252214.19455839-.70016883.0069117-.19329987-.19329988-.19100584-.50899493.00277731-.70277808z" fill-rule="evenodd">
      </path>
     </symbol>
     <symbol id="icon-video" viewbox="0 0 18 18">
      <path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-8.30912922 2.24944486 4.60460462 2.73982242c.9365543.55726659.9290753 1.46522435 0 2.01804082l-4.60460462 2.7398224c-.93655425.5572666-1.69578148.1645632-1.69578148-.8937585v-5.71016863c0-1.05087579.76670616-1.446575 1.69578148-.89375851zm-.67492769.96085624v5.5750128c0 .2995102-.10753745.2442517.16578928.0847713l4.58452283-2.67497259c.3050619-.17799716.3051624-.21655446 0-.39461026l-4.58452283-2.67497264c-.26630747-.15538481-.16578928-.20699944-.16578928.08477139z" fill-rule="evenodd">
      </path>
     </symbol>
     <symbol id="icon-warning" viewbox="0 0 18 18">
      <path d="m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd">
      </path>
     </symbol>
     <symbol id="icon-altmetric">
      <path d="M12 1c6.075 0 11 4.925 11 11s-4.925 11-11 11S1 18.075 1 12 5.925 1 12 1Zm-1.886 9.684-1.101 1.845a1 1 0 0 1-.728.479l-.13.008H3.056a9.001 9.001 0 0 0 17.886 0l-4.564-.001-2.779 4.156c-.454.68-1.467.55-1.758-.179l-.038-.113-1.69-6.195ZM12 3a9.001 9.001 0 0 0-8.947 8.016h4.533l2.017-3.375c.452-.757 1.592-.6 1.824.25l1.73 6.345 1.858-2.777a1 1 0 0 1 .707-.436l.124-.008h5.1A9.001 9.001 0 0 0 12 3Z" fill-rule="nonzero">
      </path>
     </symbol>
     <symbol id="icon-checklist-banner" viewbox="0 0 56.69 56.69">
      <path d="M0 0h56.69v56.69H0z" style="fill:none">
      </path>
      <clippath id="b">
       <use style="overflow:visible" xlink:href="#a">
       </use>
      </clippath>
      <path d="M21.14 34.46c0-6.77 5.48-12.26 12.24-12.26s12.24 5.49 12.24 12.26-5.48 12.26-12.24 12.26c-6.76-.01-12.24-5.49-12.24-12.26zm19.33 10.66 10.23 9.22s1.21 1.09 2.3-.12l2.09-2.32s1.09-1.21-.12-2.3l-10.23-9.22m-19.29-5.92c0-4.38 3.55-7.94 7.93-7.94s7.93 3.55 7.93 7.94c0 4.38-3.55 7.94-7.93 7.94-4.38-.01-7.93-3.56-7.93-7.94zm17.58 12.99 4.14-4.81" style="clip-path:url(#b);fill:none;stroke:#01324b;stroke-width:2;stroke-linecap:round">
      </path>
      <path d="M8.26 9.75H28.6M8.26 15.98H28.6m-20.34 6.2h12.5m14.42-5.2V4.86s0-2.93-2.93-2.93H4.13s-2.93 0-2.93 2.93v37.57s0 2.93 2.93 2.93h15.01M8.26 9.75H28.6M8.26 15.98H28.6m-20.34 6.2h12.5" style="clip-path:url(#b);fill:none;stroke:#01324b;stroke-width:2;stroke-linecap:round;stroke-linejoin:round">
      </path>
     </symbol>
     <symbol id="icon-chevron-down" viewbox="0 0 16 16">
      <path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 1)">
      </path>
     </symbol>
     <symbol id="icon-citations">
      <path d="M15.59 1a1 1 0 0 1 .706.291l5.41 5.385a1 1 0 0 1 .294.709v13.077c0 .674-.269 1.32-.747 1.796a2.549 2.549 0 0 1-1.798.742h-5.843a1 1 0 1 1 0-2h5.843a.549.549 0 0 0 .387-.16.535.535 0 0 0 .158-.378V7.8L15.178 3H5.545a.543.543 0 0 0-.538.451L5 3.538v8.607a1 1 0 0 1-2 0V3.538A2.542 2.542 0 0 1 5.545 1h10.046ZM5.483 14.35c.197.26.17.62-.049.848l-.095.083-.016.011c-.36.24-.628.45-.804.634-.393.409-.59.93-.59 1.562.077-.019.192-.028.345-.028.442 0 .84.158 1.195.474.355.316.532.716.532 1.2 0 .501-.173.9-.518 1.198-.345.298-.767.446-1.266.446-.672 0-1.209-.195-1.612-.585-.403-.39-.604-.976-.604-1.757 0-.744.11-1.39.33-1.938.222-.549.49-1.009.807-1.38a4.28 4.28 0 0 1 .992-.88c.07-.043.148-.087.232-.133a.881.881 0 0 1 1.121.245Zm5 0c.197.26.17.62-.049.848l-.095.083-.016.011c-.36.24-.628.45-.804.634-.393.409-.59.93-.59 1.562.077-.019.192-.028.345-.028.442 0 .84.158 1.195.474.355.316.532.716.532 1.2 0 .501-.173.9-.518 1.198-.345.298-.767.446-1.266.446-.672 0-1.209-.195-1.612-.585-.403-.39-.604-.976-.604-1.757 0-.744.11-1.39.33-1.938.222-.549.49-1.009.807-1.38a4.28 4.28 0 0 1 .992-.88c.07-.043.148-.087.232-.133a.881.881 0 0 1 1.121.245Z" fill-rule="evenodd">
      </path>
     </symbol>
     <symbol id="icon-eds-checklist" viewbox="0 0 32 32">
      <path d="M19.2 1.333a3.468 3.468 0 0 1 3.381 2.699L24.667 4C26.515 4 28 5.52 28 7.38v19.906c0 1.86-1.485 3.38-3.333 3.38H7.333c-1.848 0-3.333-1.52-3.333-3.38V7.38C4 5.52 5.485 4 7.333 4h2.093A3.468 3.468 0 0 1 12.8 1.333h6.4ZM9.426 6.667H7.333c-.36 0-.666.312-.666.713v19.906c0 .401.305.714.666.714h17.334c.36 0 .666-.313.666-.714V7.38c0-.4-.305-.713-.646-.714l-2.121.033A3.468 3.468 0 0 1 19.2 9.333h-6.4a3.468 3.468 0 0 1-3.374-2.666Zm12.715 5.606c.586.446.7 1.283.253 1.868l-7.111 9.334a1.333 1.333 0 0 1-1.792.306l-3.556-2.333a1.333 1.333 0 1 1 1.463-2.23l2.517 1.651 6.358-8.344a1.333 1.333 0 0 1 1.868-.252ZM19.2 4h-6.4a.8.8 0 0 0-.8.8v1.067a.8.8 0 0 0 .8.8h6.4a.8.8 0 0 0 .8-.8V4.8a.8.8 0 0 0-.8-.8Z">
      </path>
     </symbol>
     <symbol id="icon-eds-i-external-link-medium" viewbox="0 0 24 24">
      <path d="M9 2a1 1 0 1 1 0 2H4.6c-.371 0-.6.209-.6.5v15c0 .291.229.5.6.5h14.8c.371 0 .6-.209.6-.5V15a1 1 0 0 1 2 0v4.5c0 1.438-1.162 2.5-2.6 2.5H4.6C3.162 22 2 20.938 2 19.5v-15C2 3.062 3.162 2 4.6 2H9Zm6 0h6l.075.003.126.017.111.03.111.044.098.052.096.067.09.08c.036.035.068.073.097.112l.071.11.054.114.035.105.03.148L22 3v6a1 1 0 0 1-2 0V5.414l-6.693 6.693a1 1 0 0 1-1.414-1.414L18.584 4H15a1 1 0 0 1-.993-.883L14 3a1 1 0 0 1 1-1Z">
      </path>
     </symbol>
     <symbol id="icon-eds-i-info-filled-medium" viewbox="0 0 24 24">
      <path d="M12 1c6.075 0 11 4.925 11 11s-4.925 11-11 11S1 18.075 1 12 5.925 1 12 1Zm0 9h-1.5a1 1 0 0 0-1 1l.007.117A1 1 0 0 0 10.5 12h.5v4H9.5a1 1 0 0 0 0 2h5a1 1 0 0 0 0-2H13v-5a1 1 0 0 0-1-1Zm0-4.5a1.5 1.5 0 0 0 0 3l.144-.007A1.5 1.5 0 0 0 12 5.5Z">
      </path>
     </symbol>
     <symbol id="icon-eds-menu" viewbox="0 0 24 24">
      <path d="M21.09 5c.503 0 .91.448.91 1s-.407 1-.91 1H2.91C2.406 7 2 6.552 2 6s.407-1 .91-1h18.18Zm-3.817 6c.401 0 .727.448.727 1s-.326 1-.727 1H2.727C2.326 13 2 12.552 2 12s.326-1 .727-1h14.546Zm3.818 6c.502 0 .909.448.909 1s-.407 1-.91 1H2.91c-.503 0-.91-.448-.91-1s.407-1 .91-1h18.18Z" fill-rule="nonzero">
      </path>
     </symbol>
     <symbol id="icon-eds-search" viewbox="0 0 24 24">
      <path d="M11 1c5.523 0 10 4.477 10 10 0 2.4-.846 4.604-2.256 6.328l3.963 3.965a1 1 0 0 1-1.414 1.414l-3.965-3.963A9.959 9.959 0 0 1 11 21C5.477 21 1 16.523 1 11S5.477 1 11 1Zm0 2a8 8 0 1 0 0 16 8 8 0 0 0 0-16Z" fill-rule="nonzero">
      </path>
     </symbol>
     <symbol id="icon-eds-small-arrow-right" viewbox="0 0 16 16">
      <g fill-rule="evenodd" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="2">
       <path d="M2 8.092h12M8 2l6 6.092M8 14.127l6-6.035">
       </path>
      </g>
     </symbol>
     <symbol id="icon-eds-user-single" viewbox="0 0 24 24">
      <path d="M12 12c5.498 0 10 4.001 10 9a1 1 0 0 1-2 0c0-3.838-3.557-7-8-7s-8 3.162-8 7a1 1 0 0 1-2 0c0-4.999 4.502-9 10-9Zm0-11a5 5 0 1 0 0 10 5 5 0 0 0 0-10Zm0 2a3 3 0 1 1 0 6 3 3 0 0 1 0-6Z" fill-rule="nonzero">
      </path>
     </symbol>
     <symbol id="icon-email-new" viewbox="0 0 24 24">
      <path d="m19.462 0c1.413 0 2.538 1.184 2.538 2.619v12.762c0 1.435-1.125 2.619-2.538 2.619h-16.924c-1.413 0-2.538-1.184-2.538-2.619v-12.762c0-1.435 1.125-2.619 2.538-2.619zm.538 5.158-7.378 6.258a2.549 2.549 0 0 1 -3.253-.008l-7.369-6.248v10.222c0 .353.253.619.538.619h16.924c.285 0 .538-.266.538-.619zm-.538-3.158h-16.924c-.264 0-.5.228-.534.542l8.65 7.334c.2.165.492.165.684.007l8.656-7.342-.001-.025c-.044-.3-.274-.516-.531-.516z">
      </path>
     </symbol>
     <symbol id="icon-expand-image" viewbox="0 0 18 18">
      <path d="m7.49754099 11.9178212c.38955542-.3895554.38761957-1.0207846-.00290473-1.4113089-.39324695-.3932469-1.02238878-.3918247-1.41130883-.0029047l-4.10273549 4.1027355.00055454-3.5103985c.00008852-.5603185-.44832171-1.006032-1.00155062-1.0059446-.53903074.0000852-.97857527.4487442-.97866268 1.0021075l-.00093318 5.9072465c-.00008751.553948.44841131 1.001882 1.00174994 1.0017946l5.906983-.0009331c.5539233-.0000875 1.00197907-.4486389 1.00206646-1.0018679.00008515-.5390307-.45026621-.9784332-1.00588841-.9783454l-3.51010549.0005545zm3.00571741-5.83449376c-.3895554.38955541-.3876196 1.02078454.0029047 1.41130883.393247.39324696 1.0223888.39182478 1.4113089.00290473l4.1027355-4.10273549-.0005546 3.5103985c-.0000885.56031852.4483217 1.006032 1.0015506 1.00594461.5390308-.00008516.9785753-.44874418.9786627-1.00210749l.0009332-5.9072465c.0000875-.553948-.4484113-1.00188204-1.0017499-1.00179463l-5.906983.00093313c-.5539233.00008751-1.0019791.44863892-1.0020665 1.00186784-.0000852.53903074.4502662.97843325 1.0058884.97834547l3.5101055-.00055449z" fill-rule="evenodd">
      </path>
     </symbol>
     <symbol id="icon-github" viewbox="0 0 100 100">
      <path clip-rule="evenodd" d="M48.854 0C21.839 0 0 22 0 49.217c0 21.756 13.993 40.172 33.405 46.69 2.427.49 3.316-1.059 3.316-2.362 0-1.141-.08-5.052-.08-9.127-13.59 2.934-16.42-5.867-16.42-5.867-2.184-5.704-5.42-7.17-5.42-7.17-4.448-3.015.324-3.015.324-3.015 4.934.326 7.523 5.052 7.523 5.052 4.367 7.496 11.404 5.378 14.235 4.074.404-3.178 1.699-5.378 3.074-6.6-10.839-1.141-22.243-5.378-22.243-24.283 0-5.378 1.94-9.778 5.014-13.2-.485-1.222-2.184-6.275.486-13.038 0 0 4.125-1.304 13.426 5.052a46.97 46.97 0 0 1 12.214-1.63c4.125 0 8.33.571 12.213 1.63 9.302-6.356 13.427-5.052 13.427-5.052 2.67 6.763.97 11.816.485 13.038 3.155 3.422 5.015 7.822 5.015 13.2 0 18.905-11.404 23.06-22.324 24.283 1.78 1.548 3.316 4.481 3.316 9.126 0 6.6-.08 11.897-.08 13.526 0 1.304.89 2.853 3.316 2.364 19.412-6.52 33.405-24.935 33.405-46.691C97.707 22 75.788 0 48.854 0z" fill-rule="evenodd">
      </path>
     </symbol>
     <symbol id="icon-mentions">
      <g fill-rule="evenodd" stroke="#000" stroke-linecap="round" stroke-linejoin="round" stroke-width="2">
       <path d="M22 15.255A9.373 9.373 0 0 1 8.745 2L22 15.255ZM15.477 8.523l4.215-4.215">
       </path>
       <path d="m7 13-5 9h10l-1-5">
       </path>
      </g>
     </symbol>
     <symbol id="icon-metrics-accesses">
      <path d="M15.59 1a1 1 0 0 1 .706.291l5.41 5.385a1 1 0 0 1 .294.709v13.077c0 .674-.269 1.32-.747 1.796a2.549 2.549 0 0 1-1.798.742h-5.843a1 1 0 1 1 0-2h5.843a.549.549 0 0 0 .387-.16.535.535 0 0 0 .158-.378V7.8L15.178 3H5.545a.543.543 0 0 0-.538.451L5 3.538v8.607a1 1 0 0 1-2 0V3.538A2.542 2.542 0 0 1 5.545 1h10.046ZM7.708 13.308c2.004 0 3.969 1.198 5.802 2.995l.23.23a2.285 2.285 0 0 1 .009 3.233C11.853 21.693 9.799 23 7.707 23c-2.091 0-4.14-1.305-6.033-3.226a2.285 2.285 0 0 1-.007-3.233c1.9-1.93 3.949-3.233 6.04-3.233Zm0 2c-1.396 0-3.064 1.062-4.623 2.644a.285.285 0 0 0 .007.41C4.642 19.938 6.311 21 7.707 21c1.397 0 3.069-1.065 4.623-2.644a.285.285 0 0 0 0-.404l-.23-.229c-1.487-1.451-3.064-2.415-4.393-2.415Zm-.036 1.077a1.77 1.77 0 1 1 .126 3.537 1.77 1.77 0 0 1-.126-3.537Zm.072 1.538a.23.23 0 1 0-.017.461.23.23 0 0 0 .017-.46Z" fill-rule="nonzero">
      </path>
     </symbol>
     <symbol id="icon-metrics">
      <path d="M3 22a1 1 0 0 1-1-1V3a1 1 0 0 1 1-1h6a1 1 0 0 1 1 1v7h4V8a1 1 0 0 1 1-1h6a1 1 0 0 1 1 1v13a1 1 0 0 1-.883.993L21 22H3Zm17-2V9h-4v11h4Zm-6-8h-4v8h4v-8ZM8 4H4v16h4V4Z" fill-rule="nonzero">
      </path>
     </symbol>
     <symbol id="icon-springer-arrow-left">
      <path d="M15 7a1 1 0 000-2H3.385l2.482-2.482a.994.994 0 00.02-1.403 1.001 1.001 0 00-1.417 0L.294 5.292a1.001 1.001 0 000 1.416l4.176 4.177a.991.991 0 001.4.016 1 1 0 00-.003-1.42L3.385 7H15z">
      </path>
     </symbol>
     <symbol id="icon-springer-arrow-right">
      <path d="M1 7a1 1 0 010-2h11.615l-2.482-2.482a.994.994 0 01-.02-1.403 1.001 1.001 0 011.417 0l4.176 4.177a1.001 1.001 0 010 1.416l-4.176 4.177a.991.991 0 01-1.4.016 1 1 0 01.003-1.42L12.615 7H1z">
      </path>
     </symbol>
     <symbol id="icon-submit-open" viewbox="0 0 16 17">
      <path d="M12 0c1.10457 0 2 .895431 2 2v5c0 .276142-.223858.5-.5.5S13 7.276142 13 7V2c0-.512836-.38604-.935507-.883379-.993272L12 1H6v3c0 1.10457-.89543 2-2 2H1v8c0 .512836.38604.935507.883379.993272L2 15h6.5c.276142 0 .5.223858.5.5s-.223858.5-.5.5H2c-1.104569 0-2-.89543-2-2V5.828427c0-.530433.210714-1.039141.585786-1.414213L4.414214.585786C4.789286.210714 5.297994 0 5.828427 0H12Zm3.41 11.14c.250899.250899.250274.659726 0 .91-.242954.242954-.649606.245216-.9-.01l-1.863671-1.900337.001043 5.869492c0 .356992-.289839.637138-.647372.637138-.347077 0-.647371-.285256-.647371-.637138l-.001043-5.869492L9.5 12.04c-.253166.258042-.649726.260274-.9.01-.242954-.242954-.252269-.657731 0-.91l2.942184-2.951303c.250908-.250909.66127-.252277.91353-.000017L15.41 11.14ZM5 1.413 1.413 5H4c.552285 0 1-.447715 1-1V1.413ZM11 3c.276142 0 .5.223858.5.5s-.223858.5-.5.5H7.5c-.276142 0-.5-.223858-.5-.5s.223858-.5.5-.5H11Zm0 2c.276142 0 .5.223858.5.5s-.223858.5-.5.5H7.5c-.276142 0-.5-.223858-.5-.5s.223858-.5.5-.5H11Z" fill-rule="nonzero">
      </path>
     </symbol>
    </svg>
   </div>
  </footer>
  <div class="c-site-messages message u-hide u-hide-print c-site-messages--nature-briefing c-site-messages--nature-briefing-email-variant c-site-messages--nature-briefing-redesign-2020 sans-serif" data-component-expirydays="30" data-component-id="nature-briefing-banner" data-component-trigger-scroll-percentage="15" data-track="in-view" data-track-action="in-view" data-track-category="nature briefing" data-track-label="Briefing banner visible: Flagship">
   <div class="c-site-messages__banner-large">
    <div class="c-site-messages__close-container">
     <button class="c-site-messages__close" data-track="click" data-track-category="nature briefing" data-track-label="Briefing banner dismiss: Flagship">
      <svg aria-hidden="true" focusable="false" height="25px" version="1.1" viewbox="0 0 25 25" width="25px" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
       <title>
        Close banner
       </title>
       <defs>
       </defs>
       <g fill="none" fill-rule="evenodd" stroke="none" stroke-width="1">
        <rect height="25" opacity="0" width="25" x="0" y="0">
        </rect>
        <path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#ffffff">
        </path>
       </g>
      </svg>
      <span class="visually-hidden">
       Close
      </span>
     </button>
    </div>
    <div class="c-site-messages__form-container">
     <div class="grid grid-12 last">
      <div class="grid grid-4">
       <img alt="Nature Briefing" height="40" src="/static/images/logos/nature-briefing-logo-n150-white-d81c9da3ec.svg" width="250"/>
       <p class="c-site-messages--nature-briefing__strapline extra-tight-line-height">
        Sign up for the
        <em>
         Nature Briefing
        </em>
        newsletter — what matters in science, free to your inbox daily.
       </p>
      </div>
      <div class="grid grid-8 last">
       <form action="https://briefer.public.springernature.app/" data-location="banner" data-track="submit||nature_briefing_sign_up" data-track-action="transmit-form" data-track-category="nature briefing" data-track-label="Briefing banner submit: Flagship" method="post">
        <input id="briefing-banner-signup-form-input-track-originReferralPoint" name="track_originReferralPoint" type="hidden" value="MainBriefingBanner"/>
        <input id="briefing-banner-signup-form-input-track-formType" name="track_formType" type="hidden" value="DirectEmailBanner"/>
        <input id="gdpr_tick" name="gdpr_tick" type="hidden" value="false"/>
        <input id="marketing" name="marketing" type="hidden" value="false"/>
        <input id="marketing_tick" name="marketing_tick" type="hidden" value="false"/>
        <input id="brieferEntryPoint" name="brieferEntryPoint" type="hidden" value="MainBriefingBanner"/>
        <label class="nature-briefing-banner__email-label" for="emailAddress">
         Email address
        </label>
        <div class="nature-briefing-banner__email-wrapper">
         <input class="nature-briefing-banner__email-input box-sizing text14" data-test-element="briefing-emailbanner-email-input" id="emailAddress" name="emailAddress" placeholder="e.g. jo.smith@university.ac.uk" required="" type="email" value=""/>
         <input id="defaultNewsletter" name="N:nature_briefing_daily" type="hidden" value="true"/>
         <button class="nature-briefing-banner__submit-button box-sizing text14" data-test-element="briefing-emailbanner-signup-button" type="submit">
          Sign up
         </button>
        </div>
        <div class="nature-briefing-banner__checkbox-wrapper grid grid-12 last">
         <input class="nature-briefing-banner__checkbox-checkbox" data-test-element="briefing-emailbanner-gdpr-checkbox" id="gdpr-briefing-banner-checkbox" name="gdpr" required="" type="checkbox" value="true"/>
         <label class="nature-briefing-banner__checkbox-label box-sizing text13 sans-serif block tighten-line-height" for="gdpr-briefing-banner-checkbox">
          I agree my information will be processed in accordance with the
          <em>
           Nature
          </em>
          and Springer Nature Limited
          <a href="https://www-nature-com.proxy.lib.ohio-state.edu/info/privacy">
           Privacy Policy
          </a>
          .
         </label>
        </div>
       </form>
      </div>
     </div>
    </div>
   </div>
   <div class="c-site-messages__banner-small">
    <div class="c-site-messages__close-container">
     <button class="c-site-messages__close" data-track="click" data-track-category="nature briefing" data-track-label="Briefing banner dismiss: Flagship">
      <svg aria-hidden="true" focusable="false" height="25px" version="1.1" viewbox="0 0 25 25" width="25px" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
       <title>
        Close banner
       </title>
       <defs>
       </defs>
       <g fill="none" fill-rule="evenodd" stroke="none" stroke-width="1">
        <rect height="25" opacity="0" width="25" x="0" y="0">
        </rect>
        <path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#ffffff">
        </path>
       </g>
      </svg>
      <span class="visually-hidden">
       Close
      </span>
     </button>
    </div>
    <div class="c-site-messages__content text14">
     <span class="c-site-messages--nature-briefing__strapline strong">
      Get the most important science stories of the day, free in your inbox.
     </span>
     <a class="nature-briefing__link text14 sans-serif" data-test-element="briefing-banner-link" data-track="click" data-track-category="nature briefing" data-track-label="Small-screen banner CTA to site" href="https://www-nature-com.proxy.lib.ohio-state.edu/briefing/signup/?brieferEntryPoint=MainBriefingBanner" rel="noreferrer noopener" target="_blank">
      Sign up for Nature Briefing
     </a>
    </div>
   </div>
  </div>
  <noscript>
   <img alt="" height="0" hidden="" src="https://verify-nature-com.proxy.lib.ohio-state.edu/verify/nature.png" style="display: none" width="0"/>
  </noscript>
  <script async="" src="//content.readcube.com/ping?doi=10.1038/s41586-021-03819-2&amp;format=js&amp;last_modified=2021-07-15">
  </script>
  <img alt="" class="u-visually-hidden" height="1" src="/m1su833m/article/s41586-021-03819-2" width="1"/>
  <div class="c-cookie-banner">
   <div class="c-cookie-banner__container">
    <p>
     This website sets only cookies which are necessary for it to function. They are used to enable core functionality such as security, network management and accessibility. These cookies cannot be switched off in our systems. You may disable these by changing your browser settings, but this may affect how the website functions. Please view our privacy policy for further details on how we process your information.
     <button class="c-cookie-banner__dismiss">
      Dismiss
     </button>
    </p>
   </div>
  </div>
  <script src="https://verify-nature-com.proxy.lib.ohio-state.edu/verify/nature.min.js">
  </script>
 </body>
</html>
